{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###### <span style=\"color:#ff1414\"> BEDtools analysis. </span>\n",
    "\n",
    "This is a script to answer research questions outlined elsewhere. In summary, this script:\n",
    "\n",
    "1. compares methylation results between different methylation-callers, and between different methylation sequencing methods.\n",
    "\n",
    "2. compares methylation between genes and non-gene regions\n",
    "\n",
    "3. compares methylation between transposons and non-repetitive regions\n",
    "\n",
    "4. compares transposons and genes\n",
    "\n",
    "\n",
    "Note:\n",
    "- PB/pb = PacBio\n",
    "- ONT/ont = Oxford Nanopore Technology\n",
    "- NP = Nanopolish"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1453,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pybedtools\n",
    "from pybedtools import BedTool\n",
    "import os\n",
    "import glob\n",
    "import pprint\n",
    "#import numpy # need for p-value stats\n",
    "import scipy\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "from matplotlib_venn import venn2\n",
    "from scipy.stats import wilcoxon\n",
    "from scipy.stats import spearmanr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1747,
   "metadata": {},
   "outputs": [],
   "source": [
    "#First we need to define the base dirs\n",
    "DIRS ={}\n",
    "DIRS['BASE1'] = '/home/anjuni/methylation_calling/pacbio'\n",
    "DIRS['BASE2'] = '/home/anjuni/analysis'\n",
    "DIRS['BED_INPUT'] = os.path.join(DIRS['BASE2'], 'bedtools_output', 'sequencing_comparison')\n",
    "DIRS['GFF_INPUT'] = os.path.join(DIRS['BASE2'], 'gff_output')\n",
    "DIRS['WINDOW_OUTPUT'] = os.path.join(DIRS['BASE2'], 'windows')\n",
    "DIRS['WINDOW_INPUT'] = os.path.join(DIRS['BASE2'], 'input_for_windows')\n",
    "DIRS['REF'] = '/home/anjuni/Pst_104_v13_assembly/'\n",
    "DIRS['FIGURES'] = os.path.join(DIRS['BASE2'], 'figures')\n",
    "DIRS['FEATURES'] = os.path.join(DIRS['BASE2'], 'coverage', 'feature_files')\n",
    "DIRS['I_FROM_C'] = os.path.join(DIRS['BASE2'], 'bedtools_output', 'intersects_from_cutoffs')\n",
    "DIRS['BED_CUTOFFS'] = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', 'cutoffs')\n",
    "DIRS['6MA_CUTOFFS'] = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', 'cutoffs_6mA')\n",
    "DIRS['5MC_CUTOFFS'] = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', 'cutoffs_5mC')\n",
    "DIRS['COVERAGE'] = os.path.join(DIRS['BASE2'], 'coverage')\n",
    "DIRS['TE_SF'] = os.path.join(DIRS['COVERAGE'], 'superfamily_files')\n",
    "DIRS['RNA'] = os.path.join(DIRS['BASE2'], 'rna_counts')\n",
    "DIRS['edgeR'] = os.path.join(DIRS['RNA'], 'edgeR_output')\n",
    "DIRS['TRIALS'] = os.path.join(DIRS['RNA'], 'trials_tsv')\n",
    "DIRS['MEAN_STD'] = os.path.join(DIRS['RNA'], 'average_and_stdev')\n",
    "DIRS['GENE'] = os.path.join(DIRS['COVERAGE'], 'gene_level')\n",
    "DIRS['GENE_ANNO'] = os.path.join(DIRS['GENE'], 'gene_anno')\n",
    "DIRS['BOTH_U_D'] = os.path.join(DIRS['GENE'], 'both_upstream_downstream')\n",
    "DIRS['DOWN_STR'] = os.path.join(DIRS['GENE'], 'downstream')\n",
    "DIRS['UP_STR'] = os.path.join(DIRS['GENE'], 'upstream')\n",
    "DIRS['TSS'] = os.path.join(DIRS['GENE'], 'tss_6mA_only')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 438,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5MC_CUTOFFS': '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC',\n",
      " '6MA_CUTOFFS': '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA',\n",
      " 'BASE1': '/home/anjuni/methylation_calling/pacbio',\n",
      " 'BASE2': '/home/anjuni/analysis',\n",
      " 'BED_CUTOFFS': '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs',\n",
      " 'BED_INPUT': '/home/anjuni/analysis/bedtools_output/sequencing_comparison',\n",
      " 'FEATURES': '/home/anjuni/analysis/coverage/feature_files',\n",
      " 'GFF_INPUT': '/home/anjuni/analysis/gff_output',\n",
      " 'I_FROM_C': '/home/anjuni/analysis/bedtools_output/intersects_from_cutoffs',\n",
      " 'METHYL': '/home/anjuni/analysis/input_for_windows/methylation_files',\n",
      " 'REF': '/home/anjuni/Pst_104_v13_assembly/',\n",
      " 'TEST_COV': '/home/anjuni/analysis/coverage/test_feature_files',\n",
      " 'WINDOW_INPUT': '/home/anjuni/analysis/input_for_windows',\n",
      " 'WINDOW_OUTPUT': '/home/anjuni/analysis/windows'}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(DIRS) # add the additional ones below to the cell above maybe?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Quick chech if directories exist\n",
    "for value in DIRS.values():\n",
    "    if not os.path.exists(value):\n",
    "        print('%s does not exist' % value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Make filepaths\n",
    "bed_file_list = [fn for fn in glob.iglob('%s/*.bed' % DIRS['BED_INPUT'], recursive=True)]\n",
    "gff_file_list = [fn for fn in glob.iglob('%s/*anno.gff3' % DIRS['GFF_INPUT'], recursive=True)]\n",
    "te_file_list = [fn for fn in glob.iglob('%s/*.gff' % DIRS['GFF_INPUT'], recursive=True)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_CpG_tombo_np.bed\n",
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_tombo_np.bed\n",
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/6mA_pb_ont.bed\n",
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_hc_tombo_sorted.bed\n",
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_CpG_np_tombo.bed\n",
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/6mA_ont_pb.bed\n",
      "/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_np_tombo.bed\n",
      "/home/anjuni/analysis/gff_output/Pst_104E_v13_p_ctg_combined_sorted_anno.gff3\n",
      "/home/anjuni/analysis/gff_output/Pst_104E_v13_h_ctg_combined_sorted_anno.gff3\n",
      "/home/anjuni/analysis/gff_output/Pst_104E_v13_h_ctg.REPET.sorted.filtered.superfamily.gff\n",
      "/home/anjuni/analysis/gff_output/Pst_104E_v13_p_ctg.REPET.sorted.filtered.superfamily.gff\n",
      "/home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.REPET.sorted.filtered.superfamily.gff\n"
     ]
    }
   ],
   "source": [
    "#Check that the list works\n",
    "print(*bed_file_list, sep='\\n')\n",
    "print(*gff_file_list, sep='\\n')\n",
    "print(*te_file_list, sep='\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:deeppink'> 1. Comparing methylation sequencing methods <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_prob_smrtlink_sorted.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_hc_tombo_sorted.bed\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "# find overlap between 6mA from PacBio and Nanopore for 6mA data\n",
    "\n",
    "pb=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_prob_smrtlink_sorted.bed # use basecall accuracy instead of Phred score\n",
    "ont=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_hc_tombo_sorted.bed # use sites with non-zero methylation\n",
    "\n",
    "out1=/home/anjuni/analysis/bedtools_output/sequencing_comparison/6mA_pb_ont.bed\n",
    "out2=/home/anjuni/analysis/bedtools_output/sequencing_comparison/6mA_ont_pb.bed\n",
    "\n",
    "echo $pb\n",
    "echo $ont\n",
    "\n",
    "bedtools intersect -a $pb -b $ont > $out1\n",
    "bedtools intersect -a $ont -b $pb > $out2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PacBio sites:\n",
      "88932\n",
      "Overlapping sites:\n",
      "84733\n",
      "Nanopore sites:\n",
      "83451878\n",
      "Overlapping sites:\n",
      "84733\n",
      "Total adenine sites:\n",
      "85779879\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "#check how many overlapping sites there were\n",
    "\n",
    "cd /home/anjuni/analysis/bedtools_output/sequencing_comparison/\n",
    "echo PacBio sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_prob_smrtlink_sorted.bed | wc -l\n",
    "\n",
    "echo Overlapping sites:\n",
    "less 6mA_pb_ont.bed | wc -l\n",
    "\n",
    "echo Nanopore sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_hc_tombo_sorted.bed | wc -l\n",
    "\n",
    "echo Overlapping sites:\n",
    "less 6mA_ont_pb.bed | wc -l\n",
    "\n",
    "echo Total adenine sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_tombo_sorted.bed | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 362,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Percentage overlap between PacBio and Nanopore as a proportion of PB sites: 95.278%\n",
      "Percentage overlap between PacBio and Nanopore as a proportation of ONT sites: 0.102%\n",
      "Percentage adenine methylation from overlap: 0.099%\n"
     ]
    }
   ],
   "source": [
    "# Descriptive Statistics\n",
    "\n",
    "print('Percentage overlap between PacBio and Nanopore as a proportion of PB sites:', \"{0:.3f}%\".format(100*84733/88932))\n",
    "# overlap between pb and ont, divided by total PacBio sites\n",
    "\n",
    "print('Percentage overlap between PacBio and Nanopore as a proportation of ONT sites:', \"{0:.3f}%\".format(100*84733/83451878))\n",
    "# = overlap between pb and ont, divided by total Nanopore sites\n",
    "\n",
    "print('Percentage adenine methylation from overlap:', \"{0:.3f}%\".format(100*84733/85779879))\n",
    "# = overlapping sites, divided by total number of adenines (gained from number of lines on tombo file. tombo counts all adenines)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <span style='color:deeppink'> Observations <span/>\n",
    "\n",
    "Very high similarity between Nanopore and PacBio, when compared to PacBio. But PacBio sites are only a small fraction of Tombo sites, and only include highly accurate sites.\n",
    "\n",
    "When overlapping PacBio and all Nanopore (Tombo) sites, there was a higher overlap (88932) than when overlapping only non-zero PB and ONT sites (84733). This indicates PB detected sites that Nanopore did not, and these were high probability sites that were missed, as PB only had high probability (>99% basecall accuracy) sites.\n",
    "\n",
    "There are also more overlapped sites when using the zero-probability sites from tombo, compared to only using only high confidence sites from both. This also suggests that Tombo/Nanopore had missed some methylated sites."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#ff14ff'> 2. Comparing methylation detection methods <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.bed\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "# compare overlap between Tombo and Nanopolish for 5mC data\n",
    "np=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed\n",
    "tombo=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.bed\n",
    "\n",
    "out1=/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_np_tombo.bed\n",
    "out2=/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_tombo_np.bed\n",
    "\n",
    "echo $np\n",
    "echo $tombo\n",
    "\n",
    "bedtools intersect -a $np -b $tombo > $out1\n",
    "bedtools intersect -a $tombo -b $np > $out2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 370,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nanopolish sites:\n",
      "3783438\n",
      "Tombo sites:\n",
      "67308386\n",
      "Overlapping sites:\n",
      "1681653\n",
      "Total cytosine sites:\n",
      "68536018\n",
      "Total CpG sites:\n",
      "5302131\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "#check how many overlapping sites there were\n",
    "\n",
    "cd /home/anjuni/analysis/bedtools_output/sequencing_comparison/\n",
    "\n",
    "echo Nanopolish sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed | wc -l\n",
    "\n",
    "echo Tombo sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.bed | wc -l\n",
    "\n",
    "echo Overlapping sites:\n",
    "less 5mC_np_tombo.bed | wc -l\n",
    "\n",
    "echo Total cytosine sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_tombo_sorted.bed | wc -l\n",
    "\n",
    "echo Total CpG sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_nanopolish_sorted.bed | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 363,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Percentage overlap between Nanopolish and Tombo as a proportion of NP sites: 44.448%\n",
      "Percentage overlap between Nanopolish and Tombo as a proportation of Tombo sites: 2.498%\n",
      "Percentage cytosine methylation from overlap: 2.454%\n",
      "Percentage of CpG sites methylated from overlap: 31.717%\n"
     ]
    }
   ],
   "source": [
    "# Descriptive Statistics\n",
    "\n",
    "print('Percentage overlap between Nanopolish and Tombo as a proportion of NP sites:', \"{0:.3f}%\".format(100*1681653/3783438))\n",
    "# overlap between np and tombo, divided by total NP sites\n",
    "\n",
    "print('Percentage overlap between Nanopolish and Tombo as a proportation of Tombo sites:', \"{0:.3f}%\".format(100*1681653/67308386))\n",
    "# = overlap between np and tombo, divided by total Tombo sites\n",
    "\n",
    "print('Percentage cytosine methylation from overlap:', \"{0:.3f}%\".format(100*1681653/68536018))\n",
    "# = overlapping sites, divided by total number of cytosines (gained from number of lines on tombo file. tombo counts all cytosines)\n",
    "\n",
    "print('Percentage of CpG sites methylated from overlap:', \"{0:.3f}%\".format(100*1681653/5302131))\n",
    "# = overlapping sites, divided by total number of CpG sites (gained from number of lines on np file. np counts all cpg sites)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <span style='color:#ff14ff'> Observations <span/>\n",
    "While adenine methylation had high similarity between ONT and PB, cytosine methylation had only 44% similarity between NP and tombo. This is likely because NP only has CpG sites, and Tombo has all cytosine sites, so Tombo will detect far more potentially methylated sites, even those that are not CpG sites, so it will have far more sites than NP to begin with."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#ff14ff'> 2.A Taking CpG sites into account when comparing methylation detection methods <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CpG sites:\n",
      "5302131\n",
      "Cytosine sites:\n",
      "68536018\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "#check how many cytosine sites and CpG sites there are\n",
    "\n",
    "cd /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/\n",
    "\n",
    "echo CpG sites:\n",
    "less 5mC_nanopolish_sorted.bed | wc -l\n",
    "\n",
    "echo Cytosine sites:\n",
    "less 5mC_tombo_sorted.bed | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 364,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Percentage of CpG sites as a proportion of cytosine sites: 7.736%\n"
     ]
    }
   ],
   "source": [
    "# Descriptive Statistics\n",
    "\n",
    "print('Percentage of CpG sites as a proportion of cytosine sites:', \"{0:.3f}%\".format(100*5302131/68536018))\n",
    "# np sites divided by tombo sites"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <span style='color:#ff14ff'> Solution <span/>\n",
    "1. Make a file of methylated CpG sites detected by Tombo.\n",
    "2. Intersect this with methylated (CpG) sites detected by Nanopolish.\n",
    "\n",
    "I will be overlapping CpG sites from Tombo and NP, because NP only has CpG sites and Tombo has all cytosine sites.\n",
    "So from the start, the overlap wouldn't have been accurate, because tombo considers sites that NP does not."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <span style='color:#ff14ff'> CpG Sites <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 160,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "# make a file of CpG sites from tombo\n",
    "all_cpg=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_nanopolish_sorted.bed\n",
    "all_tombo=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.bed\n",
    "tombo_cpg=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.bed\n",
    "\n",
    "bedtools intersect -a $all_tombo -b $all_cpg > $tombo_cpg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 161,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "# intersect Tombo and NP CpG sites\n",
    "tombo_cpg=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.bed\n",
    "np_cpg=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed\n",
    "m_tombo_cpg=/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_CpG_tombo_np.bed\n",
    "m_np_cpg=/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_CpG_np_tombo.bed\n",
    "\n",
    "bedtools intersect -a $tombo_cpg -b $np_cpg > $m_tombo_cpg\n",
    "bedtools intersect -a $np_cpg -b $tombo_cpg > $m_np_cpg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 369,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nanopolish methylated CpG sites:\n",
      "3783438\n",
      "Tombo methylated CpG sites:\n",
      "5167951\n",
      "Overlapping methylated CpG sites:\n",
      "3708640\n",
      "Total CpG sites:\n",
      "5302131\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "#check how many overlapping sites there were\n",
    "cd /home/anjuni/analysis/bedtools_output/sequencing_comparison/\n",
    "\n",
    "echo Nanopolish methylated CpG sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed | wc -l\n",
    "\n",
    "echo Tombo methylated CpG sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.bed | wc -l\n",
    "\n",
    "echo Overlapping methylated CpG sites:\n",
    "less 5mC_CpG_tombo_np.bed | wc -l\n",
    "\n",
    "echo Total CpG sites:\n",
    "less /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_nanopolish_sorted.bed | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 371,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Percentage overlap between Nanopolish and Tombo as a proportion of NP CpG sites: 44.448%\n",
      "Percentage overlap between Nanopolish and Tombo as a proportation of Tombo CpG sites: 32.540%\n",
      "Percentage of CpG sites methylated: 31.717%\n"
     ]
    }
   ],
   "source": [
    "# Descriptive Statistics\n",
    "\n",
    "print('Percentage overlap between Nanopolish and Tombo as a proportion of NP CpG sites:', \"{0:.3f}%\".format(100*1681653/3783438))\n",
    "# overlap between np and tombo, divided by total NP CpG sites\n",
    "\n",
    "print('Percentage overlap between Nanopolish and Tombo as a proportation of Tombo CpG sites:', \"{0:.3f}%\".format(100*1681653/5167951))\n",
    "# = overlap between np and tombo, divided by total Tombo CpG sites\n",
    "\n",
    "print('Percentage of CpG sites methylated:', \"{0:.3f}%\".format(100*1681653/5302131))\n",
    "# = overlapping sites, divided by total number of CpG sites (gained from number of lines on np file. np counts all cpg sites)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <span style='color:#ff14ff'> Observations <span/>\n",
    "While the overlap between Tombo and NP didn't change after only comparing CpG sites, the nearly 50% overlap may be because NP only considers one strand while Tomb considers both.\n",
    "\n",
    "This may be resolved by using only the plus file from Tombo and comparing its results to NP."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#ff14ff'> 2.B Test out whether Nanopolish only detects (+) strand <span/>\n",
    "Intersect tombo 5mC (-) strand file with nanopolish file to see if there is overlap. Then intersect the tombo (+) strand with nanopolish.\n",
    "\n",
    "The 44% match may just be because I, for some reason, put all the nanopolish sites as (+) strand when making the original BED file :P"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#Sort the initial tombo bed files, because they were not sorted and slowed down the intersection\n",
    "\n",
    "tombo_minus=/home/anjuni/methylation_calling/pacbio/input/bed_files/initial_bed_files/5mC_minus_tombo.bed\n",
    "tombo_plus=/home/anjuni/methylation_calling/pacbio/input/bed_files/initial_bed_files/5mC_plus_tombo.bed\n",
    "nanopolish=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_s_nanopolish.bed\n",
    "\n",
    "sort_tombo_minus=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_minus_tombo_sorted.bed\n",
    "sort_tombo_plus=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_plus_tombo_sorted.bed\n",
    "\n",
    "sortBed -i $tombo_minus > $sort_tombo_minus\n",
    "sortBed -i $tombo_plus > $sort_tombo_plus"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#Intersect the (+) and (-) BED files with nanopolish file\n",
    "#to see whether Nanopolish has methylation-calls on both strands :)\n",
    "\n",
    "sort_tombo_minus=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_minus_tombo_sorted.bed\n",
    "sort_tombo_plus=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_plus_tombo_sorted.bed\n",
    "nanopolish=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_s_nanopolish.bed\n",
    "\n",
    "intersectBed -a $nanopolish -b $sort_tombo_minus > /home/anjuni/methylation_calling/pacbio/input/bed_files/nano_minus_tombo_overlap.bed\n",
    "intersectBed -a $nanopolish -b $sort_tombo_plus > /home/anjuni/methylation_calling/pacbio/input/bed_files/nano_plus_tombo_overlap.bed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The number of sites captured by Nanopolish:\n",
      "5302131\n",
      "The number of overlapping sites captured by Nanopolish and by Tombo on the forward strand of the reference genome:\n",
      "5221309\n",
      "The number of overlapping sites captured by Nanopolish and by Tombo on the reverse strand of the reference genome:\n",
      "0\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "#Compare the number of NP sites to Tombo sites on the forward and reverse strand\n",
    "cd /home/anjuni/methylation_calling/pacbio/input/bed_files\n",
    "echo \"The number of CpG sites captured by Nanopolish:\"\n",
    "less 5mC_s_nanopolish.bed | wc -l\n",
    "echo \"The number of overlapping sites captured by Nanopolish and by Tombo on the forward strand of the reference genome:\"\n",
    "less nano_plus_tombo_overlap.bed  | wc -l\n",
    "echo \"The number of overlapping sites captured by Nanopolish and by Tombo on the reverse strand of the reference genome:\"\n",
    "less nano_minus_tombo_overlap.bed  | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 372,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Percentage overlap between Nanopolish and Tombo (forward strand) as a proportion of NP sites: 98.476%\n"
     ]
    }
   ],
   "source": [
    "print('Percentage overlap between Nanopolish and Tombo (forward strand) as a proportion of NP sites:', \"{0:.3f}%\".format(100*5221309/5302131))\n",
    "# overlap between np and tombo, divided by total NP sites"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There is a 98% similarity between Nanopolish sites and Tombo forward strand sites, but no similarity between Nanopolish sites and Tombo reverse strand sites, indicating that the 44% similarity between Nanopolish and Tombo from earlier was because Nanopolish only detected methylation on the forward strand.\n",
    "\n",
    "I am currently re-running Nanopolish methylation-calling in case there was a mistake in the code earlier, causing it to ignore the reverse strand. Re-running the test run for pcontig_109 showed no difference in output to the previous one. They had the same number of sites (46801) and had the same output for the first 10 lines and last 10 lines. Now I think there wasn't a mistake because I only copied the index files to a new folder to save them back to the output folder, while a copy of all index files remained in the original fastq folder. This allowed methylation-calling to continue.\n",
    "\n",
    "Based on these results, I will intersect the methylated forward strand CpG sites that Tombo detected with the the Nanopolish site, to ensure only related sites intersect :)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#bedtools script to check nanopolish results\n",
    "#if the subtract file is empty, the initial nanopolish run was accurate\n",
    "\n",
    "nano1=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_s_nanopolish.bed\n",
    "nano2=/home/anjuni/methylation_calling/pacbio/input/bed_files/5mC_rerun_s_nanopolish.bed\n",
    "nano1_2_sub=/home/anjuni/methylation_calling/pacbio/input/bed_files/nanopolish_rerun_subtract.bed\n",
    "\n",
    "bedtools subtract -a $nano1 -b $nano2 > $nano1_2_sub"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 158,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of site differences between the original Nanopolish run and the re-run\n",
      "0\n"
     ]
    }
   ],
   "source": [
    "!echo \"Number of site differences between the original Nanopolish run and the re-run\"\n",
    "!less /home/anjuni/methylation_calling/pacbio/input/bed_files/nanopolish_rerun_subtract.bed | wc -l"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As there was no difference in sites between each Nanopolish run, the original run had no issues causing only the (+) strand to be detected and this issue is a result of the software."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#ff14ff'> 2.C Finding overlap between forward strand CpG Sites <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 164,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#Make a tombo file with only forward strand CpG to match the Nanopolish data\n",
    "grep '+' /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.bed > /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.plus.bed\n",
    "\n",
    "#This file ended up being exactly the same as the original CpG file for Tombo, but this new file will be used for added detail"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 165,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.plus.bed\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "\n",
    "# compare overlap between Tombo and Nanopolish for forward strand CpG 5mC data\n",
    "np=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed\n",
    "tombo=/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.plus.bed\n",
    "\n",
    "out1=/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_plus_CpG_np_tombo.bed\n",
    "out2=/home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_plus_CpG_tombo_np.bed\n",
    "\n",
    "echo $np\n",
    "echo $tombo\n",
    "\n",
    "bedtools intersect -a $np -b $tombo > $out1\n",
    "bedtools intersect -a $tombo -b $np > $out2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 374,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The number of CpG sites captured by Nanopolish:\n",
      "3783438\n",
      "The number of CpG sites captured by Tombo on the forward strand:\n",
      "5167951\n",
      "The number of overlapping sites captured by Nanopolish and by Tombo on the forward strand of the reference genome:\n",
      "3708640\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "#Compare the number of (+) strand CpG NP sites to (+) strand CpG Tombo sites on the forward strand\n",
    "cd /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files\n",
    "echo \"The number of CpG sites captured by Nanopolish:\"\n",
    "less 5mC_hc_nanopolish_sorted.bed | wc -l\n",
    "echo \"The number of CpG sites captured by Tombo on the forward strand:\"\n",
    "less 5mC_hc_tombo_sorted.CpG.plus.bed | wc -l\n",
    "echo \"The number of overlapping sites captured by Nanopolish and by Tombo on the forward strand of the reference genome:\"\n",
    "less /home/anjuni/analysis/bedtools_output/sequencing_comparison/5mC_plus_CpG_tombo_np.bed  | wc -l"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 375,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Percentage overlap between Nanopolish and Tombo (forward strand) as a proportion of NP sites: 98.023%\n",
      "Percentage overlap between Nanopolish and Tombo (forward strand) as a proportion of Tombo sites: 71.762%\n"
     ]
    }
   ],
   "source": [
    "print('Percentage overlap between Nanopolish and Tombo (forward strand) as a proportion of NP sites:', \"{0:.3f}%\".format(100*3708640/3783438))\n",
    "# overlap between np and tombo, divided by total NP sites\n",
    "print('Percentage overlap between Nanopolish and Tombo (forward strand) as a proportion of Tombo sites:', \"{0:.3f}%\".format(100*3708640/5167951))\n",
    "# overlap between np and tombo, divided by total Tombo sites"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There was a 98% overlap between Nanopolish and Tombo sites, as a proportion of Nanopolish sites, as this was the smaller dataset. Using the positive strand CpG sites increased the overlap from 44% to 98%."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1527,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1459311 74798 3708640\n"
     ]
    }
   ],
   "source": [
    "print(a,b,c)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1531,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAVQAAAD8CAYAAAAoqlyCAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMS4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvNQv5yAAAIABJREFUeJzsnXl8XFX5/99ntkwymez70iTd6Jq2lJalpZRSJIAKyCZaFUQRwS+yCHwVcRxFUVb54QJfQVHKLoqsA5RCS1u6l6b73jRJs++Tyez398e9KUNomzSd5E6S83698srMPfee89xlPvec55zzHKEoChKJRCI5eQx6GyCRSCTDBSmoEolEEiWkoEokEkmUkIIqkUgkUUIKqkQikUQJKagSiUQSJUa8oAohFCHE2AHI9xkhxH3RznekIYS4UQixRG87eiKEKBNC7NXbjmgjhKgVQszVPjuFEH/swzGrhRCLolD2PiHEmSebj57EnKAKIa4VQmwRQni0m/sXIUSK3nYNJrEuxkKIbUIIt/YXEkJ4I77/TG/79EYIYdVe1OuFECJi+0NCiCf0tO1EUBTFoSjKjwaxvDGKonwCIIT4nRDiqcEqO1rElKAKIe4Afg/cCSQDZwBFwPtCCEuUyzJFM7+RhKIokxVFSVQUJRH4GPhR93dFUX6rt30xRAlwmd5GSAaPmBFUIUQS4AT+R1EUl6IoAUVRDgJXoYrqIiFEnhCiSwiRFnHcDCFEoxDCrH3/rhBihxCiRQjxrhCiKGJfRQhxsxBiD7DnKDZcLITYJIRoF0JUCiF+GZFWrB1/gxDisBCiRnsBHI8MIcT7QogOIcSyHrZM0NKahRC7hBBXadtvAL4J3KXV+N4QQlwnhHgj4ti9QoiXI75XCiGmHy9fLS1OqyUdEkLUCSGeEELEa2nzhRBVQog7hBD12vld18v5HRUhhFFrLnaX8zchhD3CvqAQ4nohRLUQokm7Z2cKIbYKIVqFEI/0yNIghHhSuy/bhRDzIsoaJYR4Wzvf3UKI7xzHrsuEEJu1fCoia9MRdl2nXYcGIcSdEek2IcRzmn1bgBl9uBQPAL8SQnzhdyaEMAkhXtWuT6sQ4kMhxCkR6S8KIf6gPcMdQoiVPZ6fc4QQG4UQbUJtcs+KSFsthPi1EGKDlv6qECI5Iv1y7Tq2CiGWCCHGHeN6Haklauf/onadW4UQa4QQqRG7j9HKbdfuR+ox8swRQri0PJqEEEsj0mqFEHOFEJcCtwPf0X4Da7X0NCHEP7X9KoUQju5rq92/Fdr5Nggh/tnbzRkQFEWJiT+gDAgCpqOk/QN4Qfu8FPh+RNqDwBPa50uBvcBEwAT8HFgVsa8CvA+kAfER28Zqn+cDU1FfNKVAHXCpllas7fsCYNP2awAWHuN8ngE6gHlAHPAYsEJLswGVwHWanacCjcDkiGPvi8hrNNCq2ZULVADVEWktWlpv+f4BeF07fzvwBnB/xLkHgV8BZuAiwAOk9nLfPgK+12PbTcAO1BdhEvAm8FctbYJ2HR/TrstXgU7gVSAdGKWdz+na/jdqdt2k2fVtoBlI0tLXAI9qeZ2mpc05hq3nAZO1a3Wqtm9ZD7v+BFiBWYAfGB1x7T4AUlBrnruAvccox6rlNQrYCizStj/EZ8+qCfgOkKjt/xdgdUQeLwL1mp1m4F/AM1paFtCOWtkwAdeiPovJWvpq7RmZoOX/BvCUljYF9bmcD1iAe7V7ZdLSa4G52uffRRz3Y82GeK3MWYAtorxdwBjUZ3AV8MtjXJtHtXtv0sqfF5F21LIj0t8BHgcSUH8Hm4DvaGn/AX4CCM3Goz4DA65jehR6jAu9CKg9RtrvgPe1z98DlmqfBaqAzIu44NdHHGdAFYUi7bsCLOiR9xFBPUq5fwAe1T4Xa/tOiEh/AHj6GMc+A7wY8T0RCAGFwNXAxz32fxJwRBx7X4/0StQf19eB/wPWaj+Y64DXtX2Oma92rTqBMRFpZwIHtM/zgS4iXmioP+gzerlvH/FFQV0JfDfi+zTtPgg+E670iPRO4JKI728BN2qfb+y2MSK9HLgSGAd40V6OWtqjaKLVh2fuCT57oXTbldGjnO4X6mFgfkTaLfQuqAXA11Bf8iYiBPUox+QAYcCqfX8R+GNE+teAT7XP3weW9zh+E/B17fNqIgRNe246tc+/Af4ZkWZEFeMztO/HEtSbgGXAlKPYvhr4ScT324HXjnGeDwCvoL2oeqQdU1BRX86dgDli23XAO9rnl4E/Arl9ufcD9RczTX7UmlSGOLpvM1dLB/UteaYQIg+19qeg+vFAveiPac2JVtQaiADyI/KqPJYBQojTtaZXgxCiDfXHnNFjt8jjK4C845zTkX0VRXFr9uRpdp7ebadm6zdRf1THYhmq6M3TPn8EnKP9LdP2OV6+mahv9g0RaS5tezdNiqIEI757UF8EJ0oe6rXppgK11tDtqgkpitIUkd6F2hqI/B5ZblWP/Luvex7QoChKV4+0fI6CEGKOUF0v3ff3Wj5/f0OKojRGfPcAiUIIAWTzxXvfK4qi/Bu1dfE594nW5H9ICLFfCNEO7ER9VtMjdqvtaYv2uef17bbnWM95BZCgNfs/d6yiKCGgmmNcswieRn3O/qW5RH4rhDD2wdae/Ab15fShUF1Xt/dSbjdFqC+qhojn9zHU+wJwG+rzvUkIUS6iMOqgP8SSoH4C+FDfxEcQQtiAC1GbWyiK0gq8h9rc+QaqK6A7ZFYl8ANFUVIi/uIVRVkVkeXxwms9j9okLlQUJRm1BiN67FMY8XkU6sNxLI7sK4RIRBWUw5qdy3rYmagoyg+PY2O3oJ6tfV7GFwX1ePk2ogrV5Ii0ZEXtWIo2h1F/AN2M0spu7md+BT2+d1/3w0Cm0PzAEWnVx8jnZeAlPru/z/DF+/sFtOerni/e+77yc9SmdVzEtuuALwHnonbATtC292oPX7y+3fZEnndPWz2KorT1PFYTxXyOfc0AUBTFpyjKLxRFmYD6Ur8StbV0QiiK0qYoyo8VRSkCLgd+LoSYc7Rde3yvBNyoLqju5zdJUZRTtXyrFUX5Lmrl6xbgb0KIE7lHUSFmBFW72U7gcaGO8TMLIYpRmwdVwLMRuz+P6ku7XPvczRPAT4UQkwGEEMlCiCtPwAw70KwoilcIMRtVsHtyrxAiQSvjOtQf6LG4SHOyW4BfA2sURalE9SmOF0J8SztPsxBilhBionZcHapvNJJlqD++eEVRqlBr5WWoNZpN2j7HzFdRlDDwV+BRIUSWdn3yhRAXnMD16SsvAD8RaoeRHbgPeD7ixXeiFAp1PKpJq3mMQn2p7kVtlt8n1A63U1H9ks/1zECrZSai1sK9QoizUEWhr7wM3KM9U0WoTeA+oSiKC7VWGPk82VHdFU2ofscTGSb3OjBDCHGFdk2+jXpNXBH7XCuEGK+9yH/JZ8/pS8BlQoh5Qu3I/V/NhvXHK1AIsVAIMUnrBGpH9WuHTsDm7ny+KoQo0e5Hm5bH0fKpA7r3Q1GUA6iuhQeEEHYhhEEIMU58Nmb2aiFEnvaMtWp5BI+S74ASM4IKoCjKA8DPUH1N7agdDpXAeYqi+CJ2fR3Vf1anKMrmiOP/gzrs6kWtGbUVtXbbV25C7ZXtAH6B+iPqyTLUH/IHwEOKorx3nPyeR/VfNgMzUZvfKIrSgVo7+TpqjaFWs7u7BvM0MElr2rymHbMb9Q39sfa9HdgPrNSabX3J927N9tXa9VkCHOlZjiJ/Af6N2jmxTzv/vjbtjsZy1F71ZuAe4DKtpqOgtlQmoZ7rS8CdiqJ83DMDbd8bgYe0+3sX6su6r/wctZZ/CNXHe6K9yPfwmcsD1HvcoNm9BVjR14wURalD7cy7B1UMfwR8WWu9dfMs6outGtU3e4d2bDlwPapvvQG1o+6SHq6eo5EP/Be1Q2sr8DZH/330xkRUd1UH6n19SFGU1UfZ70XUJnyzEKK7hXkNaqfgTtRn4SU+a/KfierOcqPe1xsURTle63FAEP2vNIwstNryAVSn+KC/+SSSviKEWI3aobVYb1tGGjFVQ5VIJJKhjBRUiUQiiRKyyS+RSCRRQtZQJRKJJEpIQZVIJJIoIQVVIpFIooQUVIlEIokSUlAlEokkSkhBlUgkkighBVUikUiihBRUiUQiiRJSUCUSiSRKSEGVSCSSKCEFVSKRSKKEFFSJRCKJElJQJRKJJEpIQZVIJJIoIQVVIpFIooQUVIlEIokSUlAlEokkSpj0NkAycikrx4y6hHKC9t8GJGn/LairtZoBI+rL34C65HDXUf4CqMsyd6CumNvlKv3C2u4SyYAil0CRDDhl5diAdNRllEcBhdrnOEDR/oT2F0JdTz2sfQ5r6UTsZ9T+THwmtkTkZUAV2CbUpZ/rI/6aXKUnvp68RNIXpKBKoopW68xBFc4SoABI5DMx9AKd2v+BFDYjqmBbtT8zqjiHUdeq36v9bwBaZG1WEg2koEpOirJyDEAWqnBOBsag1hDDgBvwAH7dDPwiBj5zLRi1bR5gG7ALqHSV0qmTbZIhjhRUyQlTVo4FKAJKgYmoNUCFz/yXYf2s6xcWIAW1RgtQCXwK7HeV0qibVZIhhxRUSZ8oK8eEKqLTgSmoTWgv0Izq8xwuCMAOJKPWZuuAlcAuVyluPQ2TxD5SUCXHpaycbGAaMBuIR+1Rb2Jg/Z+xRCJqBxqoboF1wEFX6bB6iUiihBRUyRfQ/KKjgXmoPtEAam95LPlCBxsBZKAO8fIAq4GNrlJadLVKElNIQZUcoawcKzAJWACkovpEm3U1KjaxoHbEGYANwCpXKXX6miSJBaSgSigrJxE4HZiLKhaNIHu6+0D3CAcrsBX42FVKpb4mSfRECuoIRquRzgLO47MOmICuRg1Nut0BNuAAsMRVygF9TZLogRTUEYg2+H4acAFqR1MtI9s/Gk3SUEcIbAXec5XSoLM9kkFECuoIQutsmgRciDrusg516JMk+mSjugKWAyvkZIGRgRTUEUJZOVnAV1F77esZpj5SoYRJDrbZkkNtSZaw32xSgiaTEjSblYDZqIRMBsIijCEUEsZwSBhDIYzBoDAF/QaLv8Nod7eaUjoCBku0JiYYgVzUcbou1FEBcrjVMEYK6jBHa96fBSxErY0OiyZohr8hKStQn5kRaEjP9DfaUoIt8fZQR0JC2BNvCQf8ZiXQZVKCAQPhkFEJhQxKKGhS/yshYRRhYTCGMRpCwmAMC6MxhNHkN1jig8Jk9RqswU6DzdthTOyqt2S7m8zpHc3mtKZaS06Dx2jrj2skDjW+QQPwquy4Gr5IQR3GlJUzCvgakIkaCGRI1o5sIbe12HuwoMBblZ3vq07OCDQmJ4Q7g7ZQZ3tSqL3VEva3m5VAhyXs70gIezpMhPtdwwwDPhFn9Rqs9oCwJPoNlhSPIT7JbbIndRnik9qMSd6auLzWg9bihsq4wqrauNwTGVaWov0tAz5yleLrr52S2EQK6jBE671fCMxBHUfapq9FJ4Y57DcUew8WFHor88Z692Zm++vs9lB7U2qgpdEa9jYkhtz18Ypv0H2/YcBtsKe5jbZst8me1WpMzmw1pRgqrMWNB+JL6iusRRVN5oyOXrIxAvlAC/CyrK0OL6SgDjPKyskHvo5aE6pmiAQqMYf9hrFde4sndu4oGt+1Oyc52NqeGmxpTAy5q1ODLYeNKDF5Hh0GW1K7MWmU22TPbDalZlXH5Xu22qZU7UkYv7cXcU1GnTyxBFgufavDAymowwStB3828BXUGmmrvhb1jlDCjOvaUzTBs7N4gmdnbnqgqT0j0FCVFmjerUcN9GQJIQxNpvTCVnNqcZM5PbcyrsC9NWFK5bbEKbs6jYlHOx8jatjDQ8BLrtLYv2eS4yMFdRhQVk4CcAlqOL0qYnxwvj3Ybi11l0+a6V4/Osdf68/yN1SmBpr32JSuYRPNSRPXUS3mtNF15uzsrYlTDm9KnLGz0jrqaFNUs1FbEs+5Sjk4uJZKookU1CFOWTm5wDdRoyId1tmc45Lvq8o4tWPjlFJ3eW62v/ZwRrBxZ1qwddjPgfcIa0KjOWNCnSVn9IH4Yt8G+2l7t9qm7AoJU6Qbw466TMzrwBq5gsDQRArqEKasnFNQxdRNDDfxC72Hsue0rZw20bMjJcdfuz/LX7d1KDbpT5YwUG/OGtNgzhxfaR2VuCJ57o7NidO2RwirGXW9rXXAm3IUwNBDCuoQpayc2cBlqNNGu3Q256jk+g6nz2lbeerUzi2p+b6qnTn+2q0nM6RpONFkSs2tseROr7AW21Ykz91enli6UxNWgepXrQWel+EBhxZSUIcYWufTedpfJTHoL8301yef1bby1OnuT7PyfdV7cv015WZCshf7KDSZUvMPW/KmHbQWxy9LmV++NXHqHi0pC/Xe/t1VSr2OJkpOACmoQwhtLaevAjOBCmJsSJQ57Dec2bbq1LPbVowd5Tu0L8dXszmOgAy60gcaTOmF1XH5M8tt0/xLUxes1iYMpKG6Af7mKqVaZxMlfUAK6hBB68n/OurSzFUQW50W4zy7R53f8v5p4zy7Owp9lasTw57eBrhLehBCGA5b8qZWWIsmrko66+Cq5DkbvMb4BNQOx2dkSMDYRwrqEKCsnHjg26iBNmKqJz8l0GKb3/rh6ad1bEgv9B3akB1o2K+3TUOdThFvq7IWzN4XPyb9vdQLNuy0TWxAra0+6ypll972SY6NFNQYR5tGugi19zemmn0TO7ePvqjp7Vlju/ZWFPgq10s/aXRpMKUXVliLz1iZPKf2g9TzPvUb4jJQO6q26G2b5OhIQY1hysqJQx0WVYzazI8JLGGfaUHLB2fMbVuZW+Ldvyo92BJTQj+c8GG2HIwvnrs9YVLymxlfWV8dV2BEbf7v1ts2yReRghqjaB1Q1wBjIXYCaOT7qjIuanpr7uTObe6SrgPLZafT4FBjzj7lQPzo6e+lnl+5JumMSkUYnnKVUqG3XZLPIwU1BtFimF4NTECd5x0TTO/YOOHi5renjfHsK88L1OzQ256RhtuQYD9gLZn/cfLZwpV+4Wa/Ie6PrlJq9LZL8hlSUGOMsnIEcClwGsRGDUQoYc5tWXrGgtal+eO69nyUHOpo0tumkUoQg+GAdfS8DfaZOf/NuGRNizntYVcpjXrbJVGRghpjlJUzB/gycJAYGBplDXWZL256c/6Z7avjxnTtXTISp4zGIpWW/GlbbVMmv55xyeoD8aPvd5UOrZi3wxUpqDFEWTnjgetQfaa695inBxrtlza8tmCa+9PW0d79H8tpo7FFvSmjaHfCKXNcaResXZ80++euUrlyrd6Y9DZAolJWTg7q8KhaYkBMs/21qVfVv3zeVPeW/aP8lRv1tkfyRbKCjRVWt7fdovgusIc6nGXl5/1MRqnSF1lDjQHKyrEDN6IGxtA9alSOrybt6voXzyt1l2/PD9Rs09seyfHpMNjse+PHXWDdm7Ny4p23OvW2ZyQjBVVnysoxAdcCeai1U13J91VlXF3/0oIp7i3leYHanXrbI+kbQX98sqkxZSE7Jr7B3+/6o972jFQMehsg4RxgNDEgpoXeQ9nX1L1wXql78yYppkMLk6WrDXv7Okbv+wpX/elyve0ZqUhB1ZGyckr4LAyfruT4atKuqn95/pTOLetzAvV7ej9CEnPYOw+R0lrB6P3f5aJ/nK23OSMRKag6UVaODTV6VBMQ0tOWlECL7YqGfy2Y3LmtPCdQv09PWyQnSWrbejLrg+RV/5CyxRP1NmekIQVVB7TB+xcDVkDXMHcJoU7L5Q2vLpzSueWgnP00TEhpXU7JwTxs7h9StrhAb3NGElJQ9WEKcCo6h+IzKkHDpQ2vnTfDvbG52HdovZ62SKJIXKCD1ObNjNk7BRG+lrLFqXqbNFKQgjrIaEOkLgNq0Hkm1EVNb8+b3bE2ONq7/2M97ZAMAEnubeTUtpJ7uBT4FmWLE/Q2aSQgBXXwOQ8wovPCejPb100+q21l6piuvR/Kh2CYkty2kjH70knoHAtcSdliOZFngJG/pUGkrJxRwGx0buoXeg9lX9jsmjKma/8yGRR6GBMX6CC1ZTvjd5cgwqcAF+ht0nBHCuogUVaOEbgEdSaUbk19W8ht/Wrj63PHdu3ZmBTuaNbLDskgYXdvIbM+RE5NMjCXssVj9TZpOCMFdfA4FXVNKF2nll7c9Oa8yZ5ttXKs6QhBAEntGyiuOAVjsBW4irLFiXqbNVyRgjoIaB1RF4K+wYBnta+delrHemuR9+AnetohGWTifXUktzVQUDkOsAAXU7ZY6G3WcEQK6uBwLmpHlE8vAzL99cnntSyZVNxVscKIIsPwjTTs7esprBqNtasNmAFM1duk4YgU1AGmrJx01I4o3WqnQglT1uyaM6Zr327pNx2hxAU6SG6roPBQ9/jnr1G2OE1vs4YbUlAHnjlAANCtVnhqx4bJUzvLjfn+6k162SCJAeztm8irySexIwF1uvNllC026m3WcEIK6gASUTut08uGlECL7bzWpVNHeQ99Im/2CMcU9pLUtouiitNQn8mxqLP2JFFC/sYGFt1rp/Palp9W0nWgIiXULhdyk4C9YzOZDSnY2zNQQ0ZeRNliq95mDRekoA4QsVA7LfBWZp7asSEnz1e9QS8bJDGGgTCJHfvJPTwZdbZeAupzKokCUlAHDt1rp+e0fnRaga9qexwBuXib5DNsnVvJrsvD2mVDraWeR9niZL3NGg5IQR0AyspJBGahY+10QueOkkme7fG5frkmlKQH5pCXRHcV2bWTUV/6CurKEZKTRArqwDBZ+69L7dSoBA3zWz+aUeit2iTHnEqOSmLnNvJqSjAGTai11NMpW5yjt1lDHSmoUaasHANwNmokfl2Y1Ll93Gjvfl9WsOGAXjZIYpw4fzP29lay6iegvvi7kMFTThopqNGnCEgBPHoZcEb76gk5vtrtepUvGSLYOreTXzVe+1YPnELZ4lF6mjTUkYIafc5Ax1in4z27ikq6DogMWTuV9Iatq5KkdoWUllxtiwc4S0+ThjpSUKNIWTnJqP5T3cZ8nt6+elKu//AueWMlfcLmqSK9cYz2rQGYQtnidD1NGsrI3110mYTaY6pLvNMi78Hc8Z49tqxA/S49ypcMQeK8u8lqyEeEDajPbQg5LrXfSEGNLrMB3YKPzOjYNCHXX7NP9uxL+ozV30aCx0NqS/fqqHWoPf5y9lQ/kIIaJcrKyQSygE49yk8IdVomenbkpAca5VLQkhPD1llFRkOJ9i0ImIEJOlo0ZJGCGj3GoTaXdGGiZ8f4zEB9Q7zi8+plg2SIYu3aQ0ZjLoZQtx40A2fLINQnjhTU6HEqOi5vMt39aUmGv2mfXuVLhjCWoJsETzvpTd211A4gB8jT0aohiRTUKFBWTgrqA+jWo/xc3+H0Ud5D8enBxgo9ypcMAxI8VaQ1FUZsCaK2uiQngBTU6FCEjiuZTuzcMS7LX18pO6Mk/SbOX0lqa2bElmZgpmz2nxhSUKPDVHTqjAKY0LUjNynUdlCv8iXDgDh/C9YusLlTtS1dQCqQoaNVQw4pqCeJNnd/DNCmR/mZ/vrkLH+9OTXYouuKqpJhgNXbSFJ7fsQWBTWqv6SPSEE9eTIAEzr18Jd4D4xOCzTXyRspOWnifLWktmRFbGlF7WyV9BH5Ozx5cgDd/EwTOndkJ4Xaq/QqXzKMsPirSG6L9KN2AHmULU7Ry6ShhhTUk2c8OgVDSQh1Wgp9VWnpgSbZuy85eeICHVi9QRI7Iufyh4FinSwackhBPQnKyhGoPiZd/KejvIfyk0NtLWZCQT3KlwxDVD9q5PCpTtQYFZI+IAX15EgBEgFd1mzK8ddkJQdbdYsdIBmGWPxN2Dsim/gdyBpqn5GCenJkouP40yJvRVp8yFuvV/mSYYgx2ILNnRixJQDEU7Y4SS+ThhJSUE+ONL0KFkqYXH9tqj3ULodLSaKHxd9EfFfSUeoJWUfbXfJ5pKCeHPno1CGV76vOtIU6vTIYiiSqmMJezMEgCZ7IGqmCOppF0gtSUE+OXHRaOyo7UJdtD7VL/6kk+pgC7SR4Inv63aiTVyS9IAW1n2gzpDLRqYaaGmhOTgx1duhRtmSYE+fvwOqNdGe5gVFyXn/vSEHtP3bU66dLp1SOvy7BpAR0CxcoGcYYgy0kdkQ2+YOABXVuv+Q4SEHtPyno2MOfGmyxJ4S6WvQqXzKMMQVbSPDYjpIie/p7QQpq/0lCp+tnVIIGe6g9PiHcKWuokuhjCHuwBOKOknI0kZVEIAW1/8SjUw01I9CYHBf2+UyEZfxTSfQxBzwYgz0X6RNIQe0VKaj9Jwl10POgkxxsS4oL+3RZHUAyAjAqAYwhMAZNEVt9QPqxDpGoSEHtP0mozvpBxxr2Wi1KQJfprpIRgkHxE+dLiNjiR3ZK9YoU1P5jR6caalzYF28O+3UpWzJCMIR9mAPxEVt8qB2xkuMgBbX/JKKToFoUvyUu7JM1VMnAYQj5MAd61lCloPaCFNT+k4hOTX57sN1iJOzTo2zJCMEY8mH6XMdUEEigbLHpWIdI1KU7JL0ghPgb8GWgXlGUKdrmBKD109svP7/ug39fcda/yu+wj5vqPvDMg+P3/PHnN5kSk5sAkifP2jjzT2+9BbD5rq8vaFq95GxQRPrpCz+e9uBLHwCsv/GCr7bv2DgdIRRTQmJ76f3PPZMy7cy2ug/+k7Pj/h99x9dUOyr73Etfm/7Iq+8D2EIe89W/emfext31ixLijB2Nf1ngHPyrMnA0dvhNU3+68s5gWDGFFYyzSpI3uO6a+Ubhjz+60xcIWwE6/WF7YWrcgZ0Pnv2XUFhh3n1rr95W5Z5qNgn/A18/5Znr5uUfAjj3t+suL6/smIqCmJBn277857NfMhoEbZ6A8fzfr79mT63nFCEIf3dewWsPfeOUTd023PH8zlMfeafiB3+9fvJvvze/oALgkkc3li3b0TLXIAjfemHxi7+4dMx2fa7QifPahrrsG57edkP391ZPMOOy07Jff+lH0z4AuPyxT8//9/q6K8p/e9YdUwvt7lteWzf+vxve/lKXP9QVRjE0B9y5+85+5OGShEyzEOJm4PuoPf9/VRTlDwB4sxayAAAgAElEQVRCiOnAE4AVVYBvUhRl7aCfrI5IQe0bzwB/BP4Zsc3YvGF5SvvOTZOMCfbPzamPzx219+w39vwxcluN68W8ptVLzp7z7633m2xJwRVfm/zj+mVvbsk658v1k+594r2E/JLXATbffc2CnQ/c+uUznlvzXHx+ceeYH/7yxbr3Xp4RmVdc2Gu6ct7ozdeflfHanS/sum5gTlk/Um3m4NpfnflIYZrV1+ENGk+58+M7/7zk0NbKx+Y/2L3PxLtX3HjOhNRPAX71n31Tatt82U1PLPj5k0srS+55Zfc3rpuX/7vH36sYvae2c0z1/zvHCTDm9o/veuSdg+PvvLhk99V/LL8oJcHc0fLkefcGgmGxs6bzyJCgisauuJfX1C3ITYk70L3tlbW1uav2tM468Oi8X67b35Z81eObb7vr4uJ7rWajbpM7ToRLZ2bXXToz+9cA3kBIpN+49IEfLCjYBLB8Z3Pqpor2SXar8chz/P8uP3Xb/xs7fR3VBZvu2fNK6XOHVy4sScj0Tlxx12RUMZ2N6gZwCSHeUhRlD/AA4FQU5R0hxEXa9/mDfKq6Ipv8fUBRlOWo65QDRyL1i+333XjV2Jt+9Sqi9/GobVvW5sbnF++Py8jxG+MTwoljJu+ufu1v0wES8kuORIwKeT0WhFAAkibM6Ci8/PsVwmj63AKABhS+fs7oysI0q25LVw8kRoOgMM3qA+j0hoyhMEaD+GwaeUVjV9yBes8pd3+55FOAd8obp188LfMTo0Fw08JRB7z+cMKafa3JBiEIhBRze1fI1NoZNIcUxTg6K6EdYOXuljnP3DDlHQCzyaBMLbQfGYa26C/ll3zn7Lx3TUZxxEe+eOXhaWeNS1mXajMHvzQ1oykt0dzw9EfVJYN0SaLKQ28dnJhqMzcsmJTeDHDj37df9avLx75K5LhqoSgIxQDwRv3GWeemTVoLKE2BzonAakVRPIqiBIFlwGXaUQqfzaZKBg4PzhnFDlJQ+4c48MyD48xJaa15X170hQXyvLWVo5fOz7x3+cVjbql99+VcgNQZc6o9lfvGu/dus/kaay0dOzdN8TfVHQlAsf4H51+65Ez771o3rTx90r1Pvn7cwpWwQSj6TXsdDLyBkMi6aem9hbcue2hyfuKOG88rPFJbvP+N/TNGZyXsLMlM8AK0dAZSxmQnHJmGm5RgatlS6U65+fxR+yfm2XYV/vijB4tuXfbA1AL79stnZdfuqe2MB/j6n8ovyb75w3sm373ihg0H2uwAz3xcXdjoDqTdd8W4LZH2NLT7U/NTrUfKSLOZW/Y3eIZkJ81/NtTPOmdC2lqAn728uzQt0dy6aE7e559jQRihiFpfq2W3p3bKXSUXbwTEmPisHcA8IUS6ECIBuAjoXjLlVuBBIUQl8BDw00E7qRhBCmo/WHHppPhDL/1pztT7/vEF4cs+/4pDZ7+556cLPmr4dd7Fi5Zu+/UPbgLIXnh5bfZ5X3Ot+/6CW1d/Y9Yt1uyCKmE0HpnpdNqT77+28JOO/02ZMWfNnj/cfe5gnk8sYjUblfo/L/j19t/NuXtfvaf4xdU1ed1p729tmn3RtMzj+uaEQHnr04bM6hZfzt6H592995F5d2+vdp/yyDsHx3kDYaPbF0qdPTp5b92fzv3NpILE/dc/te3KQDAsfvHq3qse/9bEV3rmd7S3l0AMuZdamydg3F7tLr39wqINta0+yzMfV1/0jxumHv0Frgjld/vfLB1lTd87MTHfA/DJGY7dwO+B9wEXsJnPOmd/CNymKEohcBvw9ECfT6whBbUfeA7tGR1oaUxZdfWMe5eclfTbkMeduubbZ93TtnVdUkJ+ideaXegDGHuTc6sSDhk79mxJBJjifHrluR/W/eac9yofMtrsndbcUXU98y765o/Xtm1bf9y10BVhCCtCv6WrB5NxObauyfmJu19ZUzcZYFuV21bd4iv+36+UHKlBptrMrfvqPEcGnbd7gqlTChLbnl5WNeOU3IQDhWlWX2Ga1Te9yL51xa6Wkkn5iW6TQfjvv2rcpwD/c/6oDZXN3lGHmrzWhnZ/3tce23RH0veX/PZwi3f0jxfvvPmpj6qKspIsLdUt3iNlNHcGUksy43VZnPFk+N0bB6bkpsQdmlmS3PHhjubM1s5gxoyfr7o36ftLfuv2hVLP+tWae9btb0tCUbXB1VQ+64KM0nXa4QoQVhTlaUVRTlUUZR6qK2yPlv4d4N/a51dQ/awjCimo/SAcDGxZuNr96MJV7T9buKr9Z8aExJbT/7nqN8lTZrW3bV2XpIRUl2flv54sRlFE4uhJboD2nZvsAM3rPkpr37Fxxpgb7l0HUL/szSPLS1T9+6lplvSs2uOWj0BBDFtB3XyoPbG7WV7f7jdvqeqYODHfVgvw+zcPzJyYZyvPsFuODFkrm5q++a3NDWeGwgp/XnKoJM5s6Dp9TErbqHRr87Yq93iPL2To8AaN26rd4yfmJ9YaDYIJebbyR1wHxwMsXlUzISfJcnhMdkJX19/Ov6P9rwt/1v7XhT/LS7Xuf2zRhD99b35BxTfPytu8ak/rrJbOgOm9LY3pTe5A1vXz8w8c/Qxilzc/bZh93uT0dQDXnJlb7fnb+T/pPt/EOGPLql+c/ptZo5PbUYSo6XSbDnY1jP9pyVc+1Q4XQFgIkQUghBgFfA14QUs/DJyjfV7AZ0I7YpC9/H1ACPECam9lhhCiCnBcsFkJoT5gn2v2VSz+w6mNq1zzMRhCBqM5MO6W3/5VGI0AbLzlqzeGvR4bBkOo5No7X7AVn+IB2P3onV/beu+12QihmJNSm6b8+u/PAbRtXZe07nvn3hMO+K0IoSw5I3HhWa9ucfjM1uBXfuG6ZPvBphxvIJyY+L0lv7/6jJzXn/7elJWDeFkGjK1V7uTbntt1naIoBkVBnD4meX23T/Ojnc2zrj8n3xW5v+NrY7e8v61pSvqNS+8zGYX/d1eP/wfA764ev2HVnrYJhT9e5gCUyQWJ235z5bhygMe+NeHV7zy59bsPvX3walucseOZG6b+43g2XXV6Ts2zKw+vL7lt+S8NgvBtZUXPD5Ue/m5qW32WPXWeia/dOmNx73sL8Zcda4rHJeRsz7OmRk4iUYBXhRDpqBNbblYUpdu3/H3gMSGECfACNzDCEIoypJ6JmKGsnLtR1ywf9BlLFzS9M+fKhlc8Bf7Dm3rfWyLpBw3p57J5ei21uTsito4CHLgWhY512EhHNvn7jxudavhukz0QxmDRo2zJCCFktBA0RS4AaQI8UkyPjxTU/uMGzHoU7BcWv88QJwVVMnCEjXEEzJELUFoAGdC8F6QPtf90oC4jPej4DHFdfoMlrfc9JZJ+EjZY8FsiBTUOODIqxel0CqAImAh84HA4ZLAepKCeDO3oVEP1GqzeoDAfbYkKiSQ6hIUVvyVyRd+4Bqvf+2en83XUuKgTUfsP1gKfoENfQiwiBbX/6Caobabkdq8hTi5HIRkYQsJMyKgQMkVGU7N4zME24EzUMH4HUcecvo7aWpMgfagnQ1fvuxwds6fNEudu7rlmT59pNGe0+QxxcUEM8v5Jok/AnEDQ1DM8pFLUkVAHjAY+RB1zGgb+DNQ4nc4HnU7nmEG2NOaQP8j+0476QJ0wpR/99SdTlz19Z38LDglTuMOY1NVptMklKSTRJ2xIJGD29tiqAJ0Oh6MDNZTlIdRB/LOAs7T01U6n8x2n0/kVp9NpHFSbYwQpqP2nFU58+mdmxaeFJn9XringzUqu39/vRc9aTKkdXYb4IRmcQxLjBE2peGw9F4EUQBuA1gH1HdTaqXA4HHsdDsddqONUXwDuAfY5nc7/dTqdWYwg5MD+flJWjgFwoDZ9+nwRZ7z72I9FOBinGExef3xS7bZ5173cn/IXNr835+v1L3kL/NUb+nO8RHJMWlLmsHmal6rC7mfLhLri6X24FvXpWXc6nTOBm1Cnpr6FKr6fOByOYS04sobaT1ylhIEGIL63fbvJ37l8kingzQgZ49rb04s2JbZUn2ntaEzo/cgv0mxOb3MbbUm97ymRnCA+ix2vNTJoeiJQ0VcxBXA4HBscDsf1qD7XDahB2pc7nc7SqNoaY0hBPTlqUJdC6RMWb4e9etyc5wxKMN5rT2/oSCtYkVK3N6/3I79InSW7tsOYJH2okugTMCfRaWuM2JII7OtPVg6Ho8XhcDwKTACeBZY4nc6HnU6nPQqWxhxSUE+Oak6ghnpg+sVrDp8yd6chFEwIxNm6tp997au1Y8/Y25+CD1vyGt3GRGuXiOv3aAGJ5AsEDVaCZhNdCZFDoQRw3AhoveFwOMIOh+P/gCmo7oPtTqfzCm2CwLBBCurJ0dz7Ll/EEAra/Fa7p/c9j40iDNRYcpo7jEm5J5OPRPI5/JZ0uqztPbYKVPfWSeNwOOodDse1wDeBXwLvOJ3OsdHIOxaQgnpy1HOCPf3GgNdoDPmTO9IL+yXGkRyML272GONHVC+qZIAJmtJxJ0bWTi2oQVF6iuxJ4XA4lgMzgA9Qh1s5nE7nkG9tSUE9CVyltKIGSenzNNDk+v2ZIVNcc8hsPemoPfXm7IZWU4qc0y+JHn5LKm57S8SWRGBAAmk7HI6Aw+F4EDhV+1vudDozB6KswUIK6smzm89WeuyVxJbDOQFLwheWPukPh6yjqjuMSal+TLpMgZUMQ7zWTNqTqiO2JALbB7JIh8NxCLgUdZ2qFU6nc0iuJgtSUKPBXk6gYyre3ZgdsNpPysHfjcdo81daC1uazWmjopGfZITjMyfhsxpx25t6pBwc6KIdDoficDjuAf6IKqrTB7rMgUAK6slzQuJo6WrP8dpSolJDBdiRMLGm3ZRcEK38JCMYX1wBrcmRnU92oBrXokFbjNDhcDyOuhz1e06nc8FglRstpKCePI2oocv6NHfZ7HVne5KOvwjfiXDQWnyg2ZSa06+gAhJJJH5LNq2p9RFbUoCNg22Gw+F4BbgaeNHpdF412OWfDFJQTxJtxtQ+ILkv+5v9XdntGUVRq6E2WLLa6izZ/hZTqhw+JTk5vPGZtCVXRmwR9HNA/8nicDg+BM4HHnE6nf+jhw39QQpqdNiC6rw/LvHt9TYUxdiZkhvVISi74ifUtBuTi6OZp2SE4bOk0WUN47F1N+/jgWZcixqPd9hA4nA4NgNzgbucTueFetlxIkhBjQ4V9GE8anLDweygJb4OEd3Lvt02aXe9JaswFO2MJSMHn6WA1pRI8UxDh+Z+TxwOx0FgEfA3p9MZ860w+QOMAq5S2lCjTh23lprQXpcTsNqi1tzvpjYut7nCWtTZaM4ojnbekhGCJ6GQpoxDEVtMqEMCdcfhcCwDngSedTqdMa1ZMW3cEGMj6lo7x8Ta2ZLtj0+KWodUJJ8mTq9oMqePHoi8JcMcn9mOJyGRpvSD2hY76uiVGv2M+gL3oc7aultvQ46HFNTosYdemv1xnS2jPPaswwNR+M6ECbvrzVmZHmHtVzhAyQjGZx1LY2YtiqF7sEgqsOxEwvUNNA6HI4g6//9Wp9N5pt72HAspqFHCVUoj6tz+ozb7LV3tcZau9jF1JTN3DkT5HqPNvz1hUk2zOf2UgchfMoxx2wppzNivfTMBAWKkuR+Jw+GoBG4AXnA6nTG5WoUU1OiyjmM0+3P2rZ3oj7cf8NlSe67VEzU+tc/YcdiSN1Z2Tkn6jM+SiifBSnNa93CpbGA1rkUD9pyeDA6H47/AG8Bf9LblaMgfXnTZpv3/QtM/uWH/VHdqQflAFn7IWlS3O2G8u86cLWupkr7RZR1LQ2a19sgK1BrqOn2N6pW7gAVOp3Oi3ob0RApqFHGV0o46JjUjcrsIh0R8R+PUhlHTBlRQAdYknb6tJi53opw5JekTHlshjZndzf1MYDOuRScdWnIgcTgcXag11Fv1tqUnUlCjz1p6LIuSeWjzKMVg7GrNGReVIL3HY0/C+EMHrCXBBnOm7PGXHB93QhFtSWHaUrp78+OBVXqadAL8BbjK6XRm9LrnICIFNfocAlqIENW0wztKO5OzB7x22s3qpDN31FpyJg9WeZIhiidhAtUF3Z1PWcBOXIuq9DSprzgcjjrgVeBGvW2JRApqlNHm9i8notlva62Z2pIzftAEdZtt8p791jGmWnPWmMEqUzLE8FnS6EhKoT5rJ6oOxAPv6WzVifIH4Gan09nnAO8DjRTUgWE7EAaMic2VyaaAN7OuZOb+3g6KFoow8FHq/E3VcfnTZI+/5Ki4bVM4nLufsDEM5AKf4FoU9Vl8A4nD4dgKlAPX6G1LN/LHNgC4SnGj+lKzsw9snOpNTN8WNsWd9JInJ8KuhAkHt9kme2osuVMHs1zJECBgtOJOzKc2dxvq7CNQW1VDkUeA22Jl9VQpqAPHKsBkbzp0ant60aA19yNZljJ/fWVc4QQfZkvve0tGDJ22KdTlHMZn9QA5wPvRXoRvEHkP9aVwht6GgBTUAcNVSnP2/vU1Fq+7uGrCvE/1sKE6rqBxg31mbXVcwUw9ypfEIGEMuO2jqcndhtpx2knsjzs9Jg6HQ0Fdi2qu3raAFNQBpWjLu+c25U9cE7AmBvWyYXnKOesOxBePajEmD+nVJCVRosM+nfrMZjqSGlFnRb2Na5FPb7NOkk+QNdThjdPpHG8MB8+pHj/3SdRmlS60m5I9H6Sct+WQddSZsoNqhBMwWmlPHsehoo2oYrobdSLKUGc1cGYs+FHlD2zg+F/gca89YwnqdD7drvXGpNO2b7ZND1Zb8ofkSpKSKNFhn8nhvGrc9i7U5/E/uBYNh0l1B1HXdCvU2Q4pqAOB0+ksBi4BHneV0oz6BtU12vh7aV9asT9+zPg2oz1dTzskOuEzJ9GeXMihUZtQn8V/41rUqrdZ0UDzo8ZEs18K6sBwF/Ckw+Fo0b5/BIQA3QYgN1oy25eknrfloLV4bhCDvO8jjQ77TCoL9+KzJgMbgK16mxRlViMFdfjhdDrzgK8Dj3Zv08alvoWOvlSA9Umzt621z/YctBbP0dMOySDjsWbTmpJJVcE+wIfaERUzwaOjhBTUYcqdwDMOh6NnIJRPUdedSht8kz7jnfSLlm1PmJRVI0P8jQwUoMM+i4qiHYRMKcDLuBZ16m3WALAOmKb3NFQpqFHE6XQWAt8BHuqZ5iolBLwOJNGHFVIHCo/R5n8945LlexPGTZf+1BFAu30q9dmCmjw36rIm+/Q2aSBwOBydQB1QoKcdUlCjhNPpNAHPAw86HI6jrhvlKqWSGOigqrIWNryddmH5fuvoeX5MZj1tkQwgPnMSrSmT2TPuAIhtwBK9TRpgOlGDvOiGFNTo8QugC/h9L/stRe2g0vXGb7LP3LEi+ezmffFjFsjxqcOUtpS57BvbiMe2G/gXrkW6TTAZJDz0iEU82MgfUhRwOp0LgO8B33I4HMcd16d1UP0btYNK14HIrvSyj9cknS72W0efracdkgGgPXEyNTmJ1OR+CiyO1TWiokwXUlCHNk6nMwt4FviOFvS2V1ylbEMdupI3kLb1RkiYwq9lXPbBJvupKQfjRs3S0xZJFPGZ7bSkTWff2C0ohmeGy3jTPiBrqEMZp9NpAP4J/MPhcLx/goe/jfoA2KNu2AngNcYH/pV5xfvlttJR1eZcGeV/ONCaeg77S6rw2J7Ateio/vxhihTUIc4dqL32jhM90FWKB3gJSEedNqcb7aZkz6tZVyzdljhlcq05a5yetkhOkuaUWdRnGanN/ROuRbv0NmeQkYI6VHE6nWcAPwGucTgcgf7k4SrlIGo8R93nINdZclpezrzqw3Jb6Ywac/Z4ve2R9IMOWzGtKaM4UPIEb107VBbbiyZSUIciTqdzEvAKcKPD4ag4yew+Bvaisz8V1OFUL2R/44PNidOmHzbnTNLbHknfCfjjU2hPmsmB0a/y8s1v6G2PTkhBHWpoNdOlwM8cDsd/TjY/bcD/y4AbnWdRAdTE5TW9kP2N9zfZZ0yusuTL5VOGAO2GxOQtqacs3GKc+RF/v+v/9LZHR7zIcahDB6fTWQa8AVzvcDiejVa+2lCqZwErOr9hQW3+v5h1zXufJk4/pSKu8DS97ZEcm3aDPW2HbWKZK/3C9Xd+6Ue/09senUkCOvQ0QApqH3E6nd8A/gFc4nA43op2/q5S6oDnUAP/mqKd/4nSYMlqW5y96J3VSWfk7Yoft0BGqIo96k0ZxZsTS8v+m37JuuUp5zhcpQy3gCcnSjrQpKcB8kfSB5xO5y2oM6DOczgcA+bsd5WyB/gvMAqdB/0DtJpTO5/LXvT2RynzDTsSJl3kEVbda88SlUOWwhnr7aedtTjnW59sTDrtt65S+tUxOszQXVB1rwnFMtqSCr8GrgTmRqEDqi+sAbJQQ5EdHITyjovfEBd8LeOyJc2mtFnntyy5eGzXng9TQu2Nets1UgliMOyLH3POevtp2W+kf3VVqzn1IVcpQ3XF0mgjBTVWcTqdScBjwBRUMe0Zjm9AcJWilJXzNpAITAIODUa5x0MRBpalnruu2Zze/uWmN88b27Vna76/Zpvedo002g2JyQfji+d9nHy2cKVduDJgsPxJWxFCopII1OtpgBTUHmi10quBh4F3gAUOh2NQHd2uUoJl5byCGqh6PFA5mOUfiy2JpbsazJkNFze9dfbUzi15Rd6DH8crvpEwR1x3DptzJ+6PHz3t3bQvHVyXdHo18FdXKbV62xVjLHA4HNV6GiAU5Yt+bCGEAVVsdffjRRBUFCU0kAU4nc7xwJ9Qm9w/HEh/aV8oK8cCXAOMJUZEFcAc9hvmt3x4+tltHxeUeA+szgw2xYxtw40uEWetsBbP3ZYwKfHNjK+sr4nLMwJ/d5WyV2/beiKEEOi8IOUgElIU5QvRuz4nqEIIc25ublpqampSamoqJlNsVGCDwSCdnZ2iqamps7KysllRlK5o5u90OuOBnwI3Ab8BHnc4HDER6qysnDjgm0AxUKWvNZ9nvGdX0cVNb50+tmtPdaG3cq2FoOwYiSL1poyig9aS2SuS51QvTT1vS9BgTgee04LrxAxCCGG321NycnJS0tPTTXFxugbNH3DC4bBwu91Kc3Ozt7Gxscntdnu6044IqhDCPG7cuILZs2eL4uJir9kce3GHq6qqLJs2bbJs2LChSlEUT+9H9I7T6bwQ+CNq9Kfb9G4yHI2ycqzAItTe/5gSVXuw3Xpuy9IzTutYn1nkO7QpO1AfczWnoUaniE+stI46Y2/8mJT30i5YtzvhlCYgFfinNhIkZhBCiOzs7OyZM2cmTZgwwZOUlDQclqXuEzU1NebNmzdbN2/eXO3xeNwQ4UPNzc1Nmz17thg3blzM+sQKCgr8CQkJoY6OjlwhxH7laP6KPqD5SWcB/wuUAjc7HA5XNG2NJq5SvGXlPIfq2x2H2lEVE2MOO0xJ3tczL/1oW+KUgoXN7886xbNrTIGvanVS2N2mt21DjRDCUG3Jn15hLRq/MnnO/lXJcz7yG+ISUQesP+UqZTBGmZwoCaWlpcmzZ8/WdUC9HuTm5gZsNluotbU1R9OjsAlUn+mUKVPsxcXFUan1DSRpaWmhvLw84+7du+NQp5r1GafTmYZa0/se6oykJ4FvOByOmH2JdOMqpUsT1a+gvgwOoUb+jwn2xY+tOphXfPjMtlXTz25bUVbkrTiY46vZbMUf89c2Fqg3ZRRVxxWcujlxWtcHqee902DJakMdBmQAnnSVUqOziUclPz8/qaCgwK+3HXqRlJQUHjVqlNi5c6cV8HTXUM2pqamGWGzmH4309HQFMNMHQdVils5HFdGLgDeBW4DlvUXXjzVcpQTKynkNaAcWojb/Y+ZhDglTeEXKvI07bJP2nNW2auYM98ZL8n3V+3J9NZulf/XoNJjSR9XE5U49YC2xLEuZX77dNrl7Eb1s1Aj0z7hKidlxvzabLSEvLy+qfRpDjbS0tHBSUpKFCEEVZrM5JpqQfcFoNCpxcXHH7Ul0Op35wLXA9ajze/+K2rRvGXgLBw5XKWFgSVk5rcDlqCs9xlTLosmc0fFGxlc/Wps0O3VO28oZpe7NX8v3Ve/O8dduMROKic4+vWk0pRXUWPJKD8QXW1ckn71ti23qLkVd2kughnOsBl5wlRLT0faFEEOmIjZQGI1GxWw2G6CXcaiffPJJ6vvvv/8SgKIoWUBICNEE8P3vf//i3NzcftU6HnzwwbuMRmPz7bff/lR/jgeIi4v73JAup9NpBKYDC4DzgdNQozhdBWxwOBxD5oXRF1ylrC8rpw34FmpQlZgb4F1nyWn5d+blS9cknZ5xVtvK6ZM7t1+e7a89kO2v25qgeGPqJTAYhBCGBnPmmHpz1rgKa1H8yuS527ckTt0VEqbulpIFyEedLfe2qzR2Wh/94Ze//GV1fHz8k3ffffevAB5++OEbw+Gw7c4773xYT7vuu+++fxUXF/9q0aJF5b/5zW+eLSsru3nmzJlHnW3mdDrXLFy4sGzOnDl9qogdV1DPPPPMljPPPPNLAA8++OAdBoOh84477njixE9h4HA6nYuAy1Cb9XWoofWeAN7V1uoetrhK2VNWzp+Ab6CuRx5TIwC6qY4raHwl6+olH/tq0k51b5w0vePTS7IDdXUZgYYd6cGWmPQNRpNOEW9rtGRMrLXkjj5gLfGst5+2a4dt4r4IIQV1KZw01AUc1w+TQCc+r9d70cqVKx/vqyANNvfcc8+3oplfvweaPvzwwzd1dnZeAWC325+97bbb/v7f//63ePPmzX8zm82bAoHAqWazuTw9Pf3fdXV1tyuKkjZ69OibFy1aVA7g8/mm3Hfffa+Ew+HclJSUP95yyy0vejwe8fjjj//C5/OdAygZGRmP3HTTTb1FdioA/gP8j8PhGEnr5wBqlKqycp4AvgrMIMb8qpHUxuU2vx138YplKedYp7rLJ87s2HBWnv9wKNPfcCg12LLXHu4cNnPSg15lwc4AAA0JSURBVBgMjeaMkhZTakmdJTuz3FZatdk+fWl1XMHR/KHZqB2MT7pK9Z9qHEVCiYmJi1euXHnDnDlzPre8+hNPPHF+Q0PDjwGzEKJl7ty5P5o/f37jgw8+eEcwGMwLhUJFoVAoPykp6a+33Xbb3wAeeeSRG9xu99cBEhMTn7/99tufeuuttwo2bNjwvNls3hgIBKaYTKb9V1555S3jxo3zPvXUU3NramruVRTFZDabP/3Wt771054daN010PT0dO+rr776ZCgUygWM6enpf7j55ptfB1i1atV3ly5d+iVFUUyTJ0++4fLLL9/HMeiXoD777LPTOzs7L7vmmmsu8nq9xv/85z9vv/jii6vj4+O7wuHwmLFjx/7g4osv3vvQQw+929TU9OV77733kj//+c8XHTp06EfADQDBYHDit7/97a9WVFQkfvjhh+9+9NFHS7Zt23ZWIBAYf+utty7csGFD+vLly99evnz56nnz5h0z4IHD4RjpMSC7RwC8DBxAHQXgBmKyRgDQaUz0rk4+a9OapDM2jfbuLzjFs6t4Uuf2i9IDjZ2ZgYZDqYGWvTala8i1LoIYDE3m9OJWU2pRkyk9p8Ja1LrNNrl6u23Sco/RdrSXnAm1QrAfeMVVyrAbarZw4cJnXnvttQ82bdr058jtM2fOXDt9+vQvm81mHnvssWvWrFlz0/z5838FEAwGx1533XVXVlRU2JYsWfJxQ0PDP997772Jbrf76quuuuricDgs/vWvf731/PPPr05OTm4Nh8NjcnNz77j22mvX3X///Q+//vrr115xxRV/r66ufrS0tPTqyy67bP/999//2Msvv/ztY7kZly5deq7BYKi75557vg2wadOmI4tnmkym5nvvvfeCP/zhD9/ZvXv3D1GXPjoq/RLUhoaG0xMSEt7uHrP6zjvvuGpra2eXlJQsMxgMB6+88so9miF7kpKSPgbIzc3d2dTUdGTtpPj4eFdRUZGvqKjIt2LFijU7d+6c3tHRMTs5Ofk/SUlJ4XPPPbfhk08+Wbdjx45p8+bNW9ofO0cSWhNxbVk5lajjVbsnAcTsSAZFGNgXP7ZqX/zYqnfTLjCM6dpXOKFzR8kEz66vJodaO9MCzfWJIffhlGDr4VjtzGo32NM6TIkFbabkrFZTSsahuKL2bbbJVXvix61tNace76WQAiQDLmCVq5SYPL+TZdq0ae733nvvlQ8++OB6IcSRUTn79+/Pfffdd58Ih8NZgNlgMByZvhwfH/9BQUGBv6CgwP/BBx80bdy4MbOurm52fHz8OxMmTOjS9nm7rq5udnJy8ntCiMPXXnvtOoDMzMx/19XVXb9q1arlRqOx8rLLLtuvbX+lrq7uWuCogpqbm7uzoaHhFw888MA9mZmZ71933XVru9OmTJnyDkBaWlr5oUOHLjze+fZLUBVFOd4cf1/E57DBYPABGAyGsKIokat7fs5HJIQYDj4j3XGVUqO5ABYAc4FW7S+mCQlTeHfCKRW7E06peEsJGoq8FbmF3kP547r2Ts3x1861h9qbUwMtTdawr9EWctfrUYMNIQztxqR0jyEh222yZ7aaUjJbTKnhA9aSxgprUfVBa/EnvYgoqCvc5gONwJ9dpcTczLxoM2/evKdcLte7CQkJL3Vv27Nnz30ZGRn/d+ONN7739NNPn1lTU3NHd5oQIrI2HwoGg0aOH1ekp3Yovez/BS677LL9eXl5ZWvXrl1QVVX1s4ceemjZT37yk0cBEhISujUsRC+a2S9BzcrKWn3w4MHf79+//y9er9fg9XovKCoquvFE8ujq6iqrqqr6c0XF/2/v7kKjSu8wgD/vOXPmK5lkM840XWOaaAyJ2zaFLU3EUrFetLMEcaGsoiLC3lvoXW/WFr3qRUSwINsrU0ItvVhIqTJQKYYWXRY3G8+u2a0xcWUnX04ym8xkPjLnqxfvDIp13SQencnk+cFhmJAcZkLyzP+c9+P/sM4wjN6urq7f3b17N7i0tHQknU5/MDo6GjYM4yd79ux5byOvcSuL96AA4FpMx6cA3sbjfQA2RRVkCY89FeiYngp0TI80/RxBK+ttKzzcsX11urlldbrrO0ayr97KOAErv9xgZZa99mpGc4yM317N+O185kXmvFoQSkH4gwXVHzKFVlcUWmNWrW/IqnWhvBIIfe1pys34Xl966G9PfuVrHS1NwF+rJshVT/8C8O/NPoq/Vn19fUsjIyN/z+Vyx4LB4F8BwHGcUCgUmgWAR48evfNt52hubv5wcnLywsTExB8tyxL5fP6t3bt3/7p0rpbBwcEfnzp16uNkMvl2IBD4aN++fffv3bvXOjw83H748OEvk8nkrwKBwK1vOv/169ebOzs7l06fPv3BpUuXsqlU6uhG3uuGAvXkyZNjAwMDw0NDQ9cAIBQKDR49evSL4eHh9rWeQ9O0scuXLw/Ztr29qalp4MCBAwu9vb3/uHjx4psXLly4DsCJRqO/f979U3q+eA++ium4BLlZ9S8hrx4qul/kRuTUuuLndW9MfV73xlT5a01Gqr65OB/dZixEokYy8pq5tCNkrQTrrZWA1ylamm0UVMcyFMeyVNim6liW6pi2AtuyoaiWUBVbqKolFGFDUU3h0UyhBQzh8eaVQDGnBvNptaEw721eSWnb5lNa+ItZ7+vJouLbyIeSH8B3AcwCGNoKVenTent7379x48a75eeRSGTg/v37fzp37tycz+f72LKs7z3v50+cOPHZ+fPn/3blypVrgByUOn78+GdXr17doSjKxOzs7Dtnz579g6qqDw4dOjTY1ta22tLS8htd19+/c+eOR9O0sSNHjnxjH7ipqanumzdvvgfAFkKYO3fu/O1G3qdwHAdCCP/Bgwdb9+/fvykGAsbHx4PxeHxheXm56i9lq0VMRwRyJkAngCTkwFVNCplpf6O5HPI6Ra/mGJrHMT0ex9Q8jqkqju2xhWJaUC1TeExLqJYlVGtV8RXTakMm7WnIPTWd6UWokO3BC5B76+q1dq+0u7u789ixYxXLjdIo/5/PnDlzsFKvYWJiwh+Px5cWFhZS5QrVsayqWRa+JsVisWoHW6pRvAcLMR2XAXRDLsFtg6xWa27ZYMbTUMh4Giq5h4CAnAqlARiBHHSqyYUMtm07hmFgq6+WKisHqp3NboriFACQy+VEoVBgoK5TadnqeEzHPQA/BBCD3Ex7FlU6d3UT2gZ5n/QOgH/Geyrb4+hlM03TyGQyajgcrkhF1t/fn+jv769YdQoA2WxWzWazBlAKVMdxjF27dhVTqVTFfjHrkUgkBNa50xQ9Vrrs/CSmYxzAmwB+Afm3MA8G60YIAFEAAQCTAP5SpVvtuW5xcXF5bm4uEg6Ha7ICX4uZmRlRKBQKwBODUnNzc4tjY2Mte/fuXQkGg1U7hen27dt1iURi6VntB2h94j1YBXArpkOH3BLwZ5ADKIuo4XusLlLx+NJeB/CfrTbglE6ns+Pj45FoNOqJRqNb7n9S1/VgIpFYcRzHAJ5qgdLY2PhaR0dHtKWlBeFw2PJ6vVVxWW1Zlshms2oikVAmJyfT8/PzjxzHqYrXVktK7Va6IeewRgBkIcO1aj9gK8QHWZEKAB8B+DDeg1fSFbcaCSH8XV1d29vb2z2RSMT0er2Oqqo1+zdTzqPp6Wnx4MGD7MzMzFw5j/6vSZ8QwuP3+wOhUMjv8/nUZ57xFbNt215ZWSmm0+m84zir3/4T9CJiOhTIuas/hQxYE3Ii+lb+3SuQHzJ+yA+aWwA+qcXlohtRauzpb25uDmqaVh3N6F6SJ/MIQPHJziHP7HpKVBbTEYVsE9MH2eWgCBmuW+XyrgFymagDOdA0CuBhvKd6uiVQ9WCg0prEdKiQGx//qHR4ISvWFIBa2o1fQK6xbyg9n4GsRv9bq1OfyD0MVFq3mA4NcvOVH5QOP2QQrQBYRhX1ulojH+SyUA3yfUxCVqNfxnuqb+Nuql4MVHohpfutEcgVQd8HsBty9ogD2Zoli+q696oCqIfc0FmUjgyATwFMAJhmJUobxUAlV8V0eCCnErVCDmy1Ql5C25ADO0XIkM3j5d6H9UBWzuVDgQx5E7Jj7CTk5XwSQLpGdsinCmOg0ksX0+GHXEG0DXLrulbIqjYIGXLl7dYEZPCapcfy4Tz1fR7ISlPF42oYTzwqkIG9UDrmIad/JQGkSivGiFzHQKWKKQ101UEGa/mxAXLFka90eCFDVIEMUBNylVwe8pZCHrLqLZa+ngGQKW1hSPRKMVCJiFzy3N72RES0dgxUIiKXMFCJiFzCQCUicgkDlYjIJQxUIiKXMFCJiFzCQCUicgkDlYjIJQxUIiKXMFCJiFzCQCUicgkDlYjIJQxUIiKXMFCJiFzCQCUicgkDlYjIJQxUIiKXMFCJiFzCQCUicgkDlYjIJQxUIiKXMFCJiFzCQCUicgkDlYjIJQxUIiKXMFCJiFzyP/UzLWydWk1cAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7f47aa399a90>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# First way to call the 2 group Venn diagram:\n",
    "#The number of CpG sites captured by Nanopolish:\n",
    "#3783438\n",
    "#The number of CpG sites captured by Tombo on the forward strand:\n",
    "#5167951\n",
    "#The number of overlapping sites captured by Nanopolish and by Tombo on the forward strand of the reference genome:\n",
    "#3708640\n",
    "c = 3708640\n",
    "a = 5167951-c\n",
    "b = 3783438-c\n",
    "v = venn2(subsets = (a, b, c), set_labels = (' ', ' '))\n",
    "v.get_patch_by_id('01').set_color('#ff47a3')\n",
    "v.get_patch_by_id('10').set_color('#47d1ff')\n",
    "v.get_patch_by_id('11').set_color('#ffa347')\n",
    "plt.annotate('Tombo', xy=v.get_label_by_id('10').get_position() - np.array([0, 0.05]), xytext=(-70,-70), \\\n",
    "             ha='center', textcoords='offset points', bbox=dict(boxstyle='round,pad=0.5', fc='gray', alpha=0.1), \\\n",
    "             arrowprops=dict(arrowstyle='->', connectionstyle='arc3,rad=0.5',color='gray'))\n",
    "plt.annotate('Nanopolish', xy=v.get_label_by_id('01').get_position() - np.array([0, 0.05]), xytext=(40,-70), \\\n",
    "             ha='center', textcoords='offset points', bbox=dict(boxstyle='round,pad=0.5', fc='gray', alpha=0.1), \\\n",
    "             arrowprops=dict(arrowstyle='->', connectionstyle='arc3,rad=0.5',color='gray'))\n",
    "\n",
    "plt.title('Overlap between Tombo and Nanopolish sites')\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Nanopore and PacBio Venn diagrams don't look good, so use the tombo nanopolish one,\n",
    "# and a table of percentages will be used"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For the rest of the analysis, I will use the overlapped files for 5mC and 6mA data, and the Tombo files. This is because the Tombo files consider all possible A and C sites in the genome, and provide some consistency between the 6mA and 5mC data. In contrast, Nanopolish only detects 5mC at CpG sites on the forward strand, and SMRTLink does not provide a methylation-call accuracy value that is comparable to the accuracy in Nanopolis and Tombo. While Nanopolish and Tombo use the fraction of reads for the site that are methylated, SMRTLink uses a Phred score to determine methylation-call accuracy."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#8a14ff'> 3. Making cutoff files. <span/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#8a14ff'> 3.A Making cutoff files from five original methylation-calling files, and intersect the two sequencing methods and the two methyaltion-callers. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1320,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make the list of cutoffs\n",
    "cutoff_list = [1.00, 0.99, 0.95, 0.90, 0.80, 0.70, 0.60, 0.50, 0.40, 0.30, 0.20, 0.10, 0.00]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cutoff_list = sorted(cutoff_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 497,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "13"
      ]
     },
     "execution_count": 497,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(cutoff_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 196,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Define function to filter\n",
    "def score_filter(feature, L):\n",
    "    \"\"\"Returns True if feature is longer than L\"\"\"\n",
    "    return float(feature.score) >= L\n",
    "\n",
    "def filter_by_cutoffs(bed_files, cutoffs, initial_file_path, final_file_path):\n",
    "    \"\"\"Filters files by the list of cutoffs given, and renames the file according to the cutoff.\"\"\"\n",
    "    for file in bed_files:\n",
    "        pybed_object = BedTool(file)\n",
    "        for x in cutoffs:\n",
    "            filtered_file = pybed_object.filter(score_filter, x)\n",
    "            cutoff = \"{:.2f}\".format(x)\n",
    "            cutoff_name = '.cutoff.' + cutoff + '.bed'\n",
    "            out_filename = file.replace('.bed', cutoff_name)\n",
    "            out_filename = out_filename.replace(initial_file_path, final_file_path)\n",
    "            filtered_file.saveas(out_filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Run the function to filter all files\n",
    "initial_fp = '/home/anjuni/analysis/bedtools_output/sequencing_comparison/'\n",
    "final_fp = '/home/anjuni/analysis/bedtools_output/cutoffs_from_intersects/'\n",
    "filter_by_cutoffs(bed_file_list, cutoff_list, initial_fp, final_fp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 392,
   "metadata": {},
   "outputs": [],
   "source": [
    "# make file handles for the four input files for intersecting, and the hc_tombo file for later analysis\n",
    "sorted_bed_files = ['/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_nanopolish_sorted.bed', \\\n",
    "                    '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.CpG.plus.bed', \\\n",
    "                    '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/5mC_hc_tombo_sorted.bed', \\\n",
    "                    '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_prob_smrtlink_sorted.bed', \\\n",
    "                    '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/6mA_hc_tombo_sorted.bed']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Run the function to make cutoffs of all the sorted bed files\n",
    "initial_fp1 = '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/'\n",
    "final_fp1 = '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs/'\n",
    "filter_by_cutoffs(sorted_bed_files, cutoff_list, initial_fp1, final_fp1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#Move the 6mA files and 5mC files to separate folders, to make running intersections easier\n",
    "cd /home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/\n",
    "mkdir cutoffs_6mA\n",
    "mkdir cutoffs_5mC\n",
    "mv cutoffs/6mA* cutoffs_6mA\n",
    "mv cutoffs/5mC* cutoffs_5mC\n",
    "rmdir cutoffs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 437,
   "metadata": {},
   "outputs": [],
   "source": [
    "# make directories for 6mA and 5mC cutoff files\n",
    "DIRS['BED_CUTOFFS'] = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', 'cutoffs')\n",
    "DIRS['6MA_CUTOFFS'] = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', 'cutoffs_6mA')\n",
    "DIRS['5MC_CUTOFFS'] = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', 'cutoffs_5mC')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 400,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC\n"
     ]
    }
   ],
   "source": [
    "print(DIRS['BED_CUTOFFS'])\n",
    "print(DIRS['6MA_CUTOFFS'])\n",
    "print(DIRS['5MC_CUTOFFS'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 504,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.40.bed\n"
     ]
    }
   ],
   "source": [
    "# make a list of 6mA cutoff files from Nanopore and PacBio\n",
    "ont_6mA = [fn for fn in glob.iglob('%s/6mA_hc_tombo*.bed' % DIRS['6MA_CUTOFFS'], recursive=True)]\n",
    "pb_6mA = [fn for fn in glob.iglob('%s/6mA_prob_smrtlink*.bed' % DIRS['6MA_CUTOFFS'], recursive=True)]\n",
    "\n",
    "#test out these lists by printing\n",
    "print(*ont_6mA, sep='\\n')\n",
    "print(*pb_6mA, sep='\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 505,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.95.bed\n"
     ]
    }
   ],
   "source": [
    "# make a list of 5mC cutoff files from Nanopolish and Tombo\n",
    "np_5mC = [fn for fn in glob.iglob('%s/5mC_hc_nanopolish*.bed' % DIRS['5MC_CUTOFFS'], recursive=True)]\n",
    "tombo_CpG_5mC = [fn for fn in glob.iglob('%s/5mC_hc_tombo_sorted.CpG.plus*.bed' % DIRS['5MC_CUTOFFS'], recursive=True)]\n",
    "tombo_5mC = [fn for fn in glob.iglob('%s/5mC_hc_tombo_sorted.c*.bed' % DIRS['5MC_CUTOFFS'], recursive=True)]\n",
    "\n",
    "#test out these lists by printing\n",
    "print(*np_5mC, sep='\\n')\n",
    "print(*tombo_CpG_5mC, sep='\\n')\n",
    "print(*tombo_5mC, sep='\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 506,
   "metadata": {},
   "outputs": [],
   "source": [
    "# the lists are not sorted, so sort them before doing cutoffs\n",
    "ont_6mA = sorted(ont_6mA)\n",
    "pb_6mA = sorted(pb_6mA)\n",
    "np_5mC = sorted(np_5mC)\n",
    "tombo_CpG_5mC = sorted(tombo_CpG_5mC)\n",
    "tombo_5mC = sorted(tombo_5mC)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 507,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_prob_smrtlink_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_nanopolish_sorted.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.CpG.plus.cutoff.1.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.00.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.10.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.20.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.30.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.40.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.50.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.60.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.70.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.80.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.90.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.95.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.0.99.bed\n",
      "/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.1.00.bed\n"
     ]
    }
   ],
   "source": [
    "#Check if it worked. (It did!) :D\n",
    "print(*ont_6mA, sep='\\n')\n",
    "print(*pb_6mA, sep='\\n')\n",
    "print(*np_5mC, sep='\\n')\n",
    "print(*tombo_CpG_5mC, sep='\\n')\n",
    "print(*tombo_5mC, sep='\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 436,
   "metadata": {},
   "outputs": [],
   "source": [
    "# make the filepaths for intersects output\n",
    "DIRS['I_FROM_C'] = os.path.join(DIRS['BASE2'], 'bedtools_output', 'intersects_from_cutoffs')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 494,
   "metadata": {},
   "outputs": [],
   "source": [
    "# make a for loop to take a list of cutoffs, and a list of -a files and a list of -b files to intersect\n",
    "def intersect_cutoffs(list_a, list_b, n_cutoffs, mod, a_name, b_name):\n",
    "    \"\"\"Take a list of files and intersect them with another list of files, where files are matched by methylation cutoff. Other required inputs are the number of cutoffs in the list, the modification, and the names to be used in the outfile to signify the BED file from list a and b.\"\"\"\n",
    "    for i in range(n_cutoffs): # go through each element in the sorted list, pairing them up\n",
    "        a_bed = BedTool(list_a[i])\n",
    "        b_bed = BedTool(list_b[i])\n",
    "        cutoff_suffix = list_a[i][-16:] # get cutoff value from one of the files\n",
    "        out_name = mod + '_' + a_name + '_' + b_name + cutoff_suffix # make a new file name with the two intersected files, basemod type and cutoff\n",
    "        out_file = os.path.join(DIRS['I_FROM_C'], out_name) # make the file path\n",
    "        intersected_cutoff = a_bed.intersect(b_bed).saveas(out_file) # save out the intersected file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Run intersect cutoffs for 6mA and 5mC\n",
    "intersect_cutoffs(ont_6mA, pb_6mA, 13, '6mA', 'tombo', 'smrtlink')\n",
    "intersect_cutoffs(tombo_CpG_5mC, np_5mC, 13, '5mC', 'tombo', 'nanopolish')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "# Make one directory that contains cutoffs of all 4 bed files for use in downstream analysis\n",
    "cd /home/anjuni/analysis/coverage\n",
    "mkdir feature_files\n",
    "cp ../bedtools_output/intersects_from_cutoffs/* feature_files\n",
    "cp ../../methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC/5mC_hc_tombo_sorted.cutoff.* feature_files\n",
    "cp ../../methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA/6mA_hc_tombo_sorted.cutoff.* feature_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 508,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make lists of cutoffs from each of the BED files for use in analysis in the new folder\n",
    "# Remake the lists for the old ones under new names\n",
    "\n",
    "# set filepaths for feature inputs\n",
    "DIRS['FEATURES'] = os.path.join(DIRS['BASE2'], 'coverage', 'feature_files')\n",
    "\n",
    "intersected_5mC = [fn for fn in glob.iglob('%s/5mC_t*.bed' % DIRS['FEATURES'], recursive=True)]\n",
    "intersected_6mA = [fn for fn in glob.iglob('%s/6mA_t*.bed' % DIRS['FEATURES'], recursive=True)]\n",
    "hc_tombo_5mC = [fn for fn in glob.iglob('%s/5mC_h*.bed' % DIRS['FEATURES'], recursive=True)]\n",
    "hc_tombo_6mA = [fn for fn in glob.iglob('%s/6mA_h*.bed' % DIRS['FEATURES'], recursive=True)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 509,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.1.00.bed']\n",
      "['/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.1.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.99.bed']\n",
      "['/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.1.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.95.bed']\n",
      "['/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.1.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.50.bed']\n"
     ]
    }
   ],
   "source": [
    "# Print lists to check they worked\n",
    "pprint.pprint(intersected_5mC)\n",
    "pprint.pprint(intersected_6mA)\n",
    "pprint.pprint(hc_tombo_5mC)\n",
    "pprint.pprint(hc_tombo_6mA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 510,
   "metadata": {},
   "outputs": [],
   "source": [
    "# These lists are not sorted, so sort them before doing windows\n",
    "intersected_5mC.sort()\n",
    "intersected_6mA.sort()\n",
    "hc_tombo_5mC.sort()\n",
    "hc_tombo_6mA.sort()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1276,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.1.00.bed']\n",
      "['/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.1.00.bed']\n",
      "['/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.1.00.bed']\n",
      "['/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.1.00.bed']\n"
     ]
    }
   ],
   "source": [
    "# Print sorted lists to check they worked (they did! time for windows)\n",
    "pprint.pprint(intersected_5mC)\n",
    "pprint.pprint(intersected_6mA)\n",
    "pprint.pprint(hc_tombo_5mC)\n",
    "pprint.pprint(hc_tombo_6mA)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#8a14ff'> 3.B Making a table for the cutoff files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Gather the lists to be used as inputs\n",
    "pprint.pprint(np_5mC)\n",
    "pprint.pprint(tombo_CpG_5mC)\n",
    "pprint.pprint(pb_6mA)\n",
    "pprint.pprint(ont_6mA)\n",
    "pprint.pprint(intersected_5mC)\n",
    "pprint.pprint(intersected_6mA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1328,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Write the function to count sites\n",
    "def make_site_count_list(fn_list):\n",
    "    \"\"\"This is a function that returns a list of the number of lines in a file, when a list of files is given as input.\"\"\"\n",
    "    site_count_list = []\n",
    "    for fn in fn_list:\n",
    "        count = 0\n",
    "        with open(fn, 'r') as infile:\n",
    "            for line in infile:\n",
    "                count += 1\n",
    "        site_count_list.append(count)\n",
    "    return site_count_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1329,
   "metadata": {},
   "outputs": [],
   "source": [
    "np_5mC_count_list = make_site_count_list(np_5mC)\n",
    "tombo_CpG_5mC_count_list = make_site_count_list(tombo_CpG_5mC)\n",
    "pb_6mA_count_list = make_site_count_list(pb_6mA)\n",
    "ont_6mA_count_list = make_site_count_list(ont_6mA)\n",
    "intersected_5mC_count_list = make_site_count_list(intersected_5mC)\n",
    "intersected_6mA_count_list = make_site_count_list(intersected_6mA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1331,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[3783438, 2655129, 2408326, 2329892, 2287038, 2249995, 2195581, 2122684, 1995419, 1597784, 1068255, 609093, 608808]\n",
      "[5167951, 5123192, 4970628, 4660551, 4184707, 3511671, 2414941, 1380931, 681458, 249792, 140848, 120970, 120970]\n",
      "[88932, 88932, 88932, 88932, 88932, 88932, 88932, 88932, 88932, 88932, 88932, 88932, 68]\n",
      "[83451878, 80777279, 73301535, 61332775, 47837744, 34554267, 19402593, 8986431, 3668473, 1333195, 928352, 882625, 882625]\n",
      "[3708640, 2581835, 2313616, 2174002, 2025031, 1823390, 1439138, 953767, 502953, 151839, 53649, 30708, 30691]\n",
      "[84733, 77694, 66199, 53979, 42461, 31857, 19602, 10085, 4457, 1380, 622, 507, 0]\n"
     ]
    }
   ],
   "source": [
    "print(np_5mC_count_list)\n",
    "print(tombo_CpG_5mC_count_list)\n",
    "print(pb_6mA_count_list)\n",
    "print(ont_6mA_count_list)\n",
    "print(intersected_5mC_count_list)\n",
    "print(intersected_6mA_count_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1379,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a list of percentage overlap\n",
    "def make_percent_overlap_list(intersect_list, smaller_file_list):\n",
    "    \"\"\"Calculates percentage of intersection as a proportion of the smaller input file.\"\"\"\n",
    "    percent_list = []\n",
    "    for i in range(len(intersect_list)):\n",
    "        if intersect_list[i] == 0:\n",
    "            percent = float(0)\n",
    "            percent_list.append(percent)\n",
    "        else:\n",
    "            percent = intersect_list[i] / smaller_file_list[i]\n",
    "            percent = 100*percent\n",
    "            percent_list.append(percent)\n",
    "    return percent_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1380,
   "metadata": {},
   "outputs": [],
   "source": [
    "percent_overlap_5mC = make_percent_overlap_list(intersected_5mC_count_list, np_5mC_count_list)\n",
    "percent_overlap_6mA = make_percent_overlap_list(intersected_6mA_count_list, pb_6mA_count_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1367,
   "metadata": {},
   "outputs": [],
   "source": [
    "cutoff_5mC = {}\n",
    "cutoff_5mC['Tombo Sites'] = tombo_CpG_5mC_count_list\n",
    "cutoff_5mC['Nanopolish Sites'] = np_5mC_count_list\n",
    "cutoff_5mC['Overlapping Sites'] = intersected_5mC_count_list\n",
    "cutoff_5mC['Methylated Fraction Cutoff'] = cutoff_list\n",
    "cutoff_5mC['Percentage Overlap (%)'] = percent_overlap_5mC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1381,
   "metadata": {},
   "outputs": [],
   "source": [
    "cutoff_6mA = {}\n",
    "cutoff_6mA['PacBio Sites'] = pb_6mA_count_list\n",
    "cutoff_6mA['Nanopore Sites'] = ont_6mA_count_list\n",
    "cutoff_6mA['Overlapping Sites'] = intersected_6mA_count_list\n",
    "cutoff_6mA['Methylated Fraction Cutoff'] = cutoff_list\n",
    "cutoff_6mA['Percentage Overlap (%)'] = percent_overlap_6mA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1383,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Save the dataframes as csv files for 5mC and 6mA\n",
    "df = pd.DataFrame.from_dict(cutoff_5mC)\n",
    "df = df [['Methylated Fraction Cutoff', 'Tombo Sites', 'Nanopolish Sites', 'Overlapping Sites', 'Percentage Overlap (%)']]\n",
    "df.to_csv(os.path.join(DIRS['FIGURES'], '5mC_cutoff_table'), header=True, index=None, sep='\\t')\n",
    "\n",
    "df = pd.DataFrame.from_dict(cutoff_6mA)\n",
    "df = df [['Methylated Fraction Cutoff', 'Nanopore Sites', 'PacBio Sites', 'Overlapping Sites', 'Percentage Overlap (%)']]\n",
    "df.to_csv(os.path.join(DIRS['FIGURES'], '6mA_cutoff_table'), header=True, index=None, sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#8a14ff'> 3.C Making random distributions of the same size as all the cutoff files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 533,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Inputs needed:\n",
    "\n",
    "# file of all C in genome\n",
    "all_c = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', '5mC_tombo_sorted.bed')\n",
    "# file of all A in genome\n",
    "all_a = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', '6mA_tombo_sorted.bed')\n",
    "# file of all (+) strand CpG\n",
    "all_cpg = os.path.join(DIRS['BASE1'], 'input', 'sorted_bed_files', '5mC_hc_tombo_sorted.CpG.plus.bed')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 537,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function to count the number of sites in each cutoff file\n",
    "def count_sites(file_list):\n",
    "    \"\"\"Outputs a dictionary of each cutoff file and the number of sites it has.\"\"\"\n",
    "    count_dict = {}\n",
    "    for file in file_list:\n",
    "        with open(file) as in_file:\n",
    "            count = 0\n",
    "            for line in in_file:\n",
    "                count += 1\n",
    "        count_dict[file.split('/')[-1]] = count\n",
    "    return count_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 859,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function to make the list of output randomisation file for each input cutoff file\n",
    "def out_rand_files(file_list):\n",
    "    \"\"\"Outputs a dictionary of output randomisation file paths for a given list of input file paths.\"\"\"\n",
    "    rand_dict = {}\n",
    "    for file in file_list:\n",
    "        out_name = file.replace('.bed', '_rand.bed')\n",
    "        out_name = out_name.replace('feature_files', 'randomisation')\n",
    "        rand_dict[file.split('/')[-1]] = out_name\n",
    "    return rand_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 602,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make count dictionaries separated by modification type\n",
    "intersect_5mC_count_dict = count_sites(intersected_5mC)\n",
    "intersect_6mA_count_dict = count_sites(intersected_6mA)\n",
    "tombo_5mC_count_dict = count_sites(hc_tombo_5mC)\n",
    "tombo_6mA_count_dict = count_sites(hc_tombo_6mA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 611,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_tombo_nanopolish.cutoff.0.00.bed': 3708640,\n",
      " '5mC_tombo_nanopolish.cutoff.0.10.bed': 2581835,\n",
      " '5mC_tombo_nanopolish.cutoff.0.20.bed': 2313616,\n",
      " '5mC_tombo_nanopolish.cutoff.0.30.bed': 2174002,\n",
      " '5mC_tombo_nanopolish.cutoff.0.40.bed': 2025031,\n",
      " '5mC_tombo_nanopolish.cutoff.0.50.bed': 1823390,\n",
      " '5mC_tombo_nanopolish.cutoff.0.60.bed': 1439138,\n",
      " '5mC_tombo_nanopolish.cutoff.0.70.bed': 953767,\n",
      " '5mC_tombo_nanopolish.cutoff.0.80.bed': 502953,\n",
      " '5mC_tombo_nanopolish.cutoff.0.90.bed': 151839,\n",
      " '5mC_tombo_nanopolish.cutoff.0.95.bed': 53649,\n",
      " '5mC_tombo_nanopolish.cutoff.0.99.bed': 30708,\n",
      " '5mC_tombo_nanopolish.cutoff.1.00.bed': 30691}\n",
      "{'6mA_tombo_smrtlink.cutoff.0.00.bed': 84733,\n",
      " '6mA_tombo_smrtlink.cutoff.0.10.bed': 77694,\n",
      " '6mA_tombo_smrtlink.cutoff.0.20.bed': 66199,\n",
      " '6mA_tombo_smrtlink.cutoff.0.30.bed': 53979,\n",
      " '6mA_tombo_smrtlink.cutoff.0.40.bed': 42461,\n",
      " '6mA_tombo_smrtlink.cutoff.0.50.bed': 31857,\n",
      " '6mA_tombo_smrtlink.cutoff.0.60.bed': 19602,\n",
      " '6mA_tombo_smrtlink.cutoff.0.70.bed': 10085,\n",
      " '6mA_tombo_smrtlink.cutoff.0.80.bed': 4457,\n",
      " '6mA_tombo_smrtlink.cutoff.0.90.bed': 1380,\n",
      " '6mA_tombo_smrtlink.cutoff.0.95.bed': 622,\n",
      " '6mA_tombo_smrtlink.cutoff.0.99.bed': 507,\n",
      " '6mA_tombo_smrtlink.cutoff.1.00.bed': 0}\n",
      "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': 67308386,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10.bed': 65945899,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20.bed': 61792835,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30.bed': 54472350,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40.bed': 44908966,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50.bed': 33730885,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60.bed': 19107606,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70.bed': 8564581,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80.bed': 3463251,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90.bed': 1327297,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95.bed': 953483,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99.bed': 902981,\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00.bed': 902980}\n",
      "{'6mA_hc_tombo_sorted.cutoff.0.00.bed': 83451878,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10.bed': 80777279,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20.bed': 73301535,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30.bed': 61332775,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40.bed': 47837744,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50.bed': 34554267,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60.bed': 19402593,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70.bed': 8986431,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80.bed': 3668473,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90.bed': 1333195,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95.bed': 928352,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99.bed': 882625,\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00.bed': 882625}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(intersect_5mC_count_dict)\n",
    "pprint.pprint(intersect_6mA_count_dict)\n",
    "pprint.pprint(tombo_5mC_count_dict)\n",
    "pprint.pprint(tombo_6mA_count_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 860,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make dictionaries of out files for everything\n",
    "intersect_5mC_rand_dict = out_rand_files(intersected_5mC)\n",
    "intersect_6mA_rand_dict = out_rand_files(intersected_6mA)\n",
    "tombo_5mC_rand_dict = out_rand_files(hc_tombo_5mC)\n",
    "tombo_6mA_rand_dict = out_rand_files(hc_tombo_6mA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "methyl_rand_dict = {}\n",
    "methyl_rand_dict['5mC_hc_tombo_sorted.cutoff.0.00.bed'] = '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed'\n",
    "methyl_rand_dict['6mA_hc_tombo_sorted.cutoff.0.00.bed'] = '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed'\n",
    "methyl_rand_dict['5mC_hc_tombo_sorted.cutoff.0.90.bed'] = '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.90_rand.bed'\n",
    "methyl_rand_dict['6mA_hc_tombo_sorted.cutoff.0.90.bed'] = '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.90_rand.bed'\n",
    "methyl_rand_dict['5mC_hc_tombo_sorted.cutoff.0.99.bed'] = '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.99_rand.bed'\n",
    "methyl_rand_dict['6mA_hc_tombo_sorted.cutoff.0.99.bed'] = '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.99_rand.bed'\n",
    "methyl_rand_dict['5mC_hc_tombo_sorted.cutoff.0.50.bed'] = '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.50_rand.bed'\n",
    "methyl_rand_dict['6mA_hc_tombo_sorted.cutoff.0.50.bed'] = '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.50_rand.bed'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 861,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_tombo_nanopolish.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.00_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.10_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.20_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.30_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.40_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.50_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.60_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.70_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.80_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.90_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.95_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.99_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.1.00_rand.bed'}\n",
      "{'6mA_tombo_smrtlink.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.00_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.10_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.20_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.30_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.40_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.50_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.60_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.70_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.80_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.90_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.95_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.99_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.1.00_rand.bed'}\n",
      "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.10_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.20_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.30_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.40_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.50_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.60_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.70_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.80_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.90_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.95_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.99_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.1.00_rand.bed'}\n",
      "{'6mA_hc_tombo_sorted.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.10_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.20_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.30_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.40_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.50_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.60_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.70_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.80_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.90_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.95_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.99_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.1.00_rand.bed'}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(intersect_5mC_rand_dict)\n",
    "pprint.pprint(intersect_6mA_rand_dict)\n",
    "pprint.pprint(tombo_5mC_rand_dict)\n",
    "pprint.pprint(tombo_6mA_rand_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 599,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a function that takes number of subset sites, abspath of all_site file, and output name\n",
    "# Then generate a random array of length subset from all sites\n",
    "# Then loops over all_site file while outfile is open to write out lines whose number is part of the randamized array\n",
    "def subset_sites(all_site_fn, number_of_subset_sites, out_fn):\n",
    "    \"\"\"This is a function that creates a BED file containing a subset of the total sites in the genome, with the size of the subset equal to an input cutoff file.\"\"\"\n",
    "    df = pd.read_csv(all_site_fn, sep='\\t', header = None) #use up memory making a dataframe of the csv file, to avoid running through it for each base in each loop\n",
    "    if type(number_of_subset_sites) == int and type(out_fn) == str:\n",
    "            random_array = np.random.choice(df.shape[0], number_of_subset_sites)\n",
    "            random_array.sort()\n",
    "            df.iloc[random_array, :].to_csv(out_fn, header=None, index=None, sep='\\t') #iloc[rows, columns]\n",
    "    elif type(number_of_subset_sites) == dict and type(out_fn) == dict:\n",
    "        for skey, svalue in number_of_subset_sites.items():\n",
    "            for fkey, fvalue in out_fn.items():\n",
    "                if skey == fkey:\n",
    "                    random_array = np.random.choice(df.shape[0], svalue)\n",
    "                    random_array.sort()\n",
    "                    df.iloc[random_array, :].to_csv(fvalue, header=None, index=None, sep='\\t') "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 609,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make randomisation files for everything :)\n",
    "subset_sites(all_cpg, intersect_5mC_count_dict, intersect_5mC_rand_dict)\n",
    "subset_sites(all_a, intersect_6mA_count_dict, intersect_6mA_rand_dict)\n",
    "subset_sites(all_c, tombo_5mC_count_dict, tombo_5mC_rand_dict)\n",
    "subset_sites(all_a, tombo_6mA_count_dict, tombo_6mA_rand_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 616,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function that gives a dictionary of each input filehandle\n",
    "def file_name_dict(file_list):\n",
    "    \"\"\"Outputs a dictionary of input file paths for a given list of input file paths.\"\"\"\n",
    "    file_dict = {}\n",
    "    for file in file_list:\n",
    "        file_dict[file.split('/')[-1]] = file\n",
    "    return file_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 617,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make file name dict\n",
    "intersect_5mC_dict = file_name_dict(intersected_5mC)\n",
    "intersect_6mA_dict = file_name_dict(intersected_6mA)\n",
    "tombo_5mC_dict = file_name_dict(hc_tombo_5mC)\n",
    "tombo_6mA_dict = file_name_dict(hc_tombo_6mA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 618,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_tombo_nanopolish.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.00.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.10.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.20.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.30.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.40.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.50.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.60.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.70.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.80.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.90.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.95.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.99.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.1.00.bed'}\n",
      "{'6mA_tombo_smrtlink.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.00.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.10.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.20.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.30.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.40.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.50.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.60.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.70.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.80.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.90.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.95.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.99.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.1.00.bed'}\n",
      "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.1.00.bed'}\n",
      "{'6mA_hc_tombo_sorted.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.1.00.bed'}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(intersect_5mC_dict)\n",
    "pprint.pprint(intersect_6mA_dict)\n",
    "pprint.pprint(tombo_5mC_dict)\n",
    "pprint.pprint(tombo_6mA_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#144fff'> 4. Making windows. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 434,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make folder for windows. Each BED file will contain a series of windows\n",
    "#os.mkdir(DIRS['WINDOW_OUTPUT'])\n",
    "#os.mkdir()\n",
    "# Define inputs\n",
    "gene_fn = '/home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.anno.sorted.gff3'\n",
    "te_fn = '/home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.TE.sorted.gff3'\n",
    "reference_genome = os.path.join(DIRS['REF'], 'Pst_104E_v13_ph_ctg.fa')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smartmatch is experimental at /home/anjuni/myapps/gff3sort/gff3sort.pl line 68.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 1.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 2.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 3.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 4.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 5.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 6.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 7.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 8.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 9.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 10.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 11.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 12.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 13.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 14.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 15.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 16.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 17.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 18.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 19.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 20.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 21.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 22.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 23.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 24.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 25.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 26.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 27.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 28.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 29.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 30.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 31.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 32.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 33.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 34.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 35.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 36.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 37.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 38.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 39.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 40.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 41.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 42.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 43.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 44.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 45.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 46.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 47.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 48.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 49.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 50.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 51.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 52.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 53.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 54.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 55.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 56.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 57.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 58.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 59.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 60.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 61.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 62.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 63.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 64.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 65.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 66.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 67.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 68.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 69.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 70.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 71.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 72.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 73.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 74.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 75.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 76.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 77.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 78.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 79.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 80.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 81.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 82.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 83.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 84.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 85.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 86.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 87.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 88.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 89.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 90.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 91.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 92.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 93.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 94.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 95.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 96.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 97.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 98.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 99.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 100.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 101.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 102.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 103.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 104.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 105.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 106.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 107.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 108.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 109.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 110.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 111.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 112.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 113.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 114.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 115.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 116.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 117.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 118.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 119.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 120.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 121.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 122.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 123.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 124.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 125.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 126.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 127.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 128.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 129.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 130.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 131.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 132.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 133.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 134.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 135.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 136.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 137.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 138.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 139.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 140.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 141.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 142.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 143.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 144.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 145.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 146.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 147.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 148.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 149.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 150.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 151.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 152.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 153.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 154.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 155.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 156.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 157.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 158.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 159.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 160.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 161.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 162.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 163.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 164.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 165.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 166.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 167.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 168.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 169.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 170.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 171.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 172.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 173.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 174.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 175.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 176.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 177.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 178.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 179.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 180.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 181.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 182.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 183.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 184.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 185.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 186.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 187.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 188.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 189.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 190.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 191.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 192.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 193.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 194.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 195.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 196.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 197.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 198.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 199.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 200.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 201.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 202.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 203.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 204.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 205.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 206.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 207.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 208.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 209.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 210.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 211.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 212.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 213.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 214.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 215.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 216.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 217.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 218.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 219.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 220.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 221.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 222.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 223.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 224.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 225.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 226.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 227.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 228.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 229.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 230.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 231.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 232.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 233.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 234.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 235.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 236.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 237.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 238.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 239.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 240.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 241.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 242.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 243.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 244.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 245.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 246.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 247.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 248.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 249.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 250.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 251.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 252.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 253.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 254.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 255.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 256.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 257.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 258.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 259.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 260.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 261.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 262.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 263.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 264.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 265.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 266.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 267.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 268.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 269.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 270.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 271.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 272.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 273.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 274.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 275.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 276.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 277.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 278.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 279.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 280.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 281.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 282.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 283.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 284.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 285.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 286.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 287.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 288.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 289.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 290.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 291.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 292.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 293.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 294.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 295.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 296.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 297.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 298.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 299.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 300.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 301.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 302.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 303.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 304.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 305.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 306.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 307.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 308.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 309.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 310.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 311.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 312.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 313.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 314.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 315.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 316.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 317.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 318.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 319.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 320.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 321.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 322.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 323.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 324.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 325.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 326.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 327.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 328.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 329.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 330.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 331.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 332.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 333.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 334.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 335.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 336.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 337.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 338.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 339.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 340.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 341.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 342.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 343.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 344.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 345.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 346.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 347.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 348.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 349.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 350.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 351.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 352.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 353.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 354.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 355.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 356.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 357.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 358.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 359.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 360.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 361.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 362.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 363.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 364.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 365.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 366.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 367.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 368.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 369.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 370.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 371.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 372.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 373.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 374.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 375.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 376.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 377.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 378.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 379.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 380.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 381.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 382.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 383.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 384.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 385.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 386.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 387.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 388.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 389.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 390.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 391.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 392.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 393.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 394.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 395.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 396.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 397.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 398.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 399.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 400.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 401.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 402.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 403.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 404.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 405.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 406.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 407.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 408.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 409.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 410.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 411.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 412.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 413.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 414.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 415.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 416.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 417.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 418.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 419.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 420.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 421.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 422.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 423.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 424.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 425.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 426.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 427.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 428.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 429.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 430.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 431.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 432.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 433.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 434.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 435.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 436.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 437.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 438.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 439.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 440.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 441.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 442.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 443.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 444.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 445.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 446.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 447.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 448.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 449.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 450.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 451.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 452.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 453.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 454.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 455.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 456.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 457.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 458.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 459.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 460.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 461.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 462.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 463.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 464.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 465.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 466.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 467.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 468.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 469.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 470.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 471.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 472.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 473.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 474.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 475.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 476.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 477.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 478.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 479.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 480.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 481.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 482.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 483.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 484.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 485.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 486.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 487.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 488.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 489.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 490.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 491.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 492.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 493.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 494.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 495.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 496.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 497.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 498.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 499.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 500.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 501.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 502.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 503.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 504.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 505.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 506.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 507.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 508.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 509.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 510.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 511.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 512.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 513.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 514.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 515.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 516.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 517.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 518.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 519.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 520.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 521.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 522.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 523.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 524.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 525.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 526.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 527.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 528.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 529.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 530.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 531.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 532.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 533.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 534.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 535.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 536.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 537.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 538.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 539.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 540.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 541.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 542.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 543.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 544.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 545.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 546.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 547.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 548.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 549.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 550.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 551.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 552.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 553.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 554.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 555.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 556.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 557.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 558.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 559.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 560.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 561.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 562.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 563.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 564.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 565.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 566.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 567.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 568.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 569.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 570.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 571.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 572.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 573.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 574.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 575.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 576.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 577.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 578.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 579.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 580.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 581.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 582.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 583.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 584.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 585.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 586.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 587.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 588.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 589.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 590.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 591.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 592.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 593.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 594.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 595.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 596.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 597.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 598.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 599.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 600.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 601.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 602.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 603.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 604.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 605.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 606.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 607.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 608.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 609.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 610.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 611.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 612.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 613.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 614.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 615.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 616.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 617.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 618.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 619.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 620.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 621.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 622.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 623.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 624.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 625.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 626.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 627.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 628.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 629.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 630.\n",
      "Use of uninitialized value $pos in hash element at /home/anjuni/myapps/gff3sort/gff3sort.pl line 67, <$_[...]> line 631.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n",
      "Argument \"\" isn't numeric in sort at /home/anjuni/myapps/gff3sort/gff3sort.pl line 104.\n"
     ]
    }
   ],
   "source": [
    "# Make the genome size file for windows\n",
    "!samtools faidx /home/anjuni/Pst_104_v13_assembly/Pst_104E_v13_ph_ctg.fa\n",
    "!cut -f 1,2 /home/anjuni/Pst_104_v13_assembly/Pst_104E_v13_ph_ctg.fa.fai > /home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.genome_file\n",
    "# Note: this does put the p contig values before h contig ones, while annotation files put h contig before p contig\n",
    "# May be a problem in the future but probs not\n",
    "# Sorted it anyway below, as reference genome fasta had contigs in that order arbitrarily:\n",
    "!/home/anjuni/myapps/gff3sort/gff3sort.pl /home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.genome_file >  /home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.sorted.genome_file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 830,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Define all file paths for window BED files\n",
    "window_fn_dict = {}\n",
    "window_bed_dict = {}\n",
    "#window_fn_dict['100kb'] = os.path.join(DIRS['WINDOW_OUTPUT'], 'Pst_104E_v13_ph_ctg_w100kb.bed')\n",
    "#window_fn_dict['30kb'] = os.path.join(DIRS['WINDOW_OUTPUT'], 'Pst_104E_v13_ph_ctg_w30kb.bed')\n",
    "window_fn_dict['10kb'] = os.path.join(DIRS['WINDOW_OUTPUT'], 'Pst_104E_v13_ph_ctg_w10kb.bed')\n",
    "#window_fn_dict['100kb_s20kb'] = os.path.join(DIRS['WINDOW_OUTPUT'], 'Pst_104E_v13_ph_ctg_w100kb_s20kb.bed')\n",
    "#window_fn_dict['30kb_s6kb'] = os.path.join(DIRS['WINDOW_OUTPUT'], 'Pst_104E_v13_ph_ctg_w30kb_s6kb.bed')\n",
    "window_fn_dict['10kb_s2kb'] = os.path.join(DIRS['WINDOW_OUTPUT'], 'Pst_104E_v13_ph_ctg_w10kb_s2kb.bed')\n",
    "genome_size_f_fn = os.path.join(DIRS['WINDOW_INPUT'], 'Pst_104E_v13_ph_ctg.sorted.genome_file')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 831,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'10kb': '/home/anjuni/analysis/windows/Pst_104E_v13_ph_ctg_w10kb.bed',\n",
      " '10kb_s2kb': '/home/anjuni/analysis/windows/Pst_104E_v13_ph_ctg_w10kb_s2kb.bed'}\n"
     ]
    }
   ],
   "source": [
    "# Check whether the dictionary looks nice :) (it does!) :D\n",
    "pprint.pprint(window_fn_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 292,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make the actual windows! :D\n",
    "#!bedtools makewindows -g {genome_size_f_fn} -w 100000 > {window_fn_dict['100kb']}\n",
    "#!bedtools makewindows -g {genome_size_f_fn} -w 30000 > {window_fn_dict['30kb']}\n",
    "!bedtools makewindows -g {genome_size_f_fn} -w 10000 > {window_fn_dict['10kb']}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 293,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make sliding windows\n",
    "#!bedtools makewindows -g {genome_size_f_fn} -w 100000 -s 20000 > {window_fn_dict['100kb_s20kb']}\n",
    "#!bedtools makewindows -g {genome_size_f_fn} -w 30000 -s 6000 > {window_fn_dict['30kb_s6kb']}\n",
    "!bedtools makewindows -g {genome_size_f_fn} -w 10000 -s 2000 > {window_fn_dict['10kb_s2kb']}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 832,
   "metadata": {},
   "outputs": [],
   "source": [
    "#new make a bedtools window dataframe\n",
    "for key, value in window_fn_dict.items() :\n",
    "    window_bed_dict[key] = BedTool(value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 833,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'10kb': <BedTool(/home/anjuni/analysis/windows/Pst_104E_v13_ph_ctg_w10kb.bed)>,\n",
      " '10kb_s2kb': <BedTool(/home/anjuni/analysis/windows/Pst_104E_v13_ph_ctg_w10kb_s2kb.bed)>}\n"
     ]
    }
   ],
   "source": [
    "# Check whether the bed file dictionary looks nice :) (it does!) :D\n",
    "pprint.pprint(window_bed_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 842,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5MC_CUTOFFS': '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_5mC',\n",
      " '6MA_CUTOFFS': '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs_6mA',\n",
      " 'BASE1': '/home/anjuni/methylation_calling/pacbio',\n",
      " 'BASE2': '/home/anjuni/analysis',\n",
      " 'BED_CUTOFFS': '/home/anjuni/methylation_calling/pacbio/input/sorted_bed_files/cutoffs',\n",
      " 'BED_INPUT': '/home/anjuni/analysis/bedtools_output/sequencing_comparison',\n",
      " 'FEATURES': '/home/anjuni/analysis/coverage/feature_files',\n",
      " 'GFF_INPUT': '/home/anjuni/analysis/gff_output',\n",
      " 'I_FROM_C': '/home/anjuni/analysis/bedtools_output/intersects_from_cutoffs',\n",
      " 'METHYL': '/home/anjuni/analysis/input_for_windows/methylation_files',\n",
      " 'REF': '/home/anjuni/Pst_104_v13_assembly/',\n",
      " 'TEST_COV': '/home/anjuni/analysis/coverage/test_feature_files',\n",
      " 'WINDOW_INPUT': '/home/anjuni/analysis/input_for_windows',\n",
      " 'WINDOW_OUTPUT': '/home/anjuni/analysis/windows'}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(DIRS)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#148aff'> 5. Coverage analysis of methylation with gene annotation files. <span/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#148aff'> 5.A Make feature files of effector proteins, non-effectors, genes, exons and TE superfamilies. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "--2018-08-07 15:30:17--  https://raw.githubusercontent.com/BenjaminSchwessinger/Pst_104_E137_A-_genome/master/supplemental_files/Supplemental_file_9.txt\n",
      "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.80.133\n",
      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.80.133|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 49295 (48K) [text/plain]\n",
      "Saving to: 'Supplemental_file_9.txt’\n",
      "\n",
      "     0K .......... .......... .......... .......... ........  100% 2.80M=0.02s\n",
      "\n",
      "2018-08-07 15:30:22 (2.80 MB/s) - 'Supplemental_file_9.txt’ saved [49295/49295]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "# Downloading the effector file (need raw version with only the file)\n",
    "cd /home/anjuni/analysis/input_for_windows\n",
    "wget https://raw.githubusercontent.com/BenjaminSchwessinger/Pst_104_E137_A-_genome/master/supplemental_files/Supplemental_file_9.txt\n",
    "mv Supplemental_file_9.txt Candidate_effectors.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a GFF file of effector proteins\n",
    "\n",
    "# First extract all lines with genes (not exon or CDS) from the gene annotation file\n",
    "! grep 'gene' /home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.anno.sorted.gff3 > /home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.anno.sorted.genes_only.gff3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 277,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Then extract the effector lines from the gene file\n",
    "\n",
    "# First write grep function for python\n",
    "import fileinput\n",
    "import re\n",
    "import glob\n",
    "\n",
    "def grep(PAT, FILES):\n",
    "    \"\"\"Same function as 'grep' in bash.\"\"\"\n",
    "    fileinput.close() # close the file in case the iterable was previously open to prevent the \"input() already active\" error\n",
    "    for line in fileinput.input(glob.glob(FILES)):\n",
    "        if re.search(PAT, line):\n",
    "            #print(fileinput.filename(), fileinput.lineno(), line)\n",
    "            return line\n",
    "    fileinput.close()\n",
    "\n",
    "# Write function for filtering effectors from gene file using grep            \n",
    "def make_effector_gff(effector_list, gene_gff, out_gff):\n",
    "    \"\"\"Get effector protein features out of gene annotation files.\"\"\"\n",
    "    with open(out_gff, mode = 'w') as out_file:\n",
    "        for effector in effector_list:\n",
    "            print(grep(effector, gene_gff), file = out_file)\n",
    "        \n",
    "# And make a list of effectors\n",
    "def make_effector_list(input_file):\n",
    "    list_name = []\n",
    "    with open(input_file) as file:\n",
    "        for line in file:\n",
    "            line = line.strip()\n",
    "            list_name.append(line)\n",
    "    return list_name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 254,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make the effector list\n",
    "effectors = make_effector_list('/home/anjuni/analysis/input_for_windows/Candidate_effectors.txt')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['evm.TU.hcontig_000_003.1', 'evm.TU.hcontig_000_003.10', 'evm.TU.hcontig_000_003.120', 'evm.TU.hcontig_000_003.158', 'evm.TU.hcontig_000_003.2', 'evm.TU.hcontig_000_003.20', 'evm.TU.hcontig_000_003.26', 'evm.TU.hcontig_000_003.314', 'evm.TU.hcontig_000_003.340', 'evm.TU.hcontig_000_003.380', 'evm.TU.hcontig_000_003.402', 'evm.TU.hcontig_000_003.419', 'evm.TU.hcontig_000_003.421', 'evm.TU.hcontig_000_003.423', 'evm.TU.hcontig_000_003.444', 'evm.TU.hcontig_000_003.450', 'evm.TU.hcontig_000_003.90', 'evm.TU.hcontig_000_031.4', 'evm.TU.hcontig_000_050.114', 'evm.TU.hcontig_000_050.122', 'evm.TU.hcontig_000_050.141', 'evm.TU.hcontig_000_050.144', 'evm.TU.hcontig_000_050.149', 'evm.TU.hcontig_000_050.34', 'evm.TU.hcontig_000_050.85', 'evm.TU.hcontig_000_050.87', 'evm.TU.hcontig_000_050.9', 'evm.TU.hcontig_000_050.93', 'evm.TU.hcontig_000_054.13', 'evm.TU.hcontig_000_054.43', 'evm.TU.hcontig_000_054.68', 'evm.TU.hcontig_001_001.103', 'evm.TU.hcontig_001_001.128', 'evm.TU.hcontig_001_001.129', 'evm.TU.hcontig_001_001.144', 'evm.TU.hcontig_001_001.153', 'evm.TU.hcontig_001_001.34', 'evm.TU.hcontig_001_001.36', 'evm.TU.hcontig_001_001.39', 'evm.TU.hcontig_001_001.41', 'evm.TU.hcontig_001_001.87', 'evm.TU.hcontig_001_001.89', 'evm.TU.hcontig_001_002.39', 'evm.TU.hcontig_001_002.4', 'evm.TU.hcontig_001_002.47', 'evm.TU.hcontig_001_002.5', 'evm.TU.hcontig_001_019.1', 'evm.TU.hcontig_001_019.2', 'evm.TU.hcontig_001_021.110', 'evm.TU.hcontig_001_021.115', 'evm.TU.hcontig_001_021.129', 'evm.TU.hcontig_001_021.139', 'evm.TU.hcontig_001_021.15', 'evm.TU.hcontig_001_021.198', 'evm.TU.hcontig_001_021.28', 'evm.TU.hcontig_001_021.65', 'evm.TU.hcontig_001_021.9', 'evm.TU.hcontig_001_032.3', 'evm.TU.hcontig_001_032.5', 'evm.TU.hcontig_001_042.9', 'evm.TU.hcontig_001_054.18', 'evm.TU.hcontig_001_054.81', 'evm.TU.hcontig_001_054.82', 'evm.TU.hcontig_001_056.2', 'evm.TU.hcontig_001_056.3', 'evm.TU.hcontig_001_056.6', 'evm.TU.hcontig_001_056.8', 'evm.TU.hcontig_001_123.3', 'evm.TU.hcontig_001_180.3', 'evm.TU.hcontig_001_180.4', 'evm.TU.hcontig_001_180.5', 'evm.TU.hcontig_001_180.6', 'evm.TU.hcontig_001_180.7', 'evm.TU.hcontig_002_011.110', 'evm.TU.hcontig_002_011.123', 'evm.TU.hcontig_002_011.134', 'evm.TU.hcontig_002_011.146', 'evm.TU.hcontig_002_011.188', 'evm.TU.hcontig_002_011.201', 'evm.TU.hcontig_002_011.23', 'evm.TU.hcontig_002_011.234', 'evm.TU.hcontig_002_011.29', 'evm.TU.hcontig_002_011.33', 'evm.TU.hcontig_002_011.37', 'evm.TU.hcontig_002_011.66', 'evm.TU.hcontig_002_023.7', 'evm.TU.hcontig_002_028.1', 'evm.TU.hcontig_002_028.149', 'evm.TU.hcontig_002_028.150', 'evm.TU.hcontig_002_028.153', 'evm.TU.hcontig_002_028.159', 'evm.TU.hcontig_002_028.161', 'evm.TU.hcontig_002_028.162', 'evm.TU.hcontig_002_028.173', 'evm.TU.hcontig_002_028.177', 'evm.TU.hcontig_002_028.189', 'evm.TU.hcontig_002_028.206', 'evm.TU.hcontig_002_028.210', 'evm.TU.hcontig_002_028.230', 'evm.TU.hcontig_002_028.271', 'evm.TU.hcontig_002_028.276', 'evm.TU.hcontig_002_028.277', 'evm.TU.hcontig_002_028.279', 'evm.TU.hcontig_002_028.297', 'evm.TU.hcontig_002_028.300', 'evm.TU.hcontig_002_028.303', 'evm.TU.hcontig_002_028.306', 'evm.TU.hcontig_002_028.5', 'evm.TU.hcontig_002_028.54', 'evm.TU.hcontig_002_028.63', 'evm.TU.hcontig_002_028.75', 'evm.TU.hcontig_002_028.84', 'evm.TU.hcontig_003_002.113', 'evm.TU.hcontig_003_002.114', 'evm.TU.hcontig_003_002.125', 'evm.TU.hcontig_003_002.126', 'evm.TU.hcontig_003_002.127', 'evm.TU.hcontig_003_002.159', 'evm.TU.hcontig_003_002.209', 'evm.TU.hcontig_003_002.213', 'evm.TU.hcontig_003_002.215', 'evm.TU.hcontig_003_002.38', 'evm.TU.hcontig_003_002.54', 'evm.TU.hcontig_003_002.59', 'evm.TU.hcontig_003_002.61', 'evm.TU.hcontig_003_002.62', 'evm.TU.hcontig_003_002.91', 'evm.TU.hcontig_003_002.92', 'evm.TU.hcontig_003_002.93', 'evm.TU.hcontig_003_013.20', 'evm.TU.hcontig_003_013.21', 'evm.TU.hcontig_003_013.22', 'evm.TU.hcontig_003_013.23', 'evm.TU.hcontig_003_013.40', 'evm.TU.hcontig_003_013.41', 'evm.TU.hcontig_003_013.44', 'evm.TU.hcontig_003_013.5', 'evm.TU.hcontig_003_013.6', 'evm.TU.hcontig_003_014.20', 'evm.TU.hcontig_003_014.3', 'evm.TU.hcontig_003_014.46', 'evm.TU.hcontig_003_014.54', 'evm.TU.hcontig_003_014.55', 'evm.TU.hcontig_003_014.9', 'evm.TU.hcontig_003_026.4', 'evm.TU.hcontig_003_026.5', 'evm.TU.hcontig_003_028.6', 'evm.TU.hcontig_003_041.76', 'evm.TU.hcontig_003_045.11', 'evm.TU.hcontig_003_045.14', 'evm.TU.hcontig_003_045.19', 'evm.TU.hcontig_003_045.2', 'evm.TU.hcontig_003_045.20', 'evm.TU.hcontig_003_045.25', 'evm.TU.hcontig_003_045.26', 'evm.TU.hcontig_003_045.7', 'evm.TU.hcontig_003_136.1', 'evm.TU.hcontig_003_136.15', 'evm.TU.hcontig_004_019.5', 'evm.TU.hcontig_004_020.118', 'evm.TU.hcontig_004_020.12', 'evm.TU.hcontig_004_020.136', 'evm.TU.hcontig_004_020.14', 'evm.TU.hcontig_004_020.15', 'evm.TU.hcontig_004_020.166', 'evm.TU.hcontig_004_020.169', 'evm.TU.hcontig_004_020.18', 'evm.TU.hcontig_004_020.19', 'evm.TU.hcontig_004_020.190', 'evm.TU.hcontig_004_020.215', 'evm.TU.hcontig_004_020.219', 'evm.TU.hcontig_004_020.26', 'evm.TU.hcontig_004_020.27', 'evm.TU.hcontig_004_020.299', 'evm.TU.hcontig_004_020.324', 'evm.TU.hcontig_004_020.36', 'evm.TU.hcontig_004_020.391', 'evm.TU.hcontig_004_020.395', 'evm.TU.hcontig_004_020.397', 'evm.TU.hcontig_004_020.40', 'evm.TU.hcontig_004_020.55', 'evm.TU.hcontig_004_020.68', 'evm.TU.hcontig_004_021.2', 'evm.TU.hcontig_004_027.1', 'evm.TU.hcontig_004_027.2', 'evm.TU.hcontig_004_076.53', 'evm.TU.hcontig_005_002.19', 'evm.TU.hcontig_005_002.40', 'evm.TU.hcontig_005_002.82', 'evm.TU.hcontig_005_002.83', 'evm.TU.hcontig_005_002.86', 'evm.TU.hcontig_005_002.89', 'evm.TU.hcontig_005_021.1', 'evm.TU.hcontig_005_021.2', 'evm.TU.hcontig_005_021.37', 'evm.TU.hcontig_005_021.58', 'evm.TU.hcontig_005_021.65', 'evm.TU.hcontig_005_039.14', 'evm.TU.hcontig_005_043.152', 'evm.TU.hcontig_005_043.177', 'evm.TU.hcontig_005_043.178', 'evm.TU.hcontig_005_043.187', 'evm.TU.hcontig_005_043.188', 'evm.TU.hcontig_005_043.205', 'evm.TU.hcontig_005_043.218', 'evm.TU.hcontig_005_043.227', 'evm.TU.hcontig_005_043.41', 'evm.TU.hcontig_005_043.44', 'evm.TU.hcontig_005_043.46', 'evm.TU.hcontig_005_043.70', 'evm.TU.hcontig_005_043.71', 'evm.TU.hcontig_005_043.78', 'evm.TU.hcontig_005_043.84', 'evm.TU.hcontig_005_043.91', 'evm.TU.hcontig_006_002.39', 'evm.TU.hcontig_006_002.5', 'evm.TU.hcontig_006_013.11', 'evm.TU.hcontig_006_013.137', 'evm.TU.hcontig_006_013.14', 'evm.TU.hcontig_006_013.173', 'evm.TU.hcontig_006_013.222', 'evm.TU.hcontig_006_013.3', 'evm.TU.hcontig_006_013.47', 'evm.TU.hcontig_006_013.78', 'evm.TU.hcontig_006_013.90', 'evm.TU.hcontig_006_015.31', 'evm.TU.hcontig_006_021.3', 'evm.TU.hcontig_006_033.17', 'evm.TU.hcontig_006_033.2', 'evm.TU.hcontig_006_033.3', 'evm.TU.hcontig_006_033.37', 'evm.TU.hcontig_006_131.11', 'evm.TU.hcontig_006_131.13', 'evm.TU.hcontig_007_001.4', 'evm.TU.hcontig_007_005.22', 'evm.TU.hcontig_007_005.5', 'evm.TU.hcontig_007_005.64', 'evm.TU.hcontig_007_005.66', 'evm.TU.hcontig_007_005.7', 'evm.TU.hcontig_007_005.75', 'evm.TU.hcontig_007_005.78', 'evm.TU.hcontig_007_006.111', 'evm.TU.hcontig_007_006.112', 'evm.TU.hcontig_007_006.164', 'evm.TU.hcontig_007_006.196', 'evm.TU.hcontig_007_006.202', 'evm.TU.hcontig_007_006.206', 'evm.TU.hcontig_007_006.207', 'evm.TU.hcontig_007_006.210', 'evm.TU.hcontig_007_006.217', 'evm.TU.hcontig_007_006.220', 'evm.TU.hcontig_007_006.221', 'evm.TU.hcontig_007_006.223', 'evm.TU.hcontig_007_006.23', 'evm.TU.hcontig_007_007.17', 'evm.TU.hcontig_007_007.18', 'evm.TU.hcontig_007_007.43', 'evm.TU.hcontig_007_007.8', 'evm.TU.hcontig_007_031.2', 'evm.TU.hcontig_008_009.10', 'evm.TU.hcontig_008_009.111', 'evm.TU.hcontig_008_009.38', 'evm.TU.hcontig_008_009.46', 'evm.TU.hcontig_008_009.88', 'evm.TU.hcontig_008_014.43', 'evm.TU.hcontig_008_014.64', 'evm.TU.hcontig_008_014.80', 'evm.TU.hcontig_008_035.11', 'evm.TU.hcontig_008_093.37', 'evm.TU.hcontig_009_015.1', 'evm.TU.hcontig_009_024.1', 'evm.TU.hcontig_009_024.112', 'evm.TU.hcontig_009_024.117', 'evm.TU.hcontig_009_024.18', 'evm.TU.hcontig_009_024.55', 'evm.TU.hcontig_009_024.74', 'evm.TU.hcontig_009_024.78', 'evm.TU.hcontig_009_024.80', 'evm.TU.hcontig_009_029.41', 'evm.TU.hcontig_009_029.51', 'evm.TU.hcontig_009_029.53', 'evm.TU.hcontig_009_029.55', 'evm.TU.hcontig_009_029.56', 'evm.TU.hcontig_009_029.61', 'evm.TU.hcontig_009_142.10', 'evm.TU.hcontig_010_016.105', 'evm.TU.hcontig_010_016.114', 'evm.TU.hcontig_010_016.141', 'evm.TU.hcontig_010_016.186', 'evm.TU.hcontig_010_016.187', 'evm.TU.hcontig_010_016.188', 'evm.TU.hcontig_010_016.192', 'evm.TU.hcontig_010_016.199', 'evm.TU.hcontig_010_016.202', 'evm.TU.hcontig_010_016.205', 'evm.TU.hcontig_010_016.207', 'evm.TU.hcontig_010_016.220', 'evm.TU.hcontig_010_016.224', 'evm.TU.hcontig_010_016.225', 'evm.TU.hcontig_010_016.226', 'evm.TU.hcontig_010_016.236', 'evm.TU.hcontig_010_016.237', 'evm.TU.hcontig_010_016.246', 'evm.TU.hcontig_010_016.247', 'evm.TU.hcontig_010_016.270', 'evm.TU.hcontig_010_016.272', 'evm.TU.hcontig_010_016.277', 'evm.TU.hcontig_010_016.296', 'evm.TU.hcontig_010_016.297', 'evm.TU.hcontig_010_016.38', 'evm.TU.hcontig_010_016.41', 'evm.TU.hcontig_010_016.47', 'evm.TU.hcontig_010_016.48', 'evm.TU.hcontig_010_016.62', 'evm.TU.hcontig_010_016.92', 'evm.TU.hcontig_010_016.98', 'evm.TU.hcontig_010_016.99', 'evm.TU.hcontig_011_005.6', 'evm.TU.hcontig_011_013.3', 'evm.TU.hcontig_011_016.116', 'evm.TU.hcontig_011_016.117', 'evm.TU.hcontig_011_016.119', 'evm.TU.hcontig_011_016.143', 'evm.TU.hcontig_011_016.148', 'evm.TU.hcontig_011_016.149', 'evm.TU.hcontig_011_016.169', 'evm.TU.hcontig_011_016.205', 'evm.TU.hcontig_011_016.223', 'evm.TU.hcontig_011_016.240', 'evm.TU.hcontig_011_016.26', 'evm.TU.hcontig_011_016.27', 'evm.TU.hcontig_011_016.28', 'evm.TU.hcontig_011_016.30', 'evm.TU.hcontig_011_016.31', 'evm.TU.hcontig_011_016.32', 'evm.TU.hcontig_011_016.37', 'evm.TU.hcontig_011_016.91', 'evm.TU.hcontig_011_016.92', 'evm.TU.hcontig_011_017.25', 'evm.TU.hcontig_011_017.37', 'evm.TU.hcontig_011_017.43', 'evm.TU.hcontig_012_010.100', 'evm.TU.hcontig_012_010.103', 'evm.TU.hcontig_012_010.105', 'evm.TU.hcontig_012_010.20', 'evm.TU.hcontig_012_010.25', 'evm.TU.hcontig_012_010.28', 'evm.TU.hcontig_012_010.30', 'evm.TU.hcontig_012_010.60', 'evm.TU.hcontig_012_010.72', 'evm.TU.hcontig_012_024.75', 'evm.TU.hcontig_012_028.106', 'evm.TU.hcontig_012_028.128', 'evm.TU.hcontig_012_028.144', 'evm.TU.hcontig_012_028.25', 'evm.TU.hcontig_012_028.26', 'evm.TU.hcontig_012_028.28', 'evm.TU.hcontig_012_028.93', 'evm.TU.hcontig_012_028.98', 'evm.TU.hcontig_012_116.5', 'evm.TU.hcontig_012_116.7', 'evm.TU.hcontig_013_010.110', 'evm.TU.hcontig_013_010.111', 'evm.TU.hcontig_013_010.17', 'evm.TU.hcontig_013_010.19', 'evm.TU.hcontig_013_010.39', 'evm.TU.hcontig_013_010.40', 'evm.TU.hcontig_013_010.53', 'evm.TU.hcontig_013_010.57', 'evm.TU.hcontig_013_010.59', 'evm.TU.hcontig_013_010.89', 'evm.TU.hcontig_013_010.90', 'evm.TU.hcontig_013_012.64', 'evm.TU.hcontig_013_014.1', 'evm.TU.hcontig_013_018.3', 'evm.TU.hcontig_013_023.20', 'evm.TU.hcontig_013_023.23', 'evm.TU.hcontig_013_023.33', 'evm.TU.hcontig_013_023.71', 'evm.TU.hcontig_013_029.5', 'evm.TU.hcontig_013_029.6', 'evm.TU.hcontig_014_008.11', 'evm.TU.hcontig_014_017.7', 'evm.TU.hcontig_014_022.17', 'evm.TU.hcontig_014_022.19', 'evm.TU.hcontig_014_027.24', 'evm.TU.hcontig_014_117.15', 'evm.TU.hcontig_014_117.19', 'evm.TU.hcontig_014_117.25', 'evm.TU.hcontig_014_117.4', 'evm.TU.hcontig_014_192.10', 'evm.TU.hcontig_014_192.3', 'evm.TU.hcontig_016_014.15', 'evm.TU.hcontig_016_024.126', 'evm.TU.hcontig_016_024.133', 'evm.TU.hcontig_016_024.163', 'evm.TU.hcontig_016_024.22', 'evm.TU.hcontig_016_024.57', 'evm.TU.hcontig_016_024.79', 'evm.TU.hcontig_016_124.11', 'evm.TU.hcontig_017_004.103', 'evm.TU.hcontig_017_004.114', 'evm.TU.hcontig_017_004.118', 'evm.TU.hcontig_017_004.120', 'evm.TU.hcontig_017_004.140', 'evm.TU.hcontig_017_004.141', 'evm.TU.hcontig_017_004.2', 'evm.TU.hcontig_017_004.50', 'evm.TU.hcontig_017_004.51', 'evm.TU.hcontig_017_004.6', 'evm.TU.hcontig_017_004.70', 'evm.TU.hcontig_017_004.71', 'evm.TU.hcontig_017_004.73', 'evm.TU.hcontig_017_011.5', 'evm.TU.hcontig_017_011.6', 'evm.TU.hcontig_017_025.13', 'evm.TU.hcontig_017_025.2', 'evm.TU.hcontig_017_026.2', 'evm.TU.hcontig_017_160.7', 'evm.TU.hcontig_018_007.6', 'evm.TU.hcontig_018_008.1', 'evm.TU.hcontig_018_008.4', 'evm.TU.hcontig_018_010.2', 'evm.TU.hcontig_018_015.5', 'evm.TU.hcontig_018_016.123', 'evm.TU.hcontig_018_016.129', 'evm.TU.hcontig_018_016.132', 'evm.TU.hcontig_018_016.133', 'evm.TU.hcontig_018_016.48', 'evm.TU.hcontig_018_016.70', 'evm.TU.hcontig_018_016.84', 'evm.TU.hcontig_018_016.86', 'evm.TU.hcontig_018_022.106', 'evm.TU.hcontig_018_022.137', 'evm.TU.hcontig_018_022.3', 'evm.TU.hcontig_018_022.65', 'evm.TU.hcontig_018_022.68', 'evm.TU.hcontig_018_022.76', 'evm.TU.hcontig_018_022.79', 'evm.TU.hcontig_018_022.82', 'evm.TU.hcontig_018_022.83', 'evm.TU.hcontig_018_022.88', 'evm.TU.hcontig_019_001.21', 'evm.TU.hcontig_019_001.29', 'evm.TU.hcontig_019_001.33', 'evm.TU.hcontig_019_001.9', 'evm.TU.hcontig_019_013.16', 'evm.TU.hcontig_019_013.18', 'evm.TU.hcontig_019_013.24', 'evm.TU.hcontig_019_013.25', 'evm.TU.hcontig_019_013.36', 'evm.TU.hcontig_019_013.55', 'evm.TU.hcontig_019_015.122', 'evm.TU.hcontig_019_015.15', 'evm.TU.hcontig_019_015.17', 'evm.TU.hcontig_019_015.18', 'evm.TU.hcontig_019_015.25', 'evm.TU.hcontig_019_020.15', 'evm.TU.hcontig_019_020.25', 'evm.TU.hcontig_019_020.31', 'evm.TU.hcontig_019_085.2', 'evm.TU.hcontig_019_085.3', 'evm.TU.hcontig_019_085.36', 'evm.TU.hcontig_019_085.53', 'evm.TU.hcontig_019_085.57', 'evm.TU.hcontig_020_007.1', 'evm.TU.hcontig_020_018.35', 'evm.TU.hcontig_020_018.46', 'evm.TU.hcontig_020_018.5', 'evm.TU.hcontig_020_021.129', 'evm.TU.hcontig_020_021.14', 'evm.TU.hcontig_020_021.150', 'evm.TU.hcontig_020_021.151', 'evm.TU.hcontig_020_021.28', 'evm.TU.hcontig_020_021.45', 'evm.TU.hcontig_020_021.47', 'evm.TU.hcontig_020_021.59', 'evm.TU.hcontig_020_021.62', 'evm.TU.hcontig_020_021.7', 'evm.TU.hcontig_020_022.7', 'evm.TU.hcontig_020_025.32', 'evm.TU.hcontig_020_132.11', 'evm.TU.hcontig_020_132.17', 'evm.TU.hcontig_020_132.18', 'evm.TU.hcontig_021_014.14', 'evm.TU.hcontig_021_014.149', 'evm.TU.hcontig_021_014.151', 'evm.TU.hcontig_021_014.168', 'evm.TU.hcontig_021_014.169', 'evm.TU.hcontig_021_014.30', 'evm.TU.hcontig_021_014.38', 'evm.TU.hcontig_021_014.45', 'evm.TU.hcontig_021_014.63', 'evm.TU.hcontig_021_014.76', 'evm.TU.hcontig_021_014.84', 'evm.TU.hcontig_021_014.96', 'evm.TU.hcontig_021_026.2', 'evm.TU.hcontig_021_026.22', 'evm.TU.hcontig_021_026.23', 'evm.TU.hcontig_021_027.28', 'evm.TU.hcontig_021_027.3', 'evm.TU.hcontig_021_027.33', 'evm.TU.hcontig_021_027.62', 'evm.TU.hcontig_021_027.70', 'evm.TU.hcontig_021_027.72', 'evm.TU.hcontig_021_027.83', 'evm.TU.hcontig_022_007.17', 'evm.TU.hcontig_022_007.25', 'evm.TU.hcontig_022_007.29', 'evm.TU.hcontig_022_007.31', 'evm.TU.hcontig_022_007.39', 'evm.TU.hcontig_022_008.107', 'evm.TU.hcontig_022_008.14', 'evm.TU.hcontig_022_008.35', 'evm.TU.hcontig_022_008.39', 'evm.TU.hcontig_022_008.53', 'evm.TU.hcontig_022_008.79', 'evm.TU.hcontig_022_015.15', 'evm.TU.hcontig_022_015.7', 'evm.TU.hcontig_022_101.24', 'evm.TU.hcontig_023_002.107', 'evm.TU.hcontig_023_002.11', 'evm.TU.hcontig_023_002.142', 'evm.TU.hcontig_023_002.26', 'evm.TU.hcontig_023_002.27', 'evm.TU.hcontig_023_002.33', 'evm.TU.hcontig_023_002.43', 'evm.TU.hcontig_023_002.67', 'evm.TU.hcontig_023_003.3', 'evm.TU.hcontig_023_007.110', 'evm.TU.hcontig_023_007.31', 'evm.TU.hcontig_023_007.51', 'evm.TU.hcontig_023_007.77', 'evm.TU.hcontig_023_007.96', 'evm.TU.hcontig_025_002.15', 'evm.TU.hcontig_025_002.19', 'evm.TU.hcontig_025_003.23', 'evm.TU.hcontig_025_003.24', 'evm.TU.hcontig_025_003.38', 'evm.TU.hcontig_025_003.39', 'evm.TU.hcontig_025_003.4', 'evm.TU.hcontig_025_003.63', 'evm.TU.hcontig_026_002.113', 'evm.TU.hcontig_026_002.124', 'evm.TU.hcontig_026_002.137', 'evm.TU.hcontig_026_002.138', 'evm.TU.hcontig_026_002.159', 'evm.TU.hcontig_026_002.31', 'evm.TU.hcontig_026_002.32', 'evm.TU.hcontig_026_002.7', 'evm.TU.hcontig_026_002.8', 'evm.TU.hcontig_026_011.10', 'evm.TU.hcontig_026_011.11', 'evm.TU.hcontig_026_011.9', 'evm.TU.hcontig_026_018.47', 'evm.TU.hcontig_026_018.49', 'evm.TU.hcontig_026_018.50', 'evm.TU.hcontig_026_018.59', 'evm.TU.hcontig_026_018.62', 'evm.TU.hcontig_026_018.63', 'evm.TU.hcontig_027_006.107', 'evm.TU.hcontig_027_006.69', 'evm.TU.hcontig_027_011.32', 'evm.TU.hcontig_027_011.38', 'evm.TU.hcontig_027_011.39', 'evm.TU.hcontig_027_011.40', 'evm.TU.hcontig_027_011.42', 'evm.TU.hcontig_027_011.78', 'evm.TU.hcontig_027_011.79', 'evm.TU.hcontig_027_011.80', 'evm.TU.hcontig_027_011.86', 'evm.TU.hcontig_027_018.8', 'evm.TU.hcontig_027_018.9', 'evm.TU.hcontig_027_102.2', 'evm.TU.hcontig_027_102.24', 'evm.TU.hcontig_027_102.43', 'evm.TU.hcontig_027_102.45', 'evm.TU.hcontig_028_003.17', 'evm.TU.hcontig_028_003.19', 'evm.TU.hcontig_028_003.23', 'evm.TU.hcontig_028_011.104', 'evm.TU.hcontig_028_011.105', 'evm.TU.hcontig_028_011.113', 'evm.TU.hcontig_028_011.125', 'evm.TU.hcontig_028_011.6', 'evm.TU.hcontig_028_011.85', 'evm.TU.hcontig_028_015.14', 'evm.TU.hcontig_028_015.19', 'evm.TU.hcontig_028_015.2', 'evm.TU.hcontig_029_001.3', 'evm.TU.hcontig_029_010.1', 'evm.TU.hcontig_029_010.2', 'evm.TU.hcontig_029_010.62', 'evm.TU.hcontig_029_010.80', 'evm.TU.hcontig_029_010.84', 'evm.TU.hcontig_029_013.103', 'evm.TU.hcontig_029_013.104', 'evm.TU.hcontig_029_013.24', 'evm.TU.hcontig_029_013.30', 'evm.TU.hcontig_029_013.46', 'evm.TU.hcontig_029_013.74', 'evm.TU.hcontig_029_013.76', 'evm.TU.hcontig_029_013.81', 'evm.TU.hcontig_029_013.83', 'evm.TU.hcontig_029_013.90', 'evm.TU.hcontig_029_013.96', 'evm.TU.hcontig_029_014.3', 'evm.TU.hcontig_029_023.1', 'evm.TU.hcontig_029_023.2', 'evm.TU.hcontig_030_011.116', 'evm.TU.hcontig_030_011.134', 'evm.TU.hcontig_030_011.15', 'evm.TU.hcontig_030_011.151', 'evm.TU.hcontig_030_011.160', 'evm.TU.hcontig_030_011.161', 'evm.TU.hcontig_030_011.203', 'evm.TU.hcontig_030_011.25', 'evm.TU.hcontig_030_011.26', 'evm.TU.hcontig_030_011.38', 'evm.TU.hcontig_030_011.41', 'evm.TU.hcontig_030_011.42', 'evm.TU.hcontig_030_011.92', 'evm.TU.hcontig_030_155.14', 'evm.TU.hcontig_031_005.109', 'evm.TU.hcontig_031_005.112', 'evm.TU.hcontig_031_005.15', 'evm.TU.hcontig_031_005.22', 'evm.TU.hcontig_031_005.23', 'evm.TU.hcontig_031_005.29', 'evm.TU.hcontig_031_005.44', 'evm.TU.hcontig_031_005.47', 'evm.TU.hcontig_031_005.52', 'evm.TU.hcontig_031_005.53', 'evm.TU.hcontig_031_005.54', 'evm.TU.hcontig_031_005.87', 'evm.TU.hcontig_031_005.9', 'evm.TU.hcontig_031_005.93', 'evm.TU.hcontig_031_005.96', 'evm.TU.hcontig_031_005.97', 'evm.TU.hcontig_031_005.99', 'evm.TU.hcontig_031_007.24', 'evm.TU.hcontig_031_007.26', 'evm.TU.hcontig_031_007.35', 'evm.TU.hcontig_031_007.40', 'evm.TU.hcontig_031_007.42', 'evm.TU.hcontig_031_007.69', 'evm.TU.hcontig_031_007.91', 'evm.TU.hcontig_031_013.17', 'evm.TU.hcontig_031_015.2', 'evm.TU.hcontig_031_015.4', 'evm.TU.hcontig_033_010.1', 'evm.TU.hcontig_033_014.54', 'evm.TU.hcontig_033_071.16', 'evm.TU.hcontig_033_071.17', 'evm.TU.hcontig_033_071.21', 'evm.TU.hcontig_033_071.25', 'evm.TU.hcontig_033_071.59', 'evm.TU.hcontig_033_071.60', 'evm.TU.hcontig_033_071.76', 'evm.TU.hcontig_033_071.85', 'evm.TU.hcontig_034_001.109', 'evm.TU.hcontig_034_001.116', 'evm.TU.hcontig_034_001.127', 'evm.TU.hcontig_034_001.140', 'evm.TU.hcontig_034_001.142', 'evm.TU.hcontig_034_001.144', 'evm.TU.hcontig_034_001.157', 'evm.TU.hcontig_034_001.158', 'evm.TU.hcontig_034_001.160', 'evm.TU.hcontig_034_001.163', 'evm.TU.hcontig_034_001.21', 'evm.TU.hcontig_034_001.57', 'evm.TU.hcontig_034_001.60', 'evm.TU.hcontig_034_001.66', 'evm.TU.hcontig_034_001.76', 'evm.TU.hcontig_034_001.83', 'evm.TU.hcontig_034_005.1', 'evm.TU.hcontig_034_022.11', 'evm.TU.hcontig_034_022.9', 'evm.TU.hcontig_035_008.10', 'evm.TU.hcontig_035_008.11', 'evm.TU.hcontig_035_008.129', 'evm.TU.hcontig_035_008.130', 'evm.TU.hcontig_035_008.134', 'evm.TU.hcontig_035_008.137', 'evm.TU.hcontig_035_008.144', 'evm.TU.hcontig_035_008.157', 'evm.TU.hcontig_035_008.29', 'evm.TU.hcontig_035_008.31', 'evm.TU.hcontig_035_008.32', 'evm.TU.hcontig_035_008.33', 'evm.TU.hcontig_035_008.34', 'evm.TU.hcontig_035_008.4', 'evm.TU.hcontig_035_008.46', 'evm.TU.hcontig_035_008.47', 'evm.TU.hcontig_035_008.49', 'evm.TU.hcontig_035_008.61', 'evm.TU.hcontig_035_008.8', 'evm.TU.hcontig_035_013.2', 'evm.TU.hcontig_035_019.3', 'evm.TU.hcontig_035_019.4', 'evm.TU.hcontig_035_019.6', 'evm.TU.hcontig_036_002.11', 'evm.TU.hcontig_036_002.12', 'evm.TU.hcontig_036_006.13', 'evm.TU.hcontig_036_006.14', 'evm.TU.hcontig_036_006.16', 'evm.TU.hcontig_036_008.100', 'evm.TU.hcontig_036_008.15', 'evm.TU.hcontig_036_008.6', 'evm.TU.hcontig_036_008.8', 'evm.TU.hcontig_036_012.1', 'evm.TU.hcontig_036_012.14', 'evm.TU.hcontig_036_012.16', 'evm.TU.hcontig_036_012.2', 'evm.TU.hcontig_037_004.37', 'evm.TU.hcontig_037_004.55', 'evm.TU.hcontig_037_004.58', 'evm.TU.hcontig_037_004.61', 'evm.TU.hcontig_037_004.63', 'evm.TU.hcontig_037_004.70', 'evm.TU.hcontig_037_011.31', 'evm.TU.hcontig_037_011.32', 'evm.TU.hcontig_037_011.35', 'evm.TU.hcontig_037_011.73', 'evm.TU.hcontig_037_011.75', 'evm.TU.hcontig_037_011.76', 'evm.TU.hcontig_037_011.77', 'evm.TU.hcontig_037_011.78', 'evm.TU.hcontig_037_017.3', 'evm.TU.hcontig_037_161.3', 'evm.TU.hcontig_039_004.26', 'evm.TU.hcontig_039_004.36', 'evm.TU.hcontig_039_004.37', 'evm.TU.hcontig_039_004.63', 'evm.TU.hcontig_039_005.11', 'evm.TU.hcontig_039_005.22', 'evm.TU.hcontig_039_005.38', 'evm.TU.hcontig_039_005.39', 'evm.TU.hcontig_039_005.7', 'evm.TU.hcontig_039_005.83', 'evm.TU.hcontig_039_005.88', 'evm.TU.hcontig_039_005.89', 'evm.TU.hcontig_039_005.90', 'evm.TU.hcontig_039_005.94', 'evm.TU.hcontig_039_005.96', 'evm.TU.hcontig_040_001.2', 'evm.TU.hcontig_040_001.3', 'evm.TU.hcontig_040_001.6', 'evm.TU.hcontig_040_001.9', 'evm.TU.hcontig_040_002.12', 'evm.TU.hcontig_040_002.2', 'evm.TU.hcontig_040_015.13', 'evm.TU.hcontig_040_015.29', 'evm.TU.hcontig_040_015.3', 'evm.TU.hcontig_040_015.69', 'evm.TU.hcontig_040_015.79', 'evm.TU.hcontig_040_015.86', 'evm.TU.hcontig_040_015.91', 'evm.TU.hcontig_040_109.23', 'evm.TU.hcontig_040_109.31', 'evm.TU.hcontig_041_004.2', 'evm.TU.hcontig_041_004.4', 'evm.TU.hcontig_041_004.54', 'evm.TU.hcontig_041_004.55', 'evm.TU.hcontig_041_004.60', 'evm.TU.hcontig_041_004.78', 'evm.TU.hcontig_041_004.83', 'evm.TU.hcontig_041_005.12', 'evm.TU.hcontig_041_005.28', 'evm.TU.hcontig_041_005.29', 'evm.TU.hcontig_041_005.6', 'evm.TU.hcontig_041_005.7', 'evm.TU.hcontig_041_127.12', 'evm.TU.hcontig_041_127.20', 'evm.TU.hcontig_041_127.8', 'evm.TU.hcontig_042_005.2', 'evm.TU.hcontig_042_008.7', 'evm.TU.hcontig_042_009.58', 'evm.TU.hcontig_042_009.59', 'evm.TU.hcontig_042_011.5', 'evm.TU.hcontig_042_168.11', 'evm.TU.hcontig_043_003.15', 'evm.TU.hcontig_043_003.41', 'evm.TU.hcontig_043_003.52', 'evm.TU.hcontig_043_003.53', 'evm.TU.hcontig_043_006.6', 'evm.TU.hcontig_043_119.21', 'evm.TU.hcontig_043_119.4', 'evm.TU.hcontig_043_119.8', 'evm.TU.hcontig_044_003.13', 'evm.TU.hcontig_044_003.37', 'evm.TU.hcontig_044_003.39', 'evm.TU.hcontig_044_003.40', 'evm.TU.hcontig_044_003.43', 'evm.TU.hcontig_044_023.118', 'evm.TU.hcontig_044_023.121', 'evm.TU.hcontig_044_023.30', 'evm.TU.hcontig_044_023.62', 'evm.TU.hcontig_044_023.87', 'evm.TU.hcontig_044_023.88', 'evm.TU.hcontig_045_002.10', 'evm.TU.hcontig_045_002.12', 'evm.TU.hcontig_045_002.14', 'evm.TU.hcontig_045_002.3', 'evm.TU.hcontig_045_002.83', 'evm.TU.hcontig_045_002.88', 'evm.TU.hcontig_045_002.92', 'evm.TU.hcontig_045_002.96', 'evm.TU.hcontig_046_007.10', 'evm.TU.hcontig_046_007.119', 'evm.TU.hcontig_046_007.120', 'evm.TU.hcontig_046_007.42', 'evm.TU.hcontig_046_007.58', 'evm.TU.hcontig_046_007.63', 'evm.TU.hcontig_046_007.70', 'evm.TU.hcontig_046_007.80', 'evm.TU.hcontig_046_007.83', 'evm.TU.hcontig_046_007.88', 'evm.TU.hcontig_047_003.30', 'evm.TU.hcontig_047_003.37', 'evm.TU.hcontig_047_003.38', 'evm.TU.hcontig_047_006.25', 'evm.TU.hcontig_047_006.35', 'evm.TU.hcontig_047_006.51', 'evm.TU.hcontig_047_006.9', 'evm.TU.hcontig_047_007.2', 'evm.TU.hcontig_048_005.42', 'evm.TU.hcontig_048_005.62', 'evm.TU.hcontig_048_010.27', 'evm.TU.hcontig_048_010.29', 'evm.TU.hcontig_049_002.1', 'evm.TU.hcontig_049_002.37', 'evm.TU.hcontig_049_002.41', 'evm.TU.hcontig_049_002.58', 'evm.TU.hcontig_050_003.14', 'evm.TU.hcontig_050_003.20', 'evm.TU.hcontig_050_003.45', 'evm.TU.hcontig_050_010.1', 'evm.TU.hcontig_050_011.33', 'evm.TU.hcontig_050_011.39', 'evm.TU.hcontig_050_053.12', 'evm.TU.hcontig_050_053.5', 'evm.TU.hcontig_050_121.18', 'evm.TU.hcontig_050_121.9', 'evm.TU.hcontig_052_006.115', 'evm.TU.hcontig_052_006.117', 'evm.TU.hcontig_052_006.146', 'evm.TU.hcontig_052_006.155', 'evm.TU.hcontig_052_006.25', 'evm.TU.hcontig_052_006.53', 'evm.TU.hcontig_052_006.63', 'evm.TU.hcontig_052_105.25', 'evm.TU.hcontig_052_105.4', 'evm.TU.hcontig_054_001.108', 'evm.TU.hcontig_054_001.17', 'evm.TU.hcontig_054_001.25', 'evm.TU.hcontig_054_001.35', 'evm.TU.hcontig_054_001.50', 'evm.TU.hcontig_054_001.52', 'evm.TU.hcontig_054_001.91', 'evm.TU.hcontig_055_008.4', 'evm.TU.hcontig_055_009.1', 'evm.TU.hcontig_055_009.19', 'evm.TU.hcontig_055_009.22', 'evm.TU.hcontig_055_009.8', 'evm.TU.hcontig_055_009.9', 'evm.TU.hcontig_055_012.14', 'evm.TU.hcontig_055_012.91', 'evm.TU.hcontig_056_007.13', 'evm.TU.hcontig_056_007.14', 'evm.TU.hcontig_056_007.22', 'evm.TU.hcontig_056_007.24', 'evm.TU.hcontig_056_007.28', 'evm.TU.hcontig_056_007.39', 'evm.TU.hcontig_056_007.40', 'evm.TU.hcontig_056_007.58', 'evm.TU.hcontig_056_007.59', 'evm.TU.hcontig_057_001.1', 'evm.TU.hcontig_057_001.5', 'evm.TU.hcontig_057_002.5', 'evm.TU.hcontig_057_003.120', 'evm.TU.hcontig_057_003.121', 'evm.TU.hcontig_057_003.133', 'evm.TU.hcontig_057_003.36', 'evm.TU.hcontig_057_003.54', 'evm.TU.hcontig_057_003.64', 'evm.TU.hcontig_057_003.73', 'evm.TU.hcontig_057_003.74', 'evm.TU.hcontig_057_003.80', 'evm.TU.hcontig_057_003.83', 'evm.TU.hcontig_057_003.89', 'evm.TU.hcontig_057_009.4', 'evm.TU.hcontig_057_009.5', 'evm.TU.hcontig_058_004.17', 'evm.TU.hcontig_058_004.18', 'evm.TU.hcontig_058_004.19', 'evm.TU.hcontig_058_175.7', 'evm.TU.hcontig_059_005.1', 'evm.TU.hcontig_059_008.10', 'evm.TU.hcontig_059_008.101', 'evm.TU.hcontig_059_008.107', 'evm.TU.hcontig_059_008.111', 'evm.TU.hcontig_059_008.125', 'evm.TU.hcontig_059_008.19', 'evm.TU.hcontig_059_008.2', 'evm.TU.hcontig_059_008.37', 'evm.TU.hcontig_059_008.45', 'evm.TU.hcontig_059_008.54', 'evm.TU.hcontig_059_008.65', 'evm.TU.hcontig_059_008.7', 'evm.TU.hcontig_059_008.78', 'evm.TU.hcontig_060_001.2', 'evm.TU.hcontig_060_001.64', 'evm.TU.hcontig_060_001.65', 'evm.TU.hcontig_060_001.66', 'evm.TU.hcontig_060_001.67', 'evm.TU.hcontig_062_001.35', 'evm.TU.hcontig_062_001.46', 'evm.TU.hcontig_062_001.54', 'evm.TU.hcontig_062_001.62', 'evm.TU.hcontig_062_001.81', 'evm.TU.hcontig_064_007.1', 'evm.TU.hcontig_064_007.10', 'evm.TU.hcontig_064_007.5', 'evm.TU.hcontig_064_007.75', 'evm.TU.hcontig_064_007.9', 'evm.TU.hcontig_064_098.38', 'evm.TU.hcontig_065_003.39', 'evm.TU.hcontig_065_094.41', 'evm.TU.hcontig_066_004.8', 'evm.TU.hcontig_066_107.29', 'evm.TU.hcontig_067_004.24', 'evm.TU.hcontig_067_004.6', 'evm.TU.hcontig_067_005.8', 'evm.TU.hcontig_067_008.4', 'evm.TU.hcontig_067_008.6', 'evm.TU.hcontig_069_137.6', 'evm.TU.hcontig_070_006.2', 'evm.TU.hcontig_070_006.4', 'evm.TU.hcontig_070_009.13', 'evm.TU.hcontig_070_009.15', 'evm.TU.hcontig_077_002.9', 'evm.TU.hcontig_078_002.22', 'evm.TU.hcontig_078_002.8', 'evm.TU.hcontig_080_002.10', 'evm.TU.hcontig_080_002.13', 'evm.TU.hcontig_081_004.22', 'evm.TU.hcontig_081_005.11', 'evm.TU.hcontig_081_005.34', 'evm.TU.hcontig_081_005.36', 'evm.TU.hcontig_081_005.7', 'evm.TU.hcontig_082_001.6', 'evm.TU.hcontig_082_002.25', 'evm.TU.hcontig_083_002.40', 'evm.TU.hcontig_083_002.51', 'evm.TU.hcontig_083_002.52', 'evm.TU.hcontig_083_002.54', 'evm.TU.hcontig_084_003.7', 'evm.TU.hcontig_088_002.19', 'evm.TU.hcontig_088_002.29', 'evm.TU.hcontig_088_002.34', 'evm.TU.hcontig_089_005.2', 'evm.TU.hcontig_092_003.10', 'evm.TU.hcontig_095_004.15', 'evm.TU.hcontig_104_001.2', 'evm.TU.hcontig_147_001.2', 'evm.TU.hcontig_189_001.2', 'evm.TU.hcontig_225_001.1', 'evm.TU.pcontig_000.112', 'evm.TU.pcontig_000.19', 'evm.TU.pcontig_000.262', 'evm.TU.pcontig_000.276', 'evm.TU.pcontig_000.3', 'evm.TU.pcontig_000.306', 'evm.TU.pcontig_000.332', 'evm.TU.pcontig_000.378', 'evm.TU.pcontig_000.392', 'evm.TU.pcontig_000.406', 'evm.TU.pcontig_000.421', 'evm.TU.pcontig_000.423', 'evm.TU.pcontig_000.448', 'evm.TU.pcontig_000.450', 'evm.TU.pcontig_000.454', 'evm.TU.pcontig_000.479', 'evm.TU.pcontig_000.508', 'evm.TU.pcontig_000.554', 'evm.TU.pcontig_000.556', 'evm.TU.pcontig_000.558', 'evm.TU.pcontig_000.562', 'evm.TU.pcontig_000.583', 'evm.TU.pcontig_000.588', 'evm.TU.pcontig_000.593', 'evm.TU.pcontig_000.612', 'evm.TU.pcontig_000.624', 'evm.TU.pcontig_000.63', 'evm.TU.pcontig_000.679', 'evm.TU.pcontig_000.85', 'evm.TU.pcontig_000.9', 'evm.TU.pcontig_001.104', 'evm.TU.pcontig_001.169', 'evm.TU.pcontig_001.174', 'evm.TU.pcontig_001.188', 'evm.TU.pcontig_001.231', 'evm.TU.pcontig_001.252', 'evm.TU.pcontig_001.270', 'evm.TU.pcontig_001.274', 'evm.TU.pcontig_001.28', 'evm.TU.pcontig_001.288', 'evm.TU.pcontig_001.298', 'evm.TU.pcontig_001.3', 'evm.TU.pcontig_001.363', 'evm.TU.pcontig_001.365', 'evm.TU.pcontig_001.366', 'evm.TU.pcontig_001.367', 'evm.TU.pcontig_001.368', 'evm.TU.pcontig_001.371', 'evm.TU.pcontig_001.378', 'evm.TU.pcontig_001.379', 'evm.TU.pcontig_001.381', 'evm.TU.pcontig_001.4', 'evm.TU.pcontig_001.413', 'evm.TU.pcontig_001.463', 'evm.TU.pcontig_001.515', 'evm.TU.pcontig_001.518', 'evm.TU.pcontig_001.520', 'evm.TU.pcontig_001.568', 'evm.TU.pcontig_001.570', 'evm.TU.pcontig_001.583', 'evm.TU.pcontig_001.611', 'evm.TU.pcontig_001.613', 'evm.TU.pcontig_001.622', 'evm.TU.pcontig_001.632', 'evm.TU.pcontig_001.634', 'evm.TU.pcontig_001.635', 'evm.TU.pcontig_001.636', 'evm.TU.pcontig_001.638', 'evm.TU.pcontig_001.644', 'evm.TU.pcontig_001.657', 'evm.TU.pcontig_001.7', 'evm.TU.pcontig_001.83', 'evm.TU.pcontig_001.85', 'evm.TU.pcontig_002.108', 'evm.TU.pcontig_002.119', 'evm.TU.pcontig_002.131', 'evm.TU.pcontig_002.151', 'evm.TU.pcontig_002.202', 'evm.TU.pcontig_002.204', 'evm.TU.pcontig_002.223', 'evm.TU.pcontig_002.229', 'evm.TU.pcontig_002.243', 'evm.TU.pcontig_002.246', 'evm.TU.pcontig_002.28', 'evm.TU.pcontig_002.303', 'evm.TU.pcontig_002.315', 'evm.TU.pcontig_002.316', 'evm.TU.pcontig_002.332', 'evm.TU.pcontig_002.333', 'evm.TU.pcontig_002.34', 'evm.TU.pcontig_002.38', 'evm.TU.pcontig_002.40', 'evm.TU.pcontig_002.403', 'evm.TU.pcontig_002.404', 'evm.TU.pcontig_002.407', 'evm.TU.pcontig_002.413', 'evm.TU.pcontig_002.415', 'evm.TU.pcontig_002.42', 'evm.TU.pcontig_002.423', 'evm.TU.pcontig_002.426', 'evm.TU.pcontig_002.431', 'evm.TU.pcontig_002.440', 'evm.TU.pcontig_002.449', 'evm.TU.pcontig_002.453', 'evm.TU.pcontig_002.467', 'evm.TU.pcontig_002.529', 'evm.TU.pcontig_002.535', 'evm.TU.pcontig_002.536', 'evm.TU.pcontig_002.538', 'evm.TU.pcontig_002.553', 'evm.TU.pcontig_002.559', 'evm.TU.pcontig_002.562', 'evm.TU.pcontig_002.64', 'evm.TU.pcontig_002.92', 'evm.TU.pcontig_002.95', 'evm.TU.pcontig_003.140', 'evm.TU.pcontig_003.154', 'evm.TU.pcontig_003.162', 'evm.TU.pcontig_003.179', 'evm.TU.pcontig_003.208', 'evm.TU.pcontig_003.217', 'evm.TU.pcontig_003.218', 'evm.TU.pcontig_003.232', 'evm.TU.pcontig_003.236', 'evm.TU.pcontig_003.240', 'evm.TU.pcontig_003.243', 'evm.TU.pcontig_003.248', 'evm.TU.pcontig_003.249', 'evm.TU.pcontig_003.254', 'evm.TU.pcontig_003.255', 'evm.TU.pcontig_003.328', 'evm.TU.pcontig_003.333', 'evm.TU.pcontig_003.335', 'evm.TU.pcontig_003.336', 'evm.TU.pcontig_003.365', 'evm.TU.pcontig_003.366', 'evm.TU.pcontig_003.367', 'evm.TU.pcontig_003.387', 'evm.TU.pcontig_003.388', 'evm.TU.pcontig_003.402', 'evm.TU.pcontig_003.403', 'evm.TU.pcontig_003.427', 'evm.TU.pcontig_003.436', 'evm.TU.pcontig_003.471', 'evm.TU.pcontig_003.477', 'evm.TU.pcontig_003.479', 'evm.TU.pcontig_003.487', 'evm.TU.pcontig_003.505', 'evm.TU.pcontig_003.506', 'evm.TU.pcontig_003.507', 'evm.TU.pcontig_003.508', 'evm.TU.pcontig_003.521', 'evm.TU.pcontig_003.522', 'evm.TU.pcontig_003.523', 'evm.TU.pcontig_003.524', 'evm.TU.pcontig_003.525', 'evm.TU.pcontig_003.543', 'evm.TU.pcontig_003.544', 'evm.TU.pcontig_003.547', 'evm.TU.pcontig_003.87', 'evm.TU.pcontig_003.9', 'evm.TU.pcontig_004.124', 'evm.TU.pcontig_004.13', 'evm.TU.pcontig_004.142', 'evm.TU.pcontig_004.165', 'evm.TU.pcontig_004.168', 'evm.TU.pcontig_004.18', 'evm.TU.pcontig_004.19', 'evm.TU.pcontig_004.190', 'evm.TU.pcontig_004.191', 'evm.TU.pcontig_004.195', 'evm.TU.pcontig_004.214', 'evm.TU.pcontig_004.26', 'evm.TU.pcontig_004.27', 'evm.TU.pcontig_004.284', 'evm.TU.pcontig_004.304', 'evm.TU.pcontig_004.35', 'evm.TU.pcontig_004.372', 'evm.TU.pcontig_004.388', 'evm.TU.pcontig_004.39', 'evm.TU.pcontig_004.435', 'evm.TU.pcontig_004.437', 'evm.TU.pcontig_004.448', 'evm.TU.pcontig_004.53', 'evm.TU.pcontig_004.58', 'evm.TU.pcontig_004.73', 'evm.TU.pcontig_005.10', 'evm.TU.pcontig_005.126', 'evm.TU.pcontig_005.14', 'evm.TU.pcontig_005.186', 'evm.TU.pcontig_005.193', 'evm.TU.pcontig_005.242', 'evm.TU.pcontig_005.255', 'evm.TU.pcontig_005.258', 'evm.TU.pcontig_005.260', 'evm.TU.pcontig_005.289', 'evm.TU.pcontig_005.294', 'evm.TU.pcontig_005.301', 'evm.TU.pcontig_005.306', 'evm.TU.pcontig_005.33', 'evm.TU.pcontig_005.394', 'evm.TU.pcontig_005.395', 'evm.TU.pcontig_005.4', 'evm.TU.pcontig_005.418', 'evm.TU.pcontig_005.419', 'evm.TU.pcontig_005.439', 'evm.TU.pcontig_005.453', 'evm.TU.pcontig_005.461', 'evm.TU.pcontig_005.495', 'evm.TU.pcontig_005.51', 'evm.TU.pcontig_005.52', 'evm.TU.pcontig_005.6', 'evm.TU.pcontig_005.91', 'evm.TU.pcontig_005.95', 'evm.TU.pcontig_005.98', 'evm.TU.pcontig_006.108', 'evm.TU.pcontig_006.126', 'evm.TU.pcontig_006.13', 'evm.TU.pcontig_006.140', 'evm.TU.pcontig_006.142', 'evm.TU.pcontig_006.15', 'evm.TU.pcontig_006.199', 'evm.TU.pcontig_006.2', 'evm.TU.pcontig_006.278', 'evm.TU.pcontig_006.290', 'evm.TU.pcontig_006.293', 'evm.TU.pcontig_006.329', 'evm.TU.pcontig_006.35', 'evm.TU.pcontig_006.361', 'evm.TU.pcontig_006.391', 'evm.TU.pcontig_006.404', 'evm.TU.pcontig_006.433', 'evm.TU.pcontig_006.461', 'evm.TU.pcontig_006.472', 'evm.TU.pcontig_006.507', 'evm.TU.pcontig_006.90', 'evm.TU.pcontig_007.10', 'evm.TU.pcontig_007.113', 'evm.TU.pcontig_007.141', 'evm.TU.pcontig_007.151', 'evm.TU.pcontig_007.154', 'evm.TU.pcontig_007.175', 'evm.TU.pcontig_007.180', 'evm.TU.pcontig_007.223', 'evm.TU.pcontig_007.226', 'evm.TU.pcontig_007.23', 'evm.TU.pcontig_007.308', 'evm.TU.pcontig_007.309', 'evm.TU.pcontig_007.390', 'evm.TU.pcontig_007.405', 'evm.TU.pcontig_007.411', 'evm.TU.pcontig_007.412', 'evm.TU.pcontig_007.418', 'evm.TU.pcontig_007.421', 'evm.TU.pcontig_007.422', 'evm.TU.pcontig_007.424', 'evm.TU.pcontig_007.8', 'evm.TU.pcontig_007.81', 'evm.TU.pcontig_007.84', 'evm.TU.pcontig_008.133', 'evm.TU.pcontig_008.137', 'evm.TU.pcontig_008.159', 'evm.TU.pcontig_008.171', 'evm.TU.pcontig_008.209', 'evm.TU.pcontig_008.257', 'evm.TU.pcontig_008.30', 'evm.TU.pcontig_008.322', 'evm.TU.pcontig_008.343', 'evm.TU.pcontig_008.362', 'evm.TU.pcontig_008.86', 'evm.TU.pcontig_009.12', 'evm.TU.pcontig_009.126', 'evm.TU.pcontig_009.136', 'evm.TU.pcontig_009.138', 'evm.TU.pcontig_009.140', 'evm.TU.pcontig_009.141', 'evm.TU.pcontig_009.145', 'evm.TU.pcontig_009.158', 'evm.TU.pcontig_009.162', 'evm.TU.pcontig_009.163', 'evm.TU.pcontig_009.164', 'evm.TU.pcontig_009.181', 'evm.TU.pcontig_009.2', 'evm.TU.pcontig_009.21', 'evm.TU.pcontig_009.218', 'evm.TU.pcontig_009.222', 'evm.TU.pcontig_009.232', 'evm.TU.pcontig_009.236', 'evm.TU.pcontig_010.131', 'evm.TU.pcontig_010.132', 'evm.TU.pcontig_010.168', 'evm.TU.pcontig_010.21', 'evm.TU.pcontig_010.217', 'evm.TU.pcontig_010.219', 'evm.TU.pcontig_010.220', 'evm.TU.pcontig_010.231', 'evm.TU.pcontig_010.234', 'evm.TU.pcontig_010.237', 'evm.TU.pcontig_010.239', 'evm.TU.pcontig_010.240', 'evm.TU.pcontig_010.25', 'evm.TU.pcontig_010.250', 'evm.TU.pcontig_010.253', 'evm.TU.pcontig_010.254', 'evm.TU.pcontig_010.255', 'evm.TU.pcontig_010.258', 'evm.TU.pcontig_010.270', 'evm.TU.pcontig_010.271', 'evm.TU.pcontig_010.281', 'evm.TU.pcontig_010.282', 'evm.TU.pcontig_010.295', 'evm.TU.pcontig_010.304', 'evm.TU.pcontig_010.309', 'evm.TU.pcontig_010.331', 'evm.TU.pcontig_010.332', 'evm.TU.pcontig_010.345', 'evm.TU.pcontig_010.351', 'evm.TU.pcontig_010.352', 'evm.TU.pcontig_010.353', 'evm.TU.pcontig_010.376', 'evm.TU.pcontig_010.65', 'evm.TU.pcontig_010.73', 'evm.TU.pcontig_010.74', 'evm.TU.pcontig_010.89', 'evm.TU.pcontig_011.14', 'evm.TU.pcontig_011.142', 'evm.TU.pcontig_011.143', 'evm.TU.pcontig_011.147', 'evm.TU.pcontig_011.148', 'evm.TU.pcontig_011.153', 'evm.TU.pcontig_011.209', 'evm.TU.pcontig_011.210', 'evm.TU.pcontig_011.238', 'evm.TU.pcontig_011.241', 'evm.TU.pcontig_011.242', 'evm.TU.pcontig_011.263', 'evm.TU.pcontig_011.269', 'evm.TU.pcontig_011.292', 'evm.TU.pcontig_011.326', 'evm.TU.pcontig_011.347', 'evm.TU.pcontig_011.364', 'evm.TU.pcontig_011.71', 'evm.TU.pcontig_012.101', 'evm.TU.pcontig_012.122', 'evm.TU.pcontig_012.136', 'evm.TU.pcontig_012.138', 'evm.TU.pcontig_012.175', 'evm.TU.pcontig_012.176', 'evm.TU.pcontig_012.177', 'evm.TU.pcontig_012.233', 'evm.TU.pcontig_012.238', 'evm.TU.pcontig_012.246', 'evm.TU.pcontig_012.266', 'evm.TU.pcontig_012.280', 'evm.TU.pcontig_012.281', 'evm.TU.pcontig_012.293', 'evm.TU.pcontig_012.296', 'evm.TU.pcontig_012.366', 'evm.TU.pcontig_012.4', 'evm.TU.pcontig_012.52', 'evm.TU.pcontig_012.58', 'evm.TU.pcontig_012.61', 'evm.TU.pcontig_012.63', 'evm.TU.pcontig_012.85', 'evm.TU.pcontig_013.124', 'evm.TU.pcontig_013.126', 'evm.TU.pcontig_013.139', 'evm.TU.pcontig_013.140', 'evm.TU.pcontig_013.152', 'evm.TU.pcontig_013.156', 'evm.TU.pcontig_013.157', 'evm.TU.pcontig_013.162', 'evm.TU.pcontig_013.164', 'evm.TU.pcontig_013.166', 'evm.TU.pcontig_013.210', 'evm.TU.pcontig_013.215', 'evm.TU.pcontig_013.216', 'evm.TU.pcontig_013.238', 'evm.TU.pcontig_013.239', 'evm.TU.pcontig_013.247', 'evm.TU.pcontig_013.279', 'evm.TU.pcontig_013.282', 'evm.TU.pcontig_013.289', 'evm.TU.pcontig_013.290', 'evm.TU.pcontig_013.311', 'evm.TU.pcontig_013.318', 'evm.TU.pcontig_013.49', 'evm.TU.pcontig_014.101', 'evm.TU.pcontig_014.113', 'evm.TU.pcontig_014.115', 'evm.TU.pcontig_014.124', 'evm.TU.pcontig_014.128', 'evm.TU.pcontig_014.131', 'evm.TU.pcontig_014.138', 'evm.TU.pcontig_014.142', 'evm.TU.pcontig_014.193', 'evm.TU.pcontig_014.22', 'evm.TU.pcontig_014.279', 'evm.TU.pcontig_014.285', 'evm.TU.pcontig_014.295', 'evm.TU.pcontig_014.66', 'evm.TU.pcontig_014.85', 'evm.TU.pcontig_015.10', 'evm.TU.pcontig_016.119', 'evm.TU.pcontig_016.150', 'evm.TU.pcontig_016.157', 'evm.TU.pcontig_016.164', 'evm.TU.pcontig_016.165', 'evm.TU.pcontig_016.175', 'evm.TU.pcontig_016.189', 'evm.TU.pcontig_016.196', 'evm.TU.pcontig_016.198', 'evm.TU.pcontig_016.64', 'evm.TU.pcontig_016.91', 'evm.TU.pcontig_017.126', 'evm.TU.pcontig_017.150', 'evm.TU.pcontig_017.151', 'evm.TU.pcontig_017.152', 'evm.TU.pcontig_017.154', 'evm.TU.pcontig_017.184', 'evm.TU.pcontig_017.196', 'evm.TU.pcontig_017.2', 'evm.TU.pcontig_017.200', 'evm.TU.pcontig_017.202', 'evm.TU.pcontig_017.224', 'evm.TU.pcontig_017.225', 'evm.TU.pcontig_017.27', 'evm.TU.pcontig_017.3', 'evm.TU.pcontig_017.322', 'evm.TU.pcontig_017.7', 'evm.TU.pcontig_017.78', 'evm.TU.pcontig_017.82', 'evm.TU.pcontig_017.84', 'evm.TU.pcontig_017.85', 'evm.TU.pcontig_017.9', 'evm.TU.pcontig_018.118', 'evm.TU.pcontig_018.127', 'evm.TU.pcontig_018.154', 'evm.TU.pcontig_018.160', 'evm.TU.pcontig_018.163', 'evm.TU.pcontig_018.164', 'evm.TU.pcontig_018.187', 'evm.TU.pcontig_018.197', 'evm.TU.pcontig_018.218', 'evm.TU.pcontig_018.270', 'evm.TU.pcontig_018.273', 'evm.TU.pcontig_018.280', 'evm.TU.pcontig_018.281', 'evm.TU.pcontig_018.283', 'evm.TU.pcontig_018.286', 'evm.TU.pcontig_018.287', 'evm.TU.pcontig_018.292', 'evm.TU.pcontig_018.303', 'evm.TU.pcontig_018.338', 'evm.TU.pcontig_018.353', 'evm.TU.pcontig_018.5', 'evm.TU.pcontig_018.6', 'evm.TU.pcontig_018.82', 'evm.TU.pcontig_019.159', 'evm.TU.pcontig_019.167', 'evm.TU.pcontig_019.168', 'evm.TU.pcontig_019.179', 'evm.TU.pcontig_019.197', 'evm.TU.pcontig_019.202', 'evm.TU.pcontig_019.231', 'evm.TU.pcontig_019.233', 'evm.TU.pcontig_019.238', 'evm.TU.pcontig_019.245', 'evm.TU.pcontig_019.25', 'evm.TU.pcontig_019.295', 'evm.TU.pcontig_019.30', 'evm.TU.pcontig_019.31', 'evm.TU.pcontig_019.310', 'evm.TU.pcontig_019.47', 'evm.TU.pcontig_019.49', 'evm.TU.pcontig_019.50', 'evm.TU.pcontig_019.57', 'evm.TU.pcontig_020.163', 'evm.TU.pcontig_020.164', 'evm.TU.pcontig_020.182', 'evm.TU.pcontig_020.183', 'evm.TU.pcontig_020.216', 'evm.TU.pcontig_020.303', 'evm.TU.pcontig_020.329', 'evm.TU.pcontig_020.39', 'evm.TU.pcontig_020.47', 'evm.TU.pcontig_020.63', 'evm.TU.pcontig_020.67', 'evm.TU.pcontig_020.79', 'evm.TU.pcontig_020.91', 'evm.TU.pcontig_021.108', 'evm.TU.pcontig_021.117', 'evm.TU.pcontig_021.128', 'evm.TU.pcontig_021.154', 'evm.TU.pcontig_021.179', 'evm.TU.pcontig_021.181', 'evm.TU.pcontig_021.197', 'evm.TU.pcontig_021.198', 'evm.TU.pcontig_021.218', 'evm.TU.pcontig_021.222', 'evm.TU.pcontig_021.253', 'evm.TU.pcontig_021.259', 'evm.TU.pcontig_021.273', 'evm.TU.pcontig_021.282', 'evm.TU.pcontig_021.288', 'evm.TU.pcontig_021.297', 'evm.TU.pcontig_021.320', 'evm.TU.pcontig_021.323', 'evm.TU.pcontig_021.341', 'evm.TU.pcontig_021.47', 'evm.TU.pcontig_021.63', 'evm.TU.pcontig_021.71', 'evm.TU.pcontig_021.78', 'evm.TU.pcontig_021.93', 'evm.TU.pcontig_021.94', 'evm.TU.pcontig_022.106', 'evm.TU.pcontig_022.114', 'evm.TU.pcontig_022.120', 'evm.TU.pcontig_022.129', 'evm.TU.pcontig_022.143', 'evm.TU.pcontig_022.153', 'evm.TU.pcontig_022.160', 'evm.TU.pcontig_022.163', 'evm.TU.pcontig_022.189', 'evm.TU.pcontig_022.194', 'evm.TU.pcontig_022.198', 'evm.TU.pcontig_022.20', 'evm.TU.pcontig_022.218', 'evm.TU.pcontig_022.222', 'evm.TU.pcontig_022.227', 'evm.TU.pcontig_022.238', 'evm.TU.pcontig_022.245', 'evm.TU.pcontig_022.261', 'evm.TU.pcontig_022.281', 'evm.TU.pcontig_022.76', 'evm.TU.pcontig_022.81', 'evm.TU.pcontig_023.10', 'evm.TU.pcontig_023.105', 'evm.TU.pcontig_023.118', 'evm.TU.pcontig_023.133', 'evm.TU.pcontig_023.158', 'evm.TU.pcontig_023.175', 'evm.TU.pcontig_023.182', 'evm.TU.pcontig_023.192', 'evm.TU.pcontig_023.212', 'evm.TU.pcontig_023.223', 'evm.TU.pcontig_023.234', 'evm.TU.pcontig_023.265', 'evm.TU.pcontig_023.5', 'evm.TU.pcontig_023.53', 'evm.TU.pcontig_023.6', 'evm.TU.pcontig_023.73', 'evm.TU.pcontig_023.88', 'evm.TU.pcontig_023.99', 'evm.TU.pcontig_024.10', 'evm.TU.pcontig_025.10', 'evm.TU.pcontig_025.121', 'evm.TU.pcontig_025.161', 'evm.TU.pcontig_025.165', 'evm.TU.pcontig_025.33', 'evm.TU.pcontig_025.75', 'evm.TU.pcontig_025.92', 'evm.TU.pcontig_026.127', 'evm.TU.pcontig_026.128', 'evm.TU.pcontig_026.142', 'evm.TU.pcontig_026.143', 'evm.TU.pcontig_026.144', 'evm.TU.pcontig_026.251', 'evm.TU.pcontig_026.262', 'evm.TU.pcontig_026.273', 'evm.TU.pcontig_026.274', 'evm.TU.pcontig_026.56', 'evm.TU.pcontig_026.58', 'evm.TU.pcontig_026.59', 'evm.TU.pcontig_026.70', 'evm.TU.pcontig_026.73', 'evm.TU.pcontig_026.74', 'evm.TU.pcontig_026.76', 'evm.TU.pcontig_027.10', 'evm.TU.pcontig_027.11', 'evm.TU.pcontig_027.132', 'evm.TU.pcontig_027.163', 'evm.TU.pcontig_027.164', 'evm.TU.pcontig_027.202', 'evm.TU.pcontig_027.205', 'evm.TU.pcontig_027.211', 'evm.TU.pcontig_027.212', 'evm.TU.pcontig_027.222', 'evm.TU.pcontig_027.248', 'evm.TU.pcontig_027.249', 'evm.TU.pcontig_027.256', 'evm.TU.pcontig_027.265', 'evm.TU.pcontig_027.41', 'evm.TU.pcontig_027.51', 'evm.TU.pcontig_027.52', 'evm.TU.pcontig_027.9', 'evm.TU.pcontig_028.113', 'evm.TU.pcontig_028.131', 'evm.TU.pcontig_028.137', 'evm.TU.pcontig_028.214', 'evm.TU.pcontig_028.215', 'evm.TU.pcontig_028.216', 'evm.TU.pcontig_028.221', 'evm.TU.pcontig_028.71', 'evm.TU.pcontig_028.9', 'evm.TU.pcontig_028.94', 'evm.TU.pcontig_029.107', 'evm.TU.pcontig_029.108', 'evm.TU.pcontig_029.113', 'evm.TU.pcontig_029.120', 'evm.TU.pcontig_029.124', 'evm.TU.pcontig_029.139', 'evm.TU.pcontig_029.140', 'evm.TU.pcontig_029.205', 'evm.TU.pcontig_029.231', 'evm.TU.pcontig_029.34', 'evm.TU.pcontig_029.38', 'evm.TU.pcontig_029.62', 'evm.TU.pcontig_029.63', 'evm.TU.pcontig_029.90', 'evm.TU.pcontig_029.92', 'evm.TU.pcontig_029.97', 'evm.TU.pcontig_030.121', 'evm.TU.pcontig_030.175', 'evm.TU.pcontig_030.181', 'evm.TU.pcontig_030.182', 'evm.TU.pcontig_030.240', 'evm.TU.pcontig_030.256', 'evm.TU.pcontig_030.276', 'evm.TU.pcontig_030.34', 'evm.TU.pcontig_030.54', 'evm.TU.pcontig_030.55', 'evm.TU.pcontig_030.66', 'evm.TU.pcontig_030.69', 'evm.TU.pcontig_031.110', 'evm.TU.pcontig_031.113', 'evm.TU.pcontig_031.122', 'evm.TU.pcontig_031.123', 'evm.TU.pcontig_031.159', 'evm.TU.pcontig_031.164', 'evm.TU.pcontig_031.165', 'evm.TU.pcontig_031.167', 'evm.TU.pcontig_031.169', 'evm.TU.pcontig_031.170', 'evm.TU.pcontig_031.174', 'evm.TU.pcontig_031.184', 'evm.TU.pcontig_031.187', 'evm.TU.pcontig_031.189', 'evm.TU.pcontig_031.29', 'evm.TU.pcontig_031.60', 'evm.TU.pcontig_031.79', 'evm.TU.pcontig_031.81', 'evm.TU.pcontig_031.83', 'evm.TU.pcontig_031.88', 'evm.TU.pcontig_031.89', 'evm.TU.pcontig_031.96', 'evm.TU.pcontig_032.19', 'evm.TU.pcontig_033.143', 'evm.TU.pcontig_033.163', 'evm.TU.pcontig_033.179', 'evm.TU.pcontig_033.216', 'evm.TU.pcontig_033.221', 'evm.TU.pcontig_033.229', 'evm.TU.pcontig_033.240', 'evm.TU.pcontig_033.73', 'evm.TU.pcontig_034.113', 'evm.TU.pcontig_034.138', 'evm.TU.pcontig_034.143', 'evm.TU.pcontig_034.155', 'evm.TU.pcontig_034.169', 'evm.TU.pcontig_034.171', 'evm.TU.pcontig_034.173', 'evm.TU.pcontig_034.191', 'evm.TU.pcontig_034.193', 'evm.TU.pcontig_034.194', 'evm.TU.pcontig_034.199', 'evm.TU.pcontig_034.218', 'evm.TU.pcontig_034.227', 'evm.TU.pcontig_034.228', 'evm.TU.pcontig_034.51', 'evm.TU.pcontig_034.86', 'evm.TU.pcontig_034.89', 'evm.TU.pcontig_034.95', 'evm.TU.pcontig_035.163', 'evm.TU.pcontig_035.165', 'evm.TU.pcontig_035.166', 'evm.TU.pcontig_035.170', 'evm.TU.pcontig_035.174', 'evm.TU.pcontig_035.179', 'evm.TU.pcontig_035.191', 'evm.TU.pcontig_035.202', 'evm.TU.pcontig_035.203', 'evm.TU.pcontig_035.208', 'evm.TU.pcontig_035.216', 'evm.TU.pcontig_035.27', 'evm.TU.pcontig_035.46', 'evm.TU.pcontig_035.5', 'evm.TU.pcontig_035.50', 'evm.TU.pcontig_035.62', 'evm.TU.pcontig_035.63', 'evm.TU.pcontig_035.66', 'evm.TU.pcontig_035.67', 'evm.TU.pcontig_035.68', 'evm.TU.pcontig_035.69', 'evm.TU.pcontig_035.70', 'evm.TU.pcontig_035.80', 'evm.TU.pcontig_035.81', 'evm.TU.pcontig_035.83', 'evm.TU.pcontig_035.91', 'evm.TU.pcontig_036.154', 'evm.TU.pcontig_036.164', 'evm.TU.pcontig_036.173', 'evm.TU.pcontig_036.221', 'evm.TU.pcontig_036.29', 'evm.TU.pcontig_036.30', 'evm.TU.pcontig_036.31', 'evm.TU.pcontig_036.32', 'evm.TU.pcontig_036.43', 'evm.TU.pcontig_036.44', 'evm.TU.pcontig_036.46', 'evm.TU.pcontig_036.56', 'evm.TU.pcontig_036.57', 'evm.TU.pcontig_036.58', 'evm.TU.pcontig_036.73', 'evm.TU.pcontig_036.75', 'evm.TU.pcontig_036.81', 'evm.TU.pcontig_037.107', 'evm.TU.pcontig_037.109', 'evm.TU.pcontig_037.110', 'evm.TU.pcontig_037.111', 'evm.TU.pcontig_037.112', 'evm.TU.pcontig_037.137', 'evm.TU.pcontig_037.187', 'evm.TU.pcontig_037.200', 'evm.TU.pcontig_037.203', 'evm.TU.pcontig_037.206', 'evm.TU.pcontig_037.208', 'evm.TU.pcontig_037.215', 'evm.TU.pcontig_037.27', 'evm.TU.pcontig_037.43', 'evm.TU.pcontig_037.63', 'evm.TU.pcontig_037.70', 'evm.TU.pcontig_039.123', 'evm.TU.pcontig_039.169', 'evm.TU.pcontig_039.174', 'evm.TU.pcontig_039.175', 'evm.TU.pcontig_039.177', 'evm.TU.pcontig_039.181', 'evm.TU.pcontig_039.184', 'evm.TU.pcontig_039.28', 'evm.TU.pcontig_039.36', 'evm.TU.pcontig_039.37', 'evm.TU.pcontig_039.87', 'evm.TU.pcontig_039.97', 'evm.TU.pcontig_040.105', 'evm.TU.pcontig_040.135', 'evm.TU.pcontig_040.143', 'evm.TU.pcontig_040.148', 'evm.TU.pcontig_040.159', 'evm.TU.pcontig_040.165', 'evm.TU.pcontig_040.168', 'evm.TU.pcontig_040.176', 'evm.TU.pcontig_040.19', 'evm.TU.pcontig_040.45', 'evm.TU.pcontig_040.81', 'evm.TU.pcontig_041.104', 'evm.TU.pcontig_041.120', 'evm.TU.pcontig_041.125', 'evm.TU.pcontig_041.14', 'evm.TU.pcontig_041.147', 'evm.TU.pcontig_041.148', 'evm.TU.pcontig_041.15', 'evm.TU.pcontig_041.173', 'evm.TU.pcontig_041.174', 'evm.TU.pcontig_041.2', 'evm.TU.pcontig_041.20', 'evm.TU.pcontig_041.33', 'evm.TU.pcontig_041.38', 'evm.TU.pcontig_041.46', 'evm.TU.pcontig_041.48', 'evm.TU.pcontig_041.56', 'evm.TU.pcontig_041.98', 'evm.TU.pcontig_041.99', 'evm.TU.pcontig_042.129', 'evm.TU.pcontig_042.130', 'evm.TU.pcontig_042.179', 'evm.TU.pcontig_042.188', 'evm.TU.pcontig_042.31', 'evm.TU.pcontig_042.4', 'evm.TU.pcontig_042.66', 'evm.TU.pcontig_043.100', 'evm.TU.pcontig_043.101', 'evm.TU.pcontig_043.127', 'evm.TU.pcontig_043.14', 'evm.TU.pcontig_043.15', 'evm.TU.pcontig_043.29', 'evm.TU.pcontig_043.31', 'evm.TU.pcontig_043.63', 'evm.TU.pcontig_043.89', 'evm.TU.pcontig_044.100', 'evm.TU.pcontig_044.13', 'evm.TU.pcontig_044.130', 'evm.TU.pcontig_044.143', 'evm.TU.pcontig_044.165', 'evm.TU.pcontig_044.167', 'evm.TU.pcontig_044.168', 'evm.TU.pcontig_044.171', 'evm.TU.pcontig_044.4', 'evm.TU.pcontig_044.40', 'evm.TU.pcontig_044.64', 'evm.TU.pcontig_044.74', 'evm.TU.pcontig_044.92', 'evm.TU.pcontig_044.93', 'evm.TU.pcontig_044.99', 'evm.TU.pcontig_045.135', 'evm.TU.pcontig_045.140', 'evm.TU.pcontig_045.144', 'evm.TU.pcontig_045.148', 'evm.TU.pcontig_045.45', 'evm.TU.pcontig_045.51', 'evm.TU.pcontig_045.68', 'evm.TU.pcontig_046.113', 'evm.TU.pcontig_046.124', 'evm.TU.pcontig_046.127', 'evm.TU.pcontig_046.131', 'evm.TU.pcontig_046.159', 'evm.TU.pcontig_046.161', 'evm.TU.pcontig_046.162', 'evm.TU.pcontig_046.3', 'evm.TU.pcontig_046.35', 'evm.TU.pcontig_046.94', 'evm.TU.pcontig_046.99', 'evm.TU.pcontig_047.114', 'evm.TU.pcontig_047.141', 'evm.TU.pcontig_047.54', 'evm.TU.pcontig_047.55', 'evm.TU.pcontig_047.89', 'evm.TU.pcontig_048.109', 'evm.TU.pcontig_048.111', 'evm.TU.pcontig_048.112', 'evm.TU.pcontig_048.65', 'evm.TU.pcontig_049.130', 'evm.TU.pcontig_049.133', 'evm.TU.pcontig_049.139', 'evm.TU.pcontig_049.150', 'evm.TU.pcontig_049.38', 'evm.TU.pcontig_049.4', 'evm.TU.pcontig_049.42', 'evm.TU.pcontig_049.58', 'evm.TU.pcontig_049.69', 'evm.TU.pcontig_049.75', 'evm.TU.pcontig_049.79', 'evm.TU.pcontig_050.102', 'evm.TU.pcontig_050.122', 'evm.TU.pcontig_050.134', 'evm.TU.pcontig_050.19', 'evm.TU.pcontig_050.3', 'evm.TU.pcontig_050.69', 'evm.TU.pcontig_050.77', 'evm.TU.pcontig_050.92', 'evm.TU.pcontig_051.48', 'evm.TU.pcontig_051.51', 'evm.TU.pcontig_052.152', 'evm.TU.pcontig_052.170', 'evm.TU.pcontig_052.54', 'evm.TU.pcontig_052.57', 'evm.TU.pcontig_052.58', 'evm.TU.pcontig_052.8', 'evm.TU.pcontig_054.116', 'evm.TU.pcontig_054.119', 'evm.TU.pcontig_054.52', 'evm.TU.pcontig_054.58', 'evm.TU.pcontig_054.59', 'evm.TU.pcontig_054.66', 'evm.TU.pcontig_054.88', 'evm.TU.pcontig_054.89', 'evm.TU.pcontig_054.91', 'evm.TU.pcontig_055.16', 'evm.TU.pcontig_055.17', 'evm.TU.pcontig_055.28', 'evm.TU.pcontig_055.31', 'evm.TU.pcontig_055.8', 'evm.TU.pcontig_056.33', 'evm.TU.pcontig_056.42', 'evm.TU.pcontig_056.46', 'evm.TU.pcontig_056.58', 'evm.TU.pcontig_056.59', 'evm.TU.pcontig_056.82', 'evm.TU.pcontig_056.83', 'evm.TU.pcontig_057.106', 'evm.TU.pcontig_057.107', 'evm.TU.pcontig_057.120', 'evm.TU.pcontig_057.51', 'evm.TU.pcontig_057.61', 'evm.TU.pcontig_057.65', 'evm.TU.pcontig_057.66', 'evm.TU.pcontig_057.73', 'evm.TU.pcontig_057.76', 'evm.TU.pcontig_057.77', 'evm.TU.pcontig_057.82', 'evm.TU.pcontig_058.21', 'evm.TU.pcontig_058.22', 'evm.TU.pcontig_058.23', 'evm.TU.pcontig_058.64', 'evm.TU.pcontig_058.72', 'evm.TU.pcontig_059.100', 'evm.TU.pcontig_059.118', 'evm.TU.pcontig_059.123', 'evm.TU.pcontig_059.13', 'evm.TU.pcontig_059.17', 'evm.TU.pcontig_059.21', 'evm.TU.pcontig_059.33', 'evm.TU.pcontig_059.50', 'evm.TU.pcontig_059.58', 'evm.TU.pcontig_059.65', 'evm.TU.pcontig_059.67', 'evm.TU.pcontig_059.78', 'evm.TU.pcontig_059.79', 'evm.TU.pcontig_059.93', 'evm.TU.pcontig_060.10', 'evm.TU.pcontig_060.11', 'evm.TU.pcontig_060.14', 'evm.TU.pcontig_060.37', 'evm.TU.pcontig_060.94', 'evm.TU.pcontig_060.95', 'evm.TU.pcontig_060.96', 'evm.TU.pcontig_060.97', 'evm.TU.pcontig_062.10', 'evm.TU.pcontig_062.104', 'evm.TU.pcontig_062.15', 'evm.TU.pcontig_062.23', 'evm.TU.pcontig_062.26', 'evm.TU.pcontig_062.32', 'evm.TU.pcontig_062.45', 'evm.TU.pcontig_062.50', 'evm.TU.pcontig_062.61', 'evm.TU.pcontig_062.72', 'evm.TU.pcontig_062.78', 'evm.TU.pcontig_062.79', 'evm.TU.pcontig_062.84', 'evm.TU.pcontig_064.39', 'evm.TU.pcontig_064.41', 'evm.TU.pcontig_064.42', 'evm.TU.pcontig_065.36', 'evm.TU.pcontig_065.77', 'evm.TU.pcontig_066.1', 'evm.TU.pcontig_066.15', 'evm.TU.pcontig_066.53', 'evm.TU.pcontig_066.71', 'evm.TU.pcontig_066.73', 'evm.TU.pcontig_066.98', 'evm.TU.pcontig_067.100', 'evm.TU.pcontig_067.20', 'evm.TU.pcontig_067.39', 'evm.TU.pcontig_067.43', 'evm.TU.pcontig_067.45', 'evm.TU.pcontig_067.57', 'evm.TU.pcontig_067.93', 'evm.TU.pcontig_068.5', 'evm.TU.pcontig_068.8', 'evm.TU.pcontig_069.3', 'evm.TU.pcontig_069.38', 'evm.TU.pcontig_070.1', 'evm.TU.pcontig_070.20', 'evm.TU.pcontig_070.22', 'evm.TU.pcontig_070.24', 'evm.TU.pcontig_070.71', 'evm.TU.pcontig_070.73', 'evm.TU.pcontig_072.78', 'evm.TU.pcontig_073.14', 'evm.TU.pcontig_073.15', 'evm.TU.pcontig_073.4', 'evm.TU.pcontig_073.45', 'evm.TU.pcontig_073.54', 'evm.TU.pcontig_073.55', 'evm.TU.pcontig_073.8', 'evm.TU.pcontig_074.20', 'evm.TU.pcontig_075.10', 'evm.TU.pcontig_077.1', 'evm.TU.pcontig_077.13', 'evm.TU.pcontig_077.44', 'evm.TU.pcontig_077.55', 'evm.TU.pcontig_078.1', 'evm.TU.pcontig_078.2', 'evm.TU.pcontig_078.33', 'evm.TU.pcontig_078.34', 'evm.TU.pcontig_078.38', 'evm.TU.pcontig_078.45', 'evm.TU.pcontig_078.48', 'evm.TU.pcontig_078.60', 'evm.TU.pcontig_078.66', 'evm.TU.pcontig_078.67', 'evm.TU.pcontig_078.71', 'evm.TU.pcontig_080.23', 'evm.TU.pcontig_080.26', 'evm.TU.pcontig_080.29', 'evm.TU.pcontig_080.30', 'evm.TU.pcontig_080.46', 'evm.TU.pcontig_080.57', 'evm.TU.pcontig_080.58', 'evm.TU.pcontig_081.24', 'evm.TU.pcontig_081.30', 'evm.TU.pcontig_081.52', 'evm.TU.pcontig_081.53', 'evm.TU.pcontig_081.60', 'evm.TU.pcontig_081.62', 'evm.TU.pcontig_082.54', 'evm.TU.pcontig_083.37', 'evm.TU.pcontig_083.49', 'evm.TU.pcontig_084.9', 'evm.TU.pcontig_086.22', 'evm.TU.pcontig_086.52', 'evm.TU.pcontig_087.16', 'evm.TU.pcontig_087.6', 'evm.TU.pcontig_087.7', 'evm.TU.pcontig_088.33', 'evm.TU.pcontig_088.49', 'evm.TU.pcontig_088.55', 'evm.TU.pcontig_089.21', 'evm.TU.pcontig_089.47', 'evm.TU.pcontig_090.13', 'evm.TU.pcontig_091.46', 'evm.TU.pcontig_091.47', 'evm.TU.pcontig_091.55', 'evm.TU.pcontig_092.34', 'evm.TU.pcontig_092.7', 'evm.TU.pcontig_092.8', 'evm.TU.pcontig_095.44', 'evm.TU.pcontig_097.21', 'evm.TU.pcontig_100.29', 'evm.TU.pcontig_100.35', 'evm.TU.pcontig_100.42', 'evm.TU.pcontig_103.12', 'evm.TU.pcontig_103.22', 'evm.TU.pcontig_103.24', 'evm.TU.pcontig_103.6', 'evm.TU.pcontig_103.8', 'evm.TU.pcontig_104.33', 'evm.TU.pcontig_104.47', 'evm.TU.pcontig_112.1', 'evm.TU.pcontig_115.25', 'evm.TU.pcontig_120.23', 'evm.TU.pcontig_130.20', 'evm.TU.pcontig_130.9', 'evm.TU.pcontig_147.9', 'evm.TU.pcontig_148.11', 'evm.TU.pcontig_148.9', 'evm.TU.pcontig_149.10', 'evm.TU.pcontig_157.3', 'evm.TU.pcontig_157.4', 'evm.TU.pcontig_157.8', 'evm.TU.pcontig_165.8', 'evm.TU.pcontig_170.3', 'evm.TU.pcontig_173.4', 'evm.TU.pcontig_173.5', 'evm.TU.pcontig_174.1', 'evm.TU.pcontig_181.9', 'evm.TU.pcontig_186.2', 'evm.TU.pcontig_186.6', 'evm.TU.pcontig_186.7', 'evm.TU.pcontig_186.9', 'evm.TU.pcontig_188.5', 'evm.TU.pcontig_188.6', 'evm.TU.pcontig_189.7', 'evm.TU.pcontig_203.5', 'evm.TU.pcontig_203.6', 'evm.TU.pcontig_207.1', 'evm.TU.pcontig_233.6', 'evm.TU.pcontig_235.7']\n"
     ]
    }
   ],
   "source": [
    "# Check if list works (it does, but is not sorted)\n",
    "print(effectors)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['evm.TU.hcontig_000_003.1', 'evm.TU.hcontig_000_003.10', 'evm.TU.hcontig_000_003.120', 'evm.TU.hcontig_000_003.158', 'evm.TU.hcontig_000_003.2', 'evm.TU.hcontig_000_003.20', 'evm.TU.hcontig_000_003.26', 'evm.TU.hcontig_000_003.314', 'evm.TU.hcontig_000_003.340', 'evm.TU.hcontig_000_003.380', 'evm.TU.hcontig_000_003.402', 'evm.TU.hcontig_000_003.419', 'evm.TU.hcontig_000_003.421', 'evm.TU.hcontig_000_003.423', 'evm.TU.hcontig_000_003.444', 'evm.TU.hcontig_000_003.450', 'evm.TU.hcontig_000_003.90', 'evm.TU.hcontig_000_031.4', 'evm.TU.hcontig_000_050.114', 'evm.TU.hcontig_000_050.122', 'evm.TU.hcontig_000_050.141', 'evm.TU.hcontig_000_050.144', 'evm.TU.hcontig_000_050.149', 'evm.TU.hcontig_000_050.34', 'evm.TU.hcontig_000_050.85', 'evm.TU.hcontig_000_050.87', 'evm.TU.hcontig_000_050.9', 'evm.TU.hcontig_000_050.93', 'evm.TU.hcontig_000_054.13', 'evm.TU.hcontig_000_054.43', 'evm.TU.hcontig_000_054.68', 'evm.TU.hcontig_001_001.103', 'evm.TU.hcontig_001_001.128', 'evm.TU.hcontig_001_001.129', 'evm.TU.hcontig_001_001.144', 'evm.TU.hcontig_001_001.153', 'evm.TU.hcontig_001_001.34', 'evm.TU.hcontig_001_001.36', 'evm.TU.hcontig_001_001.39', 'evm.TU.hcontig_001_001.41', 'evm.TU.hcontig_001_001.87', 'evm.TU.hcontig_001_001.89', 'evm.TU.hcontig_001_002.39', 'evm.TU.hcontig_001_002.4', 'evm.TU.hcontig_001_002.47', 'evm.TU.hcontig_001_002.5', 'evm.TU.hcontig_001_019.1', 'evm.TU.hcontig_001_019.2', 'evm.TU.hcontig_001_021.110', 'evm.TU.hcontig_001_021.115', 'evm.TU.hcontig_001_021.129', 'evm.TU.hcontig_001_021.139', 'evm.TU.hcontig_001_021.15', 'evm.TU.hcontig_001_021.198', 'evm.TU.hcontig_001_021.28', 'evm.TU.hcontig_001_021.65', 'evm.TU.hcontig_001_021.9', 'evm.TU.hcontig_001_032.3', 'evm.TU.hcontig_001_032.5', 'evm.TU.hcontig_001_042.9', 'evm.TU.hcontig_001_054.18', 'evm.TU.hcontig_001_054.81', 'evm.TU.hcontig_001_054.82', 'evm.TU.hcontig_001_056.2', 'evm.TU.hcontig_001_056.3', 'evm.TU.hcontig_001_056.6', 'evm.TU.hcontig_001_056.8', 'evm.TU.hcontig_001_123.3', 'evm.TU.hcontig_001_180.3', 'evm.TU.hcontig_001_180.4', 'evm.TU.hcontig_001_180.5', 'evm.TU.hcontig_001_180.6', 'evm.TU.hcontig_001_180.7', 'evm.TU.hcontig_002_011.110', 'evm.TU.hcontig_002_011.123', 'evm.TU.hcontig_002_011.134', 'evm.TU.hcontig_002_011.146', 'evm.TU.hcontig_002_011.188', 'evm.TU.hcontig_002_011.201', 'evm.TU.hcontig_002_011.23', 'evm.TU.hcontig_002_011.234', 'evm.TU.hcontig_002_011.29', 'evm.TU.hcontig_002_011.33', 'evm.TU.hcontig_002_011.37', 'evm.TU.hcontig_002_011.66', 'evm.TU.hcontig_002_023.7', 'evm.TU.hcontig_002_028.1', 'evm.TU.hcontig_002_028.149', 'evm.TU.hcontig_002_028.150', 'evm.TU.hcontig_002_028.153', 'evm.TU.hcontig_002_028.159', 'evm.TU.hcontig_002_028.161', 'evm.TU.hcontig_002_028.162', 'evm.TU.hcontig_002_028.173', 'evm.TU.hcontig_002_028.177', 'evm.TU.hcontig_002_028.189', 'evm.TU.hcontig_002_028.206', 'evm.TU.hcontig_002_028.210', 'evm.TU.hcontig_002_028.230', 'evm.TU.hcontig_002_028.271', 'evm.TU.hcontig_002_028.276', 'evm.TU.hcontig_002_028.277', 'evm.TU.hcontig_002_028.279', 'evm.TU.hcontig_002_028.297', 'evm.TU.hcontig_002_028.300', 'evm.TU.hcontig_002_028.303', 'evm.TU.hcontig_002_028.306', 'evm.TU.hcontig_002_028.5', 'evm.TU.hcontig_002_028.54', 'evm.TU.hcontig_002_028.63', 'evm.TU.hcontig_002_028.75', 'evm.TU.hcontig_002_028.84', 'evm.TU.hcontig_003_002.113', 'evm.TU.hcontig_003_002.114', 'evm.TU.hcontig_003_002.125', 'evm.TU.hcontig_003_002.126', 'evm.TU.hcontig_003_002.127', 'evm.TU.hcontig_003_002.159', 'evm.TU.hcontig_003_002.209', 'evm.TU.hcontig_003_002.213', 'evm.TU.hcontig_003_002.215', 'evm.TU.hcontig_003_002.38', 'evm.TU.hcontig_003_002.54', 'evm.TU.hcontig_003_002.59', 'evm.TU.hcontig_003_002.61', 'evm.TU.hcontig_003_002.62', 'evm.TU.hcontig_003_002.91', 'evm.TU.hcontig_003_002.92', 'evm.TU.hcontig_003_002.93', 'evm.TU.hcontig_003_013.20', 'evm.TU.hcontig_003_013.21', 'evm.TU.hcontig_003_013.22', 'evm.TU.hcontig_003_013.23', 'evm.TU.hcontig_003_013.40', 'evm.TU.hcontig_003_013.41', 'evm.TU.hcontig_003_013.44', 'evm.TU.hcontig_003_013.5', 'evm.TU.hcontig_003_013.6', 'evm.TU.hcontig_003_014.20', 'evm.TU.hcontig_003_014.3', 'evm.TU.hcontig_003_014.46', 'evm.TU.hcontig_003_014.54', 'evm.TU.hcontig_003_014.55', 'evm.TU.hcontig_003_014.9', 'evm.TU.hcontig_003_026.4', 'evm.TU.hcontig_003_026.5', 'evm.TU.hcontig_003_028.6', 'evm.TU.hcontig_003_041.76', 'evm.TU.hcontig_003_045.11', 'evm.TU.hcontig_003_045.14', 'evm.TU.hcontig_003_045.19', 'evm.TU.hcontig_003_045.2', 'evm.TU.hcontig_003_045.20', 'evm.TU.hcontig_003_045.25', 'evm.TU.hcontig_003_045.26', 'evm.TU.hcontig_003_045.7', 'evm.TU.hcontig_003_136.1', 'evm.TU.hcontig_003_136.15', 'evm.TU.hcontig_004_019.5', 'evm.TU.hcontig_004_020.118', 'evm.TU.hcontig_004_020.12', 'evm.TU.hcontig_004_020.136', 'evm.TU.hcontig_004_020.14', 'evm.TU.hcontig_004_020.15', 'evm.TU.hcontig_004_020.166', 'evm.TU.hcontig_004_020.169', 'evm.TU.hcontig_004_020.18', 'evm.TU.hcontig_004_020.19', 'evm.TU.hcontig_004_020.190', 'evm.TU.hcontig_004_020.215', 'evm.TU.hcontig_004_020.219', 'evm.TU.hcontig_004_020.26', 'evm.TU.hcontig_004_020.27', 'evm.TU.hcontig_004_020.299', 'evm.TU.hcontig_004_020.324', 'evm.TU.hcontig_004_020.36', 'evm.TU.hcontig_004_020.391', 'evm.TU.hcontig_004_020.395', 'evm.TU.hcontig_004_020.397', 'evm.TU.hcontig_004_020.40', 'evm.TU.hcontig_004_020.55', 'evm.TU.hcontig_004_020.68', 'evm.TU.hcontig_004_021.2', 'evm.TU.hcontig_004_027.1', 'evm.TU.hcontig_004_027.2', 'evm.TU.hcontig_004_076.53', 'evm.TU.hcontig_005_002.19', 'evm.TU.hcontig_005_002.40', 'evm.TU.hcontig_005_002.82', 'evm.TU.hcontig_005_002.83', 'evm.TU.hcontig_005_002.86', 'evm.TU.hcontig_005_002.89', 'evm.TU.hcontig_005_021.1', 'evm.TU.hcontig_005_021.2', 'evm.TU.hcontig_005_021.37', 'evm.TU.hcontig_005_021.58', 'evm.TU.hcontig_005_021.65', 'evm.TU.hcontig_005_039.14', 'evm.TU.hcontig_005_043.152', 'evm.TU.hcontig_005_043.177', 'evm.TU.hcontig_005_043.178', 'evm.TU.hcontig_005_043.187', 'evm.TU.hcontig_005_043.188', 'evm.TU.hcontig_005_043.205', 'evm.TU.hcontig_005_043.218', 'evm.TU.hcontig_005_043.227', 'evm.TU.hcontig_005_043.41', 'evm.TU.hcontig_005_043.44', 'evm.TU.hcontig_005_043.46', 'evm.TU.hcontig_005_043.70', 'evm.TU.hcontig_005_043.71', 'evm.TU.hcontig_005_043.78', 'evm.TU.hcontig_005_043.84', 'evm.TU.hcontig_005_043.91', 'evm.TU.hcontig_006_002.39', 'evm.TU.hcontig_006_002.5', 'evm.TU.hcontig_006_013.11', 'evm.TU.hcontig_006_013.137', 'evm.TU.hcontig_006_013.14', 'evm.TU.hcontig_006_013.173', 'evm.TU.hcontig_006_013.222', 'evm.TU.hcontig_006_013.3', 'evm.TU.hcontig_006_013.47', 'evm.TU.hcontig_006_013.78', 'evm.TU.hcontig_006_013.90', 'evm.TU.hcontig_006_015.31', 'evm.TU.hcontig_006_021.3', 'evm.TU.hcontig_006_033.17', 'evm.TU.hcontig_006_033.2', 'evm.TU.hcontig_006_033.3', 'evm.TU.hcontig_006_033.37', 'evm.TU.hcontig_006_131.11', 'evm.TU.hcontig_006_131.13', 'evm.TU.hcontig_007_001.4', 'evm.TU.hcontig_007_005.22', 'evm.TU.hcontig_007_005.5', 'evm.TU.hcontig_007_005.64', 'evm.TU.hcontig_007_005.66', 'evm.TU.hcontig_007_005.7', 'evm.TU.hcontig_007_005.75', 'evm.TU.hcontig_007_005.78', 'evm.TU.hcontig_007_006.111', 'evm.TU.hcontig_007_006.112', 'evm.TU.hcontig_007_006.164', 'evm.TU.hcontig_007_006.196', 'evm.TU.hcontig_007_006.202', 'evm.TU.hcontig_007_006.206', 'evm.TU.hcontig_007_006.207', 'evm.TU.hcontig_007_006.210', 'evm.TU.hcontig_007_006.217', 'evm.TU.hcontig_007_006.220', 'evm.TU.hcontig_007_006.221', 'evm.TU.hcontig_007_006.223', 'evm.TU.hcontig_007_006.23', 'evm.TU.hcontig_007_007.17', 'evm.TU.hcontig_007_007.18', 'evm.TU.hcontig_007_007.43', 'evm.TU.hcontig_007_007.8', 'evm.TU.hcontig_007_031.2', 'evm.TU.hcontig_008_009.10', 'evm.TU.hcontig_008_009.111', 'evm.TU.hcontig_008_009.38', 'evm.TU.hcontig_008_009.46', 'evm.TU.hcontig_008_009.88', 'evm.TU.hcontig_008_014.43', 'evm.TU.hcontig_008_014.64', 'evm.TU.hcontig_008_014.80', 'evm.TU.hcontig_008_035.11', 'evm.TU.hcontig_008_093.37', 'evm.TU.hcontig_009_015.1', 'evm.TU.hcontig_009_024.1', 'evm.TU.hcontig_009_024.112', 'evm.TU.hcontig_009_024.117', 'evm.TU.hcontig_009_024.18', 'evm.TU.hcontig_009_024.55', 'evm.TU.hcontig_009_024.74', 'evm.TU.hcontig_009_024.78', 'evm.TU.hcontig_009_024.80', 'evm.TU.hcontig_009_029.41', 'evm.TU.hcontig_009_029.51', 'evm.TU.hcontig_009_029.53', 'evm.TU.hcontig_009_029.55', 'evm.TU.hcontig_009_029.56', 'evm.TU.hcontig_009_029.61', 'evm.TU.hcontig_009_142.10', 'evm.TU.hcontig_010_016.105', 'evm.TU.hcontig_010_016.114', 'evm.TU.hcontig_010_016.141', 'evm.TU.hcontig_010_016.186', 'evm.TU.hcontig_010_016.187', 'evm.TU.hcontig_010_016.188', 'evm.TU.hcontig_010_016.192', 'evm.TU.hcontig_010_016.199', 'evm.TU.hcontig_010_016.202', 'evm.TU.hcontig_010_016.205', 'evm.TU.hcontig_010_016.207', 'evm.TU.hcontig_010_016.220', 'evm.TU.hcontig_010_016.224', 'evm.TU.hcontig_010_016.225', 'evm.TU.hcontig_010_016.226', 'evm.TU.hcontig_010_016.236', 'evm.TU.hcontig_010_016.237', 'evm.TU.hcontig_010_016.246', 'evm.TU.hcontig_010_016.247', 'evm.TU.hcontig_010_016.270', 'evm.TU.hcontig_010_016.272', 'evm.TU.hcontig_010_016.277', 'evm.TU.hcontig_010_016.296', 'evm.TU.hcontig_010_016.297', 'evm.TU.hcontig_010_016.38', 'evm.TU.hcontig_010_016.41', 'evm.TU.hcontig_010_016.47', 'evm.TU.hcontig_010_016.48', 'evm.TU.hcontig_010_016.62', 'evm.TU.hcontig_010_016.92', 'evm.TU.hcontig_010_016.98', 'evm.TU.hcontig_010_016.99', 'evm.TU.hcontig_011_005.6', 'evm.TU.hcontig_011_013.3', 'evm.TU.hcontig_011_016.116', 'evm.TU.hcontig_011_016.117', 'evm.TU.hcontig_011_016.119', 'evm.TU.hcontig_011_016.143', 'evm.TU.hcontig_011_016.148', 'evm.TU.hcontig_011_016.149', 'evm.TU.hcontig_011_016.169', 'evm.TU.hcontig_011_016.205', 'evm.TU.hcontig_011_016.223', 'evm.TU.hcontig_011_016.240', 'evm.TU.hcontig_011_016.26', 'evm.TU.hcontig_011_016.27', 'evm.TU.hcontig_011_016.28', 'evm.TU.hcontig_011_016.30', 'evm.TU.hcontig_011_016.31', 'evm.TU.hcontig_011_016.32', 'evm.TU.hcontig_011_016.37', 'evm.TU.hcontig_011_016.91', 'evm.TU.hcontig_011_016.92', 'evm.TU.hcontig_011_017.25', 'evm.TU.hcontig_011_017.37', 'evm.TU.hcontig_011_017.43', 'evm.TU.hcontig_012_010.100', 'evm.TU.hcontig_012_010.103', 'evm.TU.hcontig_012_010.105', 'evm.TU.hcontig_012_010.20', 'evm.TU.hcontig_012_010.25', 'evm.TU.hcontig_012_010.28', 'evm.TU.hcontig_012_010.30', 'evm.TU.hcontig_012_010.60', 'evm.TU.hcontig_012_010.72', 'evm.TU.hcontig_012_024.75', 'evm.TU.hcontig_012_028.106', 'evm.TU.hcontig_012_028.128', 'evm.TU.hcontig_012_028.144', 'evm.TU.hcontig_012_028.25', 'evm.TU.hcontig_012_028.26', 'evm.TU.hcontig_012_028.28', 'evm.TU.hcontig_012_028.93', 'evm.TU.hcontig_012_028.98', 'evm.TU.hcontig_012_116.5', 'evm.TU.hcontig_012_116.7', 'evm.TU.hcontig_013_010.110', 'evm.TU.hcontig_013_010.111', 'evm.TU.hcontig_013_010.17', 'evm.TU.hcontig_013_010.19', 'evm.TU.hcontig_013_010.39', 'evm.TU.hcontig_013_010.40', 'evm.TU.hcontig_013_010.53', 'evm.TU.hcontig_013_010.57', 'evm.TU.hcontig_013_010.59', 'evm.TU.hcontig_013_010.89', 'evm.TU.hcontig_013_010.90', 'evm.TU.hcontig_013_012.64', 'evm.TU.hcontig_013_014.1', 'evm.TU.hcontig_013_018.3', 'evm.TU.hcontig_013_023.20', 'evm.TU.hcontig_013_023.23', 'evm.TU.hcontig_013_023.33', 'evm.TU.hcontig_013_023.71', 'evm.TU.hcontig_013_029.5', 'evm.TU.hcontig_013_029.6', 'evm.TU.hcontig_014_008.11', 'evm.TU.hcontig_014_017.7', 'evm.TU.hcontig_014_022.17', 'evm.TU.hcontig_014_022.19', 'evm.TU.hcontig_014_027.24', 'evm.TU.hcontig_014_117.15', 'evm.TU.hcontig_014_117.19', 'evm.TU.hcontig_014_117.25', 'evm.TU.hcontig_014_117.4', 'evm.TU.hcontig_014_192.10', 'evm.TU.hcontig_014_192.3', 'evm.TU.hcontig_016_014.15', 'evm.TU.hcontig_016_024.126', 'evm.TU.hcontig_016_024.133', 'evm.TU.hcontig_016_024.163', 'evm.TU.hcontig_016_024.22', 'evm.TU.hcontig_016_024.57', 'evm.TU.hcontig_016_024.79', 'evm.TU.hcontig_016_124.11', 'evm.TU.hcontig_017_004.103', 'evm.TU.hcontig_017_004.114', 'evm.TU.hcontig_017_004.118', 'evm.TU.hcontig_017_004.120', 'evm.TU.hcontig_017_004.140', 'evm.TU.hcontig_017_004.141', 'evm.TU.hcontig_017_004.2', 'evm.TU.hcontig_017_004.50', 'evm.TU.hcontig_017_004.51', 'evm.TU.hcontig_017_004.6', 'evm.TU.hcontig_017_004.70', 'evm.TU.hcontig_017_004.71', 'evm.TU.hcontig_017_004.73', 'evm.TU.hcontig_017_011.5', 'evm.TU.hcontig_017_011.6', 'evm.TU.hcontig_017_025.13', 'evm.TU.hcontig_017_025.2', 'evm.TU.hcontig_017_026.2', 'evm.TU.hcontig_017_160.7', 'evm.TU.hcontig_018_007.6', 'evm.TU.hcontig_018_008.1', 'evm.TU.hcontig_018_008.4', 'evm.TU.hcontig_018_010.2', 'evm.TU.hcontig_018_015.5', 'evm.TU.hcontig_018_016.123', 'evm.TU.hcontig_018_016.129', 'evm.TU.hcontig_018_016.132', 'evm.TU.hcontig_018_016.133', 'evm.TU.hcontig_018_016.48', 'evm.TU.hcontig_018_016.70', 'evm.TU.hcontig_018_016.84', 'evm.TU.hcontig_018_016.86', 'evm.TU.hcontig_018_022.106', 'evm.TU.hcontig_018_022.137', 'evm.TU.hcontig_018_022.3', 'evm.TU.hcontig_018_022.65', 'evm.TU.hcontig_018_022.68', 'evm.TU.hcontig_018_022.76', 'evm.TU.hcontig_018_022.79', 'evm.TU.hcontig_018_022.82', 'evm.TU.hcontig_018_022.83', 'evm.TU.hcontig_018_022.88', 'evm.TU.hcontig_019_001.21', 'evm.TU.hcontig_019_001.29', 'evm.TU.hcontig_019_001.33', 'evm.TU.hcontig_019_001.9', 'evm.TU.hcontig_019_013.16', 'evm.TU.hcontig_019_013.18', 'evm.TU.hcontig_019_013.24', 'evm.TU.hcontig_019_013.25', 'evm.TU.hcontig_019_013.36', 'evm.TU.hcontig_019_013.55', 'evm.TU.hcontig_019_015.122', 'evm.TU.hcontig_019_015.15', 'evm.TU.hcontig_019_015.17', 'evm.TU.hcontig_019_015.18', 'evm.TU.hcontig_019_015.25', 'evm.TU.hcontig_019_020.15', 'evm.TU.hcontig_019_020.25', 'evm.TU.hcontig_019_020.31', 'evm.TU.hcontig_019_085.2', 'evm.TU.hcontig_019_085.3', 'evm.TU.hcontig_019_085.36', 'evm.TU.hcontig_019_085.53', 'evm.TU.hcontig_019_085.57', 'evm.TU.hcontig_020_007.1', 'evm.TU.hcontig_020_018.35', 'evm.TU.hcontig_020_018.46', 'evm.TU.hcontig_020_018.5', 'evm.TU.hcontig_020_021.129', 'evm.TU.hcontig_020_021.14', 'evm.TU.hcontig_020_021.150', 'evm.TU.hcontig_020_021.151', 'evm.TU.hcontig_020_021.28', 'evm.TU.hcontig_020_021.45', 'evm.TU.hcontig_020_021.47', 'evm.TU.hcontig_020_021.59', 'evm.TU.hcontig_020_021.62', 'evm.TU.hcontig_020_021.7', 'evm.TU.hcontig_020_022.7', 'evm.TU.hcontig_020_025.32', 'evm.TU.hcontig_020_132.11', 'evm.TU.hcontig_020_132.17', 'evm.TU.hcontig_020_132.18', 'evm.TU.hcontig_021_014.14', 'evm.TU.hcontig_021_014.149', 'evm.TU.hcontig_021_014.151', 'evm.TU.hcontig_021_014.168', 'evm.TU.hcontig_021_014.169', 'evm.TU.hcontig_021_014.30', 'evm.TU.hcontig_021_014.38', 'evm.TU.hcontig_021_014.45', 'evm.TU.hcontig_021_014.63', 'evm.TU.hcontig_021_014.76', 'evm.TU.hcontig_021_014.84', 'evm.TU.hcontig_021_014.96', 'evm.TU.hcontig_021_026.2', 'evm.TU.hcontig_021_026.22', 'evm.TU.hcontig_021_026.23', 'evm.TU.hcontig_021_027.28', 'evm.TU.hcontig_021_027.3', 'evm.TU.hcontig_021_027.33', 'evm.TU.hcontig_021_027.62', 'evm.TU.hcontig_021_027.70', 'evm.TU.hcontig_021_027.72', 'evm.TU.hcontig_021_027.83', 'evm.TU.hcontig_022_007.17', 'evm.TU.hcontig_022_007.25', 'evm.TU.hcontig_022_007.29', 'evm.TU.hcontig_022_007.31', 'evm.TU.hcontig_022_007.39', 'evm.TU.hcontig_022_008.107', 'evm.TU.hcontig_022_008.14', 'evm.TU.hcontig_022_008.35', 'evm.TU.hcontig_022_008.39', 'evm.TU.hcontig_022_008.53', 'evm.TU.hcontig_022_008.79', 'evm.TU.hcontig_022_015.15', 'evm.TU.hcontig_022_015.7', 'evm.TU.hcontig_022_101.24', 'evm.TU.hcontig_023_002.107', 'evm.TU.hcontig_023_002.11', 'evm.TU.hcontig_023_002.142', 'evm.TU.hcontig_023_002.26', 'evm.TU.hcontig_023_002.27', 'evm.TU.hcontig_023_002.33', 'evm.TU.hcontig_023_002.43', 'evm.TU.hcontig_023_002.67', 'evm.TU.hcontig_023_003.3', 'evm.TU.hcontig_023_007.110', 'evm.TU.hcontig_023_007.31', 'evm.TU.hcontig_023_007.51', 'evm.TU.hcontig_023_007.77', 'evm.TU.hcontig_023_007.96', 'evm.TU.hcontig_025_002.15', 'evm.TU.hcontig_025_002.19', 'evm.TU.hcontig_025_003.23', 'evm.TU.hcontig_025_003.24', 'evm.TU.hcontig_025_003.38', 'evm.TU.hcontig_025_003.39', 'evm.TU.hcontig_025_003.4', 'evm.TU.hcontig_025_003.63', 'evm.TU.hcontig_026_002.113', 'evm.TU.hcontig_026_002.124', 'evm.TU.hcontig_026_002.137', 'evm.TU.hcontig_026_002.138', 'evm.TU.hcontig_026_002.159', 'evm.TU.hcontig_026_002.31', 'evm.TU.hcontig_026_002.32', 'evm.TU.hcontig_026_002.7', 'evm.TU.hcontig_026_002.8', 'evm.TU.hcontig_026_011.10', 'evm.TU.hcontig_026_011.11', 'evm.TU.hcontig_026_011.9', 'evm.TU.hcontig_026_018.47', 'evm.TU.hcontig_026_018.49', 'evm.TU.hcontig_026_018.50', 'evm.TU.hcontig_026_018.59', 'evm.TU.hcontig_026_018.62', 'evm.TU.hcontig_026_018.63', 'evm.TU.hcontig_027_006.107', 'evm.TU.hcontig_027_006.69', 'evm.TU.hcontig_027_011.32', 'evm.TU.hcontig_027_011.38', 'evm.TU.hcontig_027_011.39', 'evm.TU.hcontig_027_011.40', 'evm.TU.hcontig_027_011.42', 'evm.TU.hcontig_027_011.78', 'evm.TU.hcontig_027_011.79', 'evm.TU.hcontig_027_011.80', 'evm.TU.hcontig_027_011.86', 'evm.TU.hcontig_027_018.8', 'evm.TU.hcontig_027_018.9', 'evm.TU.hcontig_027_102.2', 'evm.TU.hcontig_027_102.24', 'evm.TU.hcontig_027_102.43', 'evm.TU.hcontig_027_102.45', 'evm.TU.hcontig_028_003.17', 'evm.TU.hcontig_028_003.19', 'evm.TU.hcontig_028_003.23', 'evm.TU.hcontig_028_011.104', 'evm.TU.hcontig_028_011.105', 'evm.TU.hcontig_028_011.113', 'evm.TU.hcontig_028_011.125', 'evm.TU.hcontig_028_011.6', 'evm.TU.hcontig_028_011.85', 'evm.TU.hcontig_028_015.14', 'evm.TU.hcontig_028_015.19', 'evm.TU.hcontig_028_015.2', 'evm.TU.hcontig_029_001.3', 'evm.TU.hcontig_029_010.1', 'evm.TU.hcontig_029_010.2', 'evm.TU.hcontig_029_010.62', 'evm.TU.hcontig_029_010.80', 'evm.TU.hcontig_029_010.84', 'evm.TU.hcontig_029_013.103', 'evm.TU.hcontig_029_013.104', 'evm.TU.hcontig_029_013.24', 'evm.TU.hcontig_029_013.30', 'evm.TU.hcontig_029_013.46', 'evm.TU.hcontig_029_013.74', 'evm.TU.hcontig_029_013.76', 'evm.TU.hcontig_029_013.81', 'evm.TU.hcontig_029_013.83', 'evm.TU.hcontig_029_013.90', 'evm.TU.hcontig_029_013.96', 'evm.TU.hcontig_029_014.3', 'evm.TU.hcontig_029_023.1', 'evm.TU.hcontig_029_023.2', 'evm.TU.hcontig_030_011.116', 'evm.TU.hcontig_030_011.134', 'evm.TU.hcontig_030_011.15', 'evm.TU.hcontig_030_011.151', 'evm.TU.hcontig_030_011.160', 'evm.TU.hcontig_030_011.161', 'evm.TU.hcontig_030_011.203', 'evm.TU.hcontig_030_011.25', 'evm.TU.hcontig_030_011.26', 'evm.TU.hcontig_030_011.38', 'evm.TU.hcontig_030_011.41', 'evm.TU.hcontig_030_011.42', 'evm.TU.hcontig_030_011.92', 'evm.TU.hcontig_030_155.14', 'evm.TU.hcontig_031_005.109', 'evm.TU.hcontig_031_005.112', 'evm.TU.hcontig_031_005.15', 'evm.TU.hcontig_031_005.22', 'evm.TU.hcontig_031_005.23', 'evm.TU.hcontig_031_005.29', 'evm.TU.hcontig_031_005.44', 'evm.TU.hcontig_031_005.47', 'evm.TU.hcontig_031_005.52', 'evm.TU.hcontig_031_005.53', 'evm.TU.hcontig_031_005.54', 'evm.TU.hcontig_031_005.87', 'evm.TU.hcontig_031_005.9', 'evm.TU.hcontig_031_005.93', 'evm.TU.hcontig_031_005.96', 'evm.TU.hcontig_031_005.97', 'evm.TU.hcontig_031_005.99', 'evm.TU.hcontig_031_007.24', 'evm.TU.hcontig_031_007.26', 'evm.TU.hcontig_031_007.35', 'evm.TU.hcontig_031_007.40', 'evm.TU.hcontig_031_007.42', 'evm.TU.hcontig_031_007.69', 'evm.TU.hcontig_031_007.91', 'evm.TU.hcontig_031_013.17', 'evm.TU.hcontig_031_015.2', 'evm.TU.hcontig_031_015.4', 'evm.TU.hcontig_033_010.1', 'evm.TU.hcontig_033_014.54', 'evm.TU.hcontig_033_071.16', 'evm.TU.hcontig_033_071.17', 'evm.TU.hcontig_033_071.21', 'evm.TU.hcontig_033_071.25', 'evm.TU.hcontig_033_071.59', 'evm.TU.hcontig_033_071.60', 'evm.TU.hcontig_033_071.76', 'evm.TU.hcontig_033_071.85', 'evm.TU.hcontig_034_001.109', 'evm.TU.hcontig_034_001.116', 'evm.TU.hcontig_034_001.127', 'evm.TU.hcontig_034_001.140', 'evm.TU.hcontig_034_001.142', 'evm.TU.hcontig_034_001.144', 'evm.TU.hcontig_034_001.157', 'evm.TU.hcontig_034_001.158', 'evm.TU.hcontig_034_001.160', 'evm.TU.hcontig_034_001.163', 'evm.TU.hcontig_034_001.21', 'evm.TU.hcontig_034_001.57', 'evm.TU.hcontig_034_001.60', 'evm.TU.hcontig_034_001.66', 'evm.TU.hcontig_034_001.76', 'evm.TU.hcontig_034_001.83', 'evm.TU.hcontig_034_005.1', 'evm.TU.hcontig_034_022.11', 'evm.TU.hcontig_034_022.9', 'evm.TU.hcontig_035_008.10', 'evm.TU.hcontig_035_008.11', 'evm.TU.hcontig_035_008.129', 'evm.TU.hcontig_035_008.130', 'evm.TU.hcontig_035_008.134', 'evm.TU.hcontig_035_008.137', 'evm.TU.hcontig_035_008.144', 'evm.TU.hcontig_035_008.157', 'evm.TU.hcontig_035_008.29', 'evm.TU.hcontig_035_008.31', 'evm.TU.hcontig_035_008.32', 'evm.TU.hcontig_035_008.33', 'evm.TU.hcontig_035_008.34', 'evm.TU.hcontig_035_008.4', 'evm.TU.hcontig_035_008.46', 'evm.TU.hcontig_035_008.47', 'evm.TU.hcontig_035_008.49', 'evm.TU.hcontig_035_008.61', 'evm.TU.hcontig_035_008.8', 'evm.TU.hcontig_035_013.2', 'evm.TU.hcontig_035_019.3', 'evm.TU.hcontig_035_019.4', 'evm.TU.hcontig_035_019.6', 'evm.TU.hcontig_036_002.11', 'evm.TU.hcontig_036_002.12', 'evm.TU.hcontig_036_006.13', 'evm.TU.hcontig_036_006.14', 'evm.TU.hcontig_036_006.16', 'evm.TU.hcontig_036_008.100', 'evm.TU.hcontig_036_008.15', 'evm.TU.hcontig_036_008.6', 'evm.TU.hcontig_036_008.8', 'evm.TU.hcontig_036_012.1', 'evm.TU.hcontig_036_012.14', 'evm.TU.hcontig_036_012.16', 'evm.TU.hcontig_036_012.2', 'evm.TU.hcontig_037_004.37', 'evm.TU.hcontig_037_004.55', 'evm.TU.hcontig_037_004.58', 'evm.TU.hcontig_037_004.61', 'evm.TU.hcontig_037_004.63', 'evm.TU.hcontig_037_004.70', 'evm.TU.hcontig_037_011.31', 'evm.TU.hcontig_037_011.32', 'evm.TU.hcontig_037_011.35', 'evm.TU.hcontig_037_011.73', 'evm.TU.hcontig_037_011.75', 'evm.TU.hcontig_037_011.76', 'evm.TU.hcontig_037_011.77', 'evm.TU.hcontig_037_011.78', 'evm.TU.hcontig_037_017.3', 'evm.TU.hcontig_037_161.3', 'evm.TU.hcontig_039_004.26', 'evm.TU.hcontig_039_004.36', 'evm.TU.hcontig_039_004.37', 'evm.TU.hcontig_039_004.63', 'evm.TU.hcontig_039_005.11', 'evm.TU.hcontig_039_005.22', 'evm.TU.hcontig_039_005.38', 'evm.TU.hcontig_039_005.39', 'evm.TU.hcontig_039_005.7', 'evm.TU.hcontig_039_005.83', 'evm.TU.hcontig_039_005.88', 'evm.TU.hcontig_039_005.89', 'evm.TU.hcontig_039_005.90', 'evm.TU.hcontig_039_005.94', 'evm.TU.hcontig_039_005.96', 'evm.TU.hcontig_040_001.2', 'evm.TU.hcontig_040_001.3', 'evm.TU.hcontig_040_001.6', 'evm.TU.hcontig_040_001.9', 'evm.TU.hcontig_040_002.12', 'evm.TU.hcontig_040_002.2', 'evm.TU.hcontig_040_015.13', 'evm.TU.hcontig_040_015.29', 'evm.TU.hcontig_040_015.3', 'evm.TU.hcontig_040_015.69', 'evm.TU.hcontig_040_015.79', 'evm.TU.hcontig_040_015.86', 'evm.TU.hcontig_040_015.91', 'evm.TU.hcontig_040_109.23', 'evm.TU.hcontig_040_109.31', 'evm.TU.hcontig_041_004.2', 'evm.TU.hcontig_041_004.4', 'evm.TU.hcontig_041_004.54', 'evm.TU.hcontig_041_004.55', 'evm.TU.hcontig_041_004.60', 'evm.TU.hcontig_041_004.78', 'evm.TU.hcontig_041_004.83', 'evm.TU.hcontig_041_005.12', 'evm.TU.hcontig_041_005.28', 'evm.TU.hcontig_041_005.29', 'evm.TU.hcontig_041_005.6', 'evm.TU.hcontig_041_005.7', 'evm.TU.hcontig_041_127.12', 'evm.TU.hcontig_041_127.20', 'evm.TU.hcontig_041_127.8', 'evm.TU.hcontig_042_005.2', 'evm.TU.hcontig_042_008.7', 'evm.TU.hcontig_042_009.58', 'evm.TU.hcontig_042_009.59', 'evm.TU.hcontig_042_011.5', 'evm.TU.hcontig_042_168.11', 'evm.TU.hcontig_043_003.15', 'evm.TU.hcontig_043_003.41', 'evm.TU.hcontig_043_003.52', 'evm.TU.hcontig_043_003.53', 'evm.TU.hcontig_043_006.6', 'evm.TU.hcontig_043_119.21', 'evm.TU.hcontig_043_119.4', 'evm.TU.hcontig_043_119.8', 'evm.TU.hcontig_044_003.13', 'evm.TU.hcontig_044_003.37', 'evm.TU.hcontig_044_003.39', 'evm.TU.hcontig_044_003.40', 'evm.TU.hcontig_044_003.43', 'evm.TU.hcontig_044_023.118', 'evm.TU.hcontig_044_023.121', 'evm.TU.hcontig_044_023.30', 'evm.TU.hcontig_044_023.62', 'evm.TU.hcontig_044_023.87', 'evm.TU.hcontig_044_023.88', 'evm.TU.hcontig_045_002.10', 'evm.TU.hcontig_045_002.12', 'evm.TU.hcontig_045_002.14', 'evm.TU.hcontig_045_002.3', 'evm.TU.hcontig_045_002.83', 'evm.TU.hcontig_045_002.88', 'evm.TU.hcontig_045_002.92', 'evm.TU.hcontig_045_002.96', 'evm.TU.hcontig_046_007.10', 'evm.TU.hcontig_046_007.119', 'evm.TU.hcontig_046_007.120', 'evm.TU.hcontig_046_007.42', 'evm.TU.hcontig_046_007.58', 'evm.TU.hcontig_046_007.63', 'evm.TU.hcontig_046_007.70', 'evm.TU.hcontig_046_007.80', 'evm.TU.hcontig_046_007.83', 'evm.TU.hcontig_046_007.88', 'evm.TU.hcontig_047_003.30', 'evm.TU.hcontig_047_003.37', 'evm.TU.hcontig_047_003.38', 'evm.TU.hcontig_047_006.25', 'evm.TU.hcontig_047_006.35', 'evm.TU.hcontig_047_006.51', 'evm.TU.hcontig_047_006.9', 'evm.TU.hcontig_047_007.2', 'evm.TU.hcontig_048_005.42', 'evm.TU.hcontig_048_005.62', 'evm.TU.hcontig_048_010.27', 'evm.TU.hcontig_048_010.29', 'evm.TU.hcontig_049_002.1', 'evm.TU.hcontig_049_002.37', 'evm.TU.hcontig_049_002.41', 'evm.TU.hcontig_049_002.58', 'evm.TU.hcontig_050_003.14', 'evm.TU.hcontig_050_003.20', 'evm.TU.hcontig_050_003.45', 'evm.TU.hcontig_050_010.1', 'evm.TU.hcontig_050_011.33', 'evm.TU.hcontig_050_011.39', 'evm.TU.hcontig_050_053.12', 'evm.TU.hcontig_050_053.5', 'evm.TU.hcontig_050_121.18', 'evm.TU.hcontig_050_121.9', 'evm.TU.hcontig_052_006.115', 'evm.TU.hcontig_052_006.117', 'evm.TU.hcontig_052_006.146', 'evm.TU.hcontig_052_006.155', 'evm.TU.hcontig_052_006.25', 'evm.TU.hcontig_052_006.53', 'evm.TU.hcontig_052_006.63', 'evm.TU.hcontig_052_105.25', 'evm.TU.hcontig_052_105.4', 'evm.TU.hcontig_054_001.108', 'evm.TU.hcontig_054_001.17', 'evm.TU.hcontig_054_001.25', 'evm.TU.hcontig_054_001.35', 'evm.TU.hcontig_054_001.50', 'evm.TU.hcontig_054_001.52', 'evm.TU.hcontig_054_001.91', 'evm.TU.hcontig_055_008.4', 'evm.TU.hcontig_055_009.1', 'evm.TU.hcontig_055_009.19', 'evm.TU.hcontig_055_009.22', 'evm.TU.hcontig_055_009.8', 'evm.TU.hcontig_055_009.9', 'evm.TU.hcontig_055_012.14', 'evm.TU.hcontig_055_012.91', 'evm.TU.hcontig_056_007.13', 'evm.TU.hcontig_056_007.14', 'evm.TU.hcontig_056_007.22', 'evm.TU.hcontig_056_007.24', 'evm.TU.hcontig_056_007.28', 'evm.TU.hcontig_056_007.39', 'evm.TU.hcontig_056_007.40', 'evm.TU.hcontig_056_007.58', 'evm.TU.hcontig_056_007.59', 'evm.TU.hcontig_057_001.1', 'evm.TU.hcontig_057_001.5', 'evm.TU.hcontig_057_002.5', 'evm.TU.hcontig_057_003.120', 'evm.TU.hcontig_057_003.121', 'evm.TU.hcontig_057_003.133', 'evm.TU.hcontig_057_003.36', 'evm.TU.hcontig_057_003.54', 'evm.TU.hcontig_057_003.64', 'evm.TU.hcontig_057_003.73', 'evm.TU.hcontig_057_003.74', 'evm.TU.hcontig_057_003.80', 'evm.TU.hcontig_057_003.83', 'evm.TU.hcontig_057_003.89', 'evm.TU.hcontig_057_009.4', 'evm.TU.hcontig_057_009.5', 'evm.TU.hcontig_058_004.17', 'evm.TU.hcontig_058_004.18', 'evm.TU.hcontig_058_004.19', 'evm.TU.hcontig_058_175.7', 'evm.TU.hcontig_059_005.1', 'evm.TU.hcontig_059_008.10', 'evm.TU.hcontig_059_008.101', 'evm.TU.hcontig_059_008.107', 'evm.TU.hcontig_059_008.111', 'evm.TU.hcontig_059_008.125', 'evm.TU.hcontig_059_008.19', 'evm.TU.hcontig_059_008.2', 'evm.TU.hcontig_059_008.37', 'evm.TU.hcontig_059_008.45', 'evm.TU.hcontig_059_008.54', 'evm.TU.hcontig_059_008.65', 'evm.TU.hcontig_059_008.7', 'evm.TU.hcontig_059_008.78', 'evm.TU.hcontig_060_001.2', 'evm.TU.hcontig_060_001.64', 'evm.TU.hcontig_060_001.65', 'evm.TU.hcontig_060_001.66', 'evm.TU.hcontig_060_001.67', 'evm.TU.hcontig_062_001.35', 'evm.TU.hcontig_062_001.46', 'evm.TU.hcontig_062_001.54', 'evm.TU.hcontig_062_001.62', 'evm.TU.hcontig_062_001.81', 'evm.TU.hcontig_064_007.1', 'evm.TU.hcontig_064_007.10', 'evm.TU.hcontig_064_007.5', 'evm.TU.hcontig_064_007.75', 'evm.TU.hcontig_064_007.9', 'evm.TU.hcontig_064_098.38', 'evm.TU.hcontig_065_003.39', 'evm.TU.hcontig_065_094.41', 'evm.TU.hcontig_066_004.8', 'evm.TU.hcontig_066_107.29', 'evm.TU.hcontig_067_004.24', 'evm.TU.hcontig_067_004.6', 'evm.TU.hcontig_067_005.8', 'evm.TU.hcontig_067_008.4', 'evm.TU.hcontig_067_008.6', 'evm.TU.hcontig_069_137.6', 'evm.TU.hcontig_070_006.2', 'evm.TU.hcontig_070_006.4', 'evm.TU.hcontig_070_009.13', 'evm.TU.hcontig_070_009.15', 'evm.TU.hcontig_077_002.9', 'evm.TU.hcontig_078_002.22', 'evm.TU.hcontig_078_002.8', 'evm.TU.hcontig_080_002.10', 'evm.TU.hcontig_080_002.13', 'evm.TU.hcontig_081_004.22', 'evm.TU.hcontig_081_005.11', 'evm.TU.hcontig_081_005.34', 'evm.TU.hcontig_081_005.36', 'evm.TU.hcontig_081_005.7', 'evm.TU.hcontig_082_001.6', 'evm.TU.hcontig_082_002.25', 'evm.TU.hcontig_083_002.40', 'evm.TU.hcontig_083_002.51', 'evm.TU.hcontig_083_002.52', 'evm.TU.hcontig_083_002.54', 'evm.TU.hcontig_084_003.7', 'evm.TU.hcontig_088_002.19', 'evm.TU.hcontig_088_002.29', 'evm.TU.hcontig_088_002.34', 'evm.TU.hcontig_089_005.2', 'evm.TU.hcontig_092_003.10', 'evm.TU.hcontig_095_004.15', 'evm.TU.hcontig_104_001.2', 'evm.TU.hcontig_147_001.2', 'evm.TU.hcontig_189_001.2', 'evm.TU.hcontig_225_001.1', 'evm.TU.pcontig_000.112', 'evm.TU.pcontig_000.19', 'evm.TU.pcontig_000.262', 'evm.TU.pcontig_000.276', 'evm.TU.pcontig_000.3', 'evm.TU.pcontig_000.306', 'evm.TU.pcontig_000.332', 'evm.TU.pcontig_000.378', 'evm.TU.pcontig_000.392', 'evm.TU.pcontig_000.406', 'evm.TU.pcontig_000.421', 'evm.TU.pcontig_000.423', 'evm.TU.pcontig_000.448', 'evm.TU.pcontig_000.450', 'evm.TU.pcontig_000.454', 'evm.TU.pcontig_000.479', 'evm.TU.pcontig_000.508', 'evm.TU.pcontig_000.554', 'evm.TU.pcontig_000.556', 'evm.TU.pcontig_000.558', 'evm.TU.pcontig_000.562', 'evm.TU.pcontig_000.583', 'evm.TU.pcontig_000.588', 'evm.TU.pcontig_000.593', 'evm.TU.pcontig_000.612', 'evm.TU.pcontig_000.624', 'evm.TU.pcontig_000.63', 'evm.TU.pcontig_000.679', 'evm.TU.pcontig_000.85', 'evm.TU.pcontig_000.9', 'evm.TU.pcontig_001.104', 'evm.TU.pcontig_001.169', 'evm.TU.pcontig_001.174', 'evm.TU.pcontig_001.188', 'evm.TU.pcontig_001.231', 'evm.TU.pcontig_001.252', 'evm.TU.pcontig_001.270', 'evm.TU.pcontig_001.274', 'evm.TU.pcontig_001.28', 'evm.TU.pcontig_001.288', 'evm.TU.pcontig_001.298', 'evm.TU.pcontig_001.3', 'evm.TU.pcontig_001.363', 'evm.TU.pcontig_001.365', 'evm.TU.pcontig_001.366', 'evm.TU.pcontig_001.367', 'evm.TU.pcontig_001.368', 'evm.TU.pcontig_001.371', 'evm.TU.pcontig_001.378', 'evm.TU.pcontig_001.379', 'evm.TU.pcontig_001.381', 'evm.TU.pcontig_001.4', 'evm.TU.pcontig_001.413', 'evm.TU.pcontig_001.463', 'evm.TU.pcontig_001.515', 'evm.TU.pcontig_001.518', 'evm.TU.pcontig_001.520', 'evm.TU.pcontig_001.568', 'evm.TU.pcontig_001.570', 'evm.TU.pcontig_001.583', 'evm.TU.pcontig_001.611', 'evm.TU.pcontig_001.613', 'evm.TU.pcontig_001.622', 'evm.TU.pcontig_001.632', 'evm.TU.pcontig_001.634', 'evm.TU.pcontig_001.635', 'evm.TU.pcontig_001.636', 'evm.TU.pcontig_001.638', 'evm.TU.pcontig_001.644', 'evm.TU.pcontig_001.657', 'evm.TU.pcontig_001.7', 'evm.TU.pcontig_001.83', 'evm.TU.pcontig_001.85', 'evm.TU.pcontig_002.108', 'evm.TU.pcontig_002.119', 'evm.TU.pcontig_002.131', 'evm.TU.pcontig_002.151', 'evm.TU.pcontig_002.202', 'evm.TU.pcontig_002.204', 'evm.TU.pcontig_002.223', 'evm.TU.pcontig_002.229', 'evm.TU.pcontig_002.243', 'evm.TU.pcontig_002.246', 'evm.TU.pcontig_002.28', 'evm.TU.pcontig_002.303', 'evm.TU.pcontig_002.315', 'evm.TU.pcontig_002.316', 'evm.TU.pcontig_002.332', 'evm.TU.pcontig_002.333', 'evm.TU.pcontig_002.34', 'evm.TU.pcontig_002.38', 'evm.TU.pcontig_002.40', 'evm.TU.pcontig_002.403', 'evm.TU.pcontig_002.404', 'evm.TU.pcontig_002.407', 'evm.TU.pcontig_002.413', 'evm.TU.pcontig_002.415', 'evm.TU.pcontig_002.42', 'evm.TU.pcontig_002.423', 'evm.TU.pcontig_002.426', 'evm.TU.pcontig_002.431', 'evm.TU.pcontig_002.440', 'evm.TU.pcontig_002.449', 'evm.TU.pcontig_002.453', 'evm.TU.pcontig_002.467', 'evm.TU.pcontig_002.529', 'evm.TU.pcontig_002.535', 'evm.TU.pcontig_002.536', 'evm.TU.pcontig_002.538', 'evm.TU.pcontig_002.553', 'evm.TU.pcontig_002.559', 'evm.TU.pcontig_002.562', 'evm.TU.pcontig_002.64', 'evm.TU.pcontig_002.92', 'evm.TU.pcontig_002.95', 'evm.TU.pcontig_003.140', 'evm.TU.pcontig_003.154', 'evm.TU.pcontig_003.162', 'evm.TU.pcontig_003.179', 'evm.TU.pcontig_003.208', 'evm.TU.pcontig_003.217', 'evm.TU.pcontig_003.218', 'evm.TU.pcontig_003.232', 'evm.TU.pcontig_003.236', 'evm.TU.pcontig_003.240', 'evm.TU.pcontig_003.243', 'evm.TU.pcontig_003.248', 'evm.TU.pcontig_003.249', 'evm.TU.pcontig_003.254', 'evm.TU.pcontig_003.255', 'evm.TU.pcontig_003.328', 'evm.TU.pcontig_003.333', 'evm.TU.pcontig_003.335', 'evm.TU.pcontig_003.336', 'evm.TU.pcontig_003.365', 'evm.TU.pcontig_003.366', 'evm.TU.pcontig_003.367', 'evm.TU.pcontig_003.387', 'evm.TU.pcontig_003.388', 'evm.TU.pcontig_003.402', 'evm.TU.pcontig_003.403', 'evm.TU.pcontig_003.427', 'evm.TU.pcontig_003.436', 'evm.TU.pcontig_003.471', 'evm.TU.pcontig_003.477', 'evm.TU.pcontig_003.479', 'evm.TU.pcontig_003.487', 'evm.TU.pcontig_003.505', 'evm.TU.pcontig_003.506', 'evm.TU.pcontig_003.507', 'evm.TU.pcontig_003.508', 'evm.TU.pcontig_003.521', 'evm.TU.pcontig_003.522', 'evm.TU.pcontig_003.523', 'evm.TU.pcontig_003.524', 'evm.TU.pcontig_003.525', 'evm.TU.pcontig_003.543', 'evm.TU.pcontig_003.544', 'evm.TU.pcontig_003.547', 'evm.TU.pcontig_003.87', 'evm.TU.pcontig_003.9', 'evm.TU.pcontig_004.124', 'evm.TU.pcontig_004.13', 'evm.TU.pcontig_004.142', 'evm.TU.pcontig_004.165', 'evm.TU.pcontig_004.168', 'evm.TU.pcontig_004.18', 'evm.TU.pcontig_004.19', 'evm.TU.pcontig_004.190', 'evm.TU.pcontig_004.191', 'evm.TU.pcontig_004.195', 'evm.TU.pcontig_004.214', 'evm.TU.pcontig_004.26', 'evm.TU.pcontig_004.27', 'evm.TU.pcontig_004.284', 'evm.TU.pcontig_004.304', 'evm.TU.pcontig_004.35', 'evm.TU.pcontig_004.372', 'evm.TU.pcontig_004.388', 'evm.TU.pcontig_004.39', 'evm.TU.pcontig_004.435', 'evm.TU.pcontig_004.437', 'evm.TU.pcontig_004.448', 'evm.TU.pcontig_004.53', 'evm.TU.pcontig_004.58', 'evm.TU.pcontig_004.73', 'evm.TU.pcontig_005.10', 'evm.TU.pcontig_005.126', 'evm.TU.pcontig_005.14', 'evm.TU.pcontig_005.186', 'evm.TU.pcontig_005.193', 'evm.TU.pcontig_005.242', 'evm.TU.pcontig_005.255', 'evm.TU.pcontig_005.258', 'evm.TU.pcontig_005.260', 'evm.TU.pcontig_005.289', 'evm.TU.pcontig_005.294', 'evm.TU.pcontig_005.301', 'evm.TU.pcontig_005.306', 'evm.TU.pcontig_005.33', 'evm.TU.pcontig_005.394', 'evm.TU.pcontig_005.395', 'evm.TU.pcontig_005.4', 'evm.TU.pcontig_005.418', 'evm.TU.pcontig_005.419', 'evm.TU.pcontig_005.439', 'evm.TU.pcontig_005.453', 'evm.TU.pcontig_005.461', 'evm.TU.pcontig_005.495', 'evm.TU.pcontig_005.51', 'evm.TU.pcontig_005.52', 'evm.TU.pcontig_005.6', 'evm.TU.pcontig_005.91', 'evm.TU.pcontig_005.95', 'evm.TU.pcontig_005.98', 'evm.TU.pcontig_006.108', 'evm.TU.pcontig_006.126', 'evm.TU.pcontig_006.13', 'evm.TU.pcontig_006.140', 'evm.TU.pcontig_006.142', 'evm.TU.pcontig_006.15', 'evm.TU.pcontig_006.199', 'evm.TU.pcontig_006.2', 'evm.TU.pcontig_006.278', 'evm.TU.pcontig_006.290', 'evm.TU.pcontig_006.293', 'evm.TU.pcontig_006.329', 'evm.TU.pcontig_006.35', 'evm.TU.pcontig_006.361', 'evm.TU.pcontig_006.391', 'evm.TU.pcontig_006.404', 'evm.TU.pcontig_006.433', 'evm.TU.pcontig_006.461', 'evm.TU.pcontig_006.472', 'evm.TU.pcontig_006.507', 'evm.TU.pcontig_006.90', 'evm.TU.pcontig_007.10', 'evm.TU.pcontig_007.113', 'evm.TU.pcontig_007.141', 'evm.TU.pcontig_007.151', 'evm.TU.pcontig_007.154', 'evm.TU.pcontig_007.175', 'evm.TU.pcontig_007.180', 'evm.TU.pcontig_007.223', 'evm.TU.pcontig_007.226', 'evm.TU.pcontig_007.23', 'evm.TU.pcontig_007.308', 'evm.TU.pcontig_007.309', 'evm.TU.pcontig_007.390', 'evm.TU.pcontig_007.405', 'evm.TU.pcontig_007.411', 'evm.TU.pcontig_007.412', 'evm.TU.pcontig_007.418', 'evm.TU.pcontig_007.421', 'evm.TU.pcontig_007.422', 'evm.TU.pcontig_007.424', 'evm.TU.pcontig_007.8', 'evm.TU.pcontig_007.81', 'evm.TU.pcontig_007.84', 'evm.TU.pcontig_008.133', 'evm.TU.pcontig_008.137', 'evm.TU.pcontig_008.159', 'evm.TU.pcontig_008.171', 'evm.TU.pcontig_008.209', 'evm.TU.pcontig_008.257', 'evm.TU.pcontig_008.30', 'evm.TU.pcontig_008.322', 'evm.TU.pcontig_008.343', 'evm.TU.pcontig_008.362', 'evm.TU.pcontig_008.86', 'evm.TU.pcontig_009.12', 'evm.TU.pcontig_009.126', 'evm.TU.pcontig_009.136', 'evm.TU.pcontig_009.138', 'evm.TU.pcontig_009.140', 'evm.TU.pcontig_009.141', 'evm.TU.pcontig_009.145', 'evm.TU.pcontig_009.158', 'evm.TU.pcontig_009.162', 'evm.TU.pcontig_009.163', 'evm.TU.pcontig_009.164', 'evm.TU.pcontig_009.181', 'evm.TU.pcontig_009.2', 'evm.TU.pcontig_009.21', 'evm.TU.pcontig_009.218', 'evm.TU.pcontig_009.222', 'evm.TU.pcontig_009.232', 'evm.TU.pcontig_009.236', 'evm.TU.pcontig_010.131', 'evm.TU.pcontig_010.132', 'evm.TU.pcontig_010.168', 'evm.TU.pcontig_010.21', 'evm.TU.pcontig_010.217', 'evm.TU.pcontig_010.219', 'evm.TU.pcontig_010.220', 'evm.TU.pcontig_010.231', 'evm.TU.pcontig_010.234', 'evm.TU.pcontig_010.237', 'evm.TU.pcontig_010.239', 'evm.TU.pcontig_010.240', 'evm.TU.pcontig_010.25', 'evm.TU.pcontig_010.250', 'evm.TU.pcontig_010.253', 'evm.TU.pcontig_010.254', 'evm.TU.pcontig_010.255', 'evm.TU.pcontig_010.258', 'evm.TU.pcontig_010.270', 'evm.TU.pcontig_010.271', 'evm.TU.pcontig_010.281', 'evm.TU.pcontig_010.282', 'evm.TU.pcontig_010.295', 'evm.TU.pcontig_010.304', 'evm.TU.pcontig_010.309', 'evm.TU.pcontig_010.331', 'evm.TU.pcontig_010.332', 'evm.TU.pcontig_010.345', 'evm.TU.pcontig_010.351', 'evm.TU.pcontig_010.352', 'evm.TU.pcontig_010.353', 'evm.TU.pcontig_010.376', 'evm.TU.pcontig_010.65', 'evm.TU.pcontig_010.73', 'evm.TU.pcontig_010.74', 'evm.TU.pcontig_010.89', 'evm.TU.pcontig_011.14', 'evm.TU.pcontig_011.142', 'evm.TU.pcontig_011.143', 'evm.TU.pcontig_011.147', 'evm.TU.pcontig_011.148', 'evm.TU.pcontig_011.153', 'evm.TU.pcontig_011.209', 'evm.TU.pcontig_011.210', 'evm.TU.pcontig_011.238', 'evm.TU.pcontig_011.241', 'evm.TU.pcontig_011.242', 'evm.TU.pcontig_011.263', 'evm.TU.pcontig_011.269', 'evm.TU.pcontig_011.292', 'evm.TU.pcontig_011.326', 'evm.TU.pcontig_011.347', 'evm.TU.pcontig_011.364', 'evm.TU.pcontig_011.71', 'evm.TU.pcontig_012.101', 'evm.TU.pcontig_012.122', 'evm.TU.pcontig_012.136', 'evm.TU.pcontig_012.138', 'evm.TU.pcontig_012.175', 'evm.TU.pcontig_012.176', 'evm.TU.pcontig_012.177', 'evm.TU.pcontig_012.233', 'evm.TU.pcontig_012.238', 'evm.TU.pcontig_012.246', 'evm.TU.pcontig_012.266', 'evm.TU.pcontig_012.280', 'evm.TU.pcontig_012.281', 'evm.TU.pcontig_012.293', 'evm.TU.pcontig_012.296', 'evm.TU.pcontig_012.366', 'evm.TU.pcontig_012.4', 'evm.TU.pcontig_012.52', 'evm.TU.pcontig_012.58', 'evm.TU.pcontig_012.61', 'evm.TU.pcontig_012.63', 'evm.TU.pcontig_012.85', 'evm.TU.pcontig_013.124', 'evm.TU.pcontig_013.126', 'evm.TU.pcontig_013.139', 'evm.TU.pcontig_013.140', 'evm.TU.pcontig_013.152', 'evm.TU.pcontig_013.156', 'evm.TU.pcontig_013.157', 'evm.TU.pcontig_013.162', 'evm.TU.pcontig_013.164', 'evm.TU.pcontig_013.166', 'evm.TU.pcontig_013.210', 'evm.TU.pcontig_013.215', 'evm.TU.pcontig_013.216', 'evm.TU.pcontig_013.238', 'evm.TU.pcontig_013.239', 'evm.TU.pcontig_013.247', 'evm.TU.pcontig_013.279', 'evm.TU.pcontig_013.282', 'evm.TU.pcontig_013.289', 'evm.TU.pcontig_013.290', 'evm.TU.pcontig_013.311', 'evm.TU.pcontig_013.318', 'evm.TU.pcontig_013.49', 'evm.TU.pcontig_014.101', 'evm.TU.pcontig_014.113', 'evm.TU.pcontig_014.115', 'evm.TU.pcontig_014.124', 'evm.TU.pcontig_014.128', 'evm.TU.pcontig_014.131', 'evm.TU.pcontig_014.138', 'evm.TU.pcontig_014.142', 'evm.TU.pcontig_014.193', 'evm.TU.pcontig_014.22', 'evm.TU.pcontig_014.279', 'evm.TU.pcontig_014.285', 'evm.TU.pcontig_014.295', 'evm.TU.pcontig_014.66', 'evm.TU.pcontig_014.85', 'evm.TU.pcontig_015.10', 'evm.TU.pcontig_016.119', 'evm.TU.pcontig_016.150', 'evm.TU.pcontig_016.157', 'evm.TU.pcontig_016.164', 'evm.TU.pcontig_016.165', 'evm.TU.pcontig_016.175', 'evm.TU.pcontig_016.189', 'evm.TU.pcontig_016.196', 'evm.TU.pcontig_016.198', 'evm.TU.pcontig_016.64', 'evm.TU.pcontig_016.91', 'evm.TU.pcontig_017.126', 'evm.TU.pcontig_017.150', 'evm.TU.pcontig_017.151', 'evm.TU.pcontig_017.152', 'evm.TU.pcontig_017.154', 'evm.TU.pcontig_017.184', 'evm.TU.pcontig_017.196', 'evm.TU.pcontig_017.2', 'evm.TU.pcontig_017.200', 'evm.TU.pcontig_017.202', 'evm.TU.pcontig_017.224', 'evm.TU.pcontig_017.225', 'evm.TU.pcontig_017.27', 'evm.TU.pcontig_017.3', 'evm.TU.pcontig_017.322', 'evm.TU.pcontig_017.7', 'evm.TU.pcontig_017.78', 'evm.TU.pcontig_017.82', 'evm.TU.pcontig_017.84', 'evm.TU.pcontig_017.85', 'evm.TU.pcontig_017.9', 'evm.TU.pcontig_018.118', 'evm.TU.pcontig_018.127', 'evm.TU.pcontig_018.154', 'evm.TU.pcontig_018.160', 'evm.TU.pcontig_018.163', 'evm.TU.pcontig_018.164', 'evm.TU.pcontig_018.187', 'evm.TU.pcontig_018.197', 'evm.TU.pcontig_018.218', 'evm.TU.pcontig_018.270', 'evm.TU.pcontig_018.273', 'evm.TU.pcontig_018.280', 'evm.TU.pcontig_018.281', 'evm.TU.pcontig_018.283', 'evm.TU.pcontig_018.286', 'evm.TU.pcontig_018.287', 'evm.TU.pcontig_018.292', 'evm.TU.pcontig_018.303', 'evm.TU.pcontig_018.338', 'evm.TU.pcontig_018.353', 'evm.TU.pcontig_018.5', 'evm.TU.pcontig_018.6', 'evm.TU.pcontig_018.82', 'evm.TU.pcontig_019.159', 'evm.TU.pcontig_019.167', 'evm.TU.pcontig_019.168', 'evm.TU.pcontig_019.179', 'evm.TU.pcontig_019.197', 'evm.TU.pcontig_019.202', 'evm.TU.pcontig_019.231', 'evm.TU.pcontig_019.233', 'evm.TU.pcontig_019.238', 'evm.TU.pcontig_019.245', 'evm.TU.pcontig_019.25', 'evm.TU.pcontig_019.295', 'evm.TU.pcontig_019.30', 'evm.TU.pcontig_019.31', 'evm.TU.pcontig_019.310', 'evm.TU.pcontig_019.47', 'evm.TU.pcontig_019.49', 'evm.TU.pcontig_019.50', 'evm.TU.pcontig_019.57', 'evm.TU.pcontig_020.163', 'evm.TU.pcontig_020.164', 'evm.TU.pcontig_020.182', 'evm.TU.pcontig_020.183', 'evm.TU.pcontig_020.216', 'evm.TU.pcontig_020.303', 'evm.TU.pcontig_020.329', 'evm.TU.pcontig_020.39', 'evm.TU.pcontig_020.47', 'evm.TU.pcontig_020.63', 'evm.TU.pcontig_020.67', 'evm.TU.pcontig_020.79', 'evm.TU.pcontig_020.91', 'evm.TU.pcontig_021.108', 'evm.TU.pcontig_021.117', 'evm.TU.pcontig_021.128', 'evm.TU.pcontig_021.154', 'evm.TU.pcontig_021.179', 'evm.TU.pcontig_021.181', 'evm.TU.pcontig_021.197', 'evm.TU.pcontig_021.198', 'evm.TU.pcontig_021.218', 'evm.TU.pcontig_021.222', 'evm.TU.pcontig_021.253', 'evm.TU.pcontig_021.259', 'evm.TU.pcontig_021.273', 'evm.TU.pcontig_021.282', 'evm.TU.pcontig_021.288', 'evm.TU.pcontig_021.297', 'evm.TU.pcontig_021.320', 'evm.TU.pcontig_021.323', 'evm.TU.pcontig_021.341', 'evm.TU.pcontig_021.47', 'evm.TU.pcontig_021.63', 'evm.TU.pcontig_021.71', 'evm.TU.pcontig_021.78', 'evm.TU.pcontig_021.93', 'evm.TU.pcontig_021.94', 'evm.TU.pcontig_022.106', 'evm.TU.pcontig_022.114', 'evm.TU.pcontig_022.120', 'evm.TU.pcontig_022.129', 'evm.TU.pcontig_022.143', 'evm.TU.pcontig_022.153', 'evm.TU.pcontig_022.160', 'evm.TU.pcontig_022.163', 'evm.TU.pcontig_022.189', 'evm.TU.pcontig_022.194', 'evm.TU.pcontig_022.198', 'evm.TU.pcontig_022.20', 'evm.TU.pcontig_022.218', 'evm.TU.pcontig_022.222', 'evm.TU.pcontig_022.227', 'evm.TU.pcontig_022.238', 'evm.TU.pcontig_022.245', 'evm.TU.pcontig_022.261', 'evm.TU.pcontig_022.281', 'evm.TU.pcontig_022.76', 'evm.TU.pcontig_022.81', 'evm.TU.pcontig_023.10', 'evm.TU.pcontig_023.105', 'evm.TU.pcontig_023.118', 'evm.TU.pcontig_023.133', 'evm.TU.pcontig_023.158', 'evm.TU.pcontig_023.175', 'evm.TU.pcontig_023.182', 'evm.TU.pcontig_023.192', 'evm.TU.pcontig_023.212', 'evm.TU.pcontig_023.223', 'evm.TU.pcontig_023.234', 'evm.TU.pcontig_023.265', 'evm.TU.pcontig_023.5', 'evm.TU.pcontig_023.53', 'evm.TU.pcontig_023.6', 'evm.TU.pcontig_023.73', 'evm.TU.pcontig_023.88', 'evm.TU.pcontig_023.99', 'evm.TU.pcontig_024.10', 'evm.TU.pcontig_025.10', 'evm.TU.pcontig_025.121', 'evm.TU.pcontig_025.161', 'evm.TU.pcontig_025.165', 'evm.TU.pcontig_025.33', 'evm.TU.pcontig_025.75', 'evm.TU.pcontig_025.92', 'evm.TU.pcontig_026.127', 'evm.TU.pcontig_026.128', 'evm.TU.pcontig_026.142', 'evm.TU.pcontig_026.143', 'evm.TU.pcontig_026.144', 'evm.TU.pcontig_026.251', 'evm.TU.pcontig_026.262', 'evm.TU.pcontig_026.273', 'evm.TU.pcontig_026.274', 'evm.TU.pcontig_026.56', 'evm.TU.pcontig_026.58', 'evm.TU.pcontig_026.59', 'evm.TU.pcontig_026.70', 'evm.TU.pcontig_026.73', 'evm.TU.pcontig_026.74', 'evm.TU.pcontig_026.76', 'evm.TU.pcontig_027.10', 'evm.TU.pcontig_027.11', 'evm.TU.pcontig_027.132', 'evm.TU.pcontig_027.163', 'evm.TU.pcontig_027.164', 'evm.TU.pcontig_027.202', 'evm.TU.pcontig_027.205', 'evm.TU.pcontig_027.211', 'evm.TU.pcontig_027.212', 'evm.TU.pcontig_027.222', 'evm.TU.pcontig_027.248', 'evm.TU.pcontig_027.249', 'evm.TU.pcontig_027.256', 'evm.TU.pcontig_027.265', 'evm.TU.pcontig_027.41', 'evm.TU.pcontig_027.51', 'evm.TU.pcontig_027.52', 'evm.TU.pcontig_027.9', 'evm.TU.pcontig_028.113', 'evm.TU.pcontig_028.131', 'evm.TU.pcontig_028.137', 'evm.TU.pcontig_028.214', 'evm.TU.pcontig_028.215', 'evm.TU.pcontig_028.216', 'evm.TU.pcontig_028.221', 'evm.TU.pcontig_028.71', 'evm.TU.pcontig_028.9', 'evm.TU.pcontig_028.94', 'evm.TU.pcontig_029.107', 'evm.TU.pcontig_029.108', 'evm.TU.pcontig_029.113', 'evm.TU.pcontig_029.120', 'evm.TU.pcontig_029.124', 'evm.TU.pcontig_029.139', 'evm.TU.pcontig_029.140', 'evm.TU.pcontig_029.205', 'evm.TU.pcontig_029.231', 'evm.TU.pcontig_029.34', 'evm.TU.pcontig_029.38', 'evm.TU.pcontig_029.62', 'evm.TU.pcontig_029.63', 'evm.TU.pcontig_029.90', 'evm.TU.pcontig_029.92', 'evm.TU.pcontig_029.97', 'evm.TU.pcontig_030.121', 'evm.TU.pcontig_030.175', 'evm.TU.pcontig_030.181', 'evm.TU.pcontig_030.182', 'evm.TU.pcontig_030.240', 'evm.TU.pcontig_030.256', 'evm.TU.pcontig_030.276', 'evm.TU.pcontig_030.34', 'evm.TU.pcontig_030.54', 'evm.TU.pcontig_030.55', 'evm.TU.pcontig_030.66', 'evm.TU.pcontig_030.69', 'evm.TU.pcontig_031.110', 'evm.TU.pcontig_031.113', 'evm.TU.pcontig_031.122', 'evm.TU.pcontig_031.123', 'evm.TU.pcontig_031.159', 'evm.TU.pcontig_031.164', 'evm.TU.pcontig_031.165', 'evm.TU.pcontig_031.167', 'evm.TU.pcontig_031.169', 'evm.TU.pcontig_031.170', 'evm.TU.pcontig_031.174', 'evm.TU.pcontig_031.184', 'evm.TU.pcontig_031.187', 'evm.TU.pcontig_031.189', 'evm.TU.pcontig_031.29', 'evm.TU.pcontig_031.60', 'evm.TU.pcontig_031.79', 'evm.TU.pcontig_031.81', 'evm.TU.pcontig_031.83', 'evm.TU.pcontig_031.88', 'evm.TU.pcontig_031.89', 'evm.TU.pcontig_031.96', 'evm.TU.pcontig_032.19', 'evm.TU.pcontig_033.143', 'evm.TU.pcontig_033.163', 'evm.TU.pcontig_033.179', 'evm.TU.pcontig_033.216', 'evm.TU.pcontig_033.221', 'evm.TU.pcontig_033.229', 'evm.TU.pcontig_033.240', 'evm.TU.pcontig_033.73', 'evm.TU.pcontig_034.113', 'evm.TU.pcontig_034.138', 'evm.TU.pcontig_034.143', 'evm.TU.pcontig_034.155', 'evm.TU.pcontig_034.169', 'evm.TU.pcontig_034.171', 'evm.TU.pcontig_034.173', 'evm.TU.pcontig_034.191', 'evm.TU.pcontig_034.193', 'evm.TU.pcontig_034.194', 'evm.TU.pcontig_034.199', 'evm.TU.pcontig_034.218', 'evm.TU.pcontig_034.227', 'evm.TU.pcontig_034.228', 'evm.TU.pcontig_034.51', 'evm.TU.pcontig_034.86', 'evm.TU.pcontig_034.89', 'evm.TU.pcontig_034.95', 'evm.TU.pcontig_035.163', 'evm.TU.pcontig_035.165', 'evm.TU.pcontig_035.166', 'evm.TU.pcontig_035.170', 'evm.TU.pcontig_035.174', 'evm.TU.pcontig_035.179', 'evm.TU.pcontig_035.191', 'evm.TU.pcontig_035.202', 'evm.TU.pcontig_035.203', 'evm.TU.pcontig_035.208', 'evm.TU.pcontig_035.216', 'evm.TU.pcontig_035.27', 'evm.TU.pcontig_035.46', 'evm.TU.pcontig_035.5', 'evm.TU.pcontig_035.50', 'evm.TU.pcontig_035.62', 'evm.TU.pcontig_035.63', 'evm.TU.pcontig_035.66', 'evm.TU.pcontig_035.67', 'evm.TU.pcontig_035.68', 'evm.TU.pcontig_035.69', 'evm.TU.pcontig_035.70', 'evm.TU.pcontig_035.80', 'evm.TU.pcontig_035.81', 'evm.TU.pcontig_035.83', 'evm.TU.pcontig_035.91', 'evm.TU.pcontig_036.154', 'evm.TU.pcontig_036.164', 'evm.TU.pcontig_036.173', 'evm.TU.pcontig_036.221', 'evm.TU.pcontig_036.29', 'evm.TU.pcontig_036.30', 'evm.TU.pcontig_036.31', 'evm.TU.pcontig_036.32', 'evm.TU.pcontig_036.43', 'evm.TU.pcontig_036.44', 'evm.TU.pcontig_036.46', 'evm.TU.pcontig_036.56', 'evm.TU.pcontig_036.57', 'evm.TU.pcontig_036.58', 'evm.TU.pcontig_036.73', 'evm.TU.pcontig_036.75', 'evm.TU.pcontig_036.81', 'evm.TU.pcontig_037.107', 'evm.TU.pcontig_037.109', 'evm.TU.pcontig_037.110', 'evm.TU.pcontig_037.111', 'evm.TU.pcontig_037.112', 'evm.TU.pcontig_037.137', 'evm.TU.pcontig_037.187', 'evm.TU.pcontig_037.200', 'evm.TU.pcontig_037.203', 'evm.TU.pcontig_037.206', 'evm.TU.pcontig_037.208', 'evm.TU.pcontig_037.215', 'evm.TU.pcontig_037.27', 'evm.TU.pcontig_037.43', 'evm.TU.pcontig_037.63', 'evm.TU.pcontig_037.70', 'evm.TU.pcontig_039.123', 'evm.TU.pcontig_039.169', 'evm.TU.pcontig_039.174', 'evm.TU.pcontig_039.175', 'evm.TU.pcontig_039.177', 'evm.TU.pcontig_039.181', 'evm.TU.pcontig_039.184', 'evm.TU.pcontig_039.28', 'evm.TU.pcontig_039.36', 'evm.TU.pcontig_039.37', 'evm.TU.pcontig_039.87', 'evm.TU.pcontig_039.97', 'evm.TU.pcontig_040.105', 'evm.TU.pcontig_040.135', 'evm.TU.pcontig_040.143', 'evm.TU.pcontig_040.148', 'evm.TU.pcontig_040.159', 'evm.TU.pcontig_040.165', 'evm.TU.pcontig_040.168', 'evm.TU.pcontig_040.176', 'evm.TU.pcontig_040.19', 'evm.TU.pcontig_040.45', 'evm.TU.pcontig_040.81', 'evm.TU.pcontig_041.104', 'evm.TU.pcontig_041.120', 'evm.TU.pcontig_041.125', 'evm.TU.pcontig_041.14', 'evm.TU.pcontig_041.147', 'evm.TU.pcontig_041.148', 'evm.TU.pcontig_041.15', 'evm.TU.pcontig_041.173', 'evm.TU.pcontig_041.174', 'evm.TU.pcontig_041.2', 'evm.TU.pcontig_041.20', 'evm.TU.pcontig_041.33', 'evm.TU.pcontig_041.38', 'evm.TU.pcontig_041.46', 'evm.TU.pcontig_041.48', 'evm.TU.pcontig_041.56', 'evm.TU.pcontig_041.98', 'evm.TU.pcontig_041.99', 'evm.TU.pcontig_042.129', 'evm.TU.pcontig_042.130', 'evm.TU.pcontig_042.179', 'evm.TU.pcontig_042.188', 'evm.TU.pcontig_042.31', 'evm.TU.pcontig_042.4', 'evm.TU.pcontig_042.66', 'evm.TU.pcontig_043.100', 'evm.TU.pcontig_043.101', 'evm.TU.pcontig_043.127', 'evm.TU.pcontig_043.14', 'evm.TU.pcontig_043.15', 'evm.TU.pcontig_043.29', 'evm.TU.pcontig_043.31', 'evm.TU.pcontig_043.63', 'evm.TU.pcontig_043.89', 'evm.TU.pcontig_044.100', 'evm.TU.pcontig_044.13', 'evm.TU.pcontig_044.130', 'evm.TU.pcontig_044.143', 'evm.TU.pcontig_044.165', 'evm.TU.pcontig_044.167', 'evm.TU.pcontig_044.168', 'evm.TU.pcontig_044.171', 'evm.TU.pcontig_044.4', 'evm.TU.pcontig_044.40', 'evm.TU.pcontig_044.64', 'evm.TU.pcontig_044.74', 'evm.TU.pcontig_044.92', 'evm.TU.pcontig_044.93', 'evm.TU.pcontig_044.99', 'evm.TU.pcontig_045.135', 'evm.TU.pcontig_045.140', 'evm.TU.pcontig_045.144', 'evm.TU.pcontig_045.148', 'evm.TU.pcontig_045.45', 'evm.TU.pcontig_045.51', 'evm.TU.pcontig_045.68', 'evm.TU.pcontig_046.113', 'evm.TU.pcontig_046.124', 'evm.TU.pcontig_046.127', 'evm.TU.pcontig_046.131', 'evm.TU.pcontig_046.159', 'evm.TU.pcontig_046.161', 'evm.TU.pcontig_046.162', 'evm.TU.pcontig_046.3', 'evm.TU.pcontig_046.35', 'evm.TU.pcontig_046.94', 'evm.TU.pcontig_046.99', 'evm.TU.pcontig_047.114', 'evm.TU.pcontig_047.141', 'evm.TU.pcontig_047.54', 'evm.TU.pcontig_047.55', 'evm.TU.pcontig_047.89', 'evm.TU.pcontig_048.109', 'evm.TU.pcontig_048.111', 'evm.TU.pcontig_048.112', 'evm.TU.pcontig_048.65', 'evm.TU.pcontig_049.130', 'evm.TU.pcontig_049.133', 'evm.TU.pcontig_049.139', 'evm.TU.pcontig_049.150', 'evm.TU.pcontig_049.38', 'evm.TU.pcontig_049.4', 'evm.TU.pcontig_049.42', 'evm.TU.pcontig_049.58', 'evm.TU.pcontig_049.69', 'evm.TU.pcontig_049.75', 'evm.TU.pcontig_049.79', 'evm.TU.pcontig_050.102', 'evm.TU.pcontig_050.122', 'evm.TU.pcontig_050.134', 'evm.TU.pcontig_050.19', 'evm.TU.pcontig_050.3', 'evm.TU.pcontig_050.69', 'evm.TU.pcontig_050.77', 'evm.TU.pcontig_050.92', 'evm.TU.pcontig_051.48', 'evm.TU.pcontig_051.51', 'evm.TU.pcontig_052.152', 'evm.TU.pcontig_052.170', 'evm.TU.pcontig_052.54', 'evm.TU.pcontig_052.57', 'evm.TU.pcontig_052.58', 'evm.TU.pcontig_052.8', 'evm.TU.pcontig_054.116', 'evm.TU.pcontig_054.119', 'evm.TU.pcontig_054.52', 'evm.TU.pcontig_054.58', 'evm.TU.pcontig_054.59', 'evm.TU.pcontig_054.66', 'evm.TU.pcontig_054.88', 'evm.TU.pcontig_054.89', 'evm.TU.pcontig_054.91', 'evm.TU.pcontig_055.16', 'evm.TU.pcontig_055.17', 'evm.TU.pcontig_055.28', 'evm.TU.pcontig_055.31', 'evm.TU.pcontig_055.8', 'evm.TU.pcontig_056.33', 'evm.TU.pcontig_056.42', 'evm.TU.pcontig_056.46', 'evm.TU.pcontig_056.58', 'evm.TU.pcontig_056.59', 'evm.TU.pcontig_056.82', 'evm.TU.pcontig_056.83', 'evm.TU.pcontig_057.106', 'evm.TU.pcontig_057.107', 'evm.TU.pcontig_057.120', 'evm.TU.pcontig_057.51', 'evm.TU.pcontig_057.61', 'evm.TU.pcontig_057.65', 'evm.TU.pcontig_057.66', 'evm.TU.pcontig_057.73', 'evm.TU.pcontig_057.76', 'evm.TU.pcontig_057.77', 'evm.TU.pcontig_057.82', 'evm.TU.pcontig_058.21', 'evm.TU.pcontig_058.22', 'evm.TU.pcontig_058.23', 'evm.TU.pcontig_058.64', 'evm.TU.pcontig_058.72', 'evm.TU.pcontig_059.100', 'evm.TU.pcontig_059.118', 'evm.TU.pcontig_059.123', 'evm.TU.pcontig_059.13', 'evm.TU.pcontig_059.17', 'evm.TU.pcontig_059.21', 'evm.TU.pcontig_059.33', 'evm.TU.pcontig_059.50', 'evm.TU.pcontig_059.58', 'evm.TU.pcontig_059.65', 'evm.TU.pcontig_059.67', 'evm.TU.pcontig_059.78', 'evm.TU.pcontig_059.79', 'evm.TU.pcontig_059.93', 'evm.TU.pcontig_060.10', 'evm.TU.pcontig_060.11', 'evm.TU.pcontig_060.14', 'evm.TU.pcontig_060.37', 'evm.TU.pcontig_060.94', 'evm.TU.pcontig_060.95', 'evm.TU.pcontig_060.96', 'evm.TU.pcontig_060.97', 'evm.TU.pcontig_062.10', 'evm.TU.pcontig_062.104', 'evm.TU.pcontig_062.15', 'evm.TU.pcontig_062.23', 'evm.TU.pcontig_062.26', 'evm.TU.pcontig_062.32', 'evm.TU.pcontig_062.45', 'evm.TU.pcontig_062.50', 'evm.TU.pcontig_062.61', 'evm.TU.pcontig_062.72', 'evm.TU.pcontig_062.78', 'evm.TU.pcontig_062.79', 'evm.TU.pcontig_062.84', 'evm.TU.pcontig_064.39', 'evm.TU.pcontig_064.41', 'evm.TU.pcontig_064.42', 'evm.TU.pcontig_065.36', 'evm.TU.pcontig_065.77', 'evm.TU.pcontig_066.1', 'evm.TU.pcontig_066.15', 'evm.TU.pcontig_066.53', 'evm.TU.pcontig_066.71', 'evm.TU.pcontig_066.73', 'evm.TU.pcontig_066.98', 'evm.TU.pcontig_067.100', 'evm.TU.pcontig_067.20', 'evm.TU.pcontig_067.39', 'evm.TU.pcontig_067.43', 'evm.TU.pcontig_067.45', 'evm.TU.pcontig_067.57', 'evm.TU.pcontig_067.93', 'evm.TU.pcontig_068.5', 'evm.TU.pcontig_068.8', 'evm.TU.pcontig_069.3', 'evm.TU.pcontig_069.38', 'evm.TU.pcontig_070.1', 'evm.TU.pcontig_070.20', 'evm.TU.pcontig_070.22', 'evm.TU.pcontig_070.24', 'evm.TU.pcontig_070.71', 'evm.TU.pcontig_070.73', 'evm.TU.pcontig_072.78', 'evm.TU.pcontig_073.14', 'evm.TU.pcontig_073.15', 'evm.TU.pcontig_073.4', 'evm.TU.pcontig_073.45', 'evm.TU.pcontig_073.54', 'evm.TU.pcontig_073.55', 'evm.TU.pcontig_073.8', 'evm.TU.pcontig_074.20', 'evm.TU.pcontig_075.10', 'evm.TU.pcontig_077.1', 'evm.TU.pcontig_077.13', 'evm.TU.pcontig_077.44', 'evm.TU.pcontig_077.55', 'evm.TU.pcontig_078.1', 'evm.TU.pcontig_078.2', 'evm.TU.pcontig_078.33', 'evm.TU.pcontig_078.34', 'evm.TU.pcontig_078.38', 'evm.TU.pcontig_078.45', 'evm.TU.pcontig_078.48', 'evm.TU.pcontig_078.60', 'evm.TU.pcontig_078.66', 'evm.TU.pcontig_078.67', 'evm.TU.pcontig_078.71', 'evm.TU.pcontig_080.23', 'evm.TU.pcontig_080.26', 'evm.TU.pcontig_080.29', 'evm.TU.pcontig_080.30', 'evm.TU.pcontig_080.46', 'evm.TU.pcontig_080.57', 'evm.TU.pcontig_080.58', 'evm.TU.pcontig_081.24', 'evm.TU.pcontig_081.30', 'evm.TU.pcontig_081.52', 'evm.TU.pcontig_081.53', 'evm.TU.pcontig_081.60', 'evm.TU.pcontig_081.62', 'evm.TU.pcontig_082.54', 'evm.TU.pcontig_083.37', 'evm.TU.pcontig_083.49', 'evm.TU.pcontig_084.9', 'evm.TU.pcontig_086.22', 'evm.TU.pcontig_086.52', 'evm.TU.pcontig_087.16', 'evm.TU.pcontig_087.6', 'evm.TU.pcontig_087.7', 'evm.TU.pcontig_088.33', 'evm.TU.pcontig_088.49', 'evm.TU.pcontig_088.55', 'evm.TU.pcontig_089.21', 'evm.TU.pcontig_089.47', 'evm.TU.pcontig_090.13', 'evm.TU.pcontig_091.46', 'evm.TU.pcontig_091.47', 'evm.TU.pcontig_091.55', 'evm.TU.pcontig_092.34', 'evm.TU.pcontig_092.7', 'evm.TU.pcontig_092.8', 'evm.TU.pcontig_095.44', 'evm.TU.pcontig_097.21', 'evm.TU.pcontig_100.29', 'evm.TU.pcontig_100.35', 'evm.TU.pcontig_100.42', 'evm.TU.pcontig_103.12', 'evm.TU.pcontig_103.22', 'evm.TU.pcontig_103.24', 'evm.TU.pcontig_103.6', 'evm.TU.pcontig_103.8', 'evm.TU.pcontig_104.33', 'evm.TU.pcontig_104.47', 'evm.TU.pcontig_112.1', 'evm.TU.pcontig_115.25', 'evm.TU.pcontig_120.23', 'evm.TU.pcontig_130.20', 'evm.TU.pcontig_130.9', 'evm.TU.pcontig_147.9', 'evm.TU.pcontig_148.11', 'evm.TU.pcontig_148.9', 'evm.TU.pcontig_149.10', 'evm.TU.pcontig_157.3', 'evm.TU.pcontig_157.4', 'evm.TU.pcontig_157.8', 'evm.TU.pcontig_165.8', 'evm.TU.pcontig_170.3', 'evm.TU.pcontig_173.4', 'evm.TU.pcontig_173.5', 'evm.TU.pcontig_174.1', 'evm.TU.pcontig_181.9', 'evm.TU.pcontig_186.2', 'evm.TU.pcontig_186.6', 'evm.TU.pcontig_186.7', 'evm.TU.pcontig_186.9', 'evm.TU.pcontig_188.5', 'evm.TU.pcontig_188.6', 'evm.TU.pcontig_189.7', 'evm.TU.pcontig_203.5', 'evm.TU.pcontig_203.6', 'evm.TU.pcontig_207.1', 'evm.TU.pcontig_233.6', 'evm.TU.pcontig_235.7']\n"
     ]
    }
   ],
   "source": [
    "# Sort effector list\n",
    "effectors.sort()\n",
    "print(effectors) # Check if sorting worked (it did)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 276,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Run the function to make a file of effectors\n",
    "genes_only_fn = '/home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.anno.sorted.genes_only.gff3'\n",
    "effector_fn  = '/home/anjuni/analysis/input_for_windows/Pst_104E_v13_ph_ctg.effectors.gff3'\n",
    "make_effector_gff(effectors, genes_only_fn, effector_fn)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 846,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a file of genes only\n",
    "!cp /home/anjuni/analysis/gff_output/Pst_104E_v13_ph_ctg.anno.sorted.genes_only.gff3 /home/anjuni/analysis/coverage/feature_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Move the effector file and all gene annotation files to the feature file directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 865,
   "metadata": {},
   "outputs": [],
   "source": [
    "# set filepaths for feature inputs\n",
    "DIRS['FEATURES'] = os.path.join(DIRS['BASE2'], 'coverage', 'feature_files')\n",
    "DIRS['RAND'] = os.path.join(DIRS['BASE2'], 'coverage', 'randomisation')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 849,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "# Make a file of non-effector genes only\n",
    "cd /home/anjuni/analysis/coverage/feature_files\n",
    "subtractBed -a Pst_104E_v13_ph_ctg.anno.sorted.genes_only.gff3 -b Pst_104E_v13_ph_ctg.effectors.gff3 > Pst_104E_v13_ph_ctg.non_effectors.gff3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 851,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "# Make a file of exons only and non-genes\n",
    "cd /home/anjuni/analysis/coverage/feature_files\n",
    "grep 'exon' Pst_104E_v13_ph_ctg.anno.sorted.gff3 > Pst_104E_v13_ph_ctg.anno.sorted.exons_only.gff3\n",
    "grep -v 'gene' Pst_104E_v13_ph_ctg.anno.sorted.gff3 > Pst_104E_v13_ph_ctg.anno.sorted.non_genes.gff3"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### TE files notes\n",
    "\n",
    "- files present for p, h and combined ph contigs\n",
    "- sf = superfamily"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1270,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Read in the superfamily file as a pandas dataframe\n",
    "te_sf_fn_p = os.path.join(DIRS['BASE2'], 'gff_output', 'Pst_104E_v13_p_ctg.REPET.sorted.filtered.superfamily.gff')\n",
    "te_sf_fn_ph = os.path.join(DIRS['BASE2'], 'gff_output', 'Pst_104E_v13_p_ctg.REPET.sorted.filtered.superfamily.gff')\n",
    "te_sf_fn_h = os.path.join(DIRS['BASE2'], 'gff_output', 'Pst_104E_v13_p_ctg.REPET.sorted.filtered.superfamily.gff')\n",
    "te_sf_names = ['Contig', 'Source', 'Type', 'Start', 'Stop', 'Value', 'Strand', 'Space', 'Superfamily']\n",
    "\n",
    "te_sf_df_p = pd.read_csv(te_sf_fn_p, sep='\\t', header = None, names = te_sf_names)\n",
    "te_sf_df_ph = pd.read_csv(te_sf_fn_ph, sep='\\t', header = None, names = te_sf_names)\n",
    "te_sf_df_h = pd.read_csv(te_sf_fn_h, sep='\\t', header = None, names = te_sf_names)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1271,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Contig</th>\n",
       "      <th>Source</th>\n",
       "      <th>Type</th>\n",
       "      <th>Start</th>\n",
       "      <th>Stop</th>\n",
       "      <th>Value</th>\n",
       "      <th>Strand</th>\n",
       "      <th>Space</th>\n",
       "      <th>Superfamily</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>pcontig_000</td>\n",
       "      <td>Pst79p_anno_REPET_TEs</td>\n",
       "      <td>match</td>\n",
       "      <td>4244</td>\n",
       "      <td>4312</td>\n",
       "      <td>0.0</td>\n",
       "      <td>+</td>\n",
       "      <td>.</td>\n",
       "      <td>ClassII:MITE:?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>pcontig_000</td>\n",
       "      <td>Pst79p_anno_REPET_TEs</td>\n",
       "      <td>match</td>\n",
       "      <td>10017</td>\n",
       "      <td>10551</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-</td>\n",
       "      <td>.</td>\n",
       "      <td>ClassI:LTR:?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>pcontig_000</td>\n",
       "      <td>Pst79p_anno_REPET_TEs</td>\n",
       "      <td>match</td>\n",
       "      <td>10552</td>\n",
       "      <td>10574</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-</td>\n",
       "      <td>.</td>\n",
       "      <td>ClassI:LTR:?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>pcontig_000</td>\n",
       "      <td>Pst79p_anno_REPET_TEs</td>\n",
       "      <td>match</td>\n",
       "      <td>14601</td>\n",
       "      <td>14785</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-</td>\n",
       "      <td>.</td>\n",
       "      <td>ClassII:?:?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>pcontig_000</td>\n",
       "      <td>Pst79p_anno_REPET_tblastx</td>\n",
       "      <td>match</td>\n",
       "      <td>14603</td>\n",
       "      <td>14784</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-</td>\n",
       "      <td>.</td>\n",
       "      <td>ClassII:?:?</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        Contig                     Source   Type  Start   Stop  Value Strand  \\\n",
       "0  pcontig_000      Pst79p_anno_REPET_TEs  match   4244   4312    0.0      +   \n",
       "1  pcontig_000      Pst79p_anno_REPET_TEs  match  10017  10551    0.0      -   \n",
       "2  pcontig_000      Pst79p_anno_REPET_TEs  match  10552  10574    0.0      -   \n",
       "3  pcontig_000      Pst79p_anno_REPET_TEs  match  14601  14785    0.0      -   \n",
       "4  pcontig_000  Pst79p_anno_REPET_tblastx  match  14603  14784    0.0      -   \n",
       "\n",
       "  Space     Superfamily  \n",
       "0     .  ClassII:MITE:?  \n",
       "1     .    ClassI:LTR:?  \n",
       "2     .    ClassI:LTR:?  \n",
       "3     .     ClassII:?:?  \n",
       "4     .     ClassII:?:?  "
      ]
     },
     "execution_count": 1271,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "te_sf_df_ph.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# write a function that loops over the last column in the data frame to make a list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1388,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Test to make a list of all sf\n",
    "def make_superfamily_list(df):\n",
    "    \"\"\"Goes through dataframe and makes a list of all unique TE superfamily names.\"\"\"\n",
    "    superfamilies = []\n",
    "    for sf in df['Superfamily']:\n",
    "        if sf not in superfamilies:\n",
    "            superfamilies.append(sf)\n",
    "    return superfamilies"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1391,
   "metadata": {},
   "outputs": [],
   "source": [
    "te_sf_list_ph = make_superfamily_list(te_sf_df_ph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1399,
   "metadata": {},
   "outputs": [],
   "source": [
    "# write a function to loop over sf list and make a gff file for each superfamily\n",
    "# these gff files will be used for window coverage analysis later\n",
    "def filter_superfamilies(df, sf_list):\n",
    "    \"\"\"Filter out the TE's by their superfamily into separate files.\"\"\"\n",
    "    sf_dict = {}\n",
    "    for sf in sf_list:\n",
    "        filter_list = [sf]\n",
    "        filtered_df = df[df['Superfamily'].isin(filter_list)]\n",
    "        sf_dict[sf] = filtered_df\n",
    "    return sf_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1400,
   "metadata": {},
   "outputs": [],
   "source": [
    "te_sf_dict_ph = filter_superfamilies(te_sf_df_ph, te_sf_list_ph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1415,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save out the dataframes in the dictionary as gff files\n",
    "for key, value in te_sf_dict_ph.items():\n",
    "    fn = 'Pst_104E_v13_ph_ctg.' + key + '.gff3'\n",
    "    out_fn = os.path.join(DIRS['COVERAGE'], 'superfamily_files', fn)\n",
    "    te_sf_dict_ph[key].to_csv(out_fn, sep='\\t', header=None, index=None)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#148aff'> 5.B Making dicts of feature file of effector proteins, non-effectors, genes, exons and TE superfamilies. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 863,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make filepaths for feature files for genes, effectors, TE, methylation and randomised files for methylation.\n",
    "feature_files = [fn for fn in glob.iglob('%s/*' % DIRS['FEATURES'], recursive=True)]\n",
    "feature_fn_dict = file_name_dict(feature_files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 853,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.1.00.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.00.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.10.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.20.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.30.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.40.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.50.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.60.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.70.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.80.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.90.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.95.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.0.99.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/5mC_tombo_nanopolish.cutoff.1.00.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.10.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.20.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.30.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.40.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.50.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.60.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.70.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.80.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.90.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.95.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.99.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.1.00.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.00.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.10.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.10.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.20.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.20.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.30.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.30.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.40.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.40.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.50.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.50.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.60.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.60.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.70.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.70.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.80.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.80.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.90.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.90.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.95.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.95.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.99.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.0.99.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.1.00.bed': '/home/anjuni/analysis/coverage/feature_files/6mA_tombo_smrtlink.cutoff.1.00.bed',\n",
      " 'Pst_104E_v13_ph_ctg.TE.sorted.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.TE.sorted.gff3',\n",
      " 'Pst_104E_v13_ph_ctg.anno.sorted.exons_only.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.anno.sorted.exons_only.gff3',\n",
      " 'Pst_104E_v13_ph_ctg.anno.sorted.genes_only.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.anno.sorted.genes_only.gff3',\n",
      " 'Pst_104E_v13_ph_ctg.anno.sorted.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.anno.sorted.gff3',\n",
      " 'Pst_104E_v13_ph_ctg.anno.sorted.non_genes.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.anno.sorted.non_genes.gff3',\n",
      " 'Pst_104E_v13_ph_ctg.effectors.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.effectors.gff3',\n",
      " 'Pst_104E_v13_ph_ctg.non_effectors.gff3': '/home/anjuni/analysis/coverage/feature_files/Pst_104E_v13_ph_ctg.non_effectors.gff3'}\n"
     ]
    }
   ],
   "source": [
    "# Check whether the feature file dictionary works (it does). len = 59\n",
    "pprint.pprint(feature_fn_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make dictionary of bedtools objects and check if it worked (it did!)\n",
    "feature_bed_dict = {}\n",
    "\n",
    "for key, value in feature_fn_dict.items():\n",
    "    feature_bed_dict[key] = BedTool(value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pprint.pprint(feature_bed_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 867,
   "metadata": {},
   "outputs": [],
   "source": [
    "rand_files = [fn for fn in glob.iglob('%s/*' % DIRS['RAND'], recursive=True)]\n",
    "rand_fn_dict = file_name_dict(rand_files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 869,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_hc_tombo_sorted.cutoff.0.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.10_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.20_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.30_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.40_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.50_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.60_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.70_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.80_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.90_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.95_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.99_rand.bed',\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.1.00_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.00_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.10_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.10_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.20_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.20_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.30_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.30_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.40_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.40_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.50_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.50_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.60_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.60_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.70_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.70_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.80_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.80_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.90_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.90_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.95_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.95_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.0.99_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.99_rand.bed',\n",
      " '5mC_tombo_nanopolish.cutoff.1.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.1.00_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.10_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.20_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.30_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.40_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.50_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.60_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.70_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.80_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.90_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.95_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.99_rand.bed',\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.1.00_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.00_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.10_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.10_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.20_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.20_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.30_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.30_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.40_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.40_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.50_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.50_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.60_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.60_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.70_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.70_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.80_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.80_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.90_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.90_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.95_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.95_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.0.99_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.99_rand.bed',\n",
      " '6mA_tombo_smrtlink.cutoff.1.00_rand.bed': '/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.1.00_rand.bed'}\n"
     ]
    }
   ],
   "source": [
    "# Check whether the feature file dictionary works (it does). len = 59\n",
    "pprint.pprint(rand_fn_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 870,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make dictionary of bedtools objects and check if it worked (it did!)\n",
    "rand_bed_dict = {}\n",
    "\n",
    "for key, value in rand_fn_dict.items():\n",
    "    rand_bed_dict[key] = BedTool(value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 871,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_hc_tombo_sorted.cutoff.0.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.10_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.20_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.30_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.40_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.50_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.60_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.70_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.80_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.90_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.95_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.99_rand.bed)>,\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.1.00_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.00_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.10_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.10_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.20_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.20_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.30_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.30_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.40_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.40_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.50_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.50_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.60_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.60_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.70_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.70_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.80_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.80_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.90_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.90_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.95_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.95_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.0.99_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.0.99_rand.bed)>,\n",
      " '5mC_tombo_nanopolish.cutoff.1.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_tombo_nanopolish.cutoff.1.00_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.10_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.20_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.30_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.40_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.50_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.60_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.70_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.80_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.90_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.95_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.99_rand.bed)>,\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.1.00_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.00_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.10_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.10_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.20_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.20_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.30_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.30_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.40_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.40_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.50_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.50_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.60_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.60_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.70_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.70_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.80_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.80_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.90_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.90_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.95_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.95_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.0.99_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.0.99_rand.bed)>,\n",
      " '6mA_tombo_smrtlink.cutoff.1.00_rand.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_tombo_smrtlink.cutoff.1.00_rand.bed)>}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(rand_bed_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1417,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make filepaths for feature files for TE superfamilies.\n",
    "sf_files = [fn for fn in glob.iglob('%s/*' % DIRS['TE_SF'], recursive=True)]\n",
    "sf_fn_dict = file_name_dict(sf_files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1420,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make dictionary of bedtools objects and check if it worked (it did!)\n",
    "sf_bed_dict = {}\n",
    "\n",
    "for key, value in sf_fn_dict.items():\n",
    "    sf_bed_dict[key] = BedTool(value)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#148aff'> 5.C Using Ben's pybedtools coverage function on the feature files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 353,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "# move the feature files to the correct folder\n",
    "cd /home/anjuni/analysis/coverage\n",
    "cp ../bedtools_output/intersects_from_cutoffs/* feature_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1437,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a function to do overlaps for feature files\n",
    "def coverage(windowbed_dict, featurebed_dict, featurefn_dict, old_folder_name, new_folder_name):\n",
    "    \"\"\"Create coverage files from a dictionary of window pybedtools objects and a dictionary of feature file pybedtools objects. Also outputs a dictionary of pandas dataframes for all coverage files.\"\"\"\n",
    "    feature_overlap_df_dict = {}\n",
    "    for wkey, wbed in windowbed_dict.items():\n",
    "        for fkey, fbed in featurebed_dict.items():\n",
    "            tmp_df = wbed.coverage(fbed).to_dataframe().iloc[:,[0,1,2,3,6]] # make a dataframe to put headings\n",
    "            tmp_df.rename(columns={'name': 'overlap_count', 'thickStart': 'overlap_fraction'}, inplace=True) # rename headings\n",
    "            if featurefn_dict[fkey].endswith('.gff3'): # for gene/transposon/effector/exon files\n",
    "                tmp_fn = featurefn_dict[fkey].replace('.gff3', '.%s.overlap.bed' % wkey) # change output file path\n",
    "            tmp_fn = tmp_fn.replace(old_folder_name, new_folder_name)\n",
    "            if featurefn_dict[fkey].endswith('.bed'): # for methylation files\n",
    "                tmp_fn = featurefn_dict[fkey].replace('.bed', '.%s.overlap.bed' % wkey) # change output file path\n",
    "            feature_overlap_df_dict[tmp_fn.split('/')[-1]] = tmp_df # file name as key and dataframe as value for overlap dict\n",
    "            tmp_df.to_csv(tmp_fn, sep='\\t', header=None, index=None) # save to a csv(pybedtools outputs more d.p. than BEDTools)\n",
    "    return feature_overlap_df_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Run overlaps between windows and features\n",
    "feature_coverage_dict = coverage(window_bed_dict, feature_bed_dict, feature_fn_dict, 'feature_files', 'coverage_feature')\n",
    "rand_coverage_dict = coverage(window_bed_dict, rand_bed_dict, rand_fn_dict, 'randomisation', 'coverage_random')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1438,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Run overlaps between windows and TE superfamily files\n",
    "sf_coverage_dict = coverage(window_bed_dict, sf_bed_dict, sf_fn_dict, 'superfamily_files', 'coverage_superfamily')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Run overlaps for the combined gene files for consistency with the per-gene analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#148aff'> 5.D Statistical tests on the coverage files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 958,
   "metadata": {},
   "outputs": [],
   "source": [
    "def separate_sliding_windows(input_dict, string):\n",
    "    \"\"\"Separates sliding and non-sliding windows into separate dictionaries.\"\"\"\n",
    "    sliding_dict = {}\n",
    "    non_sliding_dict = {}\n",
    "    for key, value in input_dict.items():\n",
    "        if key.endswith(string):\n",
    "            sliding_dict[key] = value\n",
    "        else:\n",
    "            non_sliding_dict[key] = value\n",
    "    return sliding_dict, non_sliding_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 959,
   "metadata": {},
   "outputs": [],
   "source": [
    "rand_s_dict, rand_ns_dict = separate_sliding_windows(rand_coverage_dict, '_s2kb.overlap.bed')\n",
    "feature_s_dict, feature_ns_dict = separate_sliding_windows(feature_coverage_dict, '_s2kb.overlap.bed')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1227,
   "metadata": {},
   "outputs": [],
   "source": [
    "def coverage_wilcoxon(obs_df_dict, exp_df_dict, obs_slice, exp_slice):\n",
    "    \"\"\"This function returns a dictionary of chi square statistic and p-value for a test of observed and randomised sites.\"\"\"\n",
    "    wilcoxon_dict = {}\n",
    "    for okey, ovalue in obs_df_dict.items():\n",
    "        for ekey, evalue in exp_df_dict.items():\n",
    "            if okey[:obs_slice] == ekey[:exp_slice]:\n",
    "                o_df = ovalue\n",
    "                e_df = evalue\n",
    "                obs = o_df['overlap_fraction']\n",
    "                exp = e_df['overlap_fraction']\n",
    "                stat, p = wilcoxon(obs, exp)\n",
    "                wilcoxon_dict[okey] = stat, p\n",
    "    return wilcoxon_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1228,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/anjuni/anaconda3/lib/python3.6/site-packages/scipy/stats/morestats.py:2385: UserWarning: Warning: sample size too small for normal approximation.\n",
      "  warnings.warn(\"Warning: sample size too small for normal approximation.\")\n",
      "/home/anjuni/anaconda3/lib/python3.6/site-packages/scipy/stats/morestats.py:2410: RuntimeWarning: invalid value encountered in double_scalars\n",
      "  z = (T - mn - correction) / se\n"
     ]
    }
   ],
   "source": [
    "wilcoxon_coverage = coverage_wilcoxon(feature_ns_dict, rand_ns_dict, -17, -22)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1233,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'5mC_hc_tombo_sorted.cutoff.0.00.10kb.overlap.bed': (87915.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.10.10kb.overlap.bed': (85739.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.20.10kb.overlap.bed': (101277.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.30.10kb.overlap.bed': (234278.0, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.40.10kb.overlap.bed': (2327553.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.50.10kb.overlap.bed': (16607427.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.60.10kb.overlap.bed': (46236315.0,\n",
      "                                                      9.9306376916492e-187),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.70.10kb.overlap.bed': (60573267.0,\n",
      "                                                      2.6349063610233442e-05),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.80.10kb.overlap.bed': (45245491.5,\n",
      "                                                      1.2543957014880141e-206),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.90.10kb.overlap.bed': (28260071.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.95.10kb.overlap.bed': (22742993.5, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed': (22678421.0, 0.0),\n",
      " '5mC_hc_tombo_sorted.cutoff.1.00.10kb.overlap.bed': (22741581.5, 0.0),\n",
      " '5mC_tombo_nanopolish.cutoff.0.00.10kb.overlap.bed': (10844108.0, 0.0),\n",
      " '5mC_tombo_nanopolish.cutoff.0.10.10kb.overlap.bed': (43145071.0,\n",
      "                                                       4.553778902654728e-242),\n",
      " '5mC_tombo_nanopolish.cutoff.0.20.10kb.overlap.bed': (47436068.0,\n",
      "                                                       5.963971678347469e-145),\n",
      " '5mC_tombo_nanopolish.cutoff.0.30.10kb.overlap.bed': (49056338.5,\n",
      "                                                       1.9209824862898895e-116),\n",
      " '5mC_tombo_nanopolish.cutoff.0.40.10kb.overlap.bed': (50401725.5,\n",
      "                                                       7.780904796466759e-94),\n",
      " '5mC_tombo_nanopolish.cutoff.0.50.10kb.overlap.bed': (52206454.5,\n",
      "                                                       6.256747390410517e-68),\n",
      " '5mC_tombo_nanopolish.cutoff.0.60.10kb.overlap.bed': (55972015.5,\n",
      "                                                       2.3119557355321374e-26),\n",
      " '5mC_tombo_nanopolish.cutoff.0.70.10kb.overlap.bed': (61285835.5,\n",
      "                                                       0.3968684092160172),\n",
      " '5mC_tombo_nanopolish.cutoff.0.80.10kb.overlap.bed': (55859127.5,\n",
      "                                                       1.5092562388783972e-19),\n",
      " '5mC_tombo_nanopolish.cutoff.0.90.10kb.overlap.bed': (46064137.0,\n",
      "                                                       1.0170925923720883e-108),\n",
      " '5mC_tombo_nanopolish.cutoff.0.95.10kb.overlap.bed': (32915851.0,\n",
      "                                                       3.43685868380614e-310),\n",
      " '5mC_tombo_nanopolish.cutoff.0.99.10kb.overlap.bed': (21577730.0, 0.0),\n",
      " '5mC_tombo_nanopolish.cutoff.1.00.10kb.overlap.bed': (21695392.0, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.00.10kb.overlap.bed': (123595.5, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.10.10kb.overlap.bed': (110543.5, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.20.10kb.overlap.bed': (86113.0, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.30.10kb.overlap.bed': (68434.0, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.40.10kb.overlap.bed': (148779.5, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.50.10kb.overlap.bed': (6381947.5, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.60.10kb.overlap.bed': (40142971.0, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.70.10kb.overlap.bed': (62540754.5,\n",
      "                                                      0.484210903911623),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.80.10kb.overlap.bed': (46297440.5,\n",
      "                                                      3.308405230559043e-182),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.90.10kb.overlap.bed': (28745766.0, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.95.10kb.overlap.bed': (23623035.0, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed': (23719120.5, 0.0),\n",
      " '6mA_hc_tombo_sorted.cutoff.1.00.10kb.overlap.bed': (23754178.0, 0.0),\n",
      " '6mA_tombo_smrtlink.cutoff.0.00.10kb.overlap.bed': (47595884.0,\n",
      "                                                     4.767778176535458e-26),\n",
      " '6mA_tombo_smrtlink.cutoff.0.10.10kb.overlap.bed': (46779618.5,\n",
      "                                                     1.0783349873119544e-23),\n",
      " '6mA_tombo_smrtlink.cutoff.0.20.10kb.overlap.bed': (45426399.5,\n",
      "                                                     8.763200632722631e-20),\n",
      " '6mA_tombo_smrtlink.cutoff.0.30.10kb.overlap.bed': (43562423.5,\n",
      "                                                     1.6437867991758577e-16),\n",
      " '6mA_tombo_smrtlink.cutoff.0.40.10kb.overlap.bed': (41230864.0,\n",
      "                                                     1.2008451600300923e-15),\n",
      " '6mA_tombo_smrtlink.cutoff.0.50.10kb.overlap.bed': (38279444.5,\n",
      "                                                     7.761436603883395e-15),\n",
      " '6mA_tombo_smrtlink.cutoff.0.60.10kb.overlap.bed': (31384716.5,\n",
      "                                                     6.0700529773129946e-12),\n",
      " '6mA_tombo_smrtlink.cutoff.0.70.10kb.overlap.bed': (19724676.5,\n",
      "                                                     6.003085591247095e-08),\n",
      " '6mA_tombo_smrtlink.cutoff.0.80.10kb.overlap.bed': (8092159.5,\n",
      "                                                     0.001988677333990722),\n",
      " '6mA_tombo_smrtlink.cutoff.0.90.10kb.overlap.bed': (1182999.5,\n",
      "                                                     0.014416027330854089),\n",
      " '6mA_tombo_smrtlink.cutoff.0.95.10kb.overlap.bed': (250210.0,\n",
      "                                                     0.01167586893927822),\n",
      " '6mA_tombo_smrtlink.cutoff.0.99.10kb.overlap.bed': (176958.0,\n",
      "                                                     0.032661011929762666),\n",
      " '6mA_tombo_smrtlink.cutoff.1.00.10kb.overlap.bed': (0.0, nan)}\n"
     ]
    }
   ],
   "source": [
    "pprint.pprint(wilcoxon_coverage)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "All the cutoffs had statistically significant results, so use any of them I guess. The lower cutoff would be best because you get more sites."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#148aff'> 5.E Correlations between the coverage files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1243,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=-0.38281643716088243, pvalue=0.0)"
      ]
     },
     "execution_count": 1243,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 5mC correlation with genes\n",
    "spearmanr(feature_coverage_dict['5mC_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         feature_coverage_dict['Pst_104E_v13_ph_ctg.anno.sorted.genes_only.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1244,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=0.25369718437406363, pvalue=1.690789556321972e-233)"
      ]
     },
     "execution_count": 1244,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 5mC correlation with TEs\n",
    "spearmanr(feature_coverage_dict['5mC_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         feature_coverage_dict['Pst_104E_v13_ph_ctg.TE.sorted.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1245,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=-0.13931099497837787, pvalue=3.5837515997418266e-70)"
      ]
     },
     "execution_count": 1245,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 5mC correlation with effectors\n",
    "spearmanr(feature_coverage_dict['5mC_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         feature_coverage_dict['Pst_104E_v13_ph_ctg.effectors.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1250,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=-0.18353279200007624, pvalue=2.970389253688614e-121)"
      ]
     },
     "execution_count": 1250,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 6mA correlation with genes\n",
    "spearmanr(feature_coverage_dict['6mA_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         feature_coverage_dict['Pst_104E_v13_ph_ctg.anno.sorted.genes_only.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1264,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=-0.07515396006471368, pvalue=1.728156969082245e-21)"
      ]
     },
     "execution_count": 1264,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 6mA correlation with effectors\n",
    "spearmanr(feature_coverage_dict['6mA_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         feature_coverage_dict['Pst_104E_v13_ph_ctg.effectors.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1262,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=0.09330526525126864, pvalue=2.775701706245244e-32)"
      ]
     },
     "execution_count": 1262,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 6mA correlation with TEs\n",
    "spearmanr(feature_coverage_dict['6mA_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         feature_coverage_dict['Pst_104E_v13_ph_ctg.TE.sorted.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Test correlation between randomised files and feature files as a control"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1440,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=0.06080346779741759, pvalue=1.3758476935372227e-14)"
      ]
     },
     "execution_count": 1440,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Test correlation between 5mC and superfamily files\n",
    "spearmanr(feature_coverage_dict['5mC_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         sf_coverage_dict['Pst_104E_v13_ph_ctg.ClassI:LTR:Gypsy.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1441,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=-0.017815652590927895, pvalue=0.02420836921035543)"
      ]
     },
     "execution_count": 1441,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "spearmanr(feature_coverage_dict['6mA_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         sf_coverage_dict['Pst_104E_v13_ph_ctg.ClassI:LTR:Gypsy.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1442,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=0.0268650466727745, pvalue=0.0006764206901908494)"
      ]
     },
     "execution_count": 1442,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "spearmanr(feature_coverage_dict['5mC_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         sf_coverage_dict['Pst_104E_v13_ph_ctg.ClassI:LTR:Copia.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1444,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "SpearmanrResult(correlation=-0.02353519583145995, pvalue=0.0029056708810170496)"
      ]
     },
     "execution_count": 1444,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "spearmanr(feature_coverage_dict['6mA_hc_tombo_sorted.cutoff.0.99.10kb.overlap.bed'].overlap_fraction,\\\n",
    "         sf_coverage_dict['Pst_104E_v13_ph_ctg.ClassI:LTR:Copia.10kb.overlap.bed'].overlap_fraction)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2180,
   "metadata": {},
   "outputs": [],
   "source": [
    "# spearman function\n",
    "def spearman_same_key(obs_df_dict, exp_df_dict):\n",
    "    \"\"\"This function returns a dictionary of Spearmans rho statistic and p-value for a test of observed and randomised sites.\"\"\"\n",
    "    spearman_dict = {}\n",
    "    for okey, ovalue in obs_df_dict.items():\n",
    "        for ekey, evalue in exp_df_dict.items():\n",
    "            if okey == ekey:\n",
    "                o_df = ovalue\n",
    "                e_df = evalue\n",
    "                obs = o_df['overlap_fraction']\n",
    "                exp = e_df['overlap_fraction']\n",
    "                stat, p = spearmanr(obs, exp)\n",
    "                spearman_dict[okey] = stat, p\n",
    "    return spearman_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Overall, very weak correlations between methylation and key features in the genome, especially TE superfamilies :)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#14c4ff'> 6. Intersecting methylation with gene annotation files. <span/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#14c4ff'> 6.A Intersecting 5mC and 6mA with gene body. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1658,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Converted the gff anno files to bed files in the anno_file_prep notebook\n",
    "# Set filepaths for the anno bed files\n",
    "\n",
    "gene_anno_dict = {}\n",
    "for fn in glob.iglob('%s/*.bed' % DIRS['GENE_ANNO'], recursive=True):\n",
    "    key = fn.split('/')[-1]\n",
    "    gene_anno_dict[key] = fn\n",
    "\n",
    "# Make dictionary of bedtools objects and check if it worked (it did!)\n",
    "gene_bed_dict = {}\n",
    "for key, value in gene_anno_dict.items():\n",
    "    gene_bed_dict[key] = BedTool(value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1744,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Pst_104E_v13_h_ctg.TE.sorted.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg.TE.sorted.bed',\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg_combined_sorted_anno.bed',\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_p_ctg.TE.sorted.bed',\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_p_ctg_combined_sorted_anno.bed',\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.TE.sorted.bed',\n",
       " 'Pst_104E_v13_ph_ctg.effectors.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.effectors.bed',\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.non_effectors.bed',\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg_combined_sorted_anno.bed'}"
      ]
     },
     "execution_count": 1744,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_anno_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2045,
   "metadata": {},
   "outputs": [],
   "source": [
    "methyl_dict = {} # lowest cutoff as it had the highest similarity between sequencers/callers\n",
    "methyl_dict['5mC_hc_tombo_sorted.cutoff.0.00.bed'] = os.path.join(DIRS['FEATURES'], '5mC_hc_tombo_sorted.cutoff.0.00.bed',)\n",
    "methyl_dict['6mA_hc_tombo_sorted.cutoff.0.00.bed'] = os.path.join(DIRS['FEATURES'], '6mA_hc_tombo_sorted.cutoff.0.00.bed',)\n",
    "\n",
    "# Make dictionary of bedtools objects and check if it worked (it did!)\n",
    "methyl_bed_dict = make_bed_dict(methyl_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1782,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a function to do overlaps for gene files and methylation\n",
    "def coverage_gene(genebed_dict, methylbed_dict, genefn_dict, old_folder_name, new_folder_name):\n",
    "    \"\"\"Create coverage files from:\n",
    "    Inputs: dictionary of gene pybedtools objects, dictionary of methylation pybedtools objects, dictionary of gene filenames, old folder name and new folder name.\n",
    "    Output: dictionary of pandas dataframes for all coverage files.\"\"\"\n",
    "    feature_overlap_df_dict = {}\n",
    "    for gkey, gbed in genebed_dict.items():\n",
    "        for mkey, mbed in methylbed_dict.items():\n",
    "            tmp_df = gbed.coverage(mbed, s=True).to_dataframe().iloc[:,[0,1,2,3,6,9]] # make a dataframe to put headings\n",
    "            tmp_df.rename(columns={'thickStart': 'overlap_count', 'blockCount': 'overlap_fraction'}, inplace=True) # rename headings\n",
    "            tmp_fn = genefn_dict[gkey].replace('.bed', '.%s.overlap.bed' % mkey[:-4]) # change output file path\n",
    "            tmp_fn = tmp_fn.replace(old_folder_name, new_folder_name)\n",
    "            feature_overlap_df_dict[tmp_fn.split('/')[-1]] = tmp_df # file name as key and dataframe as value for overlap dict\n",
    "            tmp_df.to_csv(tmp_fn, sep='\\t', header=None, index=None) # save to a csv(pybedtools outputs more d.p. than BEDTools)\n",
    "    return feature_overlap_df_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1743,
   "metadata": {},
   "outputs": [],
   "source": [
    "# run overlap over the gene body for the genes\n",
    "gene_methyl_overlap_dict = coverage_gene(gene_bed_dict, methyl_bed_dict, gene_anno_dict, 'gene_anno', 'gene_coverage')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gene_rand_methyl_overlap_dict = coverage_gene(gene_bed_dict, methyl_rand_bed_dict, gene_anno_dict, 'gene_anno', 'gene_rand_coverage')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2061,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chrom</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>name</th>\n",
       "      <th>overlap_count</th>\n",
       "      <th>overlap_fraction</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>14562</td>\n",
       "      <td>15702</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.5</td>\n",
       "      <td>18</td>\n",
       "      <td>0.015790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>17795</td>\n",
       "      <td>18752</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.6</td>\n",
       "      <td>5</td>\n",
       "      <td>0.005225</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>80525</td>\n",
       "      <td>81601</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.21</td>\n",
       "      <td>1</td>\n",
       "      <td>0.000929</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>81736</td>\n",
       "      <td>83619</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.22</td>\n",
       "      <td>1</td>\n",
       "      <td>0.000531</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>108981</td>\n",
       "      <td>112995</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.25</td>\n",
       "      <td>6</td>\n",
       "      <td>0.001495</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>180425</td>\n",
       "      <td>180836</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.45</td>\n",
       "      <td>4</td>\n",
       "      <td>0.009732</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>182250</td>\n",
       "      <td>182751</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.46</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>182796</td>\n",
       "      <td>183867</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.47</td>\n",
       "      <td>3</td>\n",
       "      <td>0.002801</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>189995</td>\n",
       "      <td>190718</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.48</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>269647</td>\n",
       "      <td>274814</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.69</td>\n",
       "      <td>10</td>\n",
       "      <td>0.001935</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>329808</td>\n",
       "      <td>331131</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.81</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>344036</td>\n",
       "      <td>344906</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.84</td>\n",
       "      <td>1</td>\n",
       "      <td>0.001149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>345369</td>\n",
       "      <td>345747</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.85</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>402060</td>\n",
       "      <td>403644</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.100</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>406700</td>\n",
       "      <td>411752</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.102</td>\n",
       "      <td>3</td>\n",
       "      <td>0.000594</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>452052</td>\n",
       "      <td>458688</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.113</td>\n",
       "      <td>3</td>\n",
       "      <td>0.000452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>461215</td>\n",
       "      <td>461695</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.114</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>464659</td>\n",
       "      <td>465209</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.115</td>\n",
       "      <td>4</td>\n",
       "      <td>0.007273</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>524253</td>\n",
       "      <td>524571</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.128</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>524807</td>\n",
       "      <td>525437</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.129</td>\n",
       "      <td>8</td>\n",
       "      <td>0.012698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>525753</td>\n",
       "      <td>527436</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.130</td>\n",
       "      <td>8</td>\n",
       "      <td>0.004753</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>530399</td>\n",
       "      <td>533666</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.132</td>\n",
       "      <td>1</td>\n",
       "      <td>0.000306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>538838</td>\n",
       "      <td>539861</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.134</td>\n",
       "      <td>1</td>\n",
       "      <td>0.000978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>547041</td>\n",
       "      <td>547965</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.136</td>\n",
       "      <td>4</td>\n",
       "      <td>0.004329</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>570436</td>\n",
       "      <td>571210</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.140</td>\n",
       "      <td>1</td>\n",
       "      <td>0.001292</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>571657</td>\n",
       "      <td>574243</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.141</td>\n",
       "      <td>1</td>\n",
       "      <td>0.000387</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>575825</td>\n",
       "      <td>576987</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.143</td>\n",
       "      <td>3</td>\n",
       "      <td>0.002582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>670734</td>\n",
       "      <td>672992</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.161</td>\n",
       "      <td>8</td>\n",
       "      <td>0.003543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>687456</td>\n",
       "      <td>688125</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.166</td>\n",
       "      <td>6</td>\n",
       "      <td>0.008969</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>hcontig_000_003</td>\n",
       "      <td>689640</td>\n",
       "      <td>690882</td>\n",
       "      <td>EVM prediction%2hcontig_0000_003.167</td>\n",
       "      <td>4</td>\n",
       "      <td>0.003221</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2848</th>\n",
       "      <td>hcontig_095_133</td>\n",
       "      <td>44683</td>\n",
       "      <td>45890</td>\n",
       "      <td>EVM prediction%2hcontig_095_0133.13</td>\n",
       "      <td>3</td>\n",
       "      <td>0.002485</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2849</th>\n",
       "      <td>hcontig_095_133</td>\n",
       "      <td>45983</td>\n",
       "      <td>46990</td>\n",
       "      <td>EVM prediction%2hcontig_095_0133.14</td>\n",
       "      <td>3</td>\n",
       "      <td>0.002979</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2850</th>\n",
       "      <td>hcontig_095_133</td>\n",
       "      <td>81291</td>\n",
       "      <td>82266</td>\n",
       "      <td>EVM prediction%2hcontig_095_0133.20</td>\n",
       "      <td>10</td>\n",
       "      <td>0.010256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2851</th>\n",
       "      <td>hcontig_095_133</td>\n",
       "      <td>82403</td>\n",
       "      <td>83261</td>\n",
       "      <td>EVM prediction%2hcontig_095_0133.21</td>\n",
       "      <td>10</td>\n",
       "      <td>0.011655</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2852</th>\n",
       "      <td>hcontig_096_004</td>\n",
       "      <td>14666</td>\n",
       "      <td>15263</td>\n",
       "      <td>EVM prediction%2hcontig_0096_004.7</td>\n",
       "      <td>2</td>\n",
       "      <td>0.003350</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2853</th>\n",
       "      <td>hcontig_096_004</td>\n",
       "      <td>64620</td>\n",
       "      <td>69315</td>\n",
       "      <td>EVM prediction%2hcontig_0096_004.1</td>\n",
       "      <td>1</td>\n",
       "      <td>0.000213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2854</th>\n",
       "      <td>hcontig_096_004</td>\n",
       "      <td>112078</td>\n",
       "      <td>117127</td>\n",
       "      <td>EVM prediction%2hcontig_0096_004.4</td>\n",
       "      <td>4</td>\n",
       "      <td>0.000792</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2855</th>\n",
       "      <td>hcontig_097_001</td>\n",
       "      <td>9031</td>\n",
       "      <td>10942</td>\n",
       "      <td>EVM prediction%2hcontig_0097_001.3</td>\n",
       "      <td>11</td>\n",
       "      <td>0.005756</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2856</th>\n",
       "      <td>hcontig_097_001</td>\n",
       "      <td>10969</td>\n",
       "      <td>12182</td>\n",
       "      <td>EVM prediction%2hcontig_0097_001.4</td>\n",
       "      <td>8</td>\n",
       "      <td>0.006595</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2857</th>\n",
       "      <td>hcontig_097_001</td>\n",
       "      <td>14055</td>\n",
       "      <td>18204</td>\n",
       "      <td>EVM prediction%2hcontig_0097_001.5</td>\n",
       "      <td>33</td>\n",
       "      <td>0.007954</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2858</th>\n",
       "      <td>hcontig_104_001</td>\n",
       "      <td>6351</td>\n",
       "      <td>8168</td>\n",
       "      <td>EVM prediction%2hcontig_0104_001.3</td>\n",
       "      <td>116</td>\n",
       "      <td>0.063841</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2859</th>\n",
       "      <td>hcontig_104_001</td>\n",
       "      <td>8459</td>\n",
       "      <td>12977</td>\n",
       "      <td>EVM prediction%2hcontig_0104_001.4</td>\n",
       "      <td>367</td>\n",
       "      <td>0.081231</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2860</th>\n",
       "      <td>hcontig_104_001</td>\n",
       "      <td>13618</td>\n",
       "      <td>16114</td>\n",
       "      <td>EVM prediction%2hcontig_0104_001.5</td>\n",
       "      <td>206</td>\n",
       "      <td>0.082532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2861</th>\n",
       "      <td>hcontig_106_001</td>\n",
       "      <td>31454</td>\n",
       "      <td>35846</td>\n",
       "      <td>EVM prediction%2hcontig_0106_001.7</td>\n",
       "      <td>5</td>\n",
       "      <td>0.001138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2862</th>\n",
       "      <td>hcontig_110_003</td>\n",
       "      <td>3534</td>\n",
       "      <td>6528</td>\n",
       "      <td>EVM prediction%2hcontig_0110_003.2</td>\n",
       "      <td>65</td>\n",
       "      <td>0.021710</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2863</th>\n",
       "      <td>hcontig_110_003</td>\n",
       "      <td>14395</td>\n",
       "      <td>18559</td>\n",
       "      <td>EVM prediction%2hcontig_0110_003.5</td>\n",
       "      <td>15</td>\n",
       "      <td>0.003602</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2864</th>\n",
       "      <td>hcontig_110_003</td>\n",
       "      <td>22108</td>\n",
       "      <td>23626</td>\n",
       "      <td>EVM prediction%2hcontig_0110_003.6</td>\n",
       "      <td>4</td>\n",
       "      <td>0.002635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2865</th>\n",
       "      <td>hcontig_110_003</td>\n",
       "      <td>24300</td>\n",
       "      <td>24739</td>\n",
       "      <td>EVM prediction%2hcontig_0110_003.7</td>\n",
       "      <td>1</td>\n",
       "      <td>0.002278</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2866</th>\n",
       "      <td>hcontig_110_003</td>\n",
       "      <td>29163</td>\n",
       "      <td>31239</td>\n",
       "      <td>EVM prediction%2hcontig_0110_003.9</td>\n",
       "      <td>10</td>\n",
       "      <td>0.004817</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2867</th>\n",
       "      <td>hcontig_110_003</td>\n",
       "      <td>41051</td>\n",
       "      <td>41714</td>\n",
       "      <td>EVM prediction%2hcontig_0110_003.11</td>\n",
       "      <td>2</td>\n",
       "      <td>0.003017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2868</th>\n",
       "      <td>hcontig_147_001</td>\n",
       "      <td>3071</td>\n",
       "      <td>6029</td>\n",
       "      <td>EVM prediction%2hcontig_0147_001.1</td>\n",
       "      <td>38</td>\n",
       "      <td>0.012847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2869</th>\n",
       "      <td>hcontig_147_001</td>\n",
       "      <td>18689</td>\n",
       "      <td>21161</td>\n",
       "      <td>EVM prediction%2hcontig_0147_001.5</td>\n",
       "      <td>6</td>\n",
       "      <td>0.002427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2870</th>\n",
       "      <td>hcontig_147_001</td>\n",
       "      <td>21248</td>\n",
       "      <td>22109</td>\n",
       "      <td>EVM prediction%2hcontig_0147_001.6</td>\n",
       "      <td>4</td>\n",
       "      <td>0.004646</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2871</th>\n",
       "      <td>hcontig_147_001</td>\n",
       "      <td>22402</td>\n",
       "      <td>23626</td>\n",
       "      <td>EVM prediction%2hcontig_0147_001.7</td>\n",
       "      <td>4</td>\n",
       "      <td>0.003268</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2872</th>\n",
       "      <td>hcontig_148_001</td>\n",
       "      <td>17106</td>\n",
       "      <td>18643</td>\n",
       "      <td>EVM prediction%2hcontig_0148_001.6</td>\n",
       "      <td>90</td>\n",
       "      <td>0.058556</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2873</th>\n",
       "      <td>hcontig_166_001</td>\n",
       "      <td>7350</td>\n",
       "      <td>11233</td>\n",
       "      <td>EVM prediction%2hcontig_0166_001.3</td>\n",
       "      <td>78</td>\n",
       "      <td>0.020088</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2874</th>\n",
       "      <td>hcontig_184_001</td>\n",
       "      <td>14584</td>\n",
       "      <td>17155</td>\n",
       "      <td>EVM prediction%2hcontig_0184_001.1</td>\n",
       "      <td>66</td>\n",
       "      <td>0.025671</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2875</th>\n",
       "      <td>hcontig_184_001</td>\n",
       "      <td>17715</td>\n",
       "      <td>18372</td>\n",
       "      <td>EVM prediction%2hcontig_0184_001.2</td>\n",
       "      <td>41</td>\n",
       "      <td>0.062405</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2876</th>\n",
       "      <td>hcontig_184_001</td>\n",
       "      <td>19820</td>\n",
       "      <td>20537</td>\n",
       "      <td>EVM prediction%2hcontig_0184_001.4</td>\n",
       "      <td>59</td>\n",
       "      <td>0.082287</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2877</th>\n",
       "      <td>hcontig_184_001</td>\n",
       "      <td>20692</td>\n",
       "      <td>21526</td>\n",
       "      <td>EVM prediction%2hcontig_0184_001.5</td>\n",
       "      <td>77</td>\n",
       "      <td>0.092326</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2878 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                chrom   start     end                                  name  \\\n",
       "0     hcontig_000_003   14562   15702    EVM prediction%2hcontig_0000_003.5   \n",
       "1     hcontig_000_003   17795   18752    EVM prediction%2hcontig_0000_003.6   \n",
       "2     hcontig_000_003   80525   81601   EVM prediction%2hcontig_0000_003.21   \n",
       "3     hcontig_000_003   81736   83619   EVM prediction%2hcontig_0000_003.22   \n",
       "4     hcontig_000_003  108981  112995   EVM prediction%2hcontig_0000_003.25   \n",
       "5     hcontig_000_003  180425  180836   EVM prediction%2hcontig_0000_003.45   \n",
       "6     hcontig_000_003  182250  182751   EVM prediction%2hcontig_0000_003.46   \n",
       "7     hcontig_000_003  182796  183867   EVM prediction%2hcontig_0000_003.47   \n",
       "8     hcontig_000_003  189995  190718   EVM prediction%2hcontig_0000_003.48   \n",
       "9     hcontig_000_003  269647  274814   EVM prediction%2hcontig_0000_003.69   \n",
       "10    hcontig_000_003  329808  331131   EVM prediction%2hcontig_0000_003.81   \n",
       "11    hcontig_000_003  344036  344906   EVM prediction%2hcontig_0000_003.84   \n",
       "12    hcontig_000_003  345369  345747   EVM prediction%2hcontig_0000_003.85   \n",
       "13    hcontig_000_003  402060  403644  EVM prediction%2hcontig_0000_003.100   \n",
       "14    hcontig_000_003  406700  411752  EVM prediction%2hcontig_0000_003.102   \n",
       "15    hcontig_000_003  452052  458688  EVM prediction%2hcontig_0000_003.113   \n",
       "16    hcontig_000_003  461215  461695  EVM prediction%2hcontig_0000_003.114   \n",
       "17    hcontig_000_003  464659  465209  EVM prediction%2hcontig_0000_003.115   \n",
       "18    hcontig_000_003  524253  524571  EVM prediction%2hcontig_0000_003.128   \n",
       "19    hcontig_000_003  524807  525437  EVM prediction%2hcontig_0000_003.129   \n",
       "20    hcontig_000_003  525753  527436  EVM prediction%2hcontig_0000_003.130   \n",
       "21    hcontig_000_003  530399  533666  EVM prediction%2hcontig_0000_003.132   \n",
       "22    hcontig_000_003  538838  539861  EVM prediction%2hcontig_0000_003.134   \n",
       "23    hcontig_000_003  547041  547965  EVM prediction%2hcontig_0000_003.136   \n",
       "24    hcontig_000_003  570436  571210  EVM prediction%2hcontig_0000_003.140   \n",
       "25    hcontig_000_003  571657  574243  EVM prediction%2hcontig_0000_003.141   \n",
       "26    hcontig_000_003  575825  576987  EVM prediction%2hcontig_0000_003.143   \n",
       "27    hcontig_000_003  670734  672992  EVM prediction%2hcontig_0000_003.161   \n",
       "28    hcontig_000_003  687456  688125  EVM prediction%2hcontig_0000_003.166   \n",
       "29    hcontig_000_003  689640  690882  EVM prediction%2hcontig_0000_003.167   \n",
       "...               ...     ...     ...                                   ...   \n",
       "2848  hcontig_095_133   44683   45890   EVM prediction%2hcontig_095_0133.13   \n",
       "2849  hcontig_095_133   45983   46990   EVM prediction%2hcontig_095_0133.14   \n",
       "2850  hcontig_095_133   81291   82266   EVM prediction%2hcontig_095_0133.20   \n",
       "2851  hcontig_095_133   82403   83261   EVM prediction%2hcontig_095_0133.21   \n",
       "2852  hcontig_096_004   14666   15263    EVM prediction%2hcontig_0096_004.7   \n",
       "2853  hcontig_096_004   64620   69315    EVM prediction%2hcontig_0096_004.1   \n",
       "2854  hcontig_096_004  112078  117127    EVM prediction%2hcontig_0096_004.4   \n",
       "2855  hcontig_097_001    9031   10942    EVM prediction%2hcontig_0097_001.3   \n",
       "2856  hcontig_097_001   10969   12182    EVM prediction%2hcontig_0097_001.4   \n",
       "2857  hcontig_097_001   14055   18204    EVM prediction%2hcontig_0097_001.5   \n",
       "2858  hcontig_104_001    6351    8168    EVM prediction%2hcontig_0104_001.3   \n",
       "2859  hcontig_104_001    8459   12977    EVM prediction%2hcontig_0104_001.4   \n",
       "2860  hcontig_104_001   13618   16114    EVM prediction%2hcontig_0104_001.5   \n",
       "2861  hcontig_106_001   31454   35846    EVM prediction%2hcontig_0106_001.7   \n",
       "2862  hcontig_110_003    3534    6528    EVM prediction%2hcontig_0110_003.2   \n",
       "2863  hcontig_110_003   14395   18559    EVM prediction%2hcontig_0110_003.5   \n",
       "2864  hcontig_110_003   22108   23626    EVM prediction%2hcontig_0110_003.6   \n",
       "2865  hcontig_110_003   24300   24739    EVM prediction%2hcontig_0110_003.7   \n",
       "2866  hcontig_110_003   29163   31239    EVM prediction%2hcontig_0110_003.9   \n",
       "2867  hcontig_110_003   41051   41714   EVM prediction%2hcontig_0110_003.11   \n",
       "2868  hcontig_147_001    3071    6029    EVM prediction%2hcontig_0147_001.1   \n",
       "2869  hcontig_147_001   18689   21161    EVM prediction%2hcontig_0147_001.5   \n",
       "2870  hcontig_147_001   21248   22109    EVM prediction%2hcontig_0147_001.6   \n",
       "2871  hcontig_147_001   22402   23626    EVM prediction%2hcontig_0147_001.7   \n",
       "2872  hcontig_148_001   17106   18643    EVM prediction%2hcontig_0148_001.6   \n",
       "2873  hcontig_166_001    7350   11233    EVM prediction%2hcontig_0166_001.3   \n",
       "2874  hcontig_184_001   14584   17155    EVM prediction%2hcontig_0184_001.1   \n",
       "2875  hcontig_184_001   17715   18372    EVM prediction%2hcontig_0184_001.2   \n",
       "2876  hcontig_184_001   19820   20537    EVM prediction%2hcontig_0184_001.4   \n",
       "2877  hcontig_184_001   20692   21526    EVM prediction%2hcontig_0184_001.5   \n",
       "\n",
       "      overlap_count  overlap_fraction  \n",
       "0                18          0.015790  \n",
       "1                 5          0.005225  \n",
       "2                 1          0.000929  \n",
       "3                 1          0.000531  \n",
       "4                 6          0.001495  \n",
       "5                 4          0.009732  \n",
       "6                 0          0.000000  \n",
       "7                 3          0.002801  \n",
       "8                 0          0.000000  \n",
       "9                10          0.001935  \n",
       "10                0          0.000000  \n",
       "11                1          0.001149  \n",
       "12                0          0.000000  \n",
       "13                0          0.000000  \n",
       "14                3          0.000594  \n",
       "15                3          0.000452  \n",
       "16                0          0.000000  \n",
       "17                4          0.007273  \n",
       "18                0          0.000000  \n",
       "19                8          0.012698  \n",
       "20                8          0.004753  \n",
       "21                1          0.000306  \n",
       "22                1          0.000978  \n",
       "23                4          0.004329  \n",
       "24                1          0.001292  \n",
       "25                1          0.000387  \n",
       "26                3          0.002582  \n",
       "27                8          0.003543  \n",
       "28                6          0.008969  \n",
       "29                4          0.003221  \n",
       "...             ...               ...  \n",
       "2848              3          0.002485  \n",
       "2849              3          0.002979  \n",
       "2850             10          0.010256  \n",
       "2851             10          0.011655  \n",
       "2852              2          0.003350  \n",
       "2853              1          0.000213  \n",
       "2854              4          0.000792  \n",
       "2855             11          0.005756  \n",
       "2856              8          0.006595  \n",
       "2857             33          0.007954  \n",
       "2858            116          0.063841  \n",
       "2859            367          0.081231  \n",
       "2860            206          0.082532  \n",
       "2861              5          0.001138  \n",
       "2862             65          0.021710  \n",
       "2863             15          0.003602  \n",
       "2864              4          0.002635  \n",
       "2865              1          0.002278  \n",
       "2866             10          0.004817  \n",
       "2867              2          0.003017  \n",
       "2868             38          0.012847  \n",
       "2869              6          0.002427  \n",
       "2870              4          0.004646  \n",
       "2871              4          0.003268  \n",
       "2872             90          0.058556  \n",
       "2873             78          0.020088  \n",
       "2874             66          0.025671  \n",
       "2875             41          0.062405  \n",
       "2876             59          0.082287  \n",
       "2877             77          0.092326  \n",
       "\n",
       "[2878 rows x 6 columns]"
      ]
     },
     "execution_count": 2061,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_methyl_overlap_dict.pop('Pst_104E_v13_h_ctg.TE.sorted.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#14c4ff'> 6.B Intersecting 5mC and 6mA with upstream and downstream regions. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for gkey, gbed in genebed_dict.items():\n",
    "        for mkey, mbed in methylbed_dict.items():\n",
    "            tmp_df = gbed.flank(g=genome_size_f_fn, l=up_window, r=down_window, s= True).to_dataframe()\n",
    "            tmp_fn = genefn_dict[gkey].replace('.bed', '.%s.bed' % suffix)\n",
    "            tmp_fn = tmp_fn.replace(old_folder_name, new_folder_name) # change output file path\n",
    "            tmp_df.to_csv(tmp_fn, sep='\\t', header=None, index=None) # save to a csv(pybedtools outputs more d.p. than BEDTools)\n",
    "            region_overlap_df_dict[tmp_fn.split('/')[-1]] = tmp_df # file name as key and dataframe as value for overlap dict\n",
    "region_overlap_df_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2264,
   "metadata": {},
   "outputs": [],
   "source": [
    "# test out flanks\n",
    "genebed_dict = {}\n",
    "genefn_dict = {}\n",
    "methylbed_dict = {}\n",
    "genebed_dict['Pst_104E_v13_h_ctg.TE.sorted.upstream.bed'] = BedTool('/home/anjuni/analysis/coverage/gene_level/upstream/input/Pst_104E_v13_h_ctg.TE.sorted.upstream.bed')\n",
    "genefn_dict['Pst_104E_v13_h_ctg.TE.sorted.upstream.bed'] = '/home/anjuni/analysis/coverage/gene_level/upstream/input/Pst_104E_v13_h_ctg.TE.sorted.upstream.bed'\n",
    "methylbed_dict['6mA_hc_tombo_sorted.cutoff.0.99.bed'] = BedTool('/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.99.bed')\n",
    "up_window = 1000\n",
    "down_window = 1000\n",
    "old_folder_name = 'input'\n",
    "new_folder_name = 'coverage'\n",
    "suffix = 'upstream'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2261,
   "metadata": {},
   "outputs": [],
   "source": [
    "gkey = 'Pst_104E_v13_h_ctg.TE.sorted.upstream.bed'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2240,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'6mA_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed)>}"
      ]
     },
     "execution_count": 2240,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "methylbed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2243,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hcontig_000_003\t7\t8\t6mA\t1.0\t+\n",
      " hcontig_000_003\t8\t9\t6mA\t1.0\t+\n",
      " hcontig_000_003\t9\t10\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t10\t11\t6mA\t0.6667\t-\n",
      " hcontig_000_003\t11\t12\t6mA\t1.0\t+\n",
      " hcontig_000_003\t15\t16\t6mA\t0.25\t-\n",
      " hcontig_000_003\t18\t19\t6mA\t0.25\t-\n",
      " hcontig_000_003\t20\t21\t6mA\t1.0\t+\n",
      " hcontig_000_003\t23\t24\t6mA\t1.0\t+\n",
      " hcontig_000_003\t24\t25\t6mA\t0.25\t-\n",
      " hcontig_000_003\t25\t26\t6mA\t0.5\t-\n",
      " hcontig_000_003\t26\t27\t6mA\t0.75\t-\n",
      " hcontig_000_003\t29\t30\t6mA\t0.75\t-\n",
      " hcontig_000_003\t30\t31\t6mA\t1.0\t-\n",
      " hcontig_000_003\t37\t38\t6mA\t0.25\t-\n",
      " hcontig_000_003\t40\t41\t6mA\t0.5\t+\n",
      " hcontig_000_003\t45\t46\t6mA\t0.25\t-\n",
      " hcontig_000_003\t47\t48\t6mA\t0.5\t-\n",
      " hcontig_000_003\t48\t49\t6mA\t0.5\t-\n",
      " hcontig_000_003\t50\t51\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t51\t52\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t52\t53\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t53\t54\t6mA\t0.75\t-\n",
      " hcontig_000_003\t54\t55\t6mA\t0.75\t-\n",
      " hcontig_000_003\t55\t56\t6mA\t0.75\t-\n",
      " hcontig_000_003\t56\t57\t6mA\t0.25\t-\n",
      " hcontig_000_003\t59\t60\t6mA\t0.5\t-\n",
      " hcontig_000_003\t60\t61\t6mA\t0.5\t-\n",
      " hcontig_000_003\t61\t62\t6mA\t1.0\t+\n",
      " hcontig_000_003\t63\t64\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t64\t65\t6mA\t0.5\t-\n",
      " hcontig_000_003\t65\t66\t6mA\t0.25\t-\n",
      " hcontig_000_003\t66\t67\t6mA\t0.5\t-\n",
      " hcontig_000_003\t68\t69\t6mA\t1.0\t+\n",
      " hcontig_000_003\t70\t71\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t71\t72\t6mA\t0.25\t-\n",
      " hcontig_000_003\t73\t74\t6mA\t1.0\t+\n",
      " hcontig_000_003\t74\t75\t6mA\t0.5\t-\n",
      " hcontig_000_003\t76\t77\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t77\t78\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t80\t81\t6mA\t0.5\t-\n",
      " hcontig_000_003\t82\t83\t6mA\t0.25\t-\n",
      " hcontig_000_003\t88\t89\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t96\t97\t6mA\t0.25\t-\n",
      " hcontig_000_003\t99\t100\t6mA\t0.5\t-\n",
      " hcontig_000_003\t102\t103\t6mA\t0.75\t-\n",
      " hcontig_000_003\t103\t104\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t104\t105\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t106\t107\t6mA\t0.5\t-\n",
      " hcontig_000_003\t107\t108\t6mA\t0.75\t-\n",
      " hcontig_000_003\t110\t111\t6mA\t0.75\t-\n",
      " hcontig_000_003\t111\t112\t6mA\t0.5\t-\n",
      " hcontig_000_003\t112\t113\t6mA\t0.5\t-\n",
      " hcontig_000_003\t114\t115\t6mA\t0.5\t-\n",
      " hcontig_000_003\t115\t116\t6mA\t0.5\t-\n",
      " hcontig_000_003\t116\t117\t6mA\t0.75\t-\n",
      " hcontig_000_003\t119\t120\t6mA\t0.25\t+\n",
      " hcontig_000_003\t127\t128\t6mA\t0.75\t+\n",
      " hcontig_000_003\t128\t129\t6mA\t0.75\t+\n",
      " hcontig_000_003\t130\t131\t6mA\t0.75\t-\n",
      " hcontig_000_003\t131\t132\t6mA\t0.75\t-\n",
      " hcontig_000_003\t134\t135\t6mA\t0.25\t-\n",
      " hcontig_000_003\t136\t137\t6mA\t1.0\t-\n",
      " hcontig_000_003\t137\t138\t6mA\t0.75\t-\n",
      " hcontig_000_003\t140\t141\t6mA\t0.25\t-\n",
      " hcontig_000_003\t141\t142\t6mA\t0.5\t-\n",
      " hcontig_000_003\t142\t143\t6mA\t0.5\t+\n",
      " hcontig_000_003\t143\t144\t6mA\t0.5\t-\n",
      " hcontig_000_003\t144\t145\t6mA\t0.25\t-\n",
      " hcontig_000_003\t150\t151\t6mA\t0.75\t+\n",
      " hcontig_000_003\t153\t154\t6mA\t0.75\t-\n",
      " hcontig_000_003\t155\t156\t6mA\t0.75\t+\n",
      " hcontig_000_003\t159\t160\t6mA\t1.0\t+\n",
      " hcontig_000_003\t160\t161\t6mA\t0.75\t+\n",
      " hcontig_000_003\t166\t167\t6mA\t0.5\t-\n",
      " hcontig_000_003\t173\t174\t6mA\t1.0\t-\n",
      " hcontig_000_003\t174\t175\t6mA\t0.5\t+\n",
      " hcontig_000_003\t189\t190\t6mA\t0.75\t+\n",
      " hcontig_000_003\t191\t192\t6mA\t0.5\t+\n",
      " hcontig_000_003\t194\t195\t6mA\t0.5\t-\n",
      " hcontig_000_003\t196\t197\t6mA\t0.25\t+\n",
      " hcontig_000_003\t197\t198\t6mA\t0.5\t+\n",
      " hcontig_000_003\t199\t200\t6mA\t1.0\t-\n",
      " hcontig_000_003\t204\t205\t6mA\t0.25\t+\n",
      " hcontig_000_003\t205\t206\t6mA\t0.25\t+\n",
      " hcontig_000_003\t209\t210\t6mA\t0.75\t+\n",
      " hcontig_000_003\t222\t223\t6mA\t0.75\t-\n",
      " hcontig_000_003\t223\t224\t6mA\t0.5\t-\n",
      " hcontig_000_003\t225\t226\t6mA\t0.75\t+\n",
      " hcontig_000_003\t227\t228\t6mA\t0.5\t-\n",
      " hcontig_000_003\t228\t229\t6mA\t0.5\t-\n",
      " hcontig_000_003\t234\t235\t6mA\t1.0\t+\n",
      " hcontig_000_003\t239\t240\t6mA\t0.5\t-\n",
      " hcontig_000_003\t249\t250\t6mA\t0.75\t-\n",
      " hcontig_000_003\t250\t251\t6mA\t0.5\t+\n",
      " hcontig_000_003\t251\t252\t6mA\t0.5\t+\n",
      " hcontig_000_003\t252\t253\t6mA\t0.75\t-\n",
      " hcontig_000_003\t253\t254\t6mA\t0.75\t-\n",
      " hcontig_000_003\t255\t256\t6mA\t0.25\t+\n",
      " hcontig_000_003\t256\t257\t6mA\t0.25\t+\n",
      " hcontig_000_003\t257\t258\t6mA\t0.5\t-\n",
      " hcontig_000_003\t258\t259\t6mA\t0.25\t-\n",
      " hcontig_000_003\t259\t260\t6mA\t0.5\t-\n",
      " hcontig_000_003\t263\t264\t6mA\t1.0\t+\n",
      " hcontig_000_003\t264\t265\t6mA\t0.25\t-\n",
      " hcontig_000_003\t265\t266\t6mA\t0.25\t-\n",
      " hcontig_000_003\t266\t267\t6mA\t0.5\t-\n",
      " hcontig_000_003\t267\t268\t6mA\t0.75\t-\n",
      " hcontig_000_003\t269\t270\t6mA\t0.75\t+\n",
      " hcontig_000_003\t270\t271\t6mA\t0.5\t-\n",
      " hcontig_000_003\t273\t274\t6mA\t0.25\t+\n",
      " hcontig_000_003\t274\t275\t6mA\t0.25\t+\n",
      " hcontig_000_003\t277\t278\t6mA\t0.25\t+\n",
      " hcontig_000_003\t279\t280\t6mA\t0.25\t-\n",
      " hcontig_000_003\t280\t281\t6mA\t0.25\t-\n",
      " hcontig_000_003\t282\t283\t6mA\t0.75\t-\n",
      " hcontig_000_003\t284\t285\t6mA\t1.0\t+\n",
      " hcontig_000_003\t286\t287\t6mA\t0.5\t+\n",
      " hcontig_000_003\t288\t289\t6mA\t0.5\t+\n",
      " hcontig_000_003\t289\t290\t6mA\t0.75\t+\n",
      " hcontig_000_003\t292\t293\t6mA\t0.25\t-\n",
      " hcontig_000_003\t294\t295\t6mA\t0.25\t+\n",
      " hcontig_000_003\t295\t296\t6mA\t0.25\t-\n",
      " hcontig_000_003\t300\t301\t6mA\t0.25\t-\n",
      " hcontig_000_003\t303\t304\t6mA\t0.25\t-\n",
      " hcontig_000_003\t306\t307\t6mA\t0.75\t+\n",
      " hcontig_000_003\t307\t308\t6mA\t0.5\t+\n",
      " hcontig_000_003\t308\t309\t6mA\t0.5\t+\n",
      " hcontig_000_003\t309\t310\t6mA\t0.75\t-\n",
      " hcontig_000_003\t316\t317\t6mA\t0.5\t+\n",
      " hcontig_000_003\t322\t323\t6mA\t0.5\t+\n",
      " hcontig_000_003\t324\t325\t6mA\t0.75\t+\n",
      " hcontig_000_003\t328\t329\t6mA\t0.25\t+\n",
      " hcontig_000_003\t330\t331\t6mA\t0.75\t-\n",
      " hcontig_000_003\t331\t332\t6mA\t0.25\t+\n",
      " hcontig_000_003\t333\t334\t6mA\t1.0\t+\n",
      " hcontig_000_003\t335\t336\t6mA\t0.5\t-\n",
      " hcontig_000_003\t336\t337\t6mA\t0.75\t-\n",
      " hcontig_000_003\t338\t339\t6mA\t1.0\t-\n",
      " hcontig_000_003\t339\t340\t6mA\t1.0\t-\n",
      " hcontig_000_003\t340\t341\t6mA\t1.0\t-\n",
      " hcontig_000_003\t341\t342\t6mA\t0.75\t-\n",
      " hcontig_000_003\t343\t344\t6mA\t0.5\t-\n",
      " hcontig_000_003\t344\t345\t6mA\t0.75\t+\n",
      " hcontig_000_003\t347\t348\t6mA\t1.0\t+\n",
      " hcontig_000_003\t349\t350\t6mA\t0.75\t+\n",
      " hcontig_000_003\t351\t352\t6mA\t0.25\t+\n",
      " hcontig_000_003\t352\t353\t6mA\t0.5\t-\n",
      " hcontig_000_003\t353\t354\t6mA\t0.75\t+\n",
      " hcontig_000_003\t355\t356\t6mA\t0.25\t-\n",
      " hcontig_000_003\t356\t357\t6mA\t0.75\t+\n",
      " hcontig_000_003\t357\t358\t6mA\t0.75\t+\n",
      " hcontig_000_003\t358\t359\t6mA\t0.25\t-\n",
      " hcontig_000_003\t360\t361\t6mA\t0.5\t+\n",
      " hcontig_000_003\t361\t362\t6mA\t0.25\t+\n",
      " hcontig_000_003\t364\t365\t6mA\t0.25\t-\n",
      " hcontig_000_003\t369\t370\t6mA\t0.25\t+\n",
      " hcontig_000_003\t370\t371\t6mA\t0.25\t+\n",
      " hcontig_000_003\t373\t374\t6mA\t0.75\t-\n",
      " hcontig_000_003\t375\t376\t6mA\t0.75\t-\n",
      " hcontig_000_003\t377\t378\t6mA\t1.0\t+\n",
      " hcontig_000_003\t379\t380\t6mA\t0.75\t+\n",
      " hcontig_000_003\t381\t382\t6mA\t0.75\t+\n",
      " hcontig_000_003\t382\t383\t6mA\t0.75\t-\n",
      " hcontig_000_003\t383\t384\t6mA\t0.75\t-\n",
      " hcontig_000_003\t385\t386\t6mA\t1.0\t+\n",
      " hcontig_000_003\t390\t391\t6mA\t0.25\t-\n",
      " hcontig_000_003\t396\t397\t6mA\t0.25\t+\n",
      " hcontig_000_003\t397\t398\t6mA\t0.75\t-\n",
      " hcontig_000_003\t398\t399\t6mA\t0.5\t+\n",
      " hcontig_000_003\t400\t401\t6mA\t0.75\t-\n",
      " hcontig_000_003\t402\t403\t6mA\t1.0\t+\n",
      " hcontig_000_003\t405\t406\t6mA\t0.25\t+\n",
      " hcontig_000_003\t408\t409\t6mA\t0.25\t-\n",
      " hcontig_000_003\t411\t412\t6mA\t0.75\t-\n",
      " hcontig_000_003\t412\t413\t6mA\t0.75\t-\n",
      " hcontig_000_003\t413\t414\t6mA\t1.0\t+\n",
      " hcontig_000_003\t414\t415\t6mA\t1.0\t-\n",
      " hcontig_000_003\t415\t416\t6mA\t1.0\t+\n",
      " hcontig_000_003\t417\t418\t6mA\t0.75\t-\n",
      " hcontig_000_003\t419\t420\t6mA\t0.25\t-\n",
      " hcontig_000_003\t420\t421\t6mA\t0.25\t-\n",
      " hcontig_000_003\t426\t427\t6mA\t0.5\t-\n",
      " hcontig_000_003\t427\t428\t6mA\t1.0\t+\n",
      " hcontig_000_003\t428\t429\t6mA\t0.5\t+\n",
      " hcontig_000_003\t430\t431\t6mA\t0.5\t-\n",
      " hcontig_000_003\t441\t442\t6mA\t0.5\t+\n",
      " hcontig_000_003\t443\t444\t6mA\t0.5\t+\n",
      " hcontig_000_003\t447\t448\t6mA\t0.25\t+\n",
      " hcontig_000_003\t450\t451\t6mA\t1.0\t+\n",
      " hcontig_000_003\t451\t452\t6mA\t0.75\t+\n",
      " hcontig_000_003\t452\t453\t6mA\t0.75\t-\n",
      " hcontig_000_003\t454\t455\t6mA\t0.75\t-\n",
      " hcontig_000_003\t455\t456\t6mA\t0.5\t+\n",
      " hcontig_000_003\t459\t460\t6mA\t0.75\t+\n",
      " hcontig_000_003\t460\t461\t6mA\t0.25\t-\n",
      " hcontig_000_003\t462\t463\t6mA\t1.0\t+\n",
      " hcontig_000_003\t465\t466\t6mA\t0.5\t-\n",
      " hcontig_000_003\t466\t467\t6mA\t1.0\t+\n",
      " hcontig_000_003\t467\t468\t6mA\t0.75\t-\n",
      " hcontig_000_003\t469\t470\t6mA\t0.75\t-\n",
      " hcontig_000_003\t470\t471\t6mA\t1.0\t-\n",
      " hcontig_000_003\t471\t472\t6mA\t0.75\t-\n",
      " hcontig_000_003\t474\t475\t6mA\t0.75\t-\n",
      " hcontig_000_003\t476\t477\t6mA\t0.75\t+\n",
      " hcontig_000_003\t477\t478\t6mA\t0.75\t+\n",
      " hcontig_000_003\t479\t480\t6mA\t0.5\t-\n",
      " hcontig_000_003\t483\t484\t6mA\t0.75\t+\n",
      " hcontig_000_003\t484\t485\t6mA\t0.5\t-\n",
      " hcontig_000_003\t485\t486\t6mA\t0.25\t-\n",
      " hcontig_000_003\t488\t489\t6mA\t0.5\t+\n",
      " hcontig_000_003\t491\t492\t6mA\t0.5\t-\n",
      " hcontig_000_003\t499\t500\t6mA\t0.75\t-\n",
      " hcontig_000_003\t506\t507\t6mA\t0.25\t-\n",
      " hcontig_000_003\t507\t508\t6mA\t1.0\t+\n",
      " hcontig_000_003\t518\t519\t6mA\t0.25\t-\n",
      " hcontig_000_003\t520\t521\t6mA\t0.75\t-\n",
      " hcontig_000_003\t521\t522\t6mA\t1.0\t-\n",
      " hcontig_000_003\t522\t523\t6mA\t0.75\t-\n",
      " hcontig_000_003\t525\t526\t6mA\t0.5\t-\n",
      " hcontig_000_003\t526\t527\t6mA\t0.25\t+\n",
      " hcontig_000_003\t528\t529\t6mA\t0.5\t+\n",
      " hcontig_000_003\t532\t533\t6mA\t0.25\t-\n",
      " hcontig_000_003\t533\t534\t6mA\t0.75\t+\n",
      " hcontig_000_003\t534\t535\t6mA\t0.5\t-\n",
      " hcontig_000_003\t535\t536\t6mA\t0.75\t+\n",
      " hcontig_000_003\t536\t537\t6mA\t0.5\t-\n",
      " hcontig_000_003\t538\t539\t6mA\t0.75\t+\n",
      " hcontig_000_003\t539\t540\t6mA\t0.25\t-\n",
      " hcontig_000_003\t540\t541\t6mA\t0.5\t+\n",
      " hcontig_000_003\t542\t543\t6mA\t0.5\t+\n",
      " hcontig_000_003\t546\t547\t6mA\t0.5\t-\n",
      " hcontig_000_003\t549\t550\t6mA\t0.75\t+\n",
      " hcontig_000_003\t550\t551\t6mA\t0.25\t+\n",
      " hcontig_000_003\t551\t552\t6mA\t0.75\t-\n",
      " hcontig_000_003\t552\t553\t6mA\t0.75\t-\n",
      " hcontig_000_003\t553\t554\t6mA\t0.25\t+\n",
      " hcontig_000_003\t554\t555\t6mA\t1.0\t-\n",
      " hcontig_000_003\t555\t556\t6mA\t0.5\t+\n",
      " hcontig_000_003\t556\t557\t6mA\t1.0\t-\n",
      " hcontig_000_003\t557\t558\t6mA\t1.0\t-\n",
      " hcontig_000_003\t559\t560\t6mA\t0.25\t+\n",
      " hcontig_000_003\t560\t561\t6mA\t0.25\t-\n",
      " hcontig_000_003\t568\t569\t6mA\t0.25\t+\n",
      " hcontig_000_003\t575\t576\t6mA\t0.5\t-\n",
      " hcontig_000_003\t578\t579\t6mA\t0.5\t+\n",
      " hcontig_000_003\t584\t585\t6mA\t0.5\t+\n",
      " hcontig_000_003\t587\t588\t6mA\t0.5\t+\n",
      " hcontig_000_003\t592\t593\t6mA\t0.2\t-\n",
      " hcontig_000_003\t595\t596\t6mA\t0.5\t+\n",
      " hcontig_000_003\t596\t597\t6mA\t0.5\t+\n",
      " hcontig_000_003\t598\t599\t6mA\t0.4\t-\n",
      " hcontig_000_003\t600\t601\t6mA\t1.0\t+\n",
      " hcontig_000_003\t602\t603\t6mA\t0.6\t-\n",
      " hcontig_000_003\t603\t604\t6mA\t0.5\t+\n",
      " hcontig_000_003\t605\t606\t6mA\t0.6\t-\n",
      " hcontig_000_003\t607\t608\t6mA\t0.4\t-\n",
      " hcontig_000_003\t609\t610\t6mA\t0.5\t+\n",
      " hcontig_000_003\t610\t611\t6mA\t0.4\t-\n",
      " hcontig_000_003\t612\t613\t6mA\t0.6\t-\n",
      " hcontig_000_003\t617\t618\t6mA\t0.4\t-\n",
      " hcontig_000_003\t621\t622\t6mA\t0.8\t-\n",
      " hcontig_000_003\t622\t623\t6mA\t0.4\t-\n",
      " hcontig_000_003\t624\t625\t6mA\t0.4\t-\n",
      " hcontig_000_003\t625\t626\t6mA\t0.75\t+\n",
      " hcontig_000_003\t628\t629\t6mA\t0.5\t+\n",
      " hcontig_000_003\t629\t630\t6mA\t0.75\t+\n",
      " hcontig_000_003\t632\t633\t6mA\t0.4\t-\n",
      " hcontig_000_003\t634\t635\t6mA\t0.25\t+\n",
      " hcontig_000_003\t640\t641\t6mA\t0.4\t-\n",
      " hcontig_000_003\t643\t644\t6mA\t0.6\t-\n",
      " hcontig_000_003\t644\t645\t6mA\t0.6\t-\n",
      " hcontig_000_003\t646\t647\t6mA\t0.4\t-\n",
      " hcontig_000_003\t647\t648\t6mA\t0.2\t-\n",
      " hcontig_000_003\t648\t649\t6mA\t0.2\t-\n",
      " hcontig_000_003\t650\t651\t6mA\t0.2\t-\n",
      " hcontig_000_003\t652\t653\t6mA\t0.25\t+\n",
      " hcontig_000_003\t654\t655\t6mA\t0.4\t-\n",
      " hcontig_000_003\t659\t660\t6mA\t0.4\t-\n",
      " hcontig_000_003\t661\t662\t6mA\t0.25\t+\n",
      " hcontig_000_003\t664\t665\t6mA\t0.6\t-\n",
      " hcontig_000_003\t666\t667\t6mA\t0.4\t-\n",
      " hcontig_000_003\t670\t671\t6mA\t0.5\t+\n",
      " hcontig_000_003\t671\t672\t6mA\t0.75\t+\n",
      " hcontig_000_003\t676\t677\t6mA\t0.8\t-\n",
      " hcontig_000_003\t680\t681\t6mA\t0.2\t-\n",
      " hcontig_000_003\t692\t693\t6mA\t0.2\t-\n",
      " hcontig_000_003\t693\t694\t6mA\t1.0\t+\n",
      " hcontig_000_003\t694\t695\t6mA\t0.2\t-\n",
      " hcontig_000_003\t696\t697\t6mA\t0.2\t-\n",
      " hcontig_000_003\t700\t701\t6mA\t0.25\t+\n",
      " hcontig_000_003\t703\t704\t6mA\t0.25\t+\n",
      " hcontig_000_003\t705\t706\t6mA\t0.4\t-\n",
      " hcontig_000_003\t710\t711\t6mA\t0.2\t-\n",
      " hcontig_000_003\t711\t712\t6mA\t0.5\t+\n",
      " hcontig_000_003\t712\t713\t6mA\t0.8\t-\n",
      " hcontig_000_003\t716\t717\t6mA\t0.75\t+\n",
      " hcontig_000_003\t717\t718\t6mA\t0.4\t-\n",
      " hcontig_000_003\t718\t719\t6mA\t0.4\t-\n",
      " hcontig_000_003\t719\t720\t6mA\t0.6\t-\n",
      " hcontig_000_003\t722\t723\t6mA\t0.5\t+\n",
      " hcontig_000_003\t726\t727\t6mA\t1.0\t+\n",
      " hcontig_000_003\t728\t729\t6mA\t0.8\t-\n",
      " hcontig_000_003\t730\t731\t6mA\t1.0\t+\n",
      " hcontig_000_003\t735\t736\t6mA\t0.2\t-\n",
      " hcontig_000_003\t736\t737\t6mA\t0.5\t+\n",
      " hcontig_000_003\t737\t738\t6mA\t0.25\t+\n",
      " hcontig_000_003\t741\t742\t6mA\t0.5\t+\n",
      " hcontig_000_003\t745\t746\t6mA\t0.25\t+\n",
      " hcontig_000_003\t750\t751\t6mA\t1.0\t+\n",
      " hcontig_000_003\t751\t752\t6mA\t1.0\t+\n",
      " hcontig_000_003\t752\t753\t6mA\t1.0\t+\n",
      " hcontig_000_003\t754\t755\t6mA\t0.75\t+\n",
      " hcontig_000_003\t755\t756\t6mA\t0.5\t+\n",
      " hcontig_000_003\t758\t759\t6mA\t0.5\t+\n",
      " hcontig_000_003\t761\t762\t6mA\t0.5\t+\n",
      " hcontig_000_003\t767\t768\t6mA\t0.4\t-\n",
      " hcontig_000_003\t768\t769\t6mA\t0.6\t-\n",
      " hcontig_000_003\t769\t770\t6mA\t0.6\t-\n",
      " hcontig_000_003\t770\t771\t6mA\t0.6\t-\n",
      " hcontig_000_003\t771\t772\t6mA\t0.6\t-\n",
      " hcontig_000_003\t772\t773\t6mA\t0.6\t-\n",
      " hcontig_000_003\t773\t774\t6mA\t0.6\t-\n",
      " hcontig_000_003\t774\t775\t6mA\t0.8\t-\n",
      " hcontig_000_003\t775\t776\t6mA\t0.6\t-\n",
      " hcontig_000_003\t781\t782\t6mA\t0.25\t+\n",
      " hcontig_000_003\t782\t783\t6mA\t1.0\t-\n",
      " hcontig_000_003\t784\t785\t6mA\t0.25\t+\n",
      " hcontig_000_003\t786\t787\t6mA\t0.75\t+\n",
      " hcontig_000_003\t787\t788\t6mA\t0.75\t+\n",
      " hcontig_000_003\t788\t789\t6mA\t0.5\t+\n",
      " hcontig_000_003\t789\t790\t6mA\t0.5\t+\n",
      " hcontig_000_003\t791\t792\t6mA\t0.6\t-\n",
      " hcontig_000_003\t792\t793\t6mA\t0.25\t+\n",
      " hcontig_000_003\t793\t794\t6mA\t0.4\t-\n",
      " hcontig_000_003\t806\t807\t6mA\t0.5\t+\n",
      " hcontig_000_003\t807\t808\t6mA\t0.5\t+\n",
      " hcontig_000_003\t810\t811\t6mA\t0.75\t+\n",
      " hcontig_000_003\t811\t812\t6mA\t0.2\t-\n",
      " hcontig_000_003\t813\t814\t6mA\t0.2\t-\n",
      " hcontig_000_003\t814\t815\t6mA\t0.2\t-\n",
      " hcontig_000_003\t818\t819\t6mA\t0.75\t+\n",
      " hcontig_000_003\t819\t820\t6mA\t0.4\t-\n",
      " hcontig_000_003\t820\t821\t6mA\t0.4\t-\n",
      " hcontig_000_003\t823\t824\t6mA\t0.2\t-\n",
      " hcontig_000_003\t826\t827\t6mA\t0.2\t-\n",
      " hcontig_000_003\t829\t830\t6mA\t0.4\t-\n",
      " hcontig_000_003\t831\t832\t6mA\t0.6\t-\n",
      " hcontig_000_003\t833\t834\t6mA\t1.0\t+\n",
      " hcontig_000_003\t837\t838\t6mA\t0.25\t+\n",
      " hcontig_000_003\t838\t839\t6mA\t0.25\t+\n",
      " hcontig_000_003\t841\t842\t6mA\t0.2\t-\n",
      " hcontig_000_003\t843\t844\t6mA\t1.0\t+\n",
      " hcontig_000_003\t847\t848\t6mA\t0.2\t-\n",
      " hcontig_000_003\t849\t850\t6mA\t0.25\t+\n",
      " hcontig_000_003\t857\t858\t6mA\t0.75\t+\n",
      " hcontig_000_003\t858\t859\t6mA\t0.4\t-\n",
      " hcontig_000_003\t861\t862\t6mA\t0.5\t+\n",
      " hcontig_000_003\t865\t866\t6mA\t1.0\t+\n",
      " hcontig_000_003\t867\t868\t6mA\t0.6\t-\n",
      " hcontig_000_003\t873\t874\t6mA\t0.2\t-\n",
      " hcontig_000_003\t874\t875\t6mA\t0.25\t+\n",
      " hcontig_000_003\t876\t877\t6mA\t0.4\t-\n",
      " hcontig_000_003\t879\t880\t6mA\t0.6\t-\n",
      " hcontig_000_003\t881\t882\t6mA\t0.6\t-\n",
      " hcontig_000_003\t882\t883\t6mA\t0.75\t+\n",
      " hcontig_000_003\t885\t886\t6mA\t0.6\t-\n",
      " hcontig_000_003\t886\t887\t6mA\t0.75\t+\n",
      " hcontig_000_003\t887\t888\t6mA\t0.2\t-\n",
      " hcontig_000_003\t891\t892\t6mA\t0.75\t+\n",
      " hcontig_000_003\t892\t893\t6mA\t0.2\t-\n",
      " hcontig_000_003\t893\t894\t6mA\t0.4\t-\n",
      " hcontig_000_003\t897\t898\t6mA\t0.2\t-\n",
      " hcontig_000_003\t898\t899\t6mA\t0.5\t+\n",
      " hcontig_000_003\t900\t901\t6mA\t0.5\t+\n",
      " hcontig_000_003\t901\t902\t6mA\t0.5\t+\n",
      " hcontig_000_003\t902\t903\t6mA\t0.5\t+\n",
      " hcontig_000_003\t904\t905\t6mA\t0.5\t+\n",
      " hcontig_000_003\t905\t906\t6mA\t0.5\t+\n",
      " hcontig_000_003\t906\t907\t6mA\t0.5\t+\n",
      " hcontig_000_003\t909\t910\t6mA\t0.2\t-\n",
      " hcontig_000_003\t914\t915\t6mA\t0.5\t+\n",
      " hcontig_000_003\t915\t916\t6mA\t0.5\t+\n",
      " hcontig_000_003\t916\t917\t6mA\t0.4\t-\n",
      " hcontig_000_003\t917\t918\t6mA\t0.4\t-\n",
      " hcontig_000_003\t918\t919\t6mA\t0.25\t+\n",
      " hcontig_000_003\t921\t922\t6mA\t0.5\t+\n",
      " hcontig_000_003\t922\t923\t6mA\t0.2\t-\n",
      " hcontig_000_003\t929\t930\t6mA\t0.5\t+\n",
      " hcontig_000_003\t931\t932\t6mA\t0.8\t-\n",
      " hcontig_000_003\t935\t936\t6mA\t0.6\t-\n",
      " hcontig_000_003\t938\t939\t6mA\t0.25\t+\n",
      " hcontig_000_003\t940\t941\t6mA\t0.5\t+\n",
      " hcontig_000_003\t943\t944\t6mA\t0.75\t+\n",
      " hcontig_000_003\t945\t946\t6mA\t0.8\t-\n",
      " hcontig_000_003\t946\t947\t6mA\t0.4\t-\n",
      " hcontig_000_003\t947\t948\t6mA\t0.8\t-\n",
      " hcontig_000_003\t948\t949\t6mA\t0.8\t-\n",
      " hcontig_000_003\t950\t951\t6mA\t0.75\t+\n",
      " hcontig_000_003\t952\t953\t6mA\t0.8\t-\n",
      " hcontig_000_003\t955\t956\t6mA\t0.4\t-\n",
      " hcontig_000_003\t956\t957\t6mA\t0.2\t-\n",
      " hcontig_000_003\t958\t959\t6mA\t0.2\t-\n",
      " hcontig_000_003\t961\t962\t6mA\t0.6\t-\n",
      " hcontig_000_003\t963\t964\t6mA\t0.4\t-\n",
      " hcontig_000_003\t965\t966\t6mA\t0.4\t-\n",
      " hcontig_000_003\t966\t967\t6mA\t0.6\t-\n",
      " hcontig_000_003\t969\t970\t6mA\t0.4\t-\n",
      " hcontig_000_003\t971\t972\t6mA\t0.75\t+\n",
      " hcontig_000_003\t982\t983\t6mA\t0.6\t-\n",
      " hcontig_000_003\t989\t990\t6mA\t0.25\t+\n",
      " hcontig_000_003\t990\t991\t6mA\t0.75\t+\n",
      " hcontig_000_003\t991\t992\t6mA\t0.6\t-\n",
      " hcontig_000_003\t993\t994\t6mA\t1.0\t+\n",
      " hcontig_000_003\t996\t997\t6mA\t0.4\t-\n",
      " hcontig_000_003\t997\t998\t6mA\t0.2\t-\n",
      " hcontig_000_003\t999\t1000\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1000\t1001\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1002\t1003\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1009\t1010\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1010\t1011\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1014\t1015\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1015\t1016\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1020\t1021\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1022\t1023\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1023\t1024\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1025\t1026\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1027\t1028\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1028\t1029\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1031\t1032\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1032\t1033\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1036\t1037\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1039\t1040\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1041\t1042\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1042\t1043\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1044\t1045\t6mA\t1.0\t-\n",
      " hcontig_000_003\t1047\t1048\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1048\t1049\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1050\t1051\t6mA\t1.0\t-\n",
      " hcontig_000_003\t1053\t1054\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1055\t1056\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1058\t1059\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1059\t1060\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1061\t1062\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1062\t1063\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1067\t1068\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1068\t1069\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1069\t1070\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1070\t1071\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1071\t1072\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1072\t1073\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1074\t1075\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1077\t1078\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1085\t1086\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1086\t1087\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1087\t1088\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1088\t1089\t6mA\t1.0\t-\n",
      " hcontig_000_003\t1090\t1091\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1091\t1092\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1092\t1093\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1093\t1094\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1094\t1095\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1096\t1097\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1097\t1098\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1098\t1099\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1100\t1101\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1101\t1102\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1105\t1106\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1106\t1107\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1116\t1117\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1120\t1121\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1121\t1122\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1123\t1124\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1124\t1125\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1125\t1126\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1130\t1131\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1131\t1132\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1133\t1134\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1140\t1141\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1141\t1142\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1142\t1143\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1143\t1144\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1144\t1145\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1147\t1148\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1149\t1150\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1150\t1151\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1151\t1152\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1152\t1153\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1154\t1155\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1155\t1156\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1156\t1157\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1160\t1161\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1165\t1166\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1166\t1167\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1167\t1168\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1168\t1169\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1169\t1170\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1171\t1172\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1175\t1176\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1177\t1178\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1180\t1181\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1187\t1188\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1189\t1190\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1191\t1192\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1194\t1195\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1195\t1196\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1199\t1200\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1200\t1201\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1213\t1214\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1217\t1218\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1218\t1219\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1219\t1220\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1221\t1222\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1222\t1223\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1223\t1224\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1226\t1227\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1227\t1228\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1230\t1231\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1238\t1239\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1239\t1240\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1249\t1250\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1251\t1252\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1254\t1255\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1257\t1258\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1260\t1261\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1261\t1262\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1262\t1263\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1263\t1264\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1267\t1268\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1269\t1270\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1272\t1273\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1275\t1276\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1278\t1279\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1285\t1286\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1289\t1290\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1293\t1294\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1297\t1298\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1298\t1299\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1301\t1302\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1302\t1303\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1308\t1309\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1310\t1311\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1317\t1318\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1318\t1319\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1319\t1320\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1320\t1321\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1321\t1322\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1322\t1323\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1323\t1324\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1324\t1325\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1325\t1326\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1327\t1328\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1329\t1330\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1330\t1331\t6mA\t1.0\t-\n",
      " hcontig_000_003\t1334\t1335\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1336\t1337\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1338\t1339\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1339\t1340\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1342\t1343\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1347\t1348\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1348\t1349\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1349\t1350\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1352\t1353\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1354\t1355\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1356\t1357\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1357\t1358\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1361\t1362\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1368\t1369\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1371\t1372\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1372\t1373\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1374\t1375\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1375\t1376\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1378\t1379\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1380\t1381\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1382\t1383\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1383\t1384\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1384\t1385\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1385\t1386\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1388\t1389\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1391\t1392\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1392\t1393\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1393\t1394\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1394\t1395\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1395\t1396\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1396\t1397\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1397\t1398\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1399\t1400\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1400\t1401\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1402\t1403\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1404\t1405\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1406\t1407\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1407\t1408\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1408\t1409\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1409\t1410\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1411\t1412\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1412\t1413\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1413\t1414\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1415\t1416\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1420\t1421\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1424\t1425\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1426\t1427\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1427\t1428\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1429\t1430\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1432\t1433\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1433\t1434\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1434\t1435\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1435\t1436\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1436\t1437\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1439\t1440\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1444\t1445\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1445\t1446\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1446\t1447\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1449\t1450\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1452\t1453\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1453\t1454\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1454\t1455\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1456\t1457\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1457\t1458\t6mA\t0.25\t+\n",
      " hcontig_000_003\t1460\t1461\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1461\t1462\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1463\t1464\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1465\t1466\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1466\t1467\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1467\t1468\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1469\t1470\t6mA\t0.75\t+\n",
      " hcontig_000_003\t1472\t1473\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1473\t1474\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1476\t1477\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1477\t1478\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1479\t1480\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1481\t1482\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1483\t1484\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1488\t1489\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1492\t1493\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1495\t1496\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1498\t1499\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1500\t1501\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1505\t1506\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1507\t1508\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1509\t1510\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1511\t1512\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1517\t1518\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1520\t1521\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1521\t1522\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1523\t1524\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1526\t1527\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1527\t1528\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1531\t1532\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1532\t1533\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1534\t1535\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1535\t1536\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1536\t1537\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1537\t1538\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1539\t1540\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1540\t1541\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1541\t1542\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1543\t1544\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1549\t1550\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1551\t1552\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1554\t1555\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1555\t1556\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1556\t1557\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1559\t1560\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1560\t1561\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1561\t1562\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1562\t1563\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1569\t1570\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1570\t1571\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1571\t1572\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1572\t1573\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1573\t1574\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1574\t1575\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1575\t1576\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1579\t1580\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1580\t1581\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1582\t1583\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1584\t1585\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1587\t1588\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1589\t1590\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1590\t1591\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1592\t1593\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1595\t1596\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1596\t1597\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1598\t1599\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1602\t1603\t6mA\t1.0\t-\n",
      " hcontig_000_003\t1604\t1605\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1605\t1606\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1609\t1610\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1610\t1611\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1612\t1613\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1613\t1614\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1614\t1615\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1620\t1621\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1621\t1622\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1622\t1623\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1623\t1624\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1625\t1626\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1626\t1627\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1628\t1629\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1631\t1632\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1635\t1636\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1640\t1641\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1641\t1642\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1642\t1643\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1645\t1646\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1646\t1647\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1648\t1649\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1650\t1651\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1653\t1654\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1655\t1656\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1656\t1657\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1657\t1658\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1658\t1659\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1664\t1665\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1665\t1666\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1667\t1668\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1669\t1670\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1670\t1671\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1671\t1672\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1674\t1675\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1678\t1679\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1679\t1680\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1682\t1683\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1684\t1685\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1687\t1688\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1689\t1690\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1690\t1691\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1692\t1693\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1693\t1694\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1695\t1696\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1697\t1698\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1698\t1699\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1699\t1700\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1701\t1702\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1702\t1703\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1703\t1704\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1704\t1705\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1706\t1707\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1709\t1710\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1711\t1712\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1712\t1713\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1715\t1716\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1716\t1717\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1717\t1718\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1723\t1724\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1726\t1727\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1729\t1730\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1730\t1731\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1733\t1734\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1734\t1735\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1735\t1736\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1740\t1741\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1741\t1742\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1742\t1743\t6mA\t1.0\t-\n",
      " hcontig_000_003\t1746\t1747\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1748\t1749\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1750\t1751\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1751\t1752\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1752\t1753\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1755\t1756\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1756\t1757\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1758\t1759\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1761\t1762\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1765\t1766\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1766\t1767\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1768\t1769\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1770\t1771\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1771\t1772\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1779\t1780\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1780\t1781\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1783\t1784\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1784\t1785\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1785\t1786\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1787\t1788\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1788\t1789\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1790\t1791\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1791\t1792\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1793\t1794\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1795\t1796\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1796\t1797\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1797\t1798\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1798\t1799\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1801\t1802\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1802\t1803\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1804\t1805\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1805\t1806\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1807\t1808\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1808\t1809\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1809\t1810\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1813\t1814\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1816\t1817\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1817\t1818\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1819\t1820\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1821\t1822\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1822\t1823\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1823\t1824\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1826\t1827\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1827\t1828\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1828\t1829\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1830\t1831\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1831\t1832\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1833\t1834\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1836\t1837\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1838\t1839\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1839\t1840\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1842\t1843\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1843\t1844\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1844\t1845\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1845\t1846\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1846\t1847\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1849\t1850\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1851\t1852\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1853\t1854\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1854\t1855\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1855\t1856\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1857\t1858\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1866\t1867\t6mA\t0.6\t+\n",
      " hcontig_000_003\t1867\t1868\t6mA\t0.8\t+\n",
      " hcontig_000_003\t1869\t1870\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1871\t1872\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1872\t1873\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1874\t1875\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1875\t1876\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1876\t1877\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1881\t1882\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1886\t1887\t6mA\t0.2\t+\n",
      " hcontig_000_003\t1891\t1892\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1893\t1894\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1894\t1895\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1896\t1897\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1897\t1898\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1898\t1899\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1900\t1901\t6mA\t0.4\t+\n",
      " hcontig_000_003\t1902\t1903\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t1903\t1904\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1904\t1905\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1905\t1906\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1907\t1908\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1909\t1910\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t1910\t1911\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1913\t1914\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t1914\t1915\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1916\t1917\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t1918\t1919\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1921\t1922\t6mA\t0.8333\t+\n",
      " hcontig_000_003\t1927\t1928\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1930\t1931\t6mA\t1.0\t+\n",
      " hcontig_000_003\t1935\t1936\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t1938\t1939\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1939\t1940\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1942\t1943\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t1943\t1944\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t1949\t1950\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1951\t1952\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1952\t1953\t6mA\t0.8\t-\n",
      " hcontig_000_003\t1953\t1954\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1954\t1955\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1958\t1959\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1962\t1963\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1968\t1969\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t1975\t1976\t6mA\t0.2\t-\n",
      " hcontig_000_003\t1980\t1981\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1981\t1982\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1983\t1984\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1984\t1985\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1985\t1986\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1986\t1987\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1987\t1988\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t1988\t1989\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t1989\t1990\t6mA\t0.8333\t+\n",
      " hcontig_000_003\t1992\t1993\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1994\t1995\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1996\t1997\t6mA\t0.5\t+\n",
      " hcontig_000_003\t1997\t1998\t6mA\t0.6\t-\n",
      " hcontig_000_003\t1998\t1999\t6mA\t0.4\t-\n",
      " hcontig_000_003\t1999\t2000\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2001\t2002\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2002\t2003\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2011\t2012\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t2013\t2014\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2014\t2015\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2016\t2017\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2017\t2018\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2018\t2019\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2019\t2020\t6mA\t0.8\t-\n",
      " hcontig_000_003\t2021\t2022\t6mA\t0.8\t-\n",
      " hcontig_000_003\t2024\t2025\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2025\t2026\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2026\t2027\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t2027\t2028\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2028\t2029\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2029\t2030\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2031\t2032\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2032\t2033\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2037\t2038\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2038\t2039\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t2040\t2041\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t2043\t2044\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2048\t2049\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2053\t2054\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2054\t2055\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2055\t2056\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2056\t2057\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2057\t2058\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2061\t2062\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t2069\t2070\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2070\t2071\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2071\t2072\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2072\t2073\t6mA\t0.8\t-\n",
      " hcontig_000_003\t2073\t2074\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2075\t2076\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2076\t2077\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t2081\t2082\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2082\t2083\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2084\t2085\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t2085\t2086\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t2087\t2088\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2090\t2091\t6mA\t0.8333\t+\n",
      " hcontig_000_003\t2094\t2095\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2096\t2097\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2098\t2099\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2099\t2100\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2101\t2102\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2102\t2103\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2106\t2107\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2107\t2108\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2108\t2109\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2109\t2110\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2111\t2112\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2112\t2113\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2114\t2115\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2116\t2117\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2118\t2119\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2119\t2120\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2121\t2122\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2122\t2123\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2123\t2124\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t2125\t2126\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2126\t2127\t6mA\t0.2\t-\n",
      " hcontig_000_003\t2129\t2130\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2133\t2134\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t2134\t2135\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2135\t2136\t6mA\t0.1667\t+\n",
      " hcontig_000_003\t2136\t2137\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2137\t2138\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2141\t2142\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2142\t2143\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2147\t2148\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2149\t2150\t6mA\t0.6\t-\n",
      " hcontig_000_003\t2150\t2151\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2152\t2153\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2155\t2156\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2156\t2157\t6mA\t0.4\t-\n",
      " hcontig_000_003\t2160\t2161\t6mA\t0.3333\t+\n",
      " hcontig_000_003\t2162\t2163\t6mA\t0.8\t-\n",
      " hcontig_000_003\t2165\t2166\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2166\t2167\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2169\t2170\t6mA\t0.5\t+\n",
      " hcontig_000_003\t2172\t2173\t6mA\t0.6667\t+\n",
      " hcontig_000_003\t2173\t2174\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2174\t2175\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2175\t2176\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2176\t2177\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2177\t2178\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2178\t2179\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2181\t2182\t6mA\t0.2857\t+\n",
      " hcontig_000_003\t2182\t2183\t6mA\t0.2857\t+\n",
      " hcontig_000_003\t2183\t2184\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2185\t2186\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2186\t2187\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2189\t2190\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2190\t2191\t6mA\t0.5714\t+\n",
      " hcontig_000_003\t2191\t2192\t6mA\t0.6667\t-\n",
      " hcontig_000_003\t2193\t2194\t6mA\t0.2857\t+\n",
      " hcontig_000_003\t2194\t2195\t6mA\t0.2857\t+\n",
      " hcontig_000_003\t2195\t2196\t6mA\t0.5\t-\n",
      " hcontig_000_003\t2196\t2197\t6mA\t0.1429\t+\n",
      " hcontig_000_003\t2197\t2198\t6mA\t0.1429\t+\n",
      " hcontig_000_003\t2198\t2199\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2199\t2200\t6mA\t0.2857\t+\n",
      " hcontig_000_003\t2200\t2201\t6mA\t0.5\t-\n",
      " hcontig_000_003\t2201\t2202\t6mA\t0.1429\t+\n",
      " hcontig_000_003\t2203\t2204\t6mA\t0.1429\t+\n",
      " hcontig_000_003\t2204\t2205\t6mA\t0.1429\t+\n",
      " hcontig_000_003\t2206\t2207\t6mA\t0.8333\t-\n",
      " hcontig_000_003\t2209\t2210\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2212\t2213\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2214\t2215\t6mA\t0.4286\t+\n",
      " hcontig_000_003\t2215\t2216\t6mA\t0.5714\t+\n",
      " hcontig_000_003\t2216\t2217\t6mA\t0.1667\t-\n",
      " hcontig_000_003\t2218\t2219\t6mA\t0.5714\t+\n",
      " hcontig_000_003\t2219\t2220\t6mA\t0.5\t-\n",
      " hcontig_000_003\t2220\t2221\t6mA\t0.5\t-\n",
      " hcontig_000_003\t2223\t2224\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2225\t2226\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2226\t2227\t6mA\t0.3333\t-\n",
      " hcontig_000_003\t2231\t2232\t6mA\t0.6667\t-\n",
      " hcontig_000_003\t2233\t2234\t6mA\t0.8571\t+\n",
      " hcontig_000_003\t2234\t2235\t6mA\t1.0\t+\n",
      " hcontig_000_003\t2236\t2237\t6mA\t0.1667\t-\n",
      " "
     ]
    },
    {
     "ename": "AttributeError",
     "evalue": "'NoneType' object has no attribute 'head'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2243-175690f36dd9>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0mmethyl_bed\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mmethylbed_dict\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'6mA_hc_tombo_sorted.cutoff.0.00.bed'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mhead\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mn\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m1000\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mmethyl_bed\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mhead\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'head'"
     ]
    }
   ],
   "source": [
    "methyl_bed = methylbed_dict['6mA_hc_tombo_sorted.cutoff.0.00.bed'].head(n=1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# the pybedtools windows() function was used to get overlaps upstream and downstream of genes\n",
    "# because windows() also detects methylation inside the gene, subtract() was run first\n",
    "# the subtracted file undergoes the window() function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2181,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Pst_104E_v13_h_ctg.TE.sorted.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg.TE.sorted.bed)>,\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg_combined_sorted_anno.bed)>,\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_p_ctg.TE.sorted.bed)>,\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_p_ctg_combined_sorted_anno.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.TE.sorted.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg.effectors.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.effectors.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.non_effectors.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg_combined_sorted_anno.bed)>}"
      ]
     },
     "execution_count": 2181,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2182,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.00.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed)>}"
      ]
     },
     "execution_count": 2182,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "methyl_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2184,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Pst_104E_v13_h_ctg.TE.sorted.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg.TE.sorted.bed',\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg_combined_sorted_anno.bed',\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_p_ctg.TE.sorted.bed',\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_p_ctg_combined_sorted_anno.bed',\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.TE.sorted.bed',\n",
       " 'Pst_104E_v13_ph_ctg.effectors.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.effectors.bed',\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg.non_effectors.bed',\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.bed': '/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_ph_ctg_combined_sorted_anno.bed'}"
      ]
     },
     "execution_count": 2184,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_anno_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2183,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hcontig_000_003\t14562\t15702\tEVM prediction%2hcontig_0000_003.5\t.\t+\n",
      " hcontig_000_003\t17795\t18752\tEVM prediction%2hcontig_0000_003.6\t.\t+\n",
      " hcontig_000_003\t80525\t81601\tEVM prediction%2hcontig_0000_003.21\t.\t+\n",
      " hcontig_000_003\t81736\t83619\tEVM prediction%2hcontig_0000_003.22\t.\t+\n",
      " hcontig_000_003\t108981\t112995\tEVM prediction%2hcontig_0000_003.25\t.\t+\n",
      " hcontig_000_003\t180425\t180836\tEVM prediction%2hcontig_0000_003.45\t.\t+\n",
      " hcontig_000_003\t182250\t182751\tEVM prediction%2hcontig_0000_003.46\t.\t+\n",
      " hcontig_000_003\t182796\t183867\tEVM prediction%2hcontig_0000_003.47\t.\t+\n",
      " hcontig_000_003\t189995\t190718\tEVM prediction%2hcontig_0000_003.48\t.\t+\n",
      " hcontig_000_003\t269647\t274814\tEVM prediction%2hcontig_0000_003.69\t.\t-\n",
      " hcontig_000_003\t329808\t331131\tEVM prediction%2hcontig_0000_003.81\t.\t-\n",
      " hcontig_000_003\t344036\t344906\tEVM prediction%2hcontig_0000_003.84\t.\t+\n",
      " hcontig_000_003\t345369\t345747\tEVM prediction%2hcontig_0000_003.85\t.\t-\n",
      " hcontig_000_003\t402060\t403644\tEVM prediction%2hcontig_0000_003.100\t.\t-\n",
      " hcontig_000_003\t406700\t411752\tEVM prediction%2hcontig_0000_003.102\t.\t-\n",
      " hcontig_000_003\t452052\t458688\tEVM prediction%2hcontig_0000_003.113\t.\t+\n",
      " hcontig_000_003\t461215\t461695\tEVM prediction%2hcontig_0000_003.114\t.\t-\n",
      " hcontig_000_003\t464659\t465209\tEVM prediction%2hcontig_0000_003.115\t.\t-\n",
      " hcontig_000_003\t524253\t524571\tEVM prediction%2hcontig_0000_003.128\t.\t-\n",
      " hcontig_000_003\t524807\t525437\tEVM prediction%2hcontig_0000_003.129\t.\t-\n",
      " hcontig_000_003\t525753\t527436\tEVM prediction%2hcontig_0000_003.130\t.\t-\n",
      " hcontig_000_003\t530399\t533666\tEVM prediction%2hcontig_0000_003.132\t.\t-\n",
      " hcontig_000_003\t538838\t539861\tEVM prediction%2hcontig_0000_003.134\t.\t-\n",
      " hcontig_000_003\t547041\t547965\tEVM prediction%2hcontig_0000_003.136\t.\t+\n",
      " hcontig_000_003\t570436\t571210\tEVM prediction%2hcontig_0000_003.140\t.\t-\n",
      " hcontig_000_003\t571657\t574243\tEVM prediction%2hcontig_0000_003.141\t.\t-\n",
      " hcontig_000_003\t575825\t576987\tEVM prediction%2hcontig_0000_003.143\t.\t+\n",
      " hcontig_000_003\t670734\t672992\tEVM prediction%2hcontig_0000_003.161\t.\t+\n",
      " hcontig_000_003\t687456\t688125\tEVM prediction%2hcontig_0000_003.166\t.\t-\n",
      " hcontig_000_003\t689640\t690882\tEVM prediction%2hcontig_0000_003.167\t.\t-\n",
      " hcontig_000_003\t692754\t693396\tEVM prediction%2hcontig_0000_003.168\t.\t+\n",
      " hcontig_000_003\t695062\t698545\tEVM prediction%2hcontig_0000_003.169\t.\t+\n",
      " hcontig_000_003\t701722\t704851\tEVM prediction%2hcontig_0000_003.171\t.\t+\n",
      " hcontig_000_003\t707654\t708197\tEVM prediction%2hcontig_0000_003.172\t.\t+\n",
      " hcontig_000_003\t709665\t710232\tEVM prediction%2hcontig_0000_003.173\t.\t+\n",
      " hcontig_000_003\t729818\t733172\tEVM prediction%2hcontig_0000_003.179\t.\t+\n",
      " hcontig_000_003\t807118\t808321\tEVM prediction%2hcontig_0000_003.197\t.\t-\n",
      " hcontig_000_003\t1050290\t1051298\tEVM prediction%2hcontig_0000_003.247\t.\t+\n",
      " hcontig_000_003\t1053136\t1053978\tEVM prediction%2hcontig_0000_003.248\t.\t+\n",
      " hcontig_000_003\t1069350\t1070533\tEVM prediction%2hcontig_0000_003.251\t.\t-\n",
      " hcontig_000_003\t1073739\t1076310\tEVM prediction%2hcontig_0000_003.252\t.\t+\n",
      " hcontig_000_003\t1080824\t1082023\tEVM prediction%2hcontig_0000_003.253\t.\t-\n",
      " hcontig_000_003\t1090907\t1095155\tEVM prediction%2hcontig_0000_003.256\t.\t+\n",
      " hcontig_000_003\t1174820\t1175384\tEVM prediction%2hcontig_0000_003.279\t.\t+\n",
      " hcontig_000_003\t1204252\t1206126\tEVM prediction%2hcontig_0000_003.285\t.\t-\n",
      " hcontig_000_003\t1219735\t1221595\tEVM prediction%2hcontig_0000_003.288\t.\t-\n",
      " hcontig_000_003\t1226550\t1228510\tEVM prediction%2hcontig_0000_003.289\t.\t+\n",
      " hcontig_000_003\t1247411\t1248422\tEVM prediction%2hcontig_0000_003.293\t.\t+\n",
      " hcontig_000_003\t1257328\t1261894\tEVM prediction%2hcontig_0000_003.297\t.\t+\n",
      " hcontig_000_003\t1266453\t1269385\tEVM prediction%2hcontig_0000_003.301\t.\t+\n",
      " hcontig_000_003\t1272184\t1273139\tEVM prediction%2hcontig_0000_003.302\t.\t+\n",
      " hcontig_000_003\t1275101\t1275422\tEVM prediction%2hcontig_0000_003.303\t.\t+\n",
      " hcontig_000_003\t1301031\t1303059\tEVM prediction%2hcontig_0000_003.310\t.\t-\n",
      " hcontig_000_003\t1437602\t1438013\tEVM prediction%2hcontig_0000_003.344\t.\t-\n",
      " hcontig_000_003\t1441056\t1441362\tEVM prediction%2hcontig_0000_003.345\t.\t-\n",
      " hcontig_000_003\t1453956\t1454940\tEVM prediction%2hcontig_0000_003.347\t.\t-\n",
      " hcontig_000_003\t1467582\t1469136\tEVM prediction%2hcontig_0000_003.349\t.\t+\n",
      " hcontig_000_003\t1472228\t1472849\tEVM prediction%2hcontig_0000_003.351\t.\t+\n",
      " hcontig_000_003\t1481052\t1481379\tEVM prediction%2hcontig_0000_003.353\t.\t+\n",
      " hcontig_000_003\t1482571\t1483321\tEVM prediction%2hcontig_0000_003.354\t.\t-\n",
      " hcontig_000_003\t1483822\t1486556\tEVM prediction%2hcontig_0000_003.355\t.\t-\n",
      " hcontig_000_003\t1488253\t1489114\tEVM prediction%2hcontig_0000_003.356\t.\t+\n",
      " hcontig_000_003\t1490645\t1491629\tEVM prediction%2hcontig_0000_003.357\t.\t+\n",
      " hcontig_000_003\t1615029\t1615458\tEVM prediction%2hcontig_0000_003.365\t.\t-\n",
      " hcontig_000_003\t1626739\t1628220\tEVM prediction%2hcontig_0000_003.366\t.\t-\n",
      " hcontig_000_003\t1631276\t1631795\tEVM prediction%2hcontig_0000_003.367\t.\t-\n",
      " hcontig_000_003\t1633584\t1635690\tEVM prediction%2hcontig_0000_003.368\t.\t-\n",
      " hcontig_000_003\t1635824\t1639456\tEVM prediction%2hcontig_0000_003.369\t.\t-\n",
      " hcontig_000_003\t1775521\t1776199\tEVM prediction%2hcontig_0000_003.396\t.\t-\n",
      " hcontig_000_003\t1779182\t1779893\tEVM prediction%2hcontig_0000_003.397\t.\t-\n",
      " hcontig_000_003\t1805292\t1808725\tEVM prediction%2hcontig_0000_003.406\t.\t-\n",
      " hcontig_000_003\t1825135\t1826215\tEVM prediction%2hcontig_0000_003.410\t.\t+\n",
      " hcontig_000_003\t1839822\t1840917\tEVM prediction%2hcontig_0000_003.411\t.\t+\n",
      " hcontig_000_003\t1957496\t1958021\tEVM prediction%2hcontig_0000_003.433\t.\t+\n",
      " hcontig_000_003\t1968886\t1971925\tEVM prediction%2hcontig_0000_003.436\t.\t-\n",
      " hcontig_000_003\t1979424\t1983594\tEVM prediction%2hcontig_0000_003.439\t.\t+\n",
      " hcontig_000_003\t1986197\t1988044\tEVM prediction%2hcontig_0000_003.441\t.\t-\n",
      " hcontig_000_005\t6041\t7043\tEVM prediction%2hcontig_0000_005.2\t.\t+\n",
      " hcontig_000_024\t32048\t34358\tEVM prediction%2hcontig_0000_024.6\t.\t-\n",
      " hcontig_000_027\t4695\t7404\tEVM prediction%2hcontig_0000_027.1\t.\t+\n",
      " hcontig_000_027\t7488\t10734\tEVM prediction%2hcontig_0000_027.2\t.\t+\n",
      " hcontig_000_027\t31337\t33137\tEVM prediction%2hcontig_0000_027.7\t.\t+\n",
      " hcontig_000_029\t3207\t4410\tEVM prediction%2hcontig_0000_029.1\t.\t-\n",
      " hcontig_000_037\t4027\t5194\tEVM prediction%2hcontig_0000_037.1\t.\t-\n",
      " hcontig_000_037\t7465\t7771\tEVM prediction%2hcontig_0000_037.2\t.\t-\n",
      " hcontig_000_050\t103841\t104504\tEVM prediction%2hcontig_0000_050.22\t.\t-\n",
      " hcontig_000_050\t105063\t107364\tEVM prediction%2hcontig_0000_050.23\t.\t+\n",
      " hcontig_000_050\t107427\t109758\tEVM prediction%2hcontig_0000_050.24\t.\t+\n",
      " hcontig_000_050\t110388\t111377\tEVM prediction%2hcontig_0000_050.25\t.\t-\n",
      " hcontig_000_050\t239038\t240234\tEVM prediction%2hcontig_0000_050.55\t.\t+\n",
      " hcontig_000_050\t241408\t242335\tEVM prediction%2hcontig_0000_050.56\t.\t-\n",
      " hcontig_000_050\t258679\t259654\tEVM prediction%2hcontig_0000_050.59\t.\t-\n",
      " hcontig_000_050\t259854\t261450\tEVM prediction%2hcontig_0000_050.60\t.\t-\n",
      " hcontig_000_050\t294020\t296815\tEVM prediction%2hcontig_0000_050.69\t.\t+\n",
      " hcontig_000_050\t306077\t308167\tEVM prediction%2hcontig_0000_050.71\t.\t+\n",
      " hcontig_000_050\t308642\t311210\tEVM prediction%2hcontig_0000_050.72\t.\t+\n",
      " hcontig_000_050\t355976\t358688\tEVM prediction%2hcontig_0000_050.86\t.\t-\n",
      " hcontig_000_050\t418383\t419424\tEVM prediction%2hcontig_0000_050.98\t.\t+\n",
      " hcontig_000_050\t462420\t464061\tEVM prediction%2hcontig_0000_050.109\t.\t-\n",
      " hcontig_000_050\t495068\t499820\tEVM prediction%2hcontig_0000_050.116\t.\t+\n",
      " hcontig_000_050\t561742\t563139\tEVM prediction%2hcontig_0000_050.132\t.\t-\n",
      " hcontig_000_050\t573162\t574026\tEVM prediction%2hcontig_0000_050.135\t.\t-\n",
      " hcontig_000_050\t574588\t575725\tEVM prediction%2hcontig_0000_050.136\t.\t+\n",
      " hcontig_000_050\t583333\t584614\tEVM prediction%2hcontig_0000_050.138\t.\t-\n",
      " hcontig_000_050\t631566\t634371\tEVM prediction%2hcontig_0000_050.143\t.\t-\n",
      " hcontig_000_050\t666556\t668467\tEVM prediction%2hcontig_0000_050.151\t.\t+\n",
      " hcontig_000_050\t669759\t670473\tEVM prediction%2hcontig_0000_050.152\t.\t+\n",
      " hcontig_000_050\t672397\t676986\tEVM prediction%2hcontig_0000_050.153\t.\t+\n",
      " hcontig_000_050\t678462\t679035\tEVM prediction%2hcontig_0000_050.154\t.\t+\n",
      " hcontig_000_054\t27\t1842\tEVM prediction%2hcontig_0000_054.1\t.\t+\n",
      " hcontig_000_054\t3570\t4125\tEVM prediction%2hcontig_0000_054.2\t.\t+\n",
      " hcontig_000_054\t10009\t15363\tEVM prediction%2hcontig_0000_054.4\t.\t-\n",
      " hcontig_000_054\t89141\t89501\tEVM prediction%2hcontig_0000_054.19\t.\t+\n",
      " hcontig_000_054\t89579\t92414\tEVM prediction%2hcontig_0000_054.20\t.\t+\n",
      " hcontig_000_054\t94340\t96748\tEVM prediction%2hcontig_0000_054.21\t.\t-\n",
      " hcontig_000_054\t99561\t101274\tEVM prediction%2hcontig_0000_054.22\t.\t-\n",
      " hcontig_000_054\t102244\t104514\tEVM prediction%2hcontig_0000_054.23\t.\t-\n",
      " hcontig_000_054\t130135\t131365\tEVM prediction%2hcontig_0000_054.29\t.\t+\n",
      " hcontig_000_054\t131970\t133227\tEVM prediction%2hcontig_0000_054.30\t.\t-\n",
      " hcontig_000_054\t135181\t136186\tEVM prediction%2hcontig_0000_054.31\t.\t+\n",
      " hcontig_000_054\t136866\t140469\tEVM prediction%2hcontig_0000_054.32\t.\t+\n",
      " hcontig_000_054\t143346\t145325\tEVM prediction%2hcontig_0000_054.33\t.\t+\n",
      " hcontig_000_054\t148576\t150790\tEVM prediction%2hcontig_0000_054.34\t.\t-\n",
      " hcontig_000_057\t9097\t11313\tEVM prediction%2hcontig_0000_057.2\t.\t+\n",
      " hcontig_001_001\t2850\t6867\tEVM prediction%2hcontig_0001_001.1\t.\t+\n",
      " hcontig_001_001\t38653\t39026\tEVM prediction%2hcontig_0001_001.8\t.\t+\n",
      " hcontig_001_001\t39305\t40933\tEVM prediction%2hcontig_0001_001.9\t.\t+\n",
      " hcontig_001_001\t144307\t145843\tEVM prediction%2hcontig_0001_001.32\t.\t+\n",
      " hcontig_001_001\t146152\t147148\tEVM prediction%2hcontig_0001_001.33\t.\t+\n",
      " hcontig_001_001\t221683\t222352\tEVM prediction%2hcontig_0001_001.48\t.\t+\n",
      " hcontig_001_001\t223001\t223640\tEVM prediction%2hcontig_0001_001.49\t.\t+\n",
      " hcontig_001_001\t241448\t244877\tEVM prediction%2hcontig_0001_001.51\t.\t-\n",
      " hcontig_001_001\t245812\t247321\tEVM prediction%2hcontig_0001_001.52\t.\t-\n",
      " hcontig_001_001\t247857\t248172\tEVM prediction%2hcontig_0001_001.53\t.\t+\n",
      " hcontig_001_001\t254699\t258134\tEVM prediction%2hcontig_0001_001.55\t.\t-\n",
      " hcontig_001_001\t259069\t259864\tEVM prediction%2hcontig_0001_001.56\t.\t-\n",
      " hcontig_001_001\t330014\t331158\tEVM prediction%2hcontig_0001_001.62\t.\t+\n",
      " hcontig_001_001\t336772\t337759\tEVM prediction%2hcontig_0001_001.63\t.\t+\n",
      " hcontig_001_001\t367688\t368594\tEVM prediction%2hcontig_0001_001.70\t.\t-\n",
      " hcontig_001_001\t373746\t376336\tEVM prediction%2hcontig_0001_001.72\t.\t+\n",
      " hcontig_001_001\t378607\t382352\tEVM prediction%2hcontig_0001_001.73\t.\t-\n",
      " hcontig_001_001\t382515\t383691\tEVM prediction%2hcontig_0001_001.74\t.\t+\n",
      " hcontig_001_001\t384119\t384746\tEVM prediction%2hcontig_0001_001.75\t.\t-\n",
      " hcontig_001_001\t393815\t395726\tEVM prediction%2hcontig_0001_001.76\t.\t-\n",
      " hcontig_001_001\t399018\t401256\tEVM prediction%2hcontig_0001_001.77\t.\t-\n",
      " hcontig_001_001\t410074\t411312\tEVM prediction%2hcontig_0001_001.80\t.\t+\n",
      " hcontig_001_001\t411581\t413236\tEVM prediction%2hcontig_0001_001.81\t.\t+\n",
      " hcontig_001_001\t420013\t422338\tEVM prediction%2hcontig_0001_001.83\t.\t-\n",
      " hcontig_001_001\t441783\t444096\tEVM prediction%2hcontig_0001_001.91\t.\t-\n",
      " hcontig_001_001\t530378\t530912\tEVM prediction%2hcontig_0001_001.108\t.\t+\n",
      " hcontig_001_001\t554914\t556141\tEVM prediction%2hcontig_0001_001.116\t.\t-\n",
      " hcontig_001_001\t605090\t608256\tEVM prediction%2hcontig_0001_001.131\t.\t-\n",
      " hcontig_001_001\t628184\t628616\tEVM prediction%2hcontig_0001_001.136\t.\t-\n",
      " hcontig_001_001\t634042\t636640\tEVM prediction%2hcontig_0001_001.137\t.\t-\n",
      " hcontig_001_001\t637619\t638030\tEVM prediction%2hcontig_0001_001.139\t.\t-\n",
      " hcontig_001_001\t663379\t664063\tEVM prediction%2hcontig_0001_001.148\t.\t+\n",
      " hcontig_001_002\t125226\t129663\tEVM prediction%2hcontig_0001_002.25\t.\t+\n",
      " hcontig_001_002\t142209\t144409\tEVM prediction%2hcontig_0001_002.28\t.\t+\n",
      " hcontig_001_002\t212290\t215524\tEVM prediction%2hcontig_0001_002.43\t.\t-\n",
      " hcontig_001_002\t234409\t238836\tEVM prediction%2hcontig_0001_002.49\t.\t+\n",
      " hcontig_001_002\t244805\t245567\tEVM prediction%2hcontig_0001_002.50\t.\t-\n",
      " hcontig_001_002\t295950\t296457\tEVM prediction%2hcontig_0001_002.59\t.\t-\n",
      " hcontig_001_002\t296487\t297421\tEVM prediction%2hcontig_0001_002.60\t.\t-\n",
      " hcontig_001_002\t323167\t324222\tEVM prediction%2hcontig_0001_002.65\t.\t-\n",
      " hcontig_001_002\t329293\t329815\tEVM prediction%2hcontig_0001_002.68\t.\t+\n",
      " hcontig_001_002\t329838\t330455\tEVM prediction%2hcontig_0001_002.69\t.\t+\n",
      " hcontig_001_002\t339400\t339877\tEVM prediction%2hcontig_0001_002.71\t.\t-\n",
      " hcontig_001_006\t12683\t14508\tEVM prediction%2hcontig_0001_006.3\t.\t+\n",
      " hcontig_001_016\t4904\t6713\tEVM prediction%2hcontig_0001_016.2\t.\t+\n",
      " hcontig_001_016\t9161\t10094\tEVM prediction%2hcontig_0001_016.3\t.\t+\n",
      " hcontig_001_021\t64898\t65778\tEVM prediction%2hcontig_0001_021.13\t.\t+\n",
      " hcontig_001_021\t65854\t68117\tEVM prediction%2hcontig_0001_021.14\t.\t+\n",
      " hcontig_001_021\t94025\t100172\tEVM prediction%2hcontig_0001_021.21\t.\t-\n",
      " hcontig_001_021\t253840\t255222\tEVM prediction%2hcontig_0001_021.57\t.\t-\n",
      " hcontig_001_021\t272938\t274747\tEVM prediction%2hcontig_0001_021.60\t.\t+\n",
      " hcontig_001_021\t276418\t278130\tEVM prediction%2hcontig_0001_021.61\t.\t+\n",
      " hcontig_001_021\t323250\t324477\tEVM prediction%2hcontig_0001_021.71\t.\t+\n",
      " hcontig_001_021\t324830\t325670\tEVM prediction%2hcontig_0001_021.72\t.\t+\n",
      " hcontig_001_021\t325711\t326878\tEVM prediction%2hcontig_0001_021.73\t.\t+\n",
      " hcontig_001_021\t327187\t327703\tEVM prediction%2hcontig_0001_021.74\t.\t+\n",
      " hcontig_001_021\t330536\t331706\tEVM prediction%2hcontig_0001_021.77\t.\t+\n",
      " hcontig_001_021\t332115\t334164\tEVM prediction%2hcontig_0001_021.78\t.\t+\n",
      " hcontig_001_021\t334473\t334989\tEVM prediction%2hcontig_0001_021.79\t.\t+\n",
      " hcontig_001_021\t337823\t339050\tEVM prediction%2hcontig_0001_021.82\t.\t+\n",
      " hcontig_001_021\t339403\t341452\tEVM prediction%2hcontig_0001_021.83\t.\t+\n",
      " hcontig_001_021\t341761\t342277\tEVM prediction%2hcontig_0001_021.84\t.\t+\n",
      " hcontig_001_021\t426378\t429543\tEVM prediction%2hcontig_0001_021.112\t.\t+\n",
      " hcontig_001_021\t430482\t432854\tEVM prediction%2hcontig_0001_021.113\t.\t-\n",
      " hcontig_001_021\t450817\t454894\tEVM prediction%2hcontig_0001_021.119\t.\t-\n",
      " hcontig_001_021\t456779\t459982\tEVM prediction%2hcontig_0001_021.120\t.\t+\n",
      " hcontig_001_021\t541098\t544118\tEVM prediction%2hcontig_0001_021.130\t.\t+\n",
      " hcontig_001_021\t546108\t547079\tEVM prediction%2hcontig_0001_021.132\t.\t-\n",
      " hcontig_001_021\t614738\t617019\tEVM prediction%2hcontig_0001_021.146\t.\t+\n",
      " hcontig_001_021\t617369\t617999\tEVM prediction%2hcontig_0001_021.147\t.\t+\n",
      " hcontig_001_021\t622687\t623929\tEVM prediction%2hcontig_0001_021.148\t.\t+\n",
      " hcontig_001_021\t703888\t705451\tEVM prediction%2hcontig_0001_021.165\t.\t-\n",
      " hcontig_001_021\t706054\t707341\tEVM prediction%2hcontig_0001_021.166\t.\t-\n",
      " hcontig_001_021\t728351\t731342\tEVM prediction%2hcontig_0001_021.170\t.\t+\n",
      " hcontig_001_021\t732342\t739223\tEVM prediction%2hcontig_0001_021.171\t.\t-\n",
      " hcontig_001_021\t741243\t744564\tEVM prediction%2hcontig_0001_021.172\t.\t+\n",
      " hcontig_001_021\t791814\t792627\tEVM prediction%2hcontig_0001_021.181\t.\t-\n",
      " hcontig_001_021\t804481\t806266\tEVM prediction%2hcontig_0001_021.183\t.\t-\n",
      " hcontig_001_021\t806302\t808996\tEVM prediction%2hcontig_0001_021.184\t.\t-\n",
      " hcontig_001_021\t820096\t821599\tEVM prediction%2hcontig_0001_021.187\t.\t-\n",
      " hcontig_001_021\t822524\t824718\tEVM prediction%2hcontig_0001_021.188\t.\t+\n",
      " hcontig_001_021\t876899\t879977\tEVM prediction%2hcontig_0001_021.206\t.\t-\n",
      " hcontig_001_023\t4024\t4546\tEVM prediction%2hcontig_0001_023.2\t.\t+\n",
      " hcontig_001_023\t4569\t5186\tEVM prediction%2hcontig_0001_023.3\t.\t+\n",
      " hcontig_001_023\t15041\t15518\tEVM prediction%2hcontig_0001_023.5\t.\t-\n",
      " hcontig_001_023\t35008\t35797\tEVM prediction%2hcontig_0001_023.11\t.\t-\n",
      " hcontig_001_023\t36494\t37832\tEVM prediction%2hcontig_0001_023.12\t.\t-\n",
      " hcontig_001_023\t38561\t39473\tEVM prediction%2hcontig_0001_023.13\t.\t-\n",
      " hcontig_001_023\t46646\t48170\tEVM prediction%2hcontig_0001_023.15\t.\t-\n",
      " hcontig_001_023\t88404\t92973\tEVM prediction%2hcontig_0001_023.20\t.\t-\n",
      " hcontig_001_042\t2585\t4505\tEVM prediction%2hcontig_0001_042.1\t.\t-\n",
      " hcontig_001_042\t11104\t16031\tEVM prediction%2hcontig_0001_042.2\t.\t+\n",
      " hcontig_001_042\t18678\t19446\tEVM prediction%2hcontig_0001_042.3\t.\t+\n",
      " hcontig_001_043\t3968\t6635\tEVM prediction%2hcontig_0001_043.2\t.\t-\n",
      " hcontig_001_043\t7891\t8782\tEVM prediction%2hcontig_0001_043.3\t.\t-\n",
      " hcontig_001_043\t13528\t16769\tEVM prediction%2hcontig_0001_043.6\t.\t-\n",
      " hcontig_001_044\t19648\t22678\tEVM prediction%2hcontig_0001_044.6\t.\t-\n",
      " hcontig_001_054\t19758\t20701\tEVM prediction%2hcontig_0001_054.5\t.\t+\n",
      " hcontig_001_054\t22627\t24191\tEVM prediction%2hcontig_0001_054.6\t.\t+\n",
      " hcontig_001_054\t30714\t32380\tEVM prediction%2hcontig_0001_054.8\t.\t-\n",
      " hcontig_001_054\t32660\t34643\tEVM prediction%2hcontig_0001_054.9\t.\t-\n",
      " hcontig_001_054\t75879\t77412\tEVM prediction%2hcontig_0001_054.21\t.\t+\n",
      " hcontig_001_054\t77544\t78531\tEVM prediction%2hcontig_0001_054.22\t.\t+\n",
      " hcontig_001_054\t82266\t83799\tEVM prediction%2hcontig_0001_054.23\t.\t+\n",
      " hcontig_001_054\t83931\t84918\tEVM prediction%2hcontig_0001_054.24\t.\t+\n",
      " hcontig_001_054\t88652\t90185\tEVM prediction%2hcontig_0001_054.25\t.\t+\n",
      " hcontig_001_054\t90317\t91304\tEVM prediction%2hcontig_0001_054.26\t.\t+\n",
      " hcontig_001_054\t145779\t147112\tEVM prediction%2hcontig_0001_054.40\t.\t-\n",
      " hcontig_001_054\t161252\t165575\tEVM prediction%2hcontig_0001_054.46\t.\t+\n",
      " hcontig_001_054\t213655\t215095\tEVM prediction%2hcontig_0001_054.58\t.\t+\n",
      " hcontig_001_054\t215363\t216975\tEVM prediction%2hcontig_0001_054.59\t.\t-\n",
      " hcontig_001_054\t258756\t262161\tEVM prediction%2hcontig_0001_054.67\t.\t-\n",
      " hcontig_001_054\t262901\t263906\tEVM prediction%2hcontig_0001_054.68\t.\t-\n",
      " hcontig_001_054\t342404\t345992\tEVM prediction%2hcontig_0001_054.90\t.\t-\n",
      " hcontig_001_054\t357076\t358825\tEVM prediction%2hcontig_0001_054.92\t.\t+\n",
      " hcontig_001_054\t396829\t399871\tEVM prediction%2hcontig_0001_054.101\t.\t-\n",
      " hcontig_001_123\t11634\t13176\tEVM prediction%2hcontig_001_0123.4\t.\t-\n",
      " hcontig_001_123\t37145\t39021\tEVM prediction%2hcontig_001_0123.10\t.\t+\n",
      " hcontig_001_123\t40209\t40683\tEVM prediction%2hcontig_001_0123.11\t.\t+\n",
      " hcontig_001_123\t40761\t41397\tEVM prediction%2hcontig_001_0123.12\t.\t+\n",
      " hcontig_001_123\t41943\t42987\tEVM prediction%2hcontig_001_0123.13\t.\t+\n",
      " hcontig_001_123\t66329\t68777\tEVM prediction%2hcontig_001_0123.21\t.\t-\n",
      " hcontig_001_123\t69267\t71069\tEVM prediction%2hcontig_001_0123.22\t.\t-\n",
      " hcontig_001_156\t13744\t18100\tEVM prediction%2hcontig_001_0156.2\t.\t-\n",
      " hcontig_001_180\t40183\t43385\tEVM prediction%2hcontig_001_0180.12\t.\t-\n",
      " hcontig_002_011\t2112\t6477\tEVM prediction%2hcontig_0002_011.1\t.\t+\n",
      " hcontig_002_011\t73612\t74017\tEVM prediction%2hcontig_0002_011.14\t.\t+\n",
      " hcontig_002_011\t74698\t76534\tEVM prediction%2hcontig_0002_011.15\t.\t+\n",
      " hcontig_002_011\t177505\t178540\tEVM prediction%2hcontig_0002_011.39\t.\t+\n",
      " hcontig_002_011\t213032\t217397\tEVM prediction%2hcontig_0002_011.52\t.\t+\n",
      " hcontig_002_011\t243512\t244801\tEVM prediction%2hcontig_0002_011.56\t.\t+\n",
      " hcontig_002_011\t245149\t247503\tEVM prediction%2hcontig_0002_011.57\t.\t-\n",
      " hcontig_002_011\t254666\t259332\tEVM prediction%2hcontig_0002_011.61\t.\t-\n",
      " hcontig_002_011\t310788\t311775\tEVM prediction%2hcontig_0002_011.81\t.\t-\n",
      " hcontig_002_011\t313214\t313661\tEVM prediction%2hcontig_0002_011.82\t.\t+\n",
      " hcontig_002_011\t390003\t391221\tEVM prediction%2hcontig_0002_011.101\t.\t-\n",
      " hcontig_002_011\t393841\t395323\tEVM prediction%2hcontig_0002_011.104\t.\t+\n",
      " hcontig_002_011\t449247\t451095\tEVM prediction%2hcontig_0002_011.121\t.\t+\n",
      " hcontig_002_011\t480463\t481702\tEVM prediction%2hcontig_0002_011.126\t.\t-\n",
      " hcontig_002_011\t491632\t493746\tEVM prediction%2hcontig_0002_011.129\t.\t-\n",
      " hcontig_002_011\t494968\t495628\tEVM prediction%2hcontig_0002_011.130\t.\t-\n",
      " hcontig_002_011\t496699\t497089\tEVM prediction%2hcontig_0002_011.131\t.\t-\n",
      " hcontig_002_011\t506422\t509631\tEVM prediction%2hcontig_0002_011.135\t.\t+\n",
      " hcontig_002_011\t514681\t517987\tEVM prediction%2hcontig_0002_011.137\t.\t-\n",
      " hcontig_002_011\t529129\t530246\tEVM prediction%2hcontig_0002_011.142\t.\t+\n",
      " hcontig_002_011\t530430\t530799\tEVM prediction%2hcontig_0002_011.143\t.\t-\n",
      " hcontig_002_011\t531328\t531763\tEVM prediction%2hcontig_0002_011.144\t.\t-\n",
      " hcontig_002_011\t538544\t539078\tEVM prediction%2hcontig_0002_011.147\t.\t-\n",
      " hcontig_002_011\t581638\t582379\tEVM prediction%2hcontig_0002_011.156\t.\t-\n",
      " hcontig_002_011\t601155\t605553\tEVM prediction%2hcontig_0002_011.166\t.\t+\n",
      " hcontig_002_011\t646360\t648080\tEVM prediction%2hcontig_0002_011.177\t.\t+\n",
      " hcontig_002_011\t719938\t722509\tEVM prediction%2hcontig_0002_011.196\t.\t-\n",
      " hcontig_002_011\t722984\t724306\tEVM prediction%2hcontig_0002_011.197\t.\t-\n",
      " hcontig_002_011\t762860\t764443\tEVM prediction%2hcontig_0002_011.205\t.\t-\n",
      " hcontig_002_011\t764562\t766036\tEVM prediction%2hcontig_0002_011.206\t.\t-\n",
      " hcontig_002_011\t854972\t855719\tEVM prediction%2hcontig_0002_011.231\t.\t-\n",
      " hcontig_002_011\t878744\t883085\tEVM prediction%2hcontig_0002_011.237\t.\t-\n",
      " hcontig_002_011\t885068\t887459\tEVM prediction%2hcontig_0002_011.239\t.\t-\n",
      " hcontig_002_019\t17178\t18284\tEVM prediction%2hcontig_0002_019.2\t.\t+\n",
      " hcontig_002_028\t38017\t38338\tEVM prediction%2hcontig_0002_028.8\t.\t+\n",
      " hcontig_002_028\t261711\t263796\tEVM prediction%2hcontig_0002_028.49\t.\t+\n",
      " hcontig_002_028\t269778\t273201\tEVM prediction%2hcontig_0002_028.51\t.\t-\n",
      " hcontig_002_028\t273941\t274931\tEVM prediction%2hcontig_0002_028.52\t.\t-\n",
      " hcontig_002_028\t343349\t345317\tEVM prediction%2hcontig_0002_028.70\t.\t-\n",
      " hcontig_002_028\t476719\t477340\tEVM prediction%2hcontig_0002_028.93\t.\t-\n",
      " hcontig_002_028\t489007\t490375\tEVM prediction%2hcontig_0002_028.96\t.\t-\n",
      " hcontig_002_028\t505925\t506942\tEVM prediction%2hcontig_0002_028.99\t.\t+\n",
      " hcontig_002_028\t507326\t508457\tEVM prediction%2hcontig_0002_028.100\t.\t+\n",
      " hcontig_002_028\t562674\t564456\tEVM prediction%2hcontig_0002_028.113\t.\t-\n",
      " hcontig_002_028\t566132\t566831\tEVM prediction%2hcontig_0002_028.114\t.\t-\n",
      " hcontig_002_028\t620804\t622020\tEVM prediction%2hcontig_0002_028.120\t.\t-\n",
      " hcontig_002_028\t626536\t630553\tEVM prediction%2hcontig_0002_028.122\t.\t-\n",
      " hcontig_002_028\t645801\t647681\tEVM prediction%2hcontig_0002_028.130\t.\t+\n",
      " hcontig_002_028\t723627\t723960\tEVM prediction%2hcontig_0002_028.145\t.\t-\n",
      " hcontig_002_028\t724456\t725022\tEVM prediction%2hcontig_0002_028.146\t.\t-\n",
      " hcontig_002_028\t727436\t727880\tEVM prediction%2hcontig_0002_028.147\t.\t-\n",
      " hcontig_002_028\t836945\t838051\tEVM prediction%2hcontig_0002_028.175\t.\t+\n",
      " hcontig_002_028\t849291\t850122\tEVM prediction%2hcontig_0002_028.180\t.\t+\n",
      " hcontig_002_028\t852815\t856403\tEVM prediction%2hcontig_0002_028.181\t.\t+\n",
      " hcontig_002_028\t861146\t861500\tEVM prediction%2hcontig_0002_028.182\t.\t-\n",
      " hcontig_002_028\t861689\t863427\tEVM prediction%2hcontig_0002_028.183\t.\t-\n",
      " hcontig_002_028\t865114\t865717\tEVM prediction%2hcontig_0002_028.184\t.\t-\n",
      " hcontig_002_028\t916046\t920048\tEVM prediction%2hcontig_0002_028.198\t.\t-\n",
      " hcontig_002_028\t920694\t921914\tEVM prediction%2hcontig_0002_028.199\t.\t+\n",
      " hcontig_002_028\t921939\t924111\tEVM prediction%2hcontig_0002_028.200\t.\t+\n",
      " hcontig_002_028\t924198\t924954\tEVM prediction%2hcontig_0002_028.201\t.\t+\n",
      " hcontig_002_028\t930865\t932257\tEVM prediction%2hcontig_0002_028.203\t.\t+\n",
      " hcontig_002_028\t968385\t969480\tEVM prediction%2hcontig_0002_028.213\t.\t-\n",
      " hcontig_002_028\t969588\t972931\tEVM prediction%2hcontig_0002_028.214\t.\t-\n",
      " hcontig_002_028\t973548\t974202\tEVM prediction%2hcontig_0002_028.215\t.\t+\n",
      " hcontig_002_028\t982230\t983714\tEVM prediction%2hcontig_0002_028.217\t.\t-\n",
      " hcontig_002_028\t983723\t985349\tEVM prediction%2hcontig_0002_028.218\t.\t-\n",
      " hcontig_002_028\t989176\t993768\tEVM prediction%2hcontig_0002_028.219\t.\t-\n",
      " hcontig_002_028\t1025690\t1027976\tEVM prediction%2hcontig_0002_028.226\t.\t-\n",
      " hcontig_002_028\t1081164\t1083207\tEVM prediction%2hcontig_0002_028.237\t.\t-\n",
      " hcontig_002_028\t1101847\t1103906\tEVM prediction%2hcontig_0002_028.242\t.\t-\n",
      " hcontig_002_028\t1107670\t1108576\tEVM prediction%2hcontig_0002_028.243\t.\t+\n",
      " hcontig_002_028\t1121903\t1122612\tEVM prediction%2hcontig_0002_028.247\t.\t-\n",
      " hcontig_002_028\t1123233\t1123824\tEVM prediction%2hcontig_0002_028.248\t.\t-\n",
      " hcontig_002_028\t1123884\t1124970\tEVM prediction%2hcontig_0002_028.249\t.\t-\n",
      " hcontig_002_028\t1125326\t1126559\tEVM prediction%2hcontig_0002_028.250\t.\t-\n",
      " hcontig_002_028\t1137017\t1141970\tEVM prediction%2hcontig_0002_028.253\t.\t+\n",
      " hcontig_002_028\t1178361\t1178784\tEVM prediction%2hcontig_0002_028.258\t.\t-\n",
      " hcontig_002_028\t1178817\t1180567\tEVM prediction%2hcontig_0002_028.259\t.\t-\n",
      " hcontig_002_028\t1323394\t1326001\tEVM prediction%2hcontig_0002_028.285\t.\t+\n",
      " hcontig_002_028\t1401654\t1402781\tEVM prediction%2hcontig_0002_028.311\t.\t+\n",
      " hcontig_002_028\t1403166\t1404254\tEVM prediction%2hcontig_0002_028.312\t.\t+\n",
      " hcontig_002_028\t1404842\t1406246\tEVM prediction%2hcontig_0002_028.313\t.\t+\n",
      " hcontig_002_028\t1407189\t1408181\tEVM prediction%2hcontig_0002_028.314\t.\t+\n",
      " hcontig_003_002\t3606\t6642\tEVM prediction%2hcontig_0003_002.2\t.\t+\n",
      " hcontig_003_002\t15465\t16319\tEVM prediction%2hcontig_0003_002.4\t.\t-\n",
      " hcontig_003_002\t46701\t47761\tEVM prediction%2hcontig_0003_002.13\t.\t+\n",
      " hcontig_003_002\t49143\t50811\tEVM prediction%2hcontig_0003_002.15\t.\t-\n",
      " hcontig_003_002\t79027\t80377\tEVM prediction%2hcontig_0003_002.22\t.\t-\n",
      " hcontig_003_002\t94823\t95722\tEVM prediction%2hcontig_0003_002.25\t.\t-\n",
      " hcontig_003_002\t101654\t101945\tEVM prediction%2hcontig_0003_002.27\t.\t-\n",
      " hcontig_003_002\t102105\t103452\tEVM prediction%2hcontig_0003_002.28\t.\t-\n",
      " hcontig_003_002\t126775\t131284\tEVM prediction%2hcontig_0003_002.34\t.\t+\n",
      " hcontig_003_002\t174830\t175922\tEVM prediction%2hcontig_0003_002.39\t.\t+\n",
      " hcontig_003_002\t179137\t180382\tEVM prediction%2hcontig_0003_002.42\t.\t-\n",
      " hcontig_003_002\t185939\t186278\tEVM prediction%2hcontig_0003_002.43\t.\t-\n",
      " hcontig_003_002\t194775\t196196\tEVM prediction%2hcontig_0003_002.45\t.\t-\n",
      " hcontig_003_002\t209970\t212811\tEVM prediction%2hcontig_0003_002.48\t.\t+\n",
      " hcontig_003_002\t230233\t231276\tEVM prediction%2hcontig_0003_002.52\t.\t-\n",
      " hcontig_003_002\t336518\t336941\tEVM prediction%2hcontig_0003_002.76\t.\t+\n",
      " hcontig_003_002\t427390\t428620\tEVM prediction%2hcontig_0003_002.101\t.\t+\n",
      " hcontig_003_002\t429526\t430510\tEVM prediction%2hcontig_0003_002.102\t.\t-\n",
      " hcontig_003_002\t459463\t461189\tEVM prediction%2hcontig_0003_002.108\t.\t+\n",
      " hcontig_003_002\t503291\t505892\tEVM prediction%2hcontig_0003_002.122\t.\t+\n",
      " hcontig_003_002\t532627\t534524\tEVM prediction%2hcontig_0003_002.130\t.\t+\n",
      " hcontig_003_002\t542680\t543058\tEVM prediction%2hcontig_0003_002.132\t.\t+\n",
      " hcontig_003_002\t544370\t545383\tEVM prediction%2hcontig_0003_002.134\t.\t-\n",
      " hcontig_003_002\t556986\t558379\tEVM prediction%2hcontig_0003_002.135\t.\t-\n",
      " hcontig_003_002\t559030\t560566\tEVM prediction%2hcontig_0003_002.136\t.\t-\n",
      " hcontig_003_002\t628060\t629078\tEVM prediction%2hcontig_0003_002.155\t.\t-\n",
      " hcontig_003_002\t701640\t702733\tEVM prediction%2hcontig_0003_002.178\t.\t+\n",
      " hcontig_003_002\t783161\t784313\tEVM prediction%2hcontig_0003_002.195\t.\t-\n",
      " hcontig_003_002\t784418\t787203\tEVM prediction%2hcontig_0003_002.196\t.\t-\n",
      " hcontig_003_002\t822441\t823956\tEVM prediction%2hcontig_0003_002.203\t.\t+\n",
      " hcontig_003_002\t823997\t826514\tEVM prediction%2hcontig_0003_002.204\t.\t+\n",
      " hcontig_003_002\t827369\t831443\tEVM prediction%2hcontig_0003_002.205\t.\t+\n",
      " hcontig_003_002\t912393\t914818\tEVM prediction%2hcontig_0003_002.223\t.\t+\n",
      " hcontig_003_002\t927711\t931430\tEVM prediction%2hcontig_0003_002.229\t.\t+\n",
      " hcontig_003_002\t946115\t946829\tEVM prediction%2hcontig_0003_002.232\t.\t-\n",
      " hcontig_003_013\t166\t1785\tEVM prediction%2hcontig_0003_013.1\t.\t+\n",
      " hcontig_003_013\t4197\t6171\tEVM prediction%2hcontig_0003_013.2\t.\t+\n",
      " hcontig_003_013\t6306\t7851\tEVM prediction%2hcontig_0003_013.3\t.\t+\n",
      " hcontig_003_013\t43797\t45116\tEVM prediction%2hcontig_0003_013.12\t.\t-\n",
      " hcontig_003_013\t98413\t101196\tEVM prediction%2hcontig_0003_013.30\t.\t-\n",
      " hcontig_003_013\t101403\t102927\tEVM prediction%2hcontig_0003_013.31\t.\t-\n",
      " hcontig_003_013\t234344\t239054\tEVM prediction%2hcontig_0003_013.55\t.\t-\n",
      " hcontig_003_013\t255030\t257306\tEVM prediction%2hcontig_0003_013.60\t.\t-\n",
      " hcontig_003_013\t257351\t258266\tEVM prediction%2hcontig_0003_013.61\t.\t-\n",
      " hcontig_003_014\t176\t1156\tEVM prediction%2hcontig_0003_014.1\t.\t+\n",
      " hcontig_003_014\t49403\t50087\tEVM prediction%2hcontig_0003_014.11\t.\t+\n",
      " hcontig_003_014\t51480\t51810\tEVM prediction%2hcontig_0003_014.12\t.\t+\n",
      " hcontig_003_014\t118374\t120873\tEVM prediction%2hcontig_0003_014.27\t.\t-\n",
      " hcontig_003_014\t190593\t192138\tEVM prediction%2hcontig_0003_014.40\t.\t+\n",
      " hcontig_003_014\t192527\t193989\tEVM prediction%2hcontig_0003_014.41\t.\t-\n",
      " hcontig_003_014\t233345\t233867\tEVM prediction%2hcontig_0003_014.50\t.\t+\n",
      " hcontig_003_015\t237\t747\tEVM prediction%2hcontig_0003_015.1\t.\t-\n",
      " hcontig_003_018\t24459\t25918\tEVM prediction%2hcontig_0003_018.5\t.\t+\n",
      " hcontig_003_018\t26377\t28048\tEVM prediction%2hcontig_0003_018.6\t.\t+\n",
      " hcontig_003_025\t5082\t6475\tEVM prediction%2hcontig_0003_025.1\t.\t-\n",
      " hcontig_003_025\t7126\t8662\tEVM prediction%2hcontig_0003_025.2\t.\t-\n",
      " hcontig_003_026\t4831\t5379\tEVM prediction%2hcontig_0003_026.1\t.\t+\n",
      " hcontig_003_028\t5332\t7526\tEVM prediction%2hcontig_0003_028.3\t.\t-\n",
      " hcontig_003_038\t753\t930\tEVM prediction%2hcontig_0003_038.1\t.\t+\n",
      " hcontig_003_041\t90141\t92799\tEVM prediction%2hcontig_0003_041.20\t.\t-\n",
      " hcontig_003_041\t92923\t93450\tEVM prediction%2hcontig_0003_041.21\t.\t-\n",
      " hcontig_003_041\t96110\t97088\tEVM prediction%2hcontig_0003_041.24\t.\t-\n",
      " hcontig_003_041\t98249\t99434\tEVM prediction%2hcontig_0003_041.25\t.\t-\n",
      " hcontig_003_041\t135890\t138305\tEVM prediction%2hcontig_0003_041.35\t.\t-\n",
      " hcontig_003_041\t164297\t165338\tEVM prediction%2hcontig_0003_041.39\t.\t+\n",
      " hcontig_003_041\t165835\t167569\tEVM prediction%2hcontig_0003_041.40\t.\t-\n",
      " hcontig_003_041\t168657\t169672\tEVM prediction%2hcontig_0003_041.41\t.\t-\n",
      " hcontig_003_041\t171519\t172613\tEVM prediction%2hcontig_0003_041.42\t.\t-\n",
      " hcontig_003_041\t173608\t174961\tEVM prediction%2hcontig_0003_041.43\t.\t-\n",
      " hcontig_003_041\t176274\t177135\tEVM prediction%2hcontig_0003_041.44\t.\t-\n",
      " hcontig_003_041\t178347\t179559\tEVM prediction%2hcontig_0003_041.45\t.\t-\n",
      " hcontig_003_041\t179594\t179915\tEVM prediction%2hcontig_0003_041.46\t.\t-\n",
      " hcontig_003_041\t180587\t182622\tEVM prediction%2hcontig_0003_041.47\t.\t+\n",
      " hcontig_003_041\t184570\t185732\tEVM prediction%2hcontig_0003_041.48\t.\t-\n",
      " hcontig_003_041\t192570\t194989\tEVM prediction%2hcontig_0003_041.50\t.\t+\n",
      " hcontig_003_041\t201588\t202320\tEVM prediction%2hcontig_0003_041.52\t.\t+\n",
      " hcontig_003_041\t216058\t216757\tEVM prediction%2hcontig_0003_041.55\t.\t-\n",
      " hcontig_003_041\t216964\t217474\tEVM prediction%2hcontig_0003_041.56\t.\t-\n",
      " hcontig_003_041\t251651\t255899\tEVM prediction%2hcontig_0003_041.61\t.\t-\n",
      " hcontig_003_041\t271637\t272210\tEVM prediction%2hcontig_0003_041.67\t.\t+\n",
      " hcontig_003_041\t283476\t285101\tEVM prediction%2hcontig_0003_041.68\t.\t-\n",
      " hcontig_003_041\t285196\t286368\tEVM prediction%2hcontig_0003_041.69\t.\t-\n",
      " hcontig_003_041\t288161\t289558\tEVM prediction%2hcontig_0003_041.71\t.\t-\n",
      " hcontig_003_044\t46\t779\tEVM prediction%2hcontig_0003_044.1\t.\t+\n",
      " hcontig_003_044\t13040\t16481\tEVM prediction%2hcontig_0003_044.5\t.\t+\n",
      " hcontig_003_045\t20472\t21702\tEVM prediction%2hcontig_0003_045.5\t.\t+\n",
      " hcontig_003_045\t64467\t65015\tEVM prediction%2hcontig_0003_045.16\t.\t+\n",
      " hcontig_003_135\t2079\t5333\tEVM prediction%2hcontig_003_0135.1\t.\t+\n",
      " hcontig_003_136\t30132\t31402\tEVM prediction%2hcontig_003_0136.5\t.\t+\n",
      " hcontig_003_136\t34048\t34672\tEVM prediction%2hcontig_003_0136.7\t.\t+\n",
      " hcontig_003_136\t75080\t76071\tEVM prediction%2hcontig_003_0136.18\t.\t+\n",
      " hcontig_003_136\t77557\t78112\tEVM prediction%2hcontig_003_0136.20\t.\t+\n",
      " hcontig_004_001\t12585\t16020\tEVM prediction%2hcontig_0004_001.3\t.\t-\n",
      " hcontig_004_008\t469\t1426\tEVM prediction%2hcontig_0004_008.1\t.\t+\n",
      " hcontig_004_020\t756\t1470\tEVM prediction%2hcontig_0004_020.1\t.\t+\n",
      " hcontig_004_020\t5194\t9970\tEVM prediction%2hcontig_0004_020.2\t.\t+\n",
      " hcontig_004_020\t80182\t82584\tEVM prediction%2hcontig_0004_020.24\t.\t+\n",
      " hcontig_004_020\t83863\t86263\tEVM prediction%2hcontig_0004_020.25\t.\t-\n",
      " hcontig_004_020\t217345\t220558\tEVM prediction%2hcontig_0004_020.61\t.\t+\n",
      " hcontig_004_020\t228483\t229858\tEVM prediction%2hcontig_0004_020.63\t.\t+\n",
      " hcontig_004_020\t235298\t235730\tEVM prediction%2hcontig_0004_020.64\t.\t-\n",
      " hcontig_004_020\t260699\t262022\tEVM prediction%2hcontig_0004_020.70\t.\t-\n",
      " hcontig_004_020\t264343\t270478\tEVM prediction%2hcontig_0004_020.71\t.\t+\n",
      " hcontig_004_020\t286000\t286951\tEVM prediction%2hcontig_0004_020.74\t.\t-\n",
      " hcontig_004_020\t288797\t289598\tEVM prediction%2hcontig_0004_020.75\t.\t-\n",
      " hcontig_004_020\t289661\t290162\tEVM prediction%2hcontig_0004_020.76\t.\t-\n",
      " hcontig_004_020\t299940\t301150\tEVM prediction%2hcontig_0004_020.80\t.\t-\n",
      " hcontig_004_020\t304583\t305635\tEVM prediction%2hcontig_0004_020.81\t.\t-\n",
      " hcontig_004_020\t341000\t345563\tEVM prediction%2hcontig_0004_020.85\t.\t-\n",
      " hcontig_004_020\t361068\t364383\tEVM prediction%2hcontig_0004_020.90\t.\t-\n",
      " hcontig_004_020\t392995\t396296\tEVM prediction%2hcontig_0004_020.99\t.\t-\n",
      " hcontig_004_020\t539194\t541450\tEVM prediction%2hcontig_0004_020.131\t.\t-\n",
      " hcontig_004_020\t541537\t542401\tEVM prediction%2hcontig_0004_020.132\t.\t-\n",
      " hcontig_004_020\t542693\t543917\tEVM prediction%2hcontig_0004_020.133\t.\t-\n",
      " hcontig_004_020\t586077\t588774\tEVM prediction%2hcontig_0004_020.140\t.\t-\n",
      " hcontig_004_020\t590498\t591356\tEVM prediction%2hcontig_0004_020.142\t.\t-\n",
      " hcontig_004_020\t592634\t593446\tEVM prediction%2hcontig_0004_020.143\t.\t-\n",
      " hcontig_004_020\t593725\t594744\tEVM prediction%2hcontig_0004_020.144\t.\t-\n",
      " hcontig_004_020\t602618\t603341\tEVM prediction%2hcontig_0004_020.146\t.\t-\n",
      " hcontig_004_020\t643751\t647006\tEVM prediction%2hcontig_0004_020.157\t.\t+\n",
      " hcontig_004_020\t651487\t652347\tEVM prediction%2hcontig_0004_020.159\t.\t+\n",
      " hcontig_004_020\t652822\t655393\tEVM prediction%2hcontig_0004_020.160\t.\t+\n",
      " hcontig_004_020\t715569\t716687\tEVM prediction%2hcontig_0004_020.172\t.\t+\n",
      " hcontig_004_020\t720964\t721678\tEVM prediction%2hcontig_0004_020.174\t.\t+\n",
      " hcontig_004_020\t732001\t734314\tEVM prediction%2hcontig_0004_020.176\t.\t+\n",
      " hcontig_004_020\t734609\t735761\tEVM prediction%2hcontig_0004_020.177\t.\t+\n",
      " hcontig_004_020\t753032\t754615\tEVM prediction%2hcontig_0004_020.182\t.\t+\n",
      " hcontig_004_020\t754694\t755285\tEVM prediction%2hcontig_0004_020.183\t.\t+\n",
      " hcontig_004_020\t760638\t763892\tEVM prediction%2hcontig_0004_020.184\t.\t+\n",
      " hcontig_004_020\t866287\t869100\tEVM prediction%2hcontig_0004_020.208\t.\t+\n",
      " hcontig_004_020\t914374\t914710\tEVM prediction%2hcontig_0004_020.221\t.\t+\n",
      " hcontig_004_020\t915692\t917852\tEVM prediction%2hcontig_0004_020.222\t.\t+\n",
      " hcontig_004_020\t997283\t999030\tEVM prediction%2hcontig_0004_020.237\t.\t-\n",
      " hcontig_004_020\t1029717\t1032123\tEVM prediction%2hcontig_0004_020.242\t.\t-\n",
      " hcontig_004_020\t1032469\t1033471\tEVM prediction%2hcontig_0004_020.243\t.\t-\n",
      " hcontig_004_020\t1123058\t1126223\tEVM prediction%2hcontig_0004_020.263\t.\t-\n",
      " hcontig_004_020\t1126453\t1128973\tEVM prediction%2hcontig_0004_020.264\t.\t-\n",
      " hcontig_004_020\t1133355\t1134345\tEVM prediction%2hcontig_0004_020.266\t.\t+\n",
      " hcontig_004_020\t1134757\t1136722\tEVM prediction%2hcontig_0004_020.267\t.\t+\n",
      " hcontig_004_020\t1154801\t1159631\tEVM prediction%2hcontig_0004_020.272\t.\t-\n",
      " hcontig_004_020\t1161404\t1165898\tEVM prediction%2hcontig_0004_020.273\t.\t+\n",
      " hcontig_004_020\t1166013\t1168026\tEVM prediction%2hcontig_0004_020.274\t.\t+\n",
      " hcontig_004_020\t1184171\t1184891\tEVM prediction%2hcontig_0004_020.277\t.\t-\n",
      " hcontig_004_020\t1214210\t1216648\tEVM prediction%2hcontig_0004_020.285\t.\t-\n",
      " hcontig_004_020\t1285498\t1286017\tEVM prediction%2hcontig_0004_020.300\t.\t+\n",
      " hcontig_004_020\t1289457\t1290102\tEVM prediction%2hcontig_0004_020.301\t.\t-\n",
      " hcontig_004_020\t1315549\t1316447\tEVM prediction%2hcontig_0004_020.304\t.\t-\n",
      " hcontig_004_020\t1337323\t1338123\tEVM prediction%2hcontig_0004_020.309\t.\t+\n",
      " hcontig_004_020\t1342448\t1343941\tEVM prediction%2hcontig_0004_020.312\t.\t+\n",
      " hcontig_004_020\t1348294\t1350082\tEVM prediction%2hcontig_0004_020.314\t.\t-\n",
      " hcontig_004_020\t1350374\t1353479\tEVM prediction%2hcontig_0004_020.315\t.\t-\n",
      " hcontig_004_020\t1357889\t1358426\tEVM prediction%2hcontig_0004_020.318\t.\t+\n",
      " hcontig_004_020\t1361423\t1361867\tEVM prediction%2hcontig_0004_020.320\t.\t+\n",
      " hcontig_004_020\t1415147\t1418086\tEVM prediction%2hcontig_0004_020.332\t.\t-\n",
      " hcontig_004_020\t1462571\t1463648\tEVM prediction%2hcontig_0004_020.342\t.\t+\n",
      " hcontig_004_020\t1464458\t1465963\tEVM prediction%2hcontig_0004_020.343\t.\t+\n",
      " hcontig_004_020\t1484142\t1485855\tEVM prediction%2hcontig_0004_020.345\t.\t-\n",
      " hcontig_004_020\t1486825\t1489095\tEVM prediction%2hcontig_0004_020.346\t.\t-\n",
      " hcontig_004_020\t1526862\t1529145\tEVM prediction%2hcontig_0004_020.356\t.\t-\n",
      " hcontig_004_020\t1530416\t1532250\tEVM prediction%2hcontig_0004_020.357\t.\t-\n",
      " hcontig_004_020\t1566863\t1567778\tEVM prediction%2hcontig_0004_020.365\t.\t+\n",
      " hcontig_004_020\t1581058\t1584556\tEVM prediction%2hcontig_0004_020.368\t.\t+\n",
      " hcontig_004_020\t1606566\t1607352\tEVM prediction%2hcontig_0004_020.370\t.\t+\n",
      " hcontig_004_020\t1607652\t1610454\tEVM prediction%2hcontig_0004_020.371\t.\t+\n",
      " hcontig_004_020\t1652135\t1653296\tEVM prediction%2hcontig_0004_020.379\t.\t+\n",
      " hcontig_004_020\t1653693\t1654637\tEVM prediction%2hcontig_0004_020.380\t.\t+\n",
      " hcontig_004_020\t1656666\t1659238\tEVM prediction%2hcontig_0004_020.381\t.\t-\n",
      " hcontig_004_020\t1755648\t1759167\tEVM prediction%2hcontig_0004_020.405\t.\t+\n",
      " hcontig_004_025\t5156\t5777\tEVM prediction%2hcontig_0004_025.2\t.\t+\n",
      " hcontig_004_025\t11423\t12068\tEVM prediction%2hcontig_0004_025.3\t.\t-\n",
      " hcontig_004_076\t10152\t12891\tEVM prediction%2hcontig_004_0076.2\t.\t+\n",
      " hcontig_004_076\t46298\t46901\tEVM prediction%2hcontig_004_0076.5\t.\t-\n",
      " hcontig_004_076\t135987\t136608\tEVM prediction%2hcontig_004_0076.17\t.\t-\n",
      " hcontig_004_076\t148963\t150817\tEVM prediction%2hcontig_004_0076.19\t.\t+\n",
      " hcontig_004_076\t201736\t202669\tEVM prediction%2hcontig_004_0076.28\t.\t-\n",
      " hcontig_004_076\t203292\t204788\tEVM prediction%2hcontig_004_0076.29\t.\t-\n",
      " hcontig_004_076\t205459\t206695\tEVM prediction%2hcontig_004_0076.30\t.\t-\n",
      " hcontig_004_076\t244764\t247060\tEVM prediction%2hcontig_004_0076.35\t.\t-\n",
      " hcontig_004_076\t314710\t316060\tEVM prediction%2hcontig_004_0076.41\t.\t-\n",
      " hcontig_004_076\t326117\t326720\tEVM prediction%2hcontig_004_0076.43\t.\t+\n",
      " hcontig_004_076\t327005\t329393\tEVM prediction%2hcontig_004_0076.44\t.\t+\n",
      " hcontig_004_076\t329457\t334014\tEVM prediction%2hcontig_004_0076.45\t.\t+\n",
      " hcontig_004_076\t336553\t338695\tEVM prediction%2hcontig_004_0076.46\t.\t+\n",
      " hcontig_004_076\t361597\t362560\tEVM prediction%2hcontig_004_0076.55\t.\t-\n",
      " hcontig_005_002\t101942\t102962\tEVM prediction%2hcontig_0005_002.25\t.\t+\n",
      " hcontig_005_002\t102992\t106802\tEVM prediction%2hcontig_0005_002.26\t.\t+\n",
      " hcontig_005_002\t138466\t139570\tEVM prediction%2hcontig_0005_002.33\t.\t+\n",
      " hcontig_005_002\t140021\t141411\tEVM prediction%2hcontig_0005_002.34\t.\t-\n",
      " hcontig_005_002\t153094\t157042\tEVM prediction%2hcontig_0005_002.38\t.\t-\n",
      " hcontig_005_002\t165651\t166695\tEVM prediction%2hcontig_0005_002.41\t.\t-\n",
      " hcontig_005_002\t167057\t170540\tEVM prediction%2hcontig_0005_002.42\t.\t-\n",
      " hcontig_005_002\t178763\t179681\tEVM prediction%2hcontig_0005_002.46\t.\t+\n",
      " hcontig_005_002\t216113\t217076\tEVM prediction%2hcontig_0005_002.50\t.\t-\n",
      " hcontig_005_002\t217143\t219366\tEVM prediction%2hcontig_0005_002.51\t.\t-\n",
      " hcontig_005_002\t219392\t221732\tEVM prediction%2hcontig_0005_002.52\t.\t-\n",
      " hcontig_005_002\t239935\t240754\tEVM prediction%2hcontig_0005_002.57\t.\t-\n",
      " hcontig_005_002\t249955\t253678\tEVM prediction%2hcontig_0005_002.60\t.\t-\n",
      " hcontig_005_002\t254623\t255073\tEVM prediction%2hcontig_0005_002.61\t.\t-\n",
      " hcontig_005_002\t261959\t262835\tEVM prediction%2hcontig_0005_002.66\t.\t+\n",
      " hcontig_005_002\t273576\t274579\tEVM prediction%2hcontig_0005_002.69\t.\t+\n",
      " hcontig_005_002\t274630\t274943\tEVM prediction%2hcontig_0005_002.70\t.\t+\n",
      " hcontig_005_002\t289260\t290118\tEVM prediction%2hcontig_0005_002.74\t.\t-\n",
      " hcontig_005_002\t293467\t294189\tEVM prediction%2hcontig_0005_002.76\t.\t-\n",
      " hcontig_005_002\t332203\t334420\tEVM prediction%2hcontig_0005_002.80\t.\t+\n",
      " hcontig_005_002\t378619\t379639\tEVM prediction%2hcontig_0005_002.93\t.\t-\n",
      " hcontig_005_002\t383807\t384770\tEVM prediction%2hcontig_0005_002.94\t.\t-\n",
      " hcontig_005_002\t389041\t390808\tEVM prediction%2hcontig_0005_002.97\t.\t-\n",
      " hcontig_005_002\t390960\t392475\tEVM prediction%2hcontig_0005_002.98\t.\t-\n",
      " hcontig_005_002\t392645\t393560\tEVM prediction%2hcontig_0005_002.99\t.\t-\n",
      " hcontig_005_002\t403419\t403683\tEVM prediction%2hcontig_0005_002.104\t.\t-\n",
      " hcontig_005_013\t34662\t38785\tEVM prediction%2hcontig_0005_013.11\t.\t+\n",
      " hcontig_005_015\t11373\t17445\tEVM prediction%2hcontig_0005_015.3\t.\t-\n",
      " hcontig_005_015\t22821\t24035\tEVM prediction%2hcontig_0005_015.6\t.\t+\n",
      " hcontig_005_021\t105727\t106798\tEVM prediction%2hcontig_0005_021.24\t.\t+\n",
      " hcontig_005_021\t107270\t108651\tEVM prediction%2hcontig_0005_021.25\t.\t-\n",
      " hcontig_005_021\t208583\t210670\tEVM prediction%2hcontig_0005_021.48\t.\t+\n",
      " hcontig_005_021\t211883\t212929\tEVM prediction%2hcontig_0005_021.49\t.\t+\n",
      " hcontig_005_021\t259863\t260484\tEVM prediction%2hcontig_0005_021.59\t.\t-\n",
      " hcontig_005_021\t261697\t265181\tEVM prediction%2hcontig_0005_021.60\t.\t+\n",
      " hcontig_005_021\t266795\t267383\tEVM prediction%2hcontig_0005_021.61\t.\t-\n",
      " hcontig_005_021\t267475\t267922\tEVM prediction%2hcontig_0005_021.62\t.\t-\n",
      " hcontig_005_021\t318759\t323487\tEVM prediction%2hcontig_0005_021.72\t.\t-\n",
      " hcontig_005_021\t342363\t343116\tEVM prediction%2hcontig_0005_021.76\t.\t+\n",
      " hcontig_005_021\t346225\t347074\tEVM prediction%2hcontig_0005_021.78\t.\t+\n",
      " hcontig_005_021\t347393\t348341\tEVM prediction%2hcontig_0005_021.79\t.\t+\n",
      " hcontig_005_021\t350160\t350649\tEVM prediction%2hcontig_0005_021.80\t.\t+\n",
      " hcontig_005_021\t356133\t356688\tEVM prediction%2hcontig_0005_021.82\t.\t-\n",
      " hcontig_005_021\t358144\t360392\tEVM prediction%2hcontig_0005_021.83\t.\t-\n",
      " hcontig_005_021\t364638\t365193\tEVM prediction%2hcontig_0005_021.85\t.\t-\n",
      " hcontig_005_021\t365283\t371532\tEVM prediction%2hcontig_0005_021.86\t.\t-\n",
      " hcontig_005_021\t373638\t376095\tEVM prediction%2hcontig_0005_021.87\t.\t+\n",
      " hcontig_005_021\t385383\t387486\tEVM prediction%2hcontig_0005_021.89\t.\t-\n",
      " hcontig_005_021\t392818\t393172\tEVM prediction%2hcontig_0005_021.92\t.\t+\n",
      " hcontig_005_028\t10835\t12922\tEVM prediction%2hcontig_0005_028.3\t.\t+\n",
      " hcontig_005_028\t14135\t15181\tEVM prediction%2hcontig_0005_028.4\t.\t+\n",
      " hcontig_005_039\t8301\t11271\tEVM prediction%2hcontig_0005_039.3\t.\t+\n",
      " hcontig_005_039\t14192\t15314\tEVM prediction%2hcontig_0005_039.5\t.\t+\n",
      " hcontig_005_039\t16557\t17714\tEVM prediction%2hcontig_0005_039.6\t.\t+\n",
      " hcontig_005_039\t18684\t20397\tEVM prediction%2hcontig_0005_039.7\t.\t+\n",
      " hcontig_005_039\t27460\t31606\tEVM prediction%2hcontig_0005_039.10\t.\t-\n",
      " hcontig_005_043\t65346\t69471\tEVM prediction%2hcontig_0005_043.18\t.\t+\n",
      " hcontig_005_043\t88469\t89450\tEVM prediction%2hcontig_0005_043.21\t.\t+\n",
      " hcontig_005_043\t90710\t93114\tEVM prediction%2hcontig_0005_043.22\t.\t+\n",
      " hcontig_005_043\t135345\t136726\tEVM prediction%2hcontig_0005_043.34\t.\t+\n",
      " hcontig_005_043\t160803\t161491\tEVM prediction%2hcontig_0005_043.40\t.\t+\n",
      " hcontig_005_043\t209312\t211207\tEVM prediction%2hcontig_0005_043.51\t.\t+\n",
      " hcontig_005_043\t217619\t220337\tEVM prediction%2hcontig_0005_043.53\t.\t-\n",
      " hcontig_005_043\t220625\t221908\tEVM prediction%2hcontig_0005_043.54\t.\t-\n",
      " hcontig_005_043\t242907\t245298\tEVM prediction%2hcontig_0005_043.58\t.\t-\n",
      " hcontig_005_043\t457837\t459361\tEVM prediction%2hcontig_0005_043.98\t.\t-\n",
      " hcontig_005_043\t491360\t492305\tEVM prediction%2hcontig_0005_043.103\t.\t-\n",
      " hcontig_005_043\t500469\t501669\tEVM prediction%2hcontig_0005_043.104\t.\t-\n",
      " hcontig_005_043\t553432\t555091\tEVM prediction%2hcontig_0005_043.111\t.\t+\n",
      " hcontig_005_043\t556673\t558386\tEVM prediction%2hcontig_0005_043.112\t.\t+\n",
      " hcontig_005_043\t578046\t580750\tEVM prediction%2hcontig_0005_043.116\t.\t+\n",
      " hcontig_005_043\t620836\t621508\tEVM prediction%2hcontig_0005_043.119\t.\t+\n",
      " hcontig_005_043\t623960\t624817\tEVM prediction%2hcontig_0005_043.120\t.\t+\n",
      " hcontig_005_043\t629266\t630979\tEVM prediction%2hcontig_0005_043.122\t.\t-\n",
      " hcontig_005_043\t632059\t635342\tEVM prediction%2hcontig_0005_043.123\t.\t-\n",
      " hcontig_005_043\t645111\t649017\tEVM prediction%2hcontig_0005_043.124\t.\t-\n",
      " hcontig_005_043\t654170\t657550\tEVM prediction%2hcontig_0005_043.126\t.\t+\n",
      " hcontig_005_043\t660828\t663957\tEVM prediction%2hcontig_0005_043.128\t.\t+\n",
      " hcontig_005_043\t665526\t666246\tEVM prediction%2hcontig_0005_043.129\t.\t-\n",
      " hcontig_005_043\t667512\t668139\tEVM prediction%2hcontig_0005_043.130\t.\t-\n",
      " hcontig_005_043\t733046\t734108\tEVM prediction%2hcontig_0005_043.149\t.\t+\n",
      " hcontig_005_043\t734554\t737686\tEVM prediction%2hcontig_0005_043.150\t.\t+\n",
      " hcontig_005_043\t785917\t786479\tEVM prediction%2hcontig_0005_043.161\t.\t+\n",
      " hcontig_005_043\t798468\t801320\tEVM prediction%2hcontig_0005_043.168\t.\t-\n",
      " hcontig_005_043\t802388\t803654\tEVM prediction%2hcontig_0005_043.170\t.\t-\n",
      " hcontig_005_043\t804791\t806553\tEVM prediction%2hcontig_0005_043.171\t.\t-\n",
      " hcontig_005_043\t813181\t814363\tEVM prediction%2hcontig_0005_043.173\t.\t-\n",
      " hcontig_005_043\t815191\t816211\tEVM prediction%2hcontig_0005_043.174\t.\t-\n",
      " hcontig_005_043\t955642\t956140\tEVM prediction%2hcontig_0005_043.212\t.\t-\n",
      " hcontig_005_043\t956203\t957879\tEVM prediction%2hcontig_0005_043.213\t.\t-\n",
      " hcontig_005_043\t958732\t959182\tEVM prediction%2hcontig_0005_043.214\t.\t+\n",
      " hcontig_005_043\t967765\t969099\tEVM prediction%2hcontig_0005_043.215\t.\t-\n",
      " hcontig_005_043\t969411\t969823\tEVM prediction%2hcontig_0005_043.216\t.\t-\n",
      " hcontig_005_043\t1017082\t1020057\tEVM prediction%2hcontig_0005_043.225\t.\t+\n",
      " hcontig_005_043\t1020347\t1020948\tEVM prediction%2hcontig_0005_043.226\t.\t+\n",
      " hcontig_005_043\t1131344\t1132941\tEVM prediction%2hcontig_0005_043.247\t.\t+\n",
      " hcontig_006_002\t77831\t78842\tEVM prediction%2hcontig_0006_002.16\t.\t+\n",
      " hcontig_006_002\t112726\t115410\tEVM prediction%2hcontig_0006_002.23\t.\t-\n",
      " hcontig_006_002\t147742\t149174\tEVM prediction%2hcontig_0006_002.35\t.\t+\n",
      " hcontig_006_013\t26769\t30375\tEVM prediction%2hcontig_0006_013.7\t.\t+\n",
      " hcontig_006_013\t67292\t71633\tEVM prediction%2hcontig_0006_013.12\t.\t+\n",
      " hcontig_006_013\t73293\t76252\tEVM prediction%2hcontig_0006_013.13\t.\t-\n",
      " hcontig_006_013\t129382\t129889\tEVM prediction%2hcontig_0006_013.27\t.\t-\n",
      " hcontig_006_013\t130822\t131890\tEVM prediction%2hcontig_0006_013.28\t.\t-\n",
      " hcontig_006_013\t134598\t135570\tEVM prediction%2hcontig_0006_013.29\t.\t-\n",
      " hcontig_006_013\t193566\t194916\tEVM prediction%2hcontig_0006_013.39\t.\t+\n",
      " hcontig_006_013\t195392\t196853\tEVM prediction%2hcontig_0006_013.40\t.\t+\n",
      " hcontig_006_013\t208985\t211260\tEVM prediction%2hcontig_0006_013.41\t.\t+\n",
      " hcontig_006_013\t230685\t231585\tEVM prediction%2hcontig_0006_013.48\t.\t+\n",
      " hcontig_006_013\t248134\t249115\tEVM prediction%2hcontig_0006_013.50\t.\t+\n",
      " hcontig_006_013\t249345\t253327\tEVM prediction%2hcontig_0006_013.51\t.\t+\n",
      " hcontig_006_013\t254565\t256367\tEVM prediction%2hcontig_0006_013.52\t.\t+\n",
      " hcontig_006_013\t257141\t258035\tEVM prediction%2hcontig_0006_013.53\t.\t+\n",
      " hcontig_006_013\t261214\t264223\tEVM prediction%2hcontig_0006_013.54\t.\t-\n",
      " hcontig_006_013\t312592\t314384\tEVM prediction%2hcontig_0006_013.67\t.\t-\n",
      " hcontig_006_013\t455449\t460108\tEVM prediction%2hcontig_0006_013.101\t.\t-\n",
      " hcontig_006_013\t516540\t516780\tEVM prediction%2hcontig_0006_013.116\t.\t+\n",
      " hcontig_006_013\t517251\t517581\tEVM prediction%2hcontig_0006_013.117\t.\t+\n",
      " hcontig_006_013\t518281\t519093\tEVM prediction%2hcontig_0006_013.118\t.\t-\n",
      " hcontig_006_013\t568712\t569786\tEVM prediction%2hcontig_0006_013.125\t.\t-\n",
      " hcontig_006_013\t569921\t571307\tEVM prediction%2hcontig_0006_013.126\t.\t-\n",
      " hcontig_006_013\t686048\t686666\tEVM prediction%2hcontig_0006_013.159\t.\t+\n",
      " hcontig_006_013\t687705\t691035\tEVM prediction%2hcontig_0006_013.160\t.\t+\n",
      " hcontig_006_013\t705128\t707952\tEVM prediction%2hcontig_0006_013.164\t.\t-\n",
      " hcontig_006_013\t795913\t799972\tEVM prediction%2hcontig_0006_013.184\t.\t+\n",
      " hcontig_006_013\t828315\t829727\tEVM prediction%2hcontig_0006_013.195\t.\t-\n",
      " hcontig_006_013\t892293\t893427\tEVM prediction%2hcontig_0006_013.206\t.\t+\n",
      " hcontig_006_013\t894432\t896535\tEVM prediction%2hcontig_0006_013.207\t.\t+\n",
      " hcontig_006_013\t898897\t903706\tEVM prediction%2hcontig_0006_013.208\t.\t+\n",
      " hcontig_006_013\t929595\t931799\tEVM prediction%2hcontig_0006_013.213\t.\t+\n",
      " hcontig_006_014\t21\t681\tEVM prediction%2hcontig_0006_014.1\t.\t+\n",
      " hcontig_006_014\t4044\t8061\tEVM prediction%2hcontig_0006_014.3\t.\t-\n",
      " hcontig_006_014\t9134\t10634\tEVM prediction%2hcontig_0006_014.4\t.\t+\n",
      " hcontig_006_014\t10938\t14248\tEVM prediction%2hcontig_0006_014.5\t.\t+\n",
      " hcontig_006_015\t18555\t19929\tEVM prediction%2hcontig_0006_015.2\t.\t-\n",
      " hcontig_006_015\t39364\t40216\tEVM prediction%2hcontig_0006_015.5\t.\t+\n",
      " hcontig_006_015\t43579\t44554\tEVM prediction%2hcontig_0006_015.7\t.\t-\n",
      " hcontig_006_015\t45161\t46156\tEVM prediction%2hcontig_0006_015.8\t.\t-\n",
      " hcontig_006_015\t46889\t47591\tEVM prediction%2hcontig_0006_015.9\t.\t-\n",
      " hcontig_006_015\t49137\t50162\tEVM prediction%2hcontig_0006_015.10\t.\t+\n",
      " hcontig_006_015\t50466\t53776\tEVM prediction%2hcontig_0006_015.11\t.\t+\n",
      " hcontig_006_015\t64071\t67522\tEVM prediction%2hcontig_0006_015.15\t.\t+\n",
      " hcontig_006_015\t74230\t79271\tEVM prediction%2hcontig_0006_015.16\t.\t-\n",
      " hcontig_006_015\t79921\t80960\tEVM prediction%2hcontig_0006_015.17\t.\t-\n",
      " hcontig_006_015\t96146\t96743\tEVM prediction%2hcontig_0006_015.20\t.\t+\n",
      " hcontig_006_015\t98366\t99023\tEVM prediction%2hcontig_0006_015.21\t.\t+\n",
      " hcontig_006_015\t275972\t277793\tEVM prediction%2hcontig_0006_015.65\t.\t-\n",
      " hcontig_006_015\t278773\t279559\tEVM prediction%2hcontig_0006_015.67\t.\t-\n",
      " hcontig_006_015\t360930\t364036\tEVM prediction%2hcontig_0006_015.90\t.\t+\n",
      " hcontig_006_015\t375127\t375790\tEVM prediction%2hcontig_0006_015.93\t.\t+\n",
      " hcontig_006_015\t396959\t401239\tEVM prediction%2hcontig_0006_015.98\t.\t+\n",
      " hcontig_006_015\t403731\t404104\tEVM prediction%2hcontig_0006_015.99\t.\t-\n",
      " hcontig_006_015\t412942\t416270\tEVM prediction%2hcontig_0006_015.101\t.\t+\n",
      " hcontig_006_033\t74327\t79415\tEVM prediction%2hcontig_0006_033.19\t.\t+\n",
      " hcontig_006_033\t173401\t174415\tEVM prediction%2hcontig_0006_033.39\t.\t+\n",
      " hcontig_006_033\t175042\t177433\tEVM prediction%2hcontig_0006_033.40\t.\t+\n",
      " hcontig_006_033\t187211\t189602\tEVM prediction%2hcontig_0006_033.44\t.\t+\n",
      " hcontig_006_033\t192510\t193574\tEVM prediction%2hcontig_0006_033.46\t.\t-\n",
      " hcontig_006_131\t1965\t7071\tEVM prediction%2hcontig_006_0131.2\t.\t-\n",
      " hcontig_006_131\t22879\t23389\tEVM prediction%2hcontig_006_0131.7\t.\t-\n",
      " hcontig_006_131\t25297\t28933\tEVM prediction%2hcontig_006_0131.8\t.\t-\n",
      " hcontig_006_131\t63181\t67525\tEVM prediction%2hcontig_006_0131.23\t.\t+\n",
      " hcontig_006_131\t69642\t72807\tEVM prediction%2hcontig_006_0131.24\t.\t-\n",
      " hcontig_006_131\t73037\t75557\tEVM prediction%2hcontig_006_0131.25\t.\t-\n",
      " hcontig_007_005\t55399\t56125\tEVM prediction%2hcontig_0007_005.16\t.\t+\n",
      " hcontig_007_005\t85516\t92130\tEVM prediction%2hcontig_0007_005.26\t.\t-\n",
      " hcontig_007_005\t95103\t95490\tEVM prediction%2hcontig_0007_005.27\t.\t+\n",
      " hcontig_007_005\t148593\t150456\tEVM prediction%2hcontig_0007_005.35\t.\t+\n",
      " hcontig_007_005\t162989\t163577\tEVM prediction%2hcontig_0007_005.38\t.\t-\n",
      " hcontig_007_005\t253612\t254839\tEVM prediction%2hcontig_0007_005.67\t.\t-\n",
      " hcontig_007_005\t254939\t257735\tEVM prediction%2hcontig_0007_005.68\t.\t-\n",
      " hcontig_007_005\t260129\t261812\tEVM prediction%2hcontig_0007_005.70\t.\t+\n",
      " hcontig_007_005\t262021\t263977\tEVM prediction%2hcontig_0007_005.71\t.\t+\n",
      " hcontig_007_005\t264852\t266535\tEVM prediction%2hcontig_0007_005.72\t.\t+\n",
      " hcontig_007_005\t266744\t268700\tEVM prediction%2hcontig_0007_005.73\t.\t+\n",
      " hcontig_007_006\t6854\t9970\tEVM prediction%2hcontig_0007_006.3\t.\t-\n",
      " hcontig_007_006\t23454\t25086\tEVM prediction%2hcontig_0007_006.8\t.\t-\n",
      " hcontig_007_006\t25209\t27528\tEVM prediction%2hcontig_0007_006.9\t.\t-\n",
      " hcontig_007_006\t79798\t80629\tEVM prediction%2hcontig_0007_006.17\t.\t+\n",
      " hcontig_007_006\t82234\t83623\tEVM prediction%2hcontig_0007_006.18\t.\t+\n",
      " hcontig_007_006\t89952\t91158\tEVM prediction%2hcontig_0007_006.20\t.\t+\n",
      " hcontig_007_006\t92050\t92936\tEVM prediction%2hcontig_0007_006.21\t.\t+\n",
      " hcontig_007_006\t95310\t98076\tEVM prediction%2hcontig_0007_006.22\t.\t+\n",
      " hcontig_007_006\t133022\t136174\tEVM prediction%2hcontig_0007_006.32\t.\t-\n",
      " hcontig_007_006\t160011\t162173\tEVM prediction%2hcontig_0007_006.36\t.\t-\n",
      " hcontig_007_006\t168125\t168569\tEVM prediction%2hcontig_0007_006.39\t.\t-\n",
      " hcontig_007_006\t168711\t169251\tEVM prediction%2hcontig_0007_006.40\t.\t-\n",
      " hcontig_007_006\t177007\t180906\tEVM prediction%2hcontig_0007_006.43\t.\t-\n",
      " hcontig_007_006\t225682\t227130\tEVM prediction%2hcontig_0007_006.57\t.\t+\n",
      " hcontig_007_006\t246416\t251174\tEVM prediction%2hcontig_0007_006.62\t.\t-\n",
      " hcontig_007_006\t284510\t285914\tEVM prediction%2hcontig_0007_006.73\t.\t-\n",
      " hcontig_007_006\t287422\t288229\tEVM prediction%2hcontig_0007_006.74\t.\t-\n",
      " hcontig_007_006\t299685\t301145\tEVM prediction%2hcontig_0007_006.77\t.\t-\n",
      " hcontig_007_006\t301202\t302015\tEVM prediction%2hcontig_0007_006.78\t.\t-\n",
      " hcontig_007_006\t404911\t405529\tEVM prediction%2hcontig_0007_006.93\t.\t+\n",
      " hcontig_007_006\t405990\t410163\tEVM prediction%2hcontig_0007_006.94\t.\t+\n",
      " hcontig_007_006\t483432\t484436\tEVM prediction%2hcontig_0007_006.108\t.\t-\n",
      " hcontig_007_006\t485328\t487486\tEVM prediction%2hcontig_0007_006.109\t.\t-\n",
      " hcontig_007_006\t507399\t509310\tEVM prediction%2hcontig_0007_006.116\t.\t+\n",
      " hcontig_007_006\t510722\t511996\tEVM prediction%2hcontig_0007_006.117\t.\t+\n",
      " hcontig_007_006\t513427\t514669\tEVM prediction%2hcontig_0007_006.118\t.\t+\n",
      " hcontig_007_006\t520644\t521067\tEVM prediction%2hcontig_0007_006.120\t.\t+\n",
      " hcontig_007_006\t534158\t538538\tEVM prediction%2hcontig_0007_006.121\t.\t+\n",
      " hcontig_007_006\t556484\t561941\tEVM prediction%2hcontig_0007_006.128\t.\t+\n",
      " hcontig_007_006\t576895\t580061\tEVM prediction%2hcontig_0007_006.132\t.\t-\n",
      " hcontig_007_006\t599598\t602768\tEVM prediction%2hcontig_0007_006.137\t.\t+\n",
      " hcontig_007_006\t623200\t625006\tEVM prediction%2hcontig_0007_006.143\t.\t-\n",
      " hcontig_007_006\t625141\t625681\tEVM prediction%2hcontig_0007_006.144\t.\t-\n",
      " hcontig_007_006\t625696\t626143\tEVM prediction%2hcontig_0007_006.145\t.\t-\n",
      " hcontig_007_006\t665030\t665393\tEVM prediction%2hcontig_0007_006.156\t.\t+\n",
      " hcontig_007_006\t671187\t672921\tEVM prediction%2hcontig_0007_006.159\t.\t-\n",
      " hcontig_007_006\t681300\t682400\tEVM prediction%2hcontig_0007_006.162\t.\t+\n",
      " hcontig_007_006\t684423\t686435\tEVM prediction%2hcontig_0007_006.163\t.\t+\n",
      " hcontig_007_006\t733784\t734276\tEVM prediction%2hcontig_0007_006.176\t.\t-\n",
      " hcontig_007_006\t752731\t753957\tEVM prediction%2hcontig_0007_006.180\t.\t+\n",
      " hcontig_007_006\t754254\t755314\tEVM prediction%2hcontig_0007_006.181\t.\t-\n",
      " hcontig_007_006\t756670\t757743\tEVM prediction%2hcontig_0007_006.182\t.\t+\n",
      " hcontig_007_006\t770762\t771392\tEVM prediction%2hcontig_0007_006.186\t.\t-\n",
      " hcontig_007_006\t789673\t790910\tEVM prediction%2hcontig_0007_006.189\t.\t+\n",
      " hcontig_007_006\t803231\t803867\tEVM prediction%2hcontig_0007_006.193\t.\t+\n",
      " hcontig_007_006\t818026\t822688\tEVM prediction%2hcontig_0007_006.195\t.\t-\n",
      " hcontig_007_006\t891889\t892438\tEVM prediction%2hcontig_0007_006.213\t.\t-\n",
      " hcontig_007_006\t898852\t899734\tEVM prediction%2hcontig_0007_006.215\t.\t+\n",
      " hcontig_007_006\t913918\t916483\tEVM prediction%2hcontig_0007_006.218\t.\t-\n",
      " hcontig_007_006\t916759\t919006\tEVM prediction%2hcontig_0007_006.219\t.\t-\n",
      " hcontig_007_006\t935403\t939052\tEVM prediction%2hcontig_0007_006.224\t.\t-\n",
      " hcontig_007_007\t3520\t4105\tEVM prediction%2hcontig_0007_007.2\t.\t+\n",
      " hcontig_007_007\t4277\t5673\tEVM prediction%2hcontig_0007_007.3\t.\t+\n",
      " hcontig_007_007\t6160\t7039\tEVM prediction%2hcontig_0007_007.4\t.\t-\n",
      " hcontig_007_007\t7779\t8440\tEVM prediction%2hcontig_0007_007.5\t.\t+\n",
      " hcontig_007_007\t8776\t9607\tEVM prediction%2hcontig_0007_007.6\t.\t+\n",
      " hcontig_007_007\t9929\t10847\tEVM prediction%2hcontig_0007_007.7\t.\t+\n",
      " hcontig_007_007\t18263\t18885\tEVM prediction%2hcontig_0007_007.9\t.\t-\n",
      " hcontig_007_007\t20083\t22273\tEVM prediction%2hcontig_0007_007.10\t.\t+\n",
      " hcontig_007_007\t22682\t24116\tEVM prediction%2hcontig_0007_007.11\t.\t+\n",
      " hcontig_007_007\t77066\t78608\tEVM prediction%2hcontig_0007_007.21\t.\t-\n",
      " hcontig_007_007\t78637\t79384\tEVM prediction%2hcontig_0007_007.22\t.\t-\n",
      " hcontig_007_007\t79455\t80319\tEVM prediction%2hcontig_0007_007.23\t.\t-\n",
      " hcontig_007_007\t80612\t81836\tEVM prediction%2hcontig_0007_007.24\t.\t-\n",
      " hcontig_007_007\t111258\t115083\tEVM prediction%2hcontig_0007_007.30\t.\t-\n",
      " hcontig_007_007\t124142\t125015\tEVM prediction%2hcontig_0007_007.32\t.\t-\n",
      " hcontig_007_007\t125414\t128489\tEVM prediction%2hcontig_0007_007.33\t.\t-\n",
      " hcontig_007_018\t256\t1267\tEVM prediction%2hcontig_0007_018.1\t.\t+\n",
      " hcontig_007_018\t2773\t4105\tEVM prediction%2hcontig_0007_018.2\t.\t+\n",
      " hcontig_007_025\t15218\t15751\tEVM prediction%2hcontig_0007_025.4\t.\t-\n",
      " hcontig_007_033\t7942\t8500\tEVM prediction%2hcontig_0007_033.2\t.\t-\n",
      " hcontig_008_009\t5460\t9363\tEVM prediction%2hcontig_0008_009.2\t.\t-\n",
      " hcontig_008_009\t61016\t65357\tEVM prediction%2hcontig_0008_009.15\t.\t-\n",
      " hcontig_008_009\t86800\t87415\tEVM prediction%2hcontig_0008_009.20\t.\t-\n",
      " hcontig_008_009\t242807\t243569\tEVM prediction%2hcontig_0008_009.55\t.\t+\n",
      " hcontig_008_009\t243599\t245126\tEVM prediction%2hcontig_0008_009.56\t.\t+\n",
      " hcontig_008_009\t245165\t245996\tEVM prediction%2hcontig_0008_009.57\t.\t+\n",
      " hcontig_008_009\t292384\t295147\tEVM prediction%2hcontig_0008_009.64\t.\t-\n",
      " hcontig_008_009\t296610\t297597\tEVM prediction%2hcontig_0008_009.65\t.\t-\n",
      " hcontig_008_009\t366671\t370290\tEVM prediction%2hcontig_0008_009.83\t.\t-\n",
      " hcontig_008_009\t373036\t376762\tEVM prediction%2hcontig_0008_009.85\t.\t-\n",
      " hcontig_008_009\t379429\t380414\tEVM prediction%2hcontig_0008_009.87\t.\t-\n",
      " hcontig_008_009\t409831\t414025\tEVM prediction%2hcontig_0008_009.93\t.\t+\n",
      " hcontig_008_009\t482074\t483310\tEVM prediction%2hcontig_0008_009.110\t.\t-\n",
      " hcontig_008_009\t497711\t499220\tEVM prediction%2hcontig_0008_009.115\t.\t+\n",
      " hcontig_008_009\t504562\t505423\tEVM prediction%2hcontig_0008_009.118\t.\t+\n",
      " hcontig_008_009\t580839\t584178\tEVM prediction%2hcontig_0008_009.133\t.\t-\n",
      " hcontig_008_014\t52762\t55147\tEVM prediction%2hcontig_0008_014.14\t.\t-\n",
      " hcontig_008_014\t57577\t58129\tEVM prediction%2hcontig_0008_014.15\t.\t+\n",
      " hcontig_008_014\t87662\t88232\tEVM prediction%2hcontig_0008_014.21\t.\t+\n",
      " hcontig_008_014\t89329\t91352\tEVM prediction%2hcontig_0008_014.22\t.\t+\n",
      " hcontig_008_014\t93697\t94484\tEVM prediction%2hcontig_0008_014.24\t.\t-\n",
      " hcontig_008_014\t103144\t104434\tEVM prediction%2hcontig_0008_014.28\t.\t-\n",
      " hcontig_008_014\t140043\t141384\tEVM prediction%2hcontig_0008_014.36\t.\t+\n",
      " hcontig_008_014\t172807\t174254\tEVM prediction%2hcontig_0008_014.45\t.\t+\n",
      " hcontig_008_014\t174631\t176352\tEVM prediction%2hcontig_0008_014.46\t.\t-\n",
      " hcontig_008_014\t182913\t183750\tEVM prediction%2hcontig_0008_014.48\t.\t-\n",
      " hcontig_008_014\t207250\t208988\tEVM prediction%2hcontig_0008_014.52\t.\t+\n",
      " hcontig_008_014\t279705\t281556\tEVM prediction%2hcontig_0008_014.69\t.\t-\n",
      " hcontig_008_014\t288936\t289341\tEVM prediction%2hcontig_0008_014.70\t.\t+\n",
      " hcontig_008_014\t331495\t332515\tEVM prediction%2hcontig_0008_014.75\t.\t+\n",
      " hcontig_008_014\t337691\t338096\tEVM prediction%2hcontig_0008_014.76\t.\t-\n",
      " hcontig_008_014\t402075\t405261\tEVM prediction%2hcontig_0008_014.88\t.\t-\n",
      " hcontig_008_014\t414000\t415305\tEVM prediction%2hcontig_0008_014.90\t.\t-\n",
      " hcontig_008_014\t422608\t423049\tEVM prediction%2hcontig_0008_014.92\t.\t-\n",
      " hcontig_008_016\t15440\t17260\tEVM prediction%2hcontig_0008_016.5\t.\t+\n",
      " hcontig_008_016\t29120\t29903\tEVM prediction%2hcontig_0008_016.9\t.\t+\n",
      " hcontig_008_016\t37305\t37734\tEVM prediction%2hcontig_0008_016.12\t.\t+\n",
      " hcontig_008_016\t47358\t47811\tEVM prediction%2hcontig_0008_016.14\t.\t+\n",
      " hcontig_008_016\t48450\t50648\tEVM prediction%2hcontig_0008_016.15\t.\t+\n",
      " hcontig_008_035\t8245\t13651\tEVM prediction%2hcontig_0008_035.2\t.\t-\n",
      " hcontig_008_035\t28718\t31014\tEVM prediction%2hcontig_0008_035.5\t.\t-\n",
      " hcontig_008_035\t75106\t76336\tEVM prediction%2hcontig_0008_035.16\t.\t-\n",
      " hcontig_008_035\t84050\t85973\tEVM prediction%2hcontig_0008_035.18\t.\t-\n",
      " hcontig_008_035\t93300\t96207\tEVM prediction%2hcontig_0008_035.21\t.\t-\n",
      " hcontig_008_035\t143597\t146487\tEVM prediction%2hcontig_0008_035.33\t.\t-\n",
      " hcontig_008_035\t153540\t155133\tEVM prediction%2hcontig_0008_035.34\t.\t+\n",
      " hcontig_008_035\t156781\t157732\tEVM prediction%2hcontig_0008_035.35\t.\t+\n",
      " hcontig_008_035\t192632\t196547\tEVM prediction%2hcontig_0008_035.40\t.\t-\n",
      " hcontig_008_093\t1049\t3294\tEVM prediction%2hcontig_008_0093.1\t.\t-\n",
      " hcontig_008_093\t5399\t6497\tEVM prediction%2hcontig_008_0093.2\t.\t-\n",
      " hcontig_008_093\t8285\t9194\tEVM prediction%2hcontig_008_0093.3\t.\t-\n",
      " hcontig_008_093\t9277\t10579\tEVM prediction%2hcontig_008_0093.4\t.\t-\n",
      " hcontig_008_093\t22046\t24508\tEVM prediction%2hcontig_008_0093.8\t.\t-\n",
      " hcontig_008_093\t24680\t30131\tEVM prediction%2hcontig_008_0093.9\t.\t+\n",
      " hcontig_008_093\t31978\t32599\tEVM prediction%2hcontig_008_0093.10\t.\t+\n",
      " hcontig_008_093\t33980\t34908\tEVM prediction%2hcontig_008_0093.11\t.\t+\n",
      " hcontig_008_093\t35267\t35735\tEVM prediction%2hcontig_008_0093.12\t.\t+\n",
      " hcontig_008_093\t35912\t37892\tEVM prediction%2hcontig_008_0093.13\t.\t+\n",
      " hcontig_008_093\t44333\t45488\tEVM prediction%2hcontig_008_0093.15\t.\t-\n",
      " hcontig_008_093\t47548\t48865\tEVM prediction%2hcontig_008_0093.16\t.\t-\n",
      " hcontig_008_093\t56819\t58184\tEVM prediction%2hcontig_008_0093.19\t.\t-\n",
      " hcontig_008_093\t59198\t59555\tEVM prediction%2hcontig_008_0093.20\t.\t+\n",
      " hcontig_008_093\t94579\t95836\tEVM prediction%2hcontig_008_0093.24\t.\t+\n",
      " hcontig_008_093\t96113\t96764\tEVM prediction%2hcontig_008_0093.25\t.\t-\n",
      " hcontig_008_093\t96871\t97135\tEVM prediction%2hcontig_008_0093.26\t.\t-\n",
      " hcontig_008_126\t886\t1450\tEVM prediction%2hcontig_008_0126.1\t.\t-\n",
      " hcontig_008_126\t64523\t65534\tEVM prediction%2hcontig_008_0126.14\t.\t-\n",
      " hcontig_008_126\t65663\t66329\tEVM prediction%2hcontig_008_0126.15\t.\t-\n",
      " hcontig_009_012\t21229\t23237\tEVM prediction%2hcontig_0009_012.6\t.\t-\n",
      " hcontig_009_020\t2005\t4359\tEVM prediction%2hcontig_0009_020.1\t.\t+\n",
      " hcontig_009_024\t75222\t75768\tEVM prediction%2hcontig_0009_024.22\t.\t-\n",
      " hcontig_009_024\t78181\t79105\tEVM prediction%2hcontig_0009_024.23\t.\t-\n",
      " hcontig_009_024\t113932\t114946\tEVM prediction%2hcontig_0009_024.30\t.\t-\n",
      " hcontig_009_024\t144028\t147246\tEVM prediction%2hcontig_0009_024.39\t.\t-\n",
      " hcontig_009_024\t271693\t278329\tEVM prediction%2hcontig_0009_024.66\t.\t+\n",
      " hcontig_009_024\t282825\t283458\tEVM prediction%2hcontig_0009_024.68\t.\t+\n",
      " hcontig_009_024\t284086\t284314\tEVM prediction%2hcontig_0009_024.69\t.\t+\n",
      " hcontig_009_024\t284609\t285636\tEVM prediction%2hcontig_0009_024.70\t.\t-\n",
      " hcontig_009_024\t296809\t299468\tEVM prediction%2hcontig_0009_024.73\t.\t+\n",
      " hcontig_009_024\t338333\t341136\tEVM prediction%2hcontig_0009_024.82\t.\t-\n",
      " hcontig_009_024\t393480\t395452\tEVM prediction%2hcontig_0009_024.100\t.\t-\n",
      " hcontig_009_024\t400038\t401782\tEVM prediction%2hcontig_0009_024.102\t.\t+\n",
      " hcontig_009_024\t402620\t404930\tEVM prediction%2hcontig_0009_024.103\t.\t+\n",
      " hcontig_009_024\t456956\t458243\tEVM prediction%2hcontig_0009_024.111\t.\t+\n",
      " hcontig_009_024\t486707\t488135\tEVM prediction%2hcontig_0009_024.118\t.\t+\n",
      " hcontig_009_024\t492108\t494786\tEVM prediction%2hcontig_0009_024.120\t.\t+\n",
      " hcontig_009_024\t618647\t621197\tEVM prediction%2hcontig_0009_024.146\t.\t-\n",
      " hcontig_009_024\t621390\t622958\tEVM prediction%2hcontig_0009_024.147\t.\t-\n",
      " hcontig_009_024\t695940\t698129\tEVM prediction%2hcontig_0009_024.164\t.\t+\n",
      " hcontig_009_024\t733175\t733745\tEVM prediction%2hcontig_0009_024.175\t.\t-\n",
      " hcontig_009_024\t734639\t735913\tEVM prediction%2hcontig_0009_024.176\t.\t-\n",
      " hcontig_009_024\t860207\t863177\tEVM prediction%2hcontig_0009_024.201\t.\t-\n",
      " hcontig_009_024\t875953\t876897\tEVM prediction%2hcontig_0009_024.203\t.\t+\n",
      " hcontig_009_024\t877418\t878026\tEVM prediction%2hcontig_0009_024.204\t.\t+\n",
      " hcontig_009_024\t918934\t919363\tEVM prediction%2hcontig_0009_024.215\t.\t+\n",
      " hcontig_009_024\t919751\t920477\tEVM prediction%2hcontig_0009_024.216\t.\t+\n",
      " hcontig_009_024\t921370\t921760\tEVM prediction%2hcontig_0009_024.217\t.\t+\n",
      " hcontig_009_024\t953367\t953826\tEVM prediction%2hcontig_0009_024.221\t.\t+\n",
      " hcontig_009_024\t993839\t995717\tEVM prediction%2hcontig_0009_024.232\t.\t-\n",
      " hcontig_009_024\t1011575\t1012571\tEVM prediction%2hcontig_0009_024.235\t.\t+\n",
      " hcontig_009_024\t1013014\t1014841\tEVM prediction%2hcontig_0009_024.236\t.\t+\n",
      " hcontig_009_024\t1015220\t1016996\tEVM prediction%2hcontig_0009_024.237\t.\t+\n",
      " hcontig_009_024\t1022104\t1024122\tEVM prediction%2hcontig_0009_024.238\t.\t+\n",
      " hcontig_009_024\t1028409\t1030785\tEVM prediction%2hcontig_0009_024.240\t.\t-\n",
      " hcontig_009_024\t1030864\t1032694\tEVM prediction%2hcontig_0009_024.241\t.\t-\n",
      " hcontig_009_024\t1032960\t1034094\tEVM prediction%2hcontig_0009_024.242\t.\t-\n",
      " hcontig_009_024\t1049454\t1050346\tEVM prediction%2hcontig_0009_024.245\t.\t+\n",
      " hcontig_009_024\t1057023\t1057938\tEVM prediction%2hcontig_0009_024.246\t.\t-\n",
      " hcontig_009_024\t1062573\t1064163\tEVM prediction%2hcontig_0009_024.248\t.\t+\n",
      " hcontig_009_024\t1064318\t1068158\tEVM prediction%2hcontig_0009_024.249\t.\t-\n",
      " hcontig_009_024\t1068544\t1069555\tEVM prediction%2hcontig_0009_024.250\t.\t-\n",
      " hcontig_009_028\t14618\t16385\tEVM prediction%2hcontig_0009_028.6\t.\t-\n",
      " hcontig_009_028\t16631\t18635\tEVM prediction%2hcontig_0009_028.7\t.\t-\n",
      " hcontig_009_028\t23552\t27659\tEVM prediction%2hcontig_0009_028.8\t.\t-\n",
      " hcontig_009_028\t32403\t32991\tEVM prediction%2hcontig_0009_028.10\t.\t-\n",
      " hcontig_009_029\t954\t6153\tEVM prediction%2hcontig_0009_029.1\t.\t-\n",
      " hcontig_009_029\t9807\t10395\tEVM prediction%2hcontig_0009_029.3\t.\t-\n",
      " hcontig_009_029\t22313\t23702\tEVM prediction%2hcontig_0009_029.6\t.\t+\n",
      " hcontig_009_029\t37125\t37536\tEVM prediction%2hcontig_0009_029.10\t.\t-\n",
      " hcontig_009_029\t45710\t52530\tEVM prediction%2hcontig_0009_029.11\t.\t-\n",
      " hcontig_009_029\t61435\t62831\tEVM prediction%2hcontig_0009_029.15\t.\t-\n",
      " hcontig_009_029\t69549\t71139\tEVM prediction%2hcontig_0009_029.16\t.\t+\n",
      " hcontig_009_029\t71277\t72453\tEVM prediction%2hcontig_0009_029.17\t.\t+\n",
      " hcontig_009_029\t72495\t73455\tEVM prediction%2hcontig_0009_029.18\t.\t+\n",
      " hcontig_009_029\t75269\t76632\tEVM prediction%2hcontig_0009_029.19\t.\t+\n",
      " hcontig_009_029\t76683\t78362\tEVM prediction%2hcontig_0009_029.20\t.\t+\n",
      " hcontig_009_029\t79131\t80388\tEVM prediction%2hcontig_0009_029.21\t.\t+\n",
      " hcontig_009_029\t82898\t84785\tEVM prediction%2hcontig_0009_029.22\t.\t+\n",
      " hcontig_009_029\t119421\t122324\tEVM prediction%2hcontig_0009_029.27\t.\t+\n",
      " hcontig_009_029\t171185\t173138\tEVM prediction%2hcontig_0009_029.38\t.\t-\n",
      " hcontig_009_029\t174331\t176298\tEVM prediction%2hcontig_0009_029.39\t.\t-\n",
      " hcontig_009_142\t63\t2391\tEVM prediction%2hcontig_009_0142.1\t.\t+\n",
      " hcontig_010_010\t13826\t16117\tEVM prediction%2hcontig_0010_010.3\t.\t-\n",
      " hcontig_010_016\t88926\t91981\tEVM prediction%2hcontig_0010_016.17\t.\t-\n",
      " hcontig_010_016\t104578\t106918\tEVM prediction%2hcontig_0010_016.19\t.\t+\n",
      " hcontig_010_016\t106944\t110196\tEVM prediction%2hcontig_0010_016.20\t.\t+\n",
      " hcontig_010_016\t117339\t118155\tEVM prediction%2hcontig_0010_016.23\t.\t-\n",
      " hcontig_010_016\t119105\t119615\tEVM prediction%2hcontig_0010_016.24\t.\t-\n",
      " hcontig_010_016\t122234\t122957\tEVM prediction%2hcontig_0010_016.25\t.\t+\n",
      " hcontig_010_016\t123023\t126893\tEVM prediction%2hcontig_0010_016.26\t.\t+\n",
      " hcontig_010_016\t128117\t129027\tEVM prediction%2hcontig_0010_016.27\t.\t-\n",
      " hcontig_010_016\t153386\t154578\tEVM prediction%2hcontig_0010_016.29\t.\t+\n",
      " hcontig_010_016\t168311\t169064\tEVM prediction%2hcontig_0010_016.31\t.\t+\n",
      " hcontig_010_016\t177650\t178842\tEVM prediction%2hcontig_0010_016.33\t.\t-\n",
      " hcontig_010_016\t204330\t208692\tEVM prediction%2hcontig_0010_016.42\t.\t+\n",
      " hcontig_010_016\t214769\t215225\tEVM prediction%2hcontig_0010_016.43\t.\t+\n",
      " hcontig_010_016\t343204\t344697\tEVM prediction%2hcontig_0010_016.66\t.\t+\n",
      " hcontig_010_016\t377966\t378473\tEVM prediction%2hcontig_0010_016.77\t.\t+\n",
      " hcontig_010_016\t440140\t441016\tEVM prediction%2hcontig_0010_016.90\t.\t+\n",
      " hcontig_010_016\t504099\t507371\tEVM prediction%2hcontig_0010_016.107\t.\t+\n",
      " hcontig_010_016\t523183\t525125\tEVM prediction%2hcontig_0010_016.111\t.\t-\n",
      " hcontig_010_016\t554576\t555380\tEVM prediction%2hcontig_0010_016.117\t.\t+\n",
      " hcontig_010_016\t560812\t561695\tEVM prediction%2hcontig_0010_016.118\t.\t+\n",
      " hcontig_010_016\t562192\t563326\tEVM prediction%2hcontig_0010_016.119\t.\t-\n",
      " hcontig_010_016\t578738\t582300\tEVM prediction%2hcontig_0010_016.124\t.\t-\n",
      " hcontig_010_016\t588575\t589373\tEVM prediction%2hcontig_0010_016.126\t.\t+\n",
      " hcontig_010_016\t589508\t590141\tEVM prediction%2hcontig_0010_016.127\t.\t+\n",
      " hcontig_010_016\t590406\t591036\tEVM prediction%2hcontig_0010_016.128\t.\t+\n",
      " hcontig_010_016\t601008\t602015\tEVM prediction%2hcontig_0010_016.129\t.\t-\n",
      " hcontig_010_016\t644790\t645072\tEVM prediction%2hcontig_0010_016.135\t.\t+\n",
      " hcontig_010_016\t645156\t645450\tEVM prediction%2hcontig_0010_016.136\t.\t+\n",
      " hcontig_010_016\t659142\t661322\tEVM prediction%2hcontig_0010_016.140\t.\t+\n",
      " hcontig_010_016\t728500\t729392\tEVM prediction%2hcontig_0010_016.144\t.\t+\n",
      " hcontig_010_016\t729584\t729953\tEVM prediction%2hcontig_0010_016.145\t.\t+\n",
      " hcontig_010_016\t730926\t731942\tEVM prediction%2hcontig_0010_016.147\t.\t-\n",
      " hcontig_010_016\t735923\t736868\tEVM prediction%2hcontig_0010_016.148\t.\t+\n",
      " hcontig_010_016\t738815\t739439\tEVM prediction%2hcontig_0010_016.149\t.\t+\n",
      " hcontig_010_016\t739812\t740762\tEVM prediction%2hcontig_0010_016.150\t.\t+\n",
      " hcontig_010_016\t745329\t746487\tEVM prediction%2hcontig_0010_016.152\t.\t-\n",
      " hcontig_010_016\t762378\t763380\tEVM prediction%2hcontig_0010_016.155\t.\t+\n",
      " hcontig_010_016\t763445\t764066\tEVM prediction%2hcontig_0010_016.156\t.\t+\n",
      " hcontig_010_016\t764354\t765527\tEVM prediction%2hcontig_0010_016.157\t.\t+\n",
      " hcontig_010_016\t783659\t784538\tEVM prediction%2hcontig_0010_016.158\t.\t+\n",
      " hcontig_010_016\t785441\t785921\tEVM prediction%2hcontig_0010_016.159\t.\t-\n",
      " hcontig_010_016\t786002\t787781\tEVM prediction%2hcontig_0010_016.160\t.\t-\n",
      " hcontig_010_016\t787841\t790703\tEVM prediction%2hcontig_0010_016.161\t.\t-\n",
      " hcontig_010_016\t795352\t798208\tEVM prediction%2hcontig_0010_016.163\t.\t-\n",
      " hcontig_010_016\t820266\t821653\tEVM prediction%2hcontig_0010_016.169\t.\t+\n",
      " hcontig_010_016\t822819\t824532\tEVM prediction%2hcontig_0010_016.170\t.\t+\n",
      " hcontig_010_016\t825975\t829041\tEVM prediction%2hcontig_0010_016.171\t.\t+\n",
      " hcontig_010_016\t843356\t850295\tEVM prediction%2hcontig_0010_016.175\t.\t+\n",
      " hcontig_010_016\t874448\t875474\tEVM prediction%2hcontig_0010_016.182\t.\t+\n",
      " hcontig_010_016\t927294\t929708\tEVM prediction%2hcontig_0010_016.194\t.\t+\n",
      " hcontig_010_016\t995210\t996469\tEVM prediction%2hcontig_0010_016.213\t.\t+\n",
      " hcontig_010_016\t996562\t997588\tEVM prediction%2hcontig_0010_016.214\t.\t+\n",
      " hcontig_010_016\t1005773\t1010049\tEVM prediction%2hcontig_0010_016.217\t.\t+\n",
      " hcontig_010_016\t1055849\t1057530\tEVM prediction%2hcontig_0010_016.229\t.\t-\n",
      " hcontig_010_016\t1135591\t1138930\tEVM prediction%2hcontig_0010_016.244\t.\t-\n",
      " hcontig_010_016\t1139283\t1140510\tEVM prediction%2hcontig_0010_016.245\t.\t-\n",
      " hcontig_010_016\t1155870\t1157066\tEVM prediction%2hcontig_0010_016.251\t.\t+\n",
      " hcontig_010_016\t1157688\t1161955\tEVM prediction%2hcontig_0010_016.252\t.\t+\n",
      " hcontig_010_016\t1171171\t1171600\tEVM prediction%2hcontig_0010_016.254\t.\t+\n",
      " hcontig_010_016\t1172000\t1173178\tEVM prediction%2hcontig_0010_016.255\t.\t+\n",
      " hcontig_010_016\t1194837\t1196954\tEVM prediction%2hcontig_0010_016.261\t.\t-\n",
      " hcontig_010_016\t1222283\t1223174\tEVM prediction%2hcontig_0010_016.266\t.\t+\n",
      " hcontig_010_016\t1223349\t1226905\tEVM prediction%2hcontig_0010_016.267\t.\t+\n",
      " hcontig_010_016\t1271247\t1274523\tEVM prediction%2hcontig_0010_016.274\t.\t-\n",
      " hcontig_010_023\t9325\t10216\tEVM prediction%2hcontig_0010_023.3\t.\t+\n",
      " hcontig_010_023\t10309\t13949\tEVM prediction%2hcontig_0010_023.4\t.\t+\n",
      " hcontig_010_030\t2189\t2969\tEVM prediction%2hcontig_0010_030.1\t.\t-\n",
      " hcontig_011_002\t209\t4627\tEVM prediction%2hcontig_0011_002.1\t.\t+\n",
      " hcontig_011_002\t7414\t9701\tEVM prediction%2hcontig_0011_002.2\t.\t+\n",
      " hcontig_011_016\t22568\t24820\tEVM prediction%2hcontig_0011_016.2\t.\t-\n",
      " hcontig_011_016\t30493\t32184\tEVM prediction%2hcontig_0011_016.3\t.\t-\n",
      " hcontig_011_016\t44208\t44544\tEVM prediction%2hcontig_0011_016.7\t.\t+\n",
      " hcontig_011_016\t44655\t45459\tEVM prediction%2hcontig_0011_016.8\t.\t+\n",
      " hcontig_011_016\t45702\t48929\tEVM prediction%2hcontig_0011_016.9\t.\t+\n",
      " hcontig_011_016\t198194\t199047\tEVM prediction%2hcontig_0011_016.41\t.\t+\n",
      " hcontig_011_016\t213943\t214447\tEVM prediction%2hcontig_0011_016.46\t.\t+\n",
      " hcontig_011_016\t242718\t243273\tEVM prediction%2hcontig_0011_016.49\t.\t+\n",
      " hcontig_011_016\t246318\t247239\tEVM prediction%2hcontig_0011_016.50\t.\t+\n",
      " hcontig_011_016\t255951\t256975\tEVM prediction%2hcontig_0011_016.54\t.\t-\n",
      " hcontig_011_016\t257140\t258165\tEVM prediction%2hcontig_0011_016.55\t.\t-\n",
      " hcontig_011_016\t272481\t273546\tEVM prediction%2hcontig_0011_016.58\t.\t-\n",
      " hcontig_011_016\t279570\t280822\tEVM prediction%2hcontig_0011_016.61\t.\t+\n",
      " hcontig_011_016\t465807\t466239\tEVM prediction%2hcontig_0011_016.94\t.\t+\n",
      " hcontig_011_016\t467222\t468582\tEVM prediction%2hcontig_0011_016.96\t.\t-\n",
      " hcontig_011_016\t470007\t472893\tEVM prediction%2hcontig_0011_016.97\t.\t-\n",
      " hcontig_011_016\t515219\t517274\tEVM prediction%2hcontig_0011_016.107\t.\t+\n",
      " hcontig_011_016\t668523\t669600\tEVM prediction%2hcontig_0011_016.137\t.\t+\n",
      " hcontig_011_016\t779254\t780964\tEVM prediction%2hcontig_0011_016.164\t.\t+\n",
      " hcontig_011_016\t781402\t782876\tEVM prediction%2hcontig_0011_016.165\t.\t+\n",
      " hcontig_011_016\t840454\t841616\tEVM prediction%2hcontig_0011_016.181\t.\t+\n",
      " hcontig_011_016\t924761\t926921\tEVM prediction%2hcontig_0011_016.195\t.\t-\n",
      " hcontig_011_016\t926941\t927958\tEVM prediction%2hcontig_0011_016.196\t.\t-\n",
      " hcontig_011_016\t1009118\t1009757\tEVM prediction%2hcontig_0011_016.215\t.\t+\n",
      " hcontig_011_016\t1010853\t1011423\tEVM prediction%2hcontig_0011_016.216\t.\t+\n",
      " hcontig_011_016\t1176987\t1178019\tEVM prediction%2hcontig_0011_016.260\t.\t+\n",
      " hcontig_011_016\t1178328\t1179969\tEVM prediction%2hcontig_0011_016.261\t.\t+\n",
      " hcontig_011_016\t1180692\t1182447\tEVM prediction%2hcontig_0011_016.262\t.\t+\n",
      " hcontig_011_017\t10057\t14230\tEVM prediction%2hcontig_0011_017.2\t.\t+\n",
      " hcontig_011_017\t18803\t21164\tEVM prediction%2hcontig_0011_017.4\t.\t+\n",
      " hcontig_011_017\t42909\t46116\tEVM prediction%2hcontig_0011_017.14\t.\t-\n",
      " hcontig_011_017\t47928\t54735\tEVM prediction%2hcontig_0011_017.15\t.\t+\n",
      " hcontig_011_017\t69128\t69401\tEVM prediction%2hcontig_0011_017.19\t.\t+\n",
      " hcontig_011_017\t70064\t70499\tEVM prediction%2hcontig_0011_017.20\t.\t+\n",
      " hcontig_011_017\t71528\t71747\tEVM prediction%2hcontig_0011_017.21\t.\t+\n",
      " "
     ]
    }
   ],
   "source": [
    "methyl_bed = BedTool('/home/anjuni/analysis/coverage/gene_level/gene_anno/Pst_104E_v13_h_ctg.TE.sorted.bed')\n",
    "methyl_bed = methyl_bed.head(n=1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2215,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make files of upstream and downstream regions\n",
    "def make_flank_files(genebed_dict, genefn_dict, genome_size_fn, up_window, down_window, old_folder_name, new_folder_name, suffix):\n",
    "    \"\"\"This function makes flanking regions upstream and downstream of the input feature file.\"\"\"\n",
    "    out_fn_dict = {}\n",
    "    for gkey, gbed in genebed_dict.items():\n",
    "            tmp_bed = gbed.flank(g=genome_size_f_fn, l=up_window, r=down_window, s= True)\n",
    "            tmp_fn = genefn_dict[gkey].replace('.bed', '.%s.bed' % suffix)\n",
    "            tmp_fn = tmp_fn.replace(old_folder_name, new_folder_name) # change output file path\n",
    "            tmp_bed.saveas(tmp_fn) # save flank to a csv\n",
    "            out_fn_dict[tmp_fn.split('/')[-1]] = tmp_fn # file name as key and dataframe as value for overlap dict\n",
    "    return out_fn_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2220,
   "metadata": {},
   "outputs": [],
   "source": [
    "upstream_flank_fn_dict = make_flank_files(gene_bed_dict, gene_anno_dict, genome_size_f_fn, 1000, 0, 'gene_anno', 'upstream/input', 'upstream')\n",
    "downstream_flank_fn_dict = make_flank_files(gene_bed_dict, gene_anno_dict, genome_size_f_fn, 0, 1000, 'gene_anno', 'downstream/input', 'downstream')\n",
    "both_u_d_flank_fn_dict = make_flank_files(gene_bed_dict, gene_anno_dict, genome_size_f_fn, 1000, 1000, 'gene_anno', 'both_upstream_downstream/input', 'both_u_d')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2225,
   "metadata": {},
   "outputs": [],
   "source": [
    "upstream_flank_bed_dict = make_bed_dict(upstream_flank_fn_dict)\n",
    "downstream_flank_bed_dict = make_bed_dict(downstream_flank_fn_dict)\n",
    "both_u_d_flank_bed_dict = make_bed_dict(both_u_d_flank_fn_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2231,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed)>}"
      ]
     },
     "execution_count": 2231,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "methyl_rand_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2273,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a function to do overlaps for gene files and methylation\n",
    "def coverage_flank(genebed_dict, methylbed_dict, genefn_dict, old_folder_name, new_folder_name, suffix):\n",
    "    \"\"\"Create coverage files from:\n",
    "    Inputs: dictionary of gene pybedtools objects, dictionary of methylation pybedtools objects, dictionary of gene filenames, old folder name and new folder name.\n",
    "    Output: dictionary of pandas dataframes for all coverage files.\"\"\"\n",
    "    feature_overlap_df_dict = {}\n",
    "    for gkey, gbed in genebed_dict.items():\n",
    "        for mkey, mbed in methylbed_dict.items():\n",
    "            tmp_df = gbed.coverage(mbed, s=True).to_dataframe().iloc[:,[0,1,2,3,6,9]] # make a dataframe to put headings\n",
    "            tmp_df.rename(columns={'thickStart': 'overlap_count', 'blockCount': 'overlap_fraction'}, inplace=True) # rename headings\n",
    "            tmp_fn = genefn_dict[gkey].replace('.%s.bed' % suffix, '.bed')\n",
    "            tmp_fn = tmp_fn.replace('.bed', '.%s.%s.overlap.bed' % (mkey[:-4], suffix)) # change output file path\n",
    "            tmp_fn = tmp_fn.replace(old_folder_name, new_folder_name)\n",
    "            feature_overlap_df_dict[tmp_fn.split('/')[-1]] = tmp_df # file name as key and dataframe as value for overlap dict\n",
    "            tmp_df.to_csv(tmp_fn, sep='\\t', header=None, index=None) # save to a csv(pybedtools outputs more d.p. than BEDTools)\n",
    "    return feature_overlap_df_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "upstream_coverage_dict = coverage_flank(upstream_flank_bed_dict, methyl_bed_dict, upstream_flank_fn_dict, 'input', 'coverage', 'upstream')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "downstream_coverage_dict = coverage_flank(downstream_flank_bed_dict, methyl_bed_dict, downstream_flank_fn_dict, 'input', 'coverage', 'downstream')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "both_u_d_coverage_dict = coverage_flank(both_u_d_flank_bed_dict, methyl_bed_dict, both_u_d_flank_fn_dict, 'input', 'coverage', 'both_u_d')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "upstream_rand_coverage_dict = coverage_flank(upstream_flank_bed_dict, methyl_rand_bed_dict, upstream_flank_fn_dict, 'input', 'rand', 'upstream')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "downstream_rand_coverage_dict = coverage_flank(downstream_flank_bed_dict, methyl_rand_bed_dict, downstream_flank_fn_dict, 'input', 'rand', 'downstream')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "both_u_d_rand_coverage_dict = coverage_flank(both_u_d_flank_bed_dict, methyl_rand_bed_dict, both_u_d_flank_fn_dict, 'input', 'rand', 'both_u_d')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#14c4ff'> 6.C Intersecting 6mA with transcription start site. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 6mA is at TSS or slightly downstream (+/- 500bp)\n",
    "# Make gene files with just the first 500bp of the gene\n",
    "# Run coverage for these"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1779,
   "metadata": {},
   "outputs": [],
   "source": [
    "# write a function to take the files in gene_anno_dict, convert to df and edit the df and save out\n",
    "def make_tss_file(fn_dict):\n",
    "    out_fn_dict = {}\n",
    "    for key, value in fn_dict.items():\n",
    "        in_fn = fn_dict[key]\n",
    "        df = pd.read_csv(in_fn, sep='\\t', header = None)\n",
    "        for index, row in df.iterrows():\n",
    "            tss = row[1] + 500\n",
    "            df.iat[index,2] = tss # change the gene end site to 500bp downstream of TSS\n",
    "        out_fn = in_fn.replace('.bed', '.tss.bed')\n",
    "        out_fn = out_fn.replace('gene_anno', 'tss_6mA_only/input') # make the outfile name\n",
    "        df.to_csv(out_fn, header=None, index=None, sep='\\t') # save the new tss df to a bed file\n",
    "        outkey = out_fn.split('/')[-1]\n",
    "        out_fn_dict[outkey] = out_fn # save the outfile names to a dictionary\n",
    "    return out_fn_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1781,
   "metadata": {},
   "outputs": [],
   "source": [
    "tss_fn_dict = make_tss_file(gene_anno_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1783,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_bed_dict(fn_dict):\n",
    "    \"\"\"Takes an input filename dictionary and outputs a dictionary of pybedtools objects for the filenames.\"\"\"\n",
    "    bed_dict = {}\n",
    "    for key, value in fn_dict.items():      \n",
    "        bed_dict[key] = BedTool(value)\n",
    "    return bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2015,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make a function to do overlaps for tss files and methylation\n",
    "def coverage_tss(genebed_dict, methylbed_dict, genefn_dict, old_folder_name, new_folder_name):\n",
    "    \"\"\"Create coverage files from:\n",
    "    Inputs: dictionary of gene pybedtools objects, dictionary of methylation pybedtools objects, dictionary of gene filenames, old folder name and new folder name.\n",
    "    Output: dictionary of pandas dataframes for all coverage files.\"\"\"\n",
    "    feature_overlap_df_dict = {}\n",
    "    for gkey, gbed in genebed_dict.items():\n",
    "        for mkey, mbed in methylbed_dict.items():\n",
    "            tmp_df = gbed.coverage(mbed, s=True).to_dataframe().iloc[:,[0,1,2,3,6,9]] # make a dataframe to put headings\n",
    "            tmp_df.rename(columns={'thickStart': 'overlap_count', 'blockCount': 'overlap_fraction'}, inplace=True) # rename headings\n",
    "            tmp_fn = genefn_dict[gkey].replace('.bed', '.%s.overlap.bed' % mkey[:-4]) # change output file path\n",
    "            tmp_fn = tmp_fn.replace(old_folder_name, new_folder_name)\n",
    "            feature_overlap_df_dict[tmp_fn.split('/')[-1]] = tmp_df # file name as key and dataframe as value for overlap dict\n",
    "            tmp_df.to_csv(tmp_fn, sep='\\t', header=None, index=None) # save to a csv(pybedtools outputs more d.p. than BEDTools)\n",
    "    return feature_overlap_df_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1784,
   "metadata": {},
   "outputs": [],
   "source": [
    "tss_bed_dict = make_bed_dict(tss_fn_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1786,
   "metadata": {},
   "outputs": [],
   "source": [
    "tss_overlap_dict = coverage_gene(tss_bed_dict, methyl_bed_dict, tss_fn_dict, 'input', 'coverage')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2043,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Randomisation\n",
    "# Generating randomised files as controls\n",
    "\n",
    "# Dictionary of filepaths \n",
    "methyl_rand_dict = {}\n",
    "methyl_rand_dict['5mC_hc_tombo_sorted.cutoff.0.00.bed'] = '/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed'\n",
    "methyl_rand_dict['6mA_hc_tombo_sorted.cutoff.0.00.bed'] = '/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed'\n",
    "\n",
    "# Make a dictionary of \n",
    "methyl_rand_bed_dict = make_bed_dict(methyl_rand_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2044,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed)>}"
      ]
     },
     "execution_count": 2044,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "methyl_rand_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2017,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.00.bed)>,\n",
       " '5mC_hc_tombo_sorted.cutoff.0.50.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.50.bed)>,\n",
       " '5mC_hc_tombo_sorted.cutoff.0.90.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.90.bed)>,\n",
       " '5mC_hc_tombo_sorted.cutoff.0.99.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/5mC_hc_tombo_sorted.cutoff.0.99.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.00.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.50.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.50.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.90.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.90.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.99.bed': <BedTool(/home/anjuni/analysis/coverage/feature_files/6mA_hc_tombo_sorted.cutoff.0.99.bed)>}"
      ]
     },
     "execution_count": 2017,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "methyl_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2038,
   "metadata": {},
   "outputs": [],
   "source": [
    "tss_rand_dict = coverage_tss(tss_bed_dict, methyl_rand_bed_dict, tss_fn_dict, 'input', 'rand')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2037,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'5mC_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.00_rand.bed)>,\n",
       " '5mC_hc_tombo_sorted.cutoff.0.50.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.50_rand.bed)>,\n",
       " '5mC_hc_tombo_sorted.cutoff.0.90.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.90_rand.bed)>,\n",
       " '5mC_hc_tombo_sorted.cutoff.0.99.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/5mC_hc_tombo_sorted.cutoff.0.99_rand.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.00.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.00_rand.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.50.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.50_rand.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.90.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.90_rand.bed)>,\n",
       " '6mA_hc_tombo_sorted.cutoff.0.99.bed': <BedTool(/home/anjuni/analysis/coverage/randomisation/6mA_hc_tombo_sorted.cutoff.0.99_rand.bed)>}"
      ]
     },
     "execution_count": 2037,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "methyl_rand_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2035,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Pst_104E_v13_h_ctg.TE.sorted.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_h_ctg.TE.sorted.tss.bed)>,\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_h_ctg_combined_sorted_anno.tss.bed)>,\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_p_ctg.TE.sorted.tss.bed)>,\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_p_ctg_combined_sorted_anno.tss.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_ph_ctg.TE.sorted.tss.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_ph_ctg.effectors.tss.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_ph_ctg.non_effectors.tss.bed)>,\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.bed': <BedTool(/home/anjuni/analysis/coverage/gene_level/tss_6mA_only/input/Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.bed)>}"
      ]
     },
     "execution_count": 2035,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tss_bed_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2023,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 0             0.000  \n",
       " 2                 1             0.002  \n",
       " 3                 1             0.002  \n",
       " 4                 1             0.002  \n",
       " 5                 2             0.004  \n",
       " 6                 2             0.004  \n",
       " 7                 0             0.000  \n",
       " 8                 4             0.008  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                1             0.002  \n",
       " 13                0             0.000  \n",
       " 14                1             0.002  \n",
       " 15                4             0.008  \n",
       " 16                0             0.000  \n",
       " 17                1             0.002  \n",
       " 18                2             0.004  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                1             0.002  \n",
       " 22                3             0.006  \n",
       " 23                1             0.002  \n",
       " 24                1             0.002  \n",
       " 25                2             0.004  \n",
       " 26                5             0.008  \n",
       " 27                1             0.002  \n",
       " 28                3             0.006  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 2848              1             0.002  \n",
       " 2849              3             0.006  \n",
       " 2850              1             0.002  \n",
       " 2851              0             0.000  \n",
       " 2852              1             0.002  \n",
       " 2853              1             0.002  \n",
       " 2854              0             0.000  \n",
       " 2855              2             0.004  \n",
       " 2856              0             0.000  \n",
       " 2857              1             0.002  \n",
       " 2858              2             0.004  \n",
       " 2859              2             0.004  \n",
       " 2860              1             0.002  \n",
       " 2861              1             0.002  \n",
       " 2862              1             0.002  \n",
       " 2863              1             0.002  \n",
       " 2864              5             0.010  \n",
       " 2865              3             0.006  \n",
       " 2866              4             0.008  \n",
       " 2867              1             0.002  \n",
       " 2868              1             0.002  \n",
       " 2869              4             0.008  \n",
       " 2870              2             0.004  \n",
       " 2871              3             0.006  \n",
       " 2872              1             0.002  \n",
       " 2873              2             0.004  \n",
       " 2874              3             0.006  \n",
       " 2875              4             0.008  \n",
       " 2876              0             0.000  \n",
       " 2877              2             0.004  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 2             0.004  \n",
       " 1                 1             0.002  \n",
       " 2                 2             0.004  \n",
       " 3                 1             0.002  \n",
       " 4                 1             0.002  \n",
       " 5                 2             0.004  \n",
       " 6                 2             0.004  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 1             0.002  \n",
       " 10                2             0.004  \n",
       " 11                3             0.006  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                3             0.006  \n",
       " 16                2             0.004  \n",
       " 17                0             0.000  \n",
       " 18                3             0.006  \n",
       " 19                1             0.002  \n",
       " 20                6             0.012  \n",
       " 21                1             0.002  \n",
       " 22                2             0.004  \n",
       " 23                0             0.000  \n",
       " 24                3             0.006  \n",
       " 25                1             0.002  \n",
       " 26                0             0.000  \n",
       " 27                3             0.006  \n",
       " 28                1             0.002  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2848              2             0.004  \n",
       " 2849              1             0.002  \n",
       " 2850              0             0.000  \n",
       " 2851              2             0.004  \n",
       " 2852              4             0.008  \n",
       " 2853              0             0.000  \n",
       " 2854              2             0.004  \n",
       " 2855              1             0.002  \n",
       " 2856              0             0.000  \n",
       " 2857              2             0.004  \n",
       " 2858              0             0.000  \n",
       " 2859              1             0.002  \n",
       " 2860              2             0.004  \n",
       " 2861              0             0.000  \n",
       " 2862              1             0.002  \n",
       " 2863              1             0.002  \n",
       " 2864              0             0.000  \n",
       " 2865              2             0.004  \n",
       " 2866              1             0.002  \n",
       " 2867              0             0.000  \n",
       " 2868              4             0.008  \n",
       " 2869              1             0.002  \n",
       " 2870              1             0.002  \n",
       " 2871              0             0.000  \n",
       " 2872              2             0.004  \n",
       " 2873              0             0.000  \n",
       " 2874              1             0.002  \n",
       " 2875              1             0.002  \n",
       " 2876              1             0.002  \n",
       " 2877              0             0.000  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  0             0.000  \n",
       " 1                  5             0.010  \n",
       " 2                  1             0.002  \n",
       " 3                  3             0.006  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  3             0.006  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  4             0.008  \n",
       " 10                 3             0.006  \n",
       " 11                 2             0.004  \n",
       " 12                 1             0.002  \n",
       " 13                 2             0.004  \n",
       " 14                 2             0.004  \n",
       " 15                 3             0.006  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 1             0.002  \n",
       " 20                 1             0.002  \n",
       " 21                 1             0.002  \n",
       " 22                 1             0.002  \n",
       " 23                 1             0.002  \n",
       " 24                 1             0.002  \n",
       " 25                 1             0.002  \n",
       " 26                 2             0.004  \n",
       " 27                 3             0.006  \n",
       " 28                 2             0.004  \n",
       " 29                 2             0.004  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170              1             0.002  \n",
       " 17171              2             0.004  \n",
       " 17172              3             0.006  \n",
       " 17173              3             0.006  \n",
       " 17174              1             0.002  \n",
       " 17175              0             0.000  \n",
       " 17176              2             0.004  \n",
       " 17177              3             0.006  \n",
       " 17178              4             0.008  \n",
       " 17179              2             0.004  \n",
       " 17180              0             0.000  \n",
       " 17181              2             0.004  \n",
       " 17182              1             0.002  \n",
       " 17183              2             0.004  \n",
       " 17184              0             0.000  \n",
       " 17185              0             0.000  \n",
       " 17186              2             0.004  \n",
       " 17187              0             0.000  \n",
       " 17188              2             0.004  \n",
       " 17189              0             0.000  \n",
       " 17190              0             0.000  \n",
       " 17191              1             0.002  \n",
       " 17192              1             0.002  \n",
       " 17193              0             0.000  \n",
       " 17194              1             0.002  \n",
       " 17195              1             0.002  \n",
       " 17196              1             0.002  \n",
       " 17197              5             0.010  \n",
       " 17198              3             0.006  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  1             0.002  \n",
       " 1                  1             0.002  \n",
       " 2                  3             0.006  \n",
       " 3                  2             0.004  \n",
       " 4                  2             0.004  \n",
       " 5                  1             0.002  \n",
       " 6                  6             0.012  \n",
       " 7                  0             0.000  \n",
       " 8                  2             0.004  \n",
       " 9                  1             0.002  \n",
       " 10                 0             0.000  \n",
       " 11                 3             0.006  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 1             0.002  \n",
       " 15                 2             0.004  \n",
       " 16                 0             0.000  \n",
       " 17                 2             0.004  \n",
       " 18                 1             0.002  \n",
       " 19                 1             0.002  \n",
       " 20                 2             0.004  \n",
       " 21                 1             0.002  \n",
       " 22                 2             0.004  \n",
       " 23                 1             0.002  \n",
       " 24                 1             0.002  \n",
       " 25                 2             0.004  \n",
       " 26                 0             0.000  \n",
       " 27                 1             0.002  \n",
       " 28                 2             0.004  \n",
       " 29                 1             0.002  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170              1             0.002  \n",
       " 17171              0             0.000  \n",
       " 17172              1             0.002  \n",
       " 17173              0             0.000  \n",
       " 17174              2             0.004  \n",
       " 17175              0             0.000  \n",
       " 17176              3             0.006  \n",
       " 17177              1             0.002  \n",
       " 17178              1             0.002  \n",
       " 17179              2             0.004  \n",
       " 17180              1             0.002  \n",
       " 17181              0             0.000  \n",
       " 17182              2             0.004  \n",
       " 17183              3             0.006  \n",
       " 17184              2             0.004  \n",
       " 17185              3             0.006  \n",
       " 17186              1             0.002  \n",
       " 17187              0             0.000  \n",
       " 17188              6             0.012  \n",
       " 17189              0             0.000  \n",
       " 17190              1             0.002  \n",
       " 17191              2             0.004  \n",
       " 17192              0             0.000  \n",
       " 17193              0             0.000  \n",
       " 17194              2             0.004  \n",
       " 17195              1             0.002  \n",
       " 17196              1             0.002  \n",
       " 17197              1             0.002  \n",
       " 17198              1             0.002  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 3             0.006  \n",
       " 1                 2             0.004  \n",
       " 2                 3             0.006  \n",
       " 3                 0             0.000  \n",
       " 4                 4             0.008  \n",
       " 5                 1             0.002  \n",
       " 6                 1             0.002  \n",
       " 7                 1             0.002  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                3             0.006  \n",
       " 11                1             0.002  \n",
       " 12                1             0.002  \n",
       " 13                0             0.000  \n",
       " 14                2             0.004  \n",
       " 15                0             0.000  \n",
       " 16                3             0.006  \n",
       " 17                1             0.002  \n",
       " 18                2             0.004  \n",
       " 19                2             0.004  \n",
       " 20                2             0.004  \n",
       " 21                3             0.006  \n",
       " 22                1             0.002  \n",
       " 23                0             0.000  \n",
       " 24                2             0.004  \n",
       " 25                2             0.004  \n",
       " 26                1             0.002  \n",
       " 27                0             0.000  \n",
       " 28                2             0.004  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 3390              3             0.006  \n",
       " 3391              4             0.008  \n",
       " 3392              6             0.010  \n",
       " 3393              4             0.008  \n",
       " 3394              1             0.002  \n",
       " 3395              4             0.008  \n",
       " 3396              0             0.000  \n",
       " 3397              7             0.014  \n",
       " 3398              2             0.004  \n",
       " 3399              2             0.004  \n",
       " 3400              1             0.002  \n",
       " 3401              3             0.006  \n",
       " 3402              3             0.006  \n",
       " 3403              1             0.002  \n",
       " 3404              7             0.014  \n",
       " 3405              3             0.006  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410              1             0.002  \n",
       " 3411              2             0.004  \n",
       " 3412              1             0.002  \n",
       " 3413              3             0.006  \n",
       " 3414              1             0.002  \n",
       " 3415              0             0.000  \n",
       " 3416              2             0.004  \n",
       " 3417              2             0.004  \n",
       " 3418              0             0.000  \n",
       " 3419              5             0.010  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 2             0.004  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 3             0.006  \n",
       " 6                 3             0.006  \n",
       " 7                 1             0.002  \n",
       " 8                 4             0.008  \n",
       " 9                 1             0.002  \n",
       " 10                6             0.012  \n",
       " 11                2             0.004  \n",
       " 12                5             0.010  \n",
       " 13                1             0.002  \n",
       " 14                2             0.004  \n",
       " 15                2             0.004  \n",
       " 16                1             0.002  \n",
       " 17                1             0.002  \n",
       " 18                1             0.002  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                1             0.002  \n",
       " 22                2             0.004  \n",
       " 23                2             0.004  \n",
       " 24                0             0.000  \n",
       " 25                1             0.002  \n",
       " 26                0             0.000  \n",
       " 27                0             0.000  \n",
       " 28                2             0.004  \n",
       " 29                2             0.004  \n",
       " ...             ...               ...  \n",
       " 3390              2             0.004  \n",
       " 3391              2             0.004  \n",
       " 3392              2             0.004  \n",
       " 3393              1             0.002  \n",
       " 3394              2             0.004  \n",
       " 3395              0             0.000  \n",
       " 3396              0             0.000  \n",
       " 3397              0             0.000  \n",
       " 3398              0             0.000  \n",
       " 3399              3             0.006  \n",
       " 3400              0             0.000  \n",
       " 3401              0             0.000  \n",
       " 3402              1             0.002  \n",
       " 3403              1             0.002  \n",
       " 3404              1             0.002  \n",
       " 3405              1             0.002  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410              2             0.004  \n",
       " 3411              3             0.006  \n",
       " 3412              0             0.000  \n",
       " 3413              0             0.000  \n",
       " 3414              0             0.000  \n",
       " 3415              0             0.000  \n",
       " 3416              2             0.004  \n",
       " 3417              3             0.006  \n",
       " 3418              3             0.006  \n",
       " 3419              4             0.008  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  2             0.004  \n",
       " 1                  2             0.004  \n",
       " 2                  2             0.004  \n",
       " 3                  4             0.008  \n",
       " 4                  2             0.004  \n",
       " 5                  3             0.006  \n",
       " 6                  2             0.004  \n",
       " 7                  1             0.002  \n",
       " 8                  1             0.002  \n",
       " 9                  2             0.004  \n",
       " 10                 5             0.010  \n",
       " 11                 3             0.006  \n",
       " 12                 2             0.004  \n",
       " 13                 2             0.004  \n",
       " 14                 1             0.002  \n",
       " 15                 1             0.002  \n",
       " 16                 2             0.004  \n",
       " 17                 2             0.004  \n",
       " 18                 2             0.004  \n",
       " 19                 2             0.004  \n",
       " 20                 5             0.010  \n",
       " 21                 4             0.008  \n",
       " 22                 2             0.004  \n",
       " 23                 1             0.002  \n",
       " 24                 1             0.002  \n",
       " 25                 1             0.002  \n",
       " 26                 2             0.004  \n",
       " 27                 3             0.006  \n",
       " 28                 0             0.000  \n",
       " 29                 1             0.002  \n",
       " ...              ...               ...  \n",
       " 19318              1             0.002  \n",
       " 19319              1             0.002  \n",
       " 19320              0             0.000  \n",
       " 19321              3             0.006  \n",
       " 19322              2             0.004  \n",
       " 19323              0             0.000  \n",
       " 19324              0             0.000  \n",
       " 19325              2             0.004  \n",
       " 19326              2             0.004  \n",
       " 19327              0             0.000  \n",
       " 19328              0             0.000  \n",
       " 19329              4             0.008  \n",
       " 19330              3             0.006  \n",
       " 19331              1             0.002  \n",
       " 19332              1             0.002  \n",
       " 19333              2             0.004  \n",
       " 19334              3             0.006  \n",
       " 19335              1             0.002  \n",
       " 19336              4             0.008  \n",
       " 19337              1             0.002  \n",
       " 19338              5             0.010  \n",
       " 19339              1             0.002  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342              1             0.002  \n",
       " 19343              1             0.002  \n",
       " 19344              2             0.004  \n",
       " 19345              2             0.004  \n",
       " 19346              1             0.002  \n",
       " 19347              1             0.002  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  2             0.004  \n",
       " 1                  1             0.002  \n",
       " 2                  0             0.000  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  1             0.002  \n",
       " 6                  2             0.004  \n",
       " 7                  2             0.004  \n",
       " 8                  1             0.002  \n",
       " 9                  1             0.002  \n",
       " 10                 1             0.002  \n",
       " 11                 2             0.004  \n",
       " 12                 1             0.002  \n",
       " 13                 0             0.000  \n",
       " 14                 1             0.002  \n",
       " 15                 1             0.002  \n",
       " 16                 2             0.004  \n",
       " 17                 2             0.004  \n",
       " 18                 3             0.006  \n",
       " 19                 3             0.006  \n",
       " 20                 1             0.002  \n",
       " 21                 2             0.004  \n",
       " 22                 1             0.002  \n",
       " 23                 2             0.004  \n",
       " 24                 1             0.002  \n",
       " 25                 1             0.002  \n",
       " 26                 2             0.004  \n",
       " 27                 0             0.000  \n",
       " 28                 4             0.008  \n",
       " 29                 2             0.004  \n",
       " ...              ...               ...  \n",
       " 19318              1             0.002  \n",
       " 19319              1             0.002  \n",
       " 19320              4             0.008  \n",
       " 19321              1             0.002  \n",
       " 19322              2             0.004  \n",
       " 19323              2             0.004  \n",
       " 19324              0             0.000  \n",
       " 19325              2             0.004  \n",
       " 19326              3             0.006  \n",
       " 19327              3             0.006  \n",
       " 19328              2             0.004  \n",
       " 19329              1             0.002  \n",
       " 19330              3             0.006  \n",
       " 19331              1             0.002  \n",
       " 19332              1             0.002  \n",
       " 19333              0             0.000  \n",
       " 19334              1             0.002  \n",
       " 19335              2             0.004  \n",
       " 19336              1             0.002  \n",
       " 19337              2             0.004  \n",
       " 19338              4             0.008  \n",
       " 19339              2             0.004  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342              3             0.006  \n",
       " 19343              1             0.002  \n",
       " 19344              1             0.002  \n",
       " 19345              1             0.002  \n",
       " 19346              3             0.006  \n",
       " 19347              1             0.002  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 0             0.000  \n",
       " 2                 1             0.002  \n",
       " 3                 1             0.002  \n",
       " 4                 1             0.002  \n",
       " 5                 2             0.004  \n",
       " 6                 2             0.004  \n",
       " 7                 0             0.000  \n",
       " 8                 4             0.008  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                1             0.002  \n",
       " 13                0             0.000  \n",
       " 14                1             0.002  \n",
       " 15                4             0.008  \n",
       " 16                0             0.000  \n",
       " 17                1             0.002  \n",
       " 18                2             0.004  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                1             0.002  \n",
       " 22                3             0.006  \n",
       " 23                1             0.002  \n",
       " 24                1             0.002  \n",
       " 25                2             0.004  \n",
       " 26                5             0.008  \n",
       " 27                1             0.002  \n",
       " 28                3             0.006  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 6268              3             0.006  \n",
       " 6269              4             0.008  \n",
       " 6270              6             0.010  \n",
       " 6271              4             0.008  \n",
       " 6272              1             0.002  \n",
       " 6273              4             0.008  \n",
       " 6274              0             0.000  \n",
       " 6275              7             0.014  \n",
       " 6276              2             0.004  \n",
       " 6277              2             0.004  \n",
       " 6278              1             0.002  \n",
       " 6279              3             0.006  \n",
       " 6280              3             0.006  \n",
       " 6281              1             0.002  \n",
       " 6282              7             0.014  \n",
       " 6283              3             0.006  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288              1             0.002  \n",
       " 6289              2             0.004  \n",
       " 6290              1             0.002  \n",
       " 6291              3             0.006  \n",
       " 6292              1             0.002  \n",
       " 6293              0             0.000  \n",
       " 6294              2             0.004  \n",
       " 6295              2             0.004  \n",
       " 6296              0             0.000  \n",
       " 6297              5             0.010  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 2             0.004  \n",
       " 1                 1             0.002  \n",
       " 2                 2             0.004  \n",
       " 3                 1             0.002  \n",
       " 4                 1             0.002  \n",
       " 5                 2             0.004  \n",
       " 6                 2             0.004  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 1             0.002  \n",
       " 10                2             0.004  \n",
       " 11                3             0.006  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                3             0.006  \n",
       " 16                2             0.004  \n",
       " 17                0             0.000  \n",
       " 18                3             0.006  \n",
       " 19                1             0.002  \n",
       " 20                6             0.012  \n",
       " 21                1             0.002  \n",
       " 22                2             0.004  \n",
       " 23                0             0.000  \n",
       " 24                3             0.006  \n",
       " 25                1             0.002  \n",
       " 26                0             0.000  \n",
       " 27                3             0.006  \n",
       " 28                1             0.002  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 6268              2             0.004  \n",
       " 6269              2             0.004  \n",
       " 6270              2             0.004  \n",
       " 6271              1             0.002  \n",
       " 6272              2             0.004  \n",
       " 6273              0             0.000  \n",
       " 6274              0             0.000  \n",
       " 6275              0             0.000  \n",
       " 6276              0             0.000  \n",
       " 6277              3             0.006  \n",
       " 6278              0             0.000  \n",
       " 6279              0             0.000  \n",
       " 6280              1             0.002  \n",
       " 6281              1             0.002  \n",
       " 6282              1             0.002  \n",
       " 6283              1             0.002  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288              2             0.004  \n",
       " 6289              3             0.006  \n",
       " 6290              0             0.000  \n",
       " 6291              0             0.000  \n",
       " 6292              0             0.000  \n",
       " 6293              0             0.000  \n",
       " 6294              2             0.004  \n",
       " 6295              3             0.006  \n",
       " 6296              3             0.006  \n",
       " 6297              4             0.008  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 4             0.008  \n",
       " 2                 0             0.000  \n",
       " 3                 2             0.004  \n",
       " 4                 5             0.010  \n",
       " 5                 1             0.002  \n",
       " 6                 1             0.002  \n",
       " 7                 1             0.002  \n",
       " 8                 2             0.004  \n",
       " 9                 1             0.002  \n",
       " 10                2             0.004  \n",
       " 11                2             0.004  \n",
       " 12                1             0.002  \n",
       " 13                3             0.006  \n",
       " 14                1             0.002  \n",
       " 15                1             0.002  \n",
       " 16                3             0.006  \n",
       " 17                3             0.006  \n",
       " 18                1             0.002  \n",
       " 19                2             0.004  \n",
       " 20                2             0.004  \n",
       " 21                1             0.002  \n",
       " 22                0             0.000  \n",
       " 23                1             0.002  \n",
       " 24                0             0.000  \n",
       " 25                2             0.004  \n",
       " 26                1             0.002  \n",
       " 27                1             0.002  \n",
       " 28                1             0.002  \n",
       " 29                2             0.004  \n",
       " ...             ...               ...  \n",
       " 2008              4             0.008  \n",
       " 2009              2             0.004  \n",
       " 2010              0             0.000  \n",
       " 2011              1             0.002  \n",
       " 2012              1             0.002  \n",
       " 2013              2             0.004  \n",
       " 2014              3             0.006  \n",
       " 2015              3             0.006  \n",
       " 2016              1             0.002  \n",
       " 2017              2             0.004  \n",
       " 2018              0             0.000  \n",
       " 2019              7             0.014  \n",
       " 2020              0             0.000  \n",
       " 2021              1             0.002  \n",
       " 2022              4             0.006  \n",
       " 2023              1             0.002  \n",
       " 2024              0             0.000  \n",
       " 2025              0             0.000  \n",
       " 2026              3             0.006  \n",
       " 2027              1             0.002  \n",
       " 2028              3             0.006  \n",
       " 2029              1             0.002  \n",
       " 2030              1             0.002  \n",
       " 2031              3             0.006  \n",
       " 2032              3             0.006  \n",
       " 2033              4             0.008  \n",
       " 2034              0             0.000  \n",
       " 2035              0             0.000  \n",
       " 2036              2             0.004  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 1             0.002  \n",
       " 1                 1             0.002  \n",
       " 2                 2             0.004  \n",
       " 3                 0             0.000  \n",
       " 4                 1             0.002  \n",
       " 5                 1             0.002  \n",
       " 6                 2             0.004  \n",
       " 7                 2             0.004  \n",
       " 8                 1             0.002  \n",
       " 9                 2             0.004  \n",
       " 10                0             0.000  \n",
       " 11                2             0.004  \n",
       " 12                1             0.002  \n",
       " 13                2             0.004  \n",
       " 14                1             0.002  \n",
       " 15                3             0.006  \n",
       " 16                1             0.002  \n",
       " 17                0             0.000  \n",
       " 18                2             0.002  \n",
       " 19                1             0.002  \n",
       " 20                2             0.004  \n",
       " 21                1             0.002  \n",
       " 22                2             0.004  \n",
       " 23                1             0.002  \n",
       " 24                0             0.000  \n",
       " 25                2             0.004  \n",
       " 26                0             0.000  \n",
       " 27                4             0.008  \n",
       " 28                1             0.002  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 2008              3             0.006  \n",
       " 2009              2             0.004  \n",
       " 2010              0             0.000  \n",
       " 2011              1             0.002  \n",
       " 2012              0             0.000  \n",
       " 2013              3             0.006  \n",
       " 2014              3             0.006  \n",
       " 2015              3             0.006  \n",
       " 2016              1             0.002  \n",
       " 2017              1             0.002  \n",
       " 2018              0             0.000  \n",
       " 2019              0             0.000  \n",
       " 2020              1             0.002  \n",
       " 2021              0             0.000  \n",
       " 2022              0             0.000  \n",
       " 2023              1             0.002  \n",
       " 2024              1             0.002  \n",
       " 2025              0             0.000  \n",
       " 2026              1             0.002  \n",
       " 2027              0             0.000  \n",
       " 2028              4             0.008  \n",
       " 2029              4             0.008  \n",
       " 2030              0             0.000  \n",
       " 2031              1             0.002  \n",
       " 2032              2             0.004  \n",
       " 2033              1             0.002  \n",
       " 2034              1             0.002  \n",
       " 2035              1             0.002  \n",
       " 2036              0             0.000  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  1             0.002  \n",
       " 1                  3             0.006  \n",
       " 2                  3             0.006  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  3             0.006  \n",
       " 6                  2             0.004  \n",
       " 7                  1             0.002  \n",
       " 8                  2             0.004  \n",
       " 9                  2             0.004  \n",
       " 10                 3             0.006  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 1             0.002  \n",
       " 15                 1             0.002  \n",
       " 16                 2             0.004  \n",
       " 17                 3             0.006  \n",
       " 18                 2             0.004  \n",
       " 19                 2             0.004  \n",
       " 20                 2             0.004  \n",
       " 21                 2             0.004  \n",
       " 22                 0             0.000  \n",
       " 23                 1             0.002  \n",
       " 24                 3             0.006  \n",
       " 25                 0             0.000  \n",
       " 26                 2             0.004  \n",
       " 27                 1             0.002  \n",
       " 28                 3             0.006  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 28181              1             0.002  \n",
       " 28182              1             0.002  \n",
       " 28183              3             0.006  \n",
       " 28184              1             0.002  \n",
       " 28185              1             0.002  \n",
       " 28186              1             0.002  \n",
       " 28187              0             0.000  \n",
       " 28188              1             0.002  \n",
       " 28189              1             0.002  \n",
       " 28190              0             0.000  \n",
       " 28191              3             0.006  \n",
       " 28192              2             0.004  \n",
       " 28193              0             0.000  \n",
       " 28194              0             0.000  \n",
       " 28195              4             0.008  \n",
       " 28196              3             0.006  \n",
       " 28197              1             0.002  \n",
       " 28198              1             0.002  \n",
       " 28199              3             0.006  \n",
       " 28200              1             0.002  \n",
       " 28201              4             0.008  \n",
       " 28202              1             0.002  \n",
       " 28203              1             0.002  \n",
       " 28204              0             0.000  \n",
       " 28205              1             0.002  \n",
       " 28206              1             0.002  \n",
       " 28207              2             0.004  \n",
       " 28208              2             0.004  \n",
       " 28209              1             0.002  \n",
       " 28210              1             0.002  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  3             0.006  \n",
       " 1                  2             0.004  \n",
       " 2                  6             0.012  \n",
       " 3                  0             0.000  \n",
       " 4                  2             0.004  \n",
       " 5                  0             0.000  \n",
       " 6                  3             0.006  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  1             0.002  \n",
       " 10                 2             0.004  \n",
       " 11                 0             0.000  \n",
       " 12                 2             0.004  \n",
       " 13                 1             0.002  \n",
       " 14                 2             0.004  \n",
       " 15                 1             0.002  \n",
       " 16                 0             0.000  \n",
       " 17                 1             0.002  \n",
       " 18                 2             0.004  \n",
       " 19                 1             0.002  \n",
       " 20                 2             0.004  \n",
       " 21                 2             0.004  \n",
       " 22                 2             0.004  \n",
       " 23                 0             0.000  \n",
       " 24                 1             0.002  \n",
       " 25                 3             0.006  \n",
       " 26                 1             0.002  \n",
       " 27                 1             0.002  \n",
       " 28                 3             0.006  \n",
       " 29                 2             0.004  \n",
       " ...              ...               ...  \n",
       " 28181              0             0.000  \n",
       " 28182              4             0.008  \n",
       " 28183              0             0.000  \n",
       " 28184              2             0.004  \n",
       " 28185              3             0.006  \n",
       " 28186              0             0.000  \n",
       " 28187              3             0.006  \n",
       " 28188              1             0.002  \n",
       " 28189              1             0.002  \n",
       " 28190              4             0.008  \n",
       " 28191              1             0.002  \n",
       " 28192              2             0.004  \n",
       " 28193              2             0.004  \n",
       " 28194              2             0.004  \n",
       " 28195              1             0.002  \n",
       " 28196              3             0.006  \n",
       " 28197              1             0.002  \n",
       " 28198              1             0.002  \n",
       " 28199              1             0.002  \n",
       " 28200              2             0.004  \n",
       " 28201              1             0.002  \n",
       " 28202              2             0.004  \n",
       " 28203              2             0.004  \n",
       " 28204              0             0.000  \n",
       " 28205              3             0.006  \n",
       " 28206              1             0.002  \n",
       " 28207              1             0.002  \n",
       " 28208              1             0.002  \n",
       " 28209              3             0.006  \n",
       " 28210              1             0.002  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  0             0.000  \n",
       " 1                  5             0.010  \n",
       " 2                  1             0.002  \n",
       " 3                  3             0.006  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  3             0.006  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  4             0.008  \n",
       " 10                 3             0.006  \n",
       " 11                 2             0.004  \n",
       " 12                 1             0.002  \n",
       " 13                 2             0.004  \n",
       " 14                 2             0.004  \n",
       " 15                 3             0.006  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 1             0.002  \n",
       " 20                 1             0.002  \n",
       " 21                 1             0.002  \n",
       " 22                 1             0.002  \n",
       " 23                 1             0.002  \n",
       " 24                 1             0.002  \n",
       " 25                 1             0.002  \n",
       " 26                 2             0.004  \n",
       " 27                 3             0.006  \n",
       " 28                 2             0.004  \n",
       " 29                 2             0.004  \n",
       " ...              ...               ...  \n",
       " 36517              1             0.002  \n",
       " 36518              1             0.002  \n",
       " 36519              0             0.000  \n",
       " 36520              3             0.006  \n",
       " 36521              2             0.004  \n",
       " 36522              0             0.000  \n",
       " 36523              0             0.000  \n",
       " 36524              2             0.004  \n",
       " 36525              2             0.004  \n",
       " 36526              0             0.000  \n",
       " 36527              0             0.000  \n",
       " 36528              4             0.008  \n",
       " 36529              3             0.006  \n",
       " 36530              1             0.002  \n",
       " 36531              1             0.002  \n",
       " 36532              2             0.004  \n",
       " 36533              3             0.006  \n",
       " 36534              1             0.002  \n",
       " 36535              4             0.008  \n",
       " 36536              1             0.002  \n",
       " 36537              5             0.010  \n",
       " 36538              1             0.002  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541              1             0.002  \n",
       " 36542              1             0.002  \n",
       " 36543              2             0.004  \n",
       " 36544              2             0.004  \n",
       " 36545              1             0.002  \n",
       " 36546              1             0.002  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  1             0.002  \n",
       " 1                  1             0.002  \n",
       " 2                  3             0.006  \n",
       " 3                  2             0.004  \n",
       " 4                  2             0.004  \n",
       " 5                  1             0.002  \n",
       " 6                  6             0.012  \n",
       " 7                  0             0.000  \n",
       " 8                  2             0.004  \n",
       " 9                  1             0.002  \n",
       " 10                 0             0.000  \n",
       " 11                 3             0.006  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 1             0.002  \n",
       " 15                 2             0.004  \n",
       " 16                 0             0.000  \n",
       " 17                 2             0.004  \n",
       " 18                 1             0.002  \n",
       " 19                 1             0.002  \n",
       " 20                 2             0.004  \n",
       " 21                 1             0.002  \n",
       " 22                 2             0.004  \n",
       " 23                 1             0.002  \n",
       " 24                 1             0.002  \n",
       " 25                 2             0.004  \n",
       " 26                 0             0.000  \n",
       " 27                 1             0.002  \n",
       " 28                 2             0.004  \n",
       " 29                 1             0.002  \n",
       " ...              ...               ...  \n",
       " 36517              1             0.002  \n",
       " 36518              1             0.002  \n",
       " 36519              4             0.008  \n",
       " 36520              1             0.002  \n",
       " 36521              2             0.004  \n",
       " 36522              2             0.004  \n",
       " 36523              0             0.000  \n",
       " 36524              2             0.004  \n",
       " 36525              3             0.006  \n",
       " 36526              3             0.006  \n",
       " 36527              2             0.004  \n",
       " 36528              1             0.002  \n",
       " 36529              3             0.006  \n",
       " 36530              1             0.002  \n",
       " 36531              1             0.002  \n",
       " 36532              0             0.000  \n",
       " 36533              1             0.002  \n",
       " 36534              2             0.004  \n",
       " 36535              1             0.002  \n",
       " 36536              2             0.004  \n",
       " 36537              4             0.008  \n",
       " 36538              2             0.004  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541              3             0.006  \n",
       " 36542              1             0.002  \n",
       " 36543              1             0.002  \n",
       " 36544              1             0.002  \n",
       " 36545              3             0.006  \n",
       " 36546              1             0.002  \n",
       " \n",
       " [36547 rows x 6 columns]}"
      ]
     },
     "execution_count": 2023,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tss_rand_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2022,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                93             0.186  \n",
       " 1               112             0.224  \n",
       " 2               123             0.246  \n",
       " 3                98             0.196  \n",
       " 4               155             0.310  \n",
       " 5               119             0.238  \n",
       " 6               127             0.254  \n",
       " 7               121             0.242  \n",
       " 8               134             0.268  \n",
       " 9                73             0.146  \n",
       " 10              130             0.260  \n",
       " 11              126             0.252  \n",
       " 12              103             0.206  \n",
       " 13              111             0.222  \n",
       " 14               88             0.176  \n",
       " 15              163             0.326  \n",
       " 16               65             0.130  \n",
       " 17               86             0.172  \n",
       " 18              103             0.206  \n",
       " 19               86             0.172  \n",
       " 20              110             0.220  \n",
       " 21               74             0.148  \n",
       " 22              133             0.266  \n",
       " 23              127             0.254  \n",
       " 24              111             0.222  \n",
       " 25              121             0.242  \n",
       " 26              117             0.234  \n",
       " 27               95             0.190  \n",
       " 28               82             0.164  \n",
       " 29               88             0.176  \n",
       " ...             ...               ...  \n",
       " 2848            141             0.282  \n",
       " 2849            123             0.246  \n",
       " 2850             84             0.168  \n",
       " 2851             82             0.164  \n",
       " 2852            117             0.234  \n",
       " 2853            167             0.334  \n",
       " 2854             88             0.176  \n",
       " 2855            176             0.352  \n",
       " 2856            146             0.292  \n",
       " 2857             84             0.168  \n",
       " 2858            114             0.228  \n",
       " 2859            162             0.324  \n",
       " 2860             95             0.190  \n",
       " 2861            141             0.282  \n",
       " 2862             45             0.090  \n",
       " 2863            115             0.230  \n",
       " 2864            132             0.264  \n",
       " 2865            158             0.316  \n",
       " 2866            168             0.336  \n",
       " 2867            107             0.214  \n",
       " 2868            104             0.208  \n",
       " 2869            131             0.262  \n",
       " 2870            130             0.260  \n",
       " 2871            122             0.244  \n",
       " 2872            111             0.222  \n",
       " 2873            137             0.274  \n",
       " 2874            110             0.220  \n",
       " 2875             24             0.048  \n",
       " 2876             65             0.130  \n",
       " 2877            123             0.246  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                66             0.132  \n",
       " 1                48             0.096  \n",
       " 2                82             0.164  \n",
       " 3                68             0.136  \n",
       " 4               121             0.242  \n",
       " 5                61             0.122  \n",
       " 6                88             0.176  \n",
       " 7                75             0.150  \n",
       " 8               100             0.200  \n",
       " 9                25             0.050  \n",
       " 10               67             0.134  \n",
       " 11               74             0.148  \n",
       " 12               60             0.120  \n",
       " 13               54             0.108  \n",
       " 14               63             0.126  \n",
       " 15               81             0.162  \n",
       " 16               35             0.070  \n",
       " 17               46             0.092  \n",
       " 18               56             0.112  \n",
       " 19               41             0.082  \n",
       " 20               47             0.094  \n",
       " 21               29             0.058  \n",
       " 22               47             0.094  \n",
       " 23               68             0.136  \n",
       " 24               61             0.122  \n",
       " 25               65             0.130  \n",
       " 26               76             0.152  \n",
       " 27               33             0.066  \n",
       " 28               15             0.030  \n",
       " 29               15             0.030  \n",
       " ...             ...               ...  \n",
       " 2848             61             0.122  \n",
       " 2849             69             0.138  \n",
       " 2850             53             0.106  \n",
       " 2851             41             0.082  \n",
       " 2852             71             0.142  \n",
       " 2853             13             0.026  \n",
       " 2854             49             0.098  \n",
       " 2855            156             0.312  \n",
       " 2856            131             0.262  \n",
       " 2857             55             0.110  \n",
       " 2858            114             0.228  \n",
       " 2859            162             0.324  \n",
       " 2860             95             0.190  \n",
       " 2861             55             0.110  \n",
       " 2862             45             0.090  \n",
       " 2863             56             0.112  \n",
       " 2864            110             0.220  \n",
       " 2865            118             0.236  \n",
       " 2866             98             0.196  \n",
       " 2867             74             0.148  \n",
       " 2868             86             0.172  \n",
       " 2869             83             0.166  \n",
       " 2870             79             0.158  \n",
       " 2871             99             0.198  \n",
       " 2872             89             0.178  \n",
       " 2873            125             0.250  \n",
       " 2874             83             0.166  \n",
       " 2875             24             0.048  \n",
       " 2876             65             0.130  \n",
       " 2877            123             0.246  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 2             0.004  \n",
       " 1                 2             0.004  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 3             0.006  \n",
       " 5                 3             0.006  \n",
       " 6                 8             0.016  \n",
       " 7                 7             0.014  \n",
       " 8                 4             0.008  \n",
       " 9                 1             0.002  \n",
       " 10                2             0.004  \n",
       " 11                1             0.002  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                1             0.002  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                4             0.008  \n",
       " 18                0             0.000  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                1             0.002  \n",
       " 23                3             0.006  \n",
       " 24                1             0.002  \n",
       " 25                0             0.000  \n",
       " 26                1             0.002  \n",
       " 27                0             0.000  \n",
       " 28                4             0.008  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2848              0             0.000  \n",
       " 2849              0             0.000  \n",
       " 2850              0             0.000  \n",
       " 2851              0             0.000  \n",
       " 2852              6             0.012  \n",
       " 2853              0             0.000  \n",
       " 2854              1             0.002  \n",
       " 2855             11             0.022  \n",
       " 2856              7             0.014  \n",
       " 2857              0             0.000  \n",
       " 2858            114             0.228  \n",
       " 2859            154             0.308  \n",
       " 2860             61             0.122  \n",
       " 2861              4             0.008  \n",
       " 2862              9             0.018  \n",
       " 2863              3             0.006  \n",
       " 2864              6             0.012  \n",
       " 2865              0             0.000  \n",
       " 2866              1             0.002  \n",
       " 2867              1             0.002  \n",
       " 2868              3             0.006  \n",
       " 2869              3             0.006  \n",
       " 2870              7             0.014  \n",
       " 2871             10             0.020  \n",
       " 2872             47             0.094  \n",
       " 2873             52             0.104  \n",
       " 2874             32             0.064  \n",
       " 2875             24             0.048  \n",
       " 2876             65             0.130  \n",
       " 2877            123             0.246  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 1             0.002  \n",
       " 6                 2             0.004  \n",
       " 7                 2             0.004  \n",
       " 8                 1             0.002  \n",
       " 9                 1             0.002  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                1             0.002  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                2             0.004  \n",
       " 24                1             0.002  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                0             0.000  \n",
       " 28                1             0.002  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2848              0             0.000  \n",
       " 2849              0             0.000  \n",
       " 2850              0             0.000  \n",
       " 2851              0             0.000  \n",
       " 2852              0             0.000  \n",
       " 2853              0             0.000  \n",
       " 2854              0             0.000  \n",
       " 2855             11             0.022  \n",
       " 2856              7             0.014  \n",
       " 2857              0             0.000  \n",
       " 2858            114             0.228  \n",
       " 2859            154             0.308  \n",
       " 2860             61             0.122  \n",
       " 2861              1             0.002  \n",
       " 2862              9             0.018  \n",
       " 2863              0             0.000  \n",
       " 2864              0             0.000  \n",
       " 2865              0             0.000  \n",
       " 2866              0             0.000  \n",
       " 2867              1             0.002  \n",
       " 2868              3             0.006  \n",
       " 2869              3             0.006  \n",
       " 2870              7             0.014  \n",
       " 2871             10             0.020  \n",
       " 2872             47             0.094  \n",
       " 2873             52             0.104  \n",
       " 2874             32             0.064  \n",
       " 2875             24             0.048  \n",
       " 2876             65             0.130  \n",
       " 2877            123             0.246  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               187             0.374  \n",
       " 1               174             0.348  \n",
       " 2               150             0.300  \n",
       " 3               163             0.326  \n",
       " 4               142             0.284  \n",
       " 5               155             0.310  \n",
       " 6               125             0.250  \n",
       " 7               127             0.254  \n",
       " 8               131             0.262  \n",
       " 9               165             0.330  \n",
       " 10              134             0.268  \n",
       " 11              173             0.346  \n",
       " 12              148             0.296  \n",
       " 13              108             0.216  \n",
       " 14              195             0.390  \n",
       " 15              117             0.234  \n",
       " 16              202             0.404  \n",
       " 17              191             0.382  \n",
       " 18              171             0.342  \n",
       " 19              164             0.328  \n",
       " 20              166             0.332  \n",
       " 21              181             0.362  \n",
       " 22              171             0.342  \n",
       " 23              151             0.302  \n",
       " 24              134             0.268  \n",
       " 25              144             0.288  \n",
       " 26              159             0.318  \n",
       " 27              153             0.306  \n",
       " 28              179             0.358  \n",
       " 29              159             0.318  \n",
       " ...             ...               ...  \n",
       " 2848            142             0.284  \n",
       " 2849            138             0.276  \n",
       " 2850            162             0.324  \n",
       " 2851            148             0.296  \n",
       " 2852            145             0.290  \n",
       " 2853            101             0.202  \n",
       " 2854            195             0.390  \n",
       " 2855            123             0.246  \n",
       " 2856             99             0.198  \n",
       " 2857            187             0.374  \n",
       " 2858             40             0.080  \n",
       " 2859             70             0.140  \n",
       " 2860            116             0.232  \n",
       " 2861            128             0.256  \n",
       " 2862            137             0.274  \n",
       " 2863            154             0.308  \n",
       " 2864            115             0.230  \n",
       " 2865             96             0.192  \n",
       " 2866            118             0.236  \n",
       " 2867            112             0.224  \n",
       " 2868            180             0.360  \n",
       " 2869             87             0.174  \n",
       " 2870            107             0.214  \n",
       " 2871            120             0.240  \n",
       " 2872            107             0.214  \n",
       " 2873            116             0.232  \n",
       " 2874             93             0.186  \n",
       " 2875             32             0.064  \n",
       " 2876             34             0.068  \n",
       " 2877             59             0.118  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               125             0.250  \n",
       " 1               104             0.208  \n",
       " 2                55             0.110  \n",
       " 3                47             0.094  \n",
       " 4                31             0.062  \n",
       " 5                79             0.158  \n",
       " 6                61             0.122  \n",
       " 7                54             0.108  \n",
       " 8                64             0.128  \n",
       " 9                60             0.120  \n",
       " 10               50             0.100  \n",
       " 11               77             0.154  \n",
       " 12               49             0.098  \n",
       " 13               30             0.060  \n",
       " 14               68             0.136  \n",
       " 15               31             0.062  \n",
       " 16              112             0.224  \n",
       " 17              119             0.238  \n",
       " 18              112             0.224  \n",
       " 19               97             0.194  \n",
       " 20              108             0.216  \n",
       " 21              106             0.212  \n",
       " 22               56             0.112  \n",
       " 23               57             0.114  \n",
       " 24               28             0.056  \n",
       " 25               45             0.090  \n",
       " 26               90             0.180  \n",
       " 27               36             0.072  \n",
       " 28               30             0.060  \n",
       " 29               37             0.074  \n",
       " ...             ...               ...  \n",
       " 2848             87             0.174  \n",
       " 2849             85             0.170  \n",
       " 2850            119             0.238  \n",
       " 2851             94             0.188  \n",
       " 2852             89             0.178  \n",
       " 2853              1             0.002  \n",
       " 2854             65             0.130  \n",
       " 2855             58             0.116  \n",
       " 2856             50             0.100  \n",
       " 2857             96             0.192  \n",
       " 2858             40             0.080  \n",
       " 2859             70             0.140  \n",
       " 2860            116             0.232  \n",
       " 2861             65             0.130  \n",
       " 2862            137             0.274  \n",
       " 2863             90             0.180  \n",
       " 2864             72             0.144  \n",
       " 2865             45             0.090  \n",
       " 2866             61             0.122  \n",
       " 2867             48             0.096  \n",
       " 2868            132             0.264  \n",
       " 2869             37             0.074  \n",
       " 2870             46             0.092  \n",
       " 2871             74             0.148  \n",
       " 2872             73             0.146  \n",
       " 2873             75             0.150  \n",
       " 2874             58             0.116  \n",
       " 2875             32             0.064  \n",
       " 2876             34             0.068  \n",
       " 2877             59             0.118  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 7             0.014  \n",
       " 1                 4             0.008  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 4             0.008  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 2             0.004  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                4             0.008  \n",
       " 18                0             0.000  \n",
       " 19                8             0.016  \n",
       " 20                2             0.004  \n",
       " 21                1             0.002  \n",
       " 22                1             0.002  \n",
       " 23                4             0.008  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                1             0.002  \n",
       " 27                1             0.002  \n",
       " 28                2             0.004  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 2848              3             0.006  \n",
       " 2849              0             0.000  \n",
       " 2850              8             0.016  \n",
       " 2851              1             0.002  \n",
       " 2852              1             0.002  \n",
       " 2853              0             0.000  \n",
       " 2854              1             0.002  \n",
       " 2855              2             0.004  \n",
       " 2856              6             0.012  \n",
       " 2857              4             0.008  \n",
       " 2858             40             0.080  \n",
       " 2859             34             0.068  \n",
       " 2860             46             0.092  \n",
       " 2861              1             0.002  \n",
       " 2862             28             0.056  \n",
       " 2863              3             0.006  \n",
       " 2864              2             0.004  \n",
       " 2865              1             0.002  \n",
       " 2866              3             0.006  \n",
       " 2867              1             0.002  \n",
       " 2868             12             0.024  \n",
       " 2869              1             0.002  \n",
       " 2870              3             0.006  \n",
       " 2871              1             0.002  \n",
       " 2872             35             0.070  \n",
       " 2873             15             0.030  \n",
       " 2874             15             0.030  \n",
       " 2875             32             0.064  \n",
       " 2876             34             0.068  \n",
       " 2877             59             0.118  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 2848  hcontig_095_133   44683   45183   EVM prediction%2hcontig_095_0133.13   \n",
       " 2849  hcontig_095_133   45983   46483   EVM prediction%2hcontig_095_0133.14   \n",
       " 2850  hcontig_095_133   81291   81791   EVM prediction%2hcontig_095_0133.20   \n",
       " 2851  hcontig_095_133   82403   82903   EVM prediction%2hcontig_095_0133.21   \n",
       " 2852  hcontig_096_004   14666   15166    EVM prediction%2hcontig_0096_004.7   \n",
       " 2853  hcontig_096_004   64620   65120    EVM prediction%2hcontig_0096_004.1   \n",
       " 2854  hcontig_096_004  112078  112578    EVM prediction%2hcontig_0096_004.4   \n",
       " 2855  hcontig_097_001    9031    9531    EVM prediction%2hcontig_0097_001.3   \n",
       " 2856  hcontig_097_001   10969   11469    EVM prediction%2hcontig_0097_001.4   \n",
       " 2857  hcontig_097_001   14055   14555    EVM prediction%2hcontig_0097_001.5   \n",
       " 2858  hcontig_104_001    6351    6851    EVM prediction%2hcontig_0104_001.3   \n",
       " 2859  hcontig_104_001    8459    8959    EVM prediction%2hcontig_0104_001.4   \n",
       " 2860  hcontig_104_001   13618   14118    EVM prediction%2hcontig_0104_001.5   \n",
       " 2861  hcontig_106_001   31454   31954    EVM prediction%2hcontig_0106_001.7   \n",
       " 2862  hcontig_110_003    3534    4034    EVM prediction%2hcontig_0110_003.2   \n",
       " 2863  hcontig_110_003   14395   14895    EVM prediction%2hcontig_0110_003.5   \n",
       " 2864  hcontig_110_003   22108   22608    EVM prediction%2hcontig_0110_003.6   \n",
       " 2865  hcontig_110_003   24300   24800    EVM prediction%2hcontig_0110_003.7   \n",
       " 2866  hcontig_110_003   29163   29663    EVM prediction%2hcontig_0110_003.9   \n",
       " 2867  hcontig_110_003   41051   41551   EVM prediction%2hcontig_0110_003.11   \n",
       " 2868  hcontig_147_001    3071    3571    EVM prediction%2hcontig_0147_001.1   \n",
       " 2869  hcontig_147_001   18689   19189    EVM prediction%2hcontig_0147_001.5   \n",
       " 2870  hcontig_147_001   21248   21748    EVM prediction%2hcontig_0147_001.6   \n",
       " 2871  hcontig_147_001   22402   22902    EVM prediction%2hcontig_0147_001.7   \n",
       " 2872  hcontig_148_001   17106   17606    EVM prediction%2hcontig_0148_001.6   \n",
       " 2873  hcontig_166_001    7350    7850    EVM prediction%2hcontig_0166_001.3   \n",
       " 2874  hcontig_184_001   14584   15084    EVM prediction%2hcontig_0184_001.1   \n",
       " 2875  hcontig_184_001   17715   18215    EVM prediction%2hcontig_0184_001.2   \n",
       " 2876  hcontig_184_001   19820   20320    EVM prediction%2hcontig_0184_001.4   \n",
       " 2877  hcontig_184_001   20692   21192    EVM prediction%2hcontig_0184_001.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 1             0.002  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                2             0.004  \n",
       " 18                0             0.000  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                2             0.004  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                1             0.002  \n",
       " 28                0             0.000  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 2848              0             0.000  \n",
       " 2849              0             0.000  \n",
       " 2850              8             0.016  \n",
       " 2851              1             0.002  \n",
       " 2852              0             0.000  \n",
       " 2853              0             0.000  \n",
       " 2854              1             0.002  \n",
       " 2855              2             0.004  \n",
       " 2856              6             0.012  \n",
       " 2857              4             0.008  \n",
       " 2858             40             0.080  \n",
       " 2859             34             0.068  \n",
       " 2860             46             0.092  \n",
       " 2861              0             0.000  \n",
       " 2862             28             0.056  \n",
       " 2863              3             0.006  \n",
       " 2864              0             0.000  \n",
       " 2865              0             0.000  \n",
       " 2866              0             0.000  \n",
       " 2867              1             0.002  \n",
       " 2868             12             0.024  \n",
       " 2869              1             0.002  \n",
       " 2870              3             0.006  \n",
       " 2871              1             0.002  \n",
       " 2872             35             0.070  \n",
       " 2873             15             0.030  \n",
       " 2874             15             0.030  \n",
       " 2875             32             0.064  \n",
       " 2876             34             0.068  \n",
       " 2877             59             0.118  \n",
       " \n",
       " [2878 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                131             0.262  \n",
       " 1                161             0.322  \n",
       " 2                177             0.354  \n",
       " 3                142             0.284  \n",
       " 4                 93             0.186  \n",
       " 5                112             0.224  \n",
       " 6                151             0.302  \n",
       " 7                113             0.226  \n",
       " 8                110             0.220  \n",
       " 9                143             0.286  \n",
       " 10               133             0.266  \n",
       " 11               127             0.254  \n",
       " 12               152             0.304  \n",
       " 13               116             0.232  \n",
       " 14               116             0.232  \n",
       " 15               116             0.232  \n",
       " 16               119             0.238  \n",
       " 17               101             0.202  \n",
       " 18               136             0.272  \n",
       " 19               112             0.224  \n",
       " 20               123             0.246  \n",
       " 21                98             0.196  \n",
       " 22               129             0.258  \n",
       " 23               117             0.234  \n",
       " 24               155             0.310  \n",
       " 25               120             0.240  \n",
       " 26               176             0.352  \n",
       " 27               123             0.246  \n",
       " 28               104             0.208  \n",
       " 29               169             0.338  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170            112             0.224  \n",
       " 17171            137             0.274  \n",
       " 17172            126             0.252  \n",
       " 17173            103             0.206  \n",
       " 17174            101             0.202  \n",
       " 17175            113             0.226  \n",
       " 17176            115             0.230  \n",
       " 17177            110             0.220  \n",
       " 17178             24             0.048  \n",
       " 17179             18             0.036  \n",
       " 17180             65             0.130  \n",
       " 17181            123             0.246  \n",
       " 17182             86             0.172  \n",
       " 17183            121             0.242  \n",
       " 17184             82             0.164  \n",
       " 17185            104             0.208  \n",
       " 17186             94             0.188  \n",
       " 17187            127             0.254  \n",
       " 17188            119             0.238  \n",
       " 17189             43             0.086  \n",
       " 17190            117             0.234  \n",
       " 17191            101             0.202  \n",
       " 17192            123             0.246  \n",
       " 17193            120             0.240  \n",
       " 17194            130             0.260  \n",
       " 17195            118             0.236  \n",
       " 17196            123             0.246  \n",
       " 17197            102             0.204  \n",
       " 17198            146             0.292  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                119             0.238  \n",
       " 1                102             0.204  \n",
       " 2                 84             0.168  \n",
       " 3                112             0.224  \n",
       " 4                 66             0.132  \n",
       " 5                 48             0.096  \n",
       " 6                121             0.242  \n",
       " 7                 57             0.114  \n",
       " 8                 50             0.100  \n",
       " 9                 86             0.172  \n",
       " 10                71             0.142  \n",
       " 11                52             0.104  \n",
       " 12               118             0.236  \n",
       " 13                21             0.042  \n",
       " 14                23             0.046  \n",
       " 15                25             0.050  \n",
       " 16                25             0.050  \n",
       " 17                13             0.026  \n",
       " 18                95             0.190  \n",
       " 19                45             0.090  \n",
       " 20                82             0.164  \n",
       " 21                68             0.136  \n",
       " 22                42             0.084  \n",
       " 23                55             0.110  \n",
       " 24               121             0.242  \n",
       " 25                76             0.152  \n",
       " 26               125             0.250  \n",
       " 27                82             0.164  \n",
       " 28                72             0.144  \n",
       " 29                 7             0.014  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             88             0.176  \n",
       " 17171            125             0.250  \n",
       " 17172             32             0.064  \n",
       " 17173             87             0.174  \n",
       " 17174             83             0.166  \n",
       " 17175             98             0.196  \n",
       " 17176             94             0.188  \n",
       " 17177             83             0.166  \n",
       " 17178             24             0.048  \n",
       " 17179             18             0.036  \n",
       " 17180             65             0.130  \n",
       " 17181            123             0.246  \n",
       " 17182             86             0.172  \n",
       " 17183             38             0.076  \n",
       " 17184             68             0.136  \n",
       " 17185             74             0.148  \n",
       " 17186             83             0.166  \n",
       " 17187            100             0.200  \n",
       " 17188            119             0.238  \n",
       " 17189             43             0.086  \n",
       " 17190             86             0.172  \n",
       " 17191             87             0.174  \n",
       " 17192            107             0.214  \n",
       " 17193            104             0.208  \n",
       " 17194            120             0.240  \n",
       " 17195             69             0.138  \n",
       " 17196             20             0.040  \n",
       " 17197            102             0.204  \n",
       " 17198            146             0.292  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 53             0.106  \n",
       " 1                  0             0.000  \n",
       " 2                  0             0.000  \n",
       " 3                  5             0.010  \n",
       " 4                  2             0.004  \n",
       " 5                  2             0.004  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  1             0.002  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 1             0.002  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 3             0.006  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             62             0.124  \n",
       " 17171             52             0.104  \n",
       " 17172              0             0.000  \n",
       " 17173             22             0.044  \n",
       " 17174              2             0.004  \n",
       " 17175             35             0.070  \n",
       " 17176             58             0.116  \n",
       " 17177             32             0.064  \n",
       " 17178             24             0.048  \n",
       " 17179             18             0.036  \n",
       " 17180             65             0.130  \n",
       " 17181            123             0.246  \n",
       " 17182             86             0.172  \n",
       " 17183              2             0.004  \n",
       " 17184              3             0.006  \n",
       " 17185              0             0.000  \n",
       " 17186              8             0.016  \n",
       " 17187              0             0.000  \n",
       " 17188             93             0.186  \n",
       " 17189             43             0.086  \n",
       " 17190              0             0.000  \n",
       " 17191              6             0.012  \n",
       " 17192              6             0.012  \n",
       " 17193              2             0.004  \n",
       " 17194             10             0.020  \n",
       " 17195              7             0.014  \n",
       " 17196              0             0.000  \n",
       " 17197              9             0.018  \n",
       " 17198             13             0.026  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 53             0.106  \n",
       " 1                  0             0.000  \n",
       " 2                  0             0.000  \n",
       " 3                  1             0.002  \n",
       " 4                  0             0.000  \n",
       " 5                  1             0.002  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 1             0.002  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             62             0.124  \n",
       " 17171             52             0.104  \n",
       " 17172              0             0.000  \n",
       " 17173              2             0.004  \n",
       " 17174              2             0.004  \n",
       " 17175             35             0.070  \n",
       " 17176             58             0.116  \n",
       " 17177             32             0.064  \n",
       " 17178             24             0.048  \n",
       " 17179             18             0.036  \n",
       " 17180             65             0.130  \n",
       " 17181            123             0.246  \n",
       " 17182             86             0.172  \n",
       " 17183              2             0.004  \n",
       " 17184              3             0.006  \n",
       " 17185              0             0.000  \n",
       " 17186              8             0.016  \n",
       " 17187              0             0.000  \n",
       " 17188             93             0.186  \n",
       " 17189             43             0.086  \n",
       " 17190              0             0.000  \n",
       " 17191              0             0.000  \n",
       " 17192              2             0.004  \n",
       " 17193              0             0.000  \n",
       " 17194             10             0.020  \n",
       " 17195              7             0.014  \n",
       " 17196              0             0.000  \n",
       " 17197              9             0.018  \n",
       " 17198             13             0.026  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                109             0.218  \n",
       " 1                104             0.208  \n",
       " 2                139             0.278  \n",
       " 3                134             0.268  \n",
       " 4                187             0.374  \n",
       " 5                174             0.348  \n",
       " 6                134             0.268  \n",
       " 7                129             0.258  \n",
       " 8                142             0.284  \n",
       " 9                116             0.232  \n",
       " 10               144             0.288  \n",
       " 11               132             0.264  \n",
       " 12               120             0.240  \n",
       " 13               140             0.280  \n",
       " 14               138             0.276  \n",
       " 15               142             0.284  \n",
       " 16               142             0.284  \n",
       " 17               157             0.314  \n",
       " 18               118             0.236  \n",
       " 19               143             0.286  \n",
       " 20               150             0.300  \n",
       " 21               163             0.326  \n",
       " 22               156             0.312  \n",
       " 23               154             0.308  \n",
       " 24               142             0.284  \n",
       " 25               127             0.254  \n",
       " 26               107             0.214  \n",
       " 27               122             0.244  \n",
       " 28               139             0.278  \n",
       " 29               122             0.244  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             79             0.158  \n",
       " 17171            116             0.232  \n",
       " 17172            118             0.236  \n",
       " 17173            149             0.298  \n",
       " 17174            140             0.280  \n",
       " 17175            132             0.264  \n",
       " 17176            111             0.222  \n",
       " 17177             93             0.186  \n",
       " 17178             32             0.064  \n",
       " 17179             74             0.148  \n",
       " 17180             34             0.068  \n",
       " 17181             59             0.118  \n",
       " 17182             49             0.098  \n",
       " 17183            116             0.232  \n",
       " 17184            140             0.280  \n",
       " 17185            148             0.296  \n",
       " 17186            171             0.342  \n",
       " 17187             90             0.180  \n",
       " 17188             82             0.164  \n",
       " 17189             23             0.046  \n",
       " 17190            144             0.288  \n",
       " 17191            142             0.284  \n",
       " 17192            142             0.284  \n",
       " 17193            139             0.278  \n",
       " 17194            138             0.276  \n",
       " 17195            119             0.238  \n",
       " 17196            159             0.318  \n",
       " 17197             88             0.176  \n",
       " 17198             78             0.156  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 74             0.148  \n",
       " 1                 46             0.092  \n",
       " 2                 45             0.090  \n",
       " 3                 66             0.132  \n",
       " 4                125             0.250  \n",
       " 5                104             0.208  \n",
       " 6                 54             0.108  \n",
       " 7                 80             0.160  \n",
       " 8                 86             0.172  \n",
       " 9                 30             0.060  \n",
       " 10                44             0.088  \n",
       " 11                83             0.166  \n",
       " 12                36             0.072  \n",
       " 13                67             0.134  \n",
       " 14                53             0.106  \n",
       " 15                66             0.132  \n",
       " 16                38             0.076  \n",
       " 17                34             0.068  \n",
       " 18                51             0.102  \n",
       " 19                61             0.122  \n",
       " 20                55             0.110  \n",
       " 21                47             0.094  \n",
       " 22                25             0.050  \n",
       " 23               107             0.214  \n",
       " 24                31             0.062  \n",
       " 25                29             0.058  \n",
       " 26                34             0.068  \n",
       " 27                48             0.096  \n",
       " 28                82             0.164  \n",
       " 29                13             0.026  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             40             0.080  \n",
       " 17171             75             0.150  \n",
       " 17172             44             0.088  \n",
       " 17173             83             0.166  \n",
       " 17174             65             0.130  \n",
       " 17175            100             0.200  \n",
       " 17176             82             0.164  \n",
       " 17177             58             0.116  \n",
       " 17178             32             0.064  \n",
       " 17179             74             0.148  \n",
       " 17180             34             0.068  \n",
       " 17181             59             0.118  \n",
       " 17182             49             0.098  \n",
       " 17183             26             0.052  \n",
       " 17184             66             0.132  \n",
       " 17185             52             0.104  \n",
       " 17186            107             0.214  \n",
       " 17187             46             0.092  \n",
       " 17188             82             0.164  \n",
       " 17189             23             0.046  \n",
       " 17190             80             0.160  \n",
       " 17191             79             0.158  \n",
       " 17192             78             0.156  \n",
       " 17193             77             0.154  \n",
       " 17194             99             0.198  \n",
       " 17195             47             0.094  \n",
       " 17196             52             0.104  \n",
       " 17197             88             0.176  \n",
       " 17198             78             0.156  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 20             0.040  \n",
       " 1                  0             0.000  \n",
       " 2                  2             0.004  \n",
       " 3                  5             0.010  \n",
       " 4                  7             0.014  \n",
       " 5                  4             0.008  \n",
       " 6                  0             0.000  \n",
       " 7                  2             0.004  \n",
       " 8                  3             0.006  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 2             0.004  \n",
       " 15                 2             0.004  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             16             0.032  \n",
       " 17171             15             0.030  \n",
       " 17172              0             0.000  \n",
       " 17173             17             0.034  \n",
       " 17174              0             0.000  \n",
       " 17175             23             0.046  \n",
       " 17176             22             0.044  \n",
       " 17177             15             0.030  \n",
       " 17178             32             0.064  \n",
       " 17179             74             0.148  \n",
       " 17180             34             0.068  \n",
       " 17181             59             0.118  \n",
       " 17182             49             0.098  \n",
       " 17183              0             0.000  \n",
       " 17184              1             0.002  \n",
       " 17185              0             0.000  \n",
       " 17186              2             0.004  \n",
       " 17187              1             0.002  \n",
       " 17188             13             0.026  \n",
       " 17189             23             0.046  \n",
       " 17190              5             0.010  \n",
       " 17191              3             0.006  \n",
       " 17192              6             0.012  \n",
       " 17193              3             0.006  \n",
       " 17194              6             0.012  \n",
       " 17195              1             0.002  \n",
       " 17196              0             0.000  \n",
       " 17197             21             0.042  \n",
       " 17198             12             0.024  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 17169  hcontig_166_001      12     512         gene_model_hcontig_0166_01.1   \n",
       " 17170  hcontig_166_001    3441    3941         gene_model_hcontig_0166_01.2   \n",
       " 17171  hcontig_166_001    7350    7850   EVM prediction%2hcontig_0166_001.3   \n",
       " 17172  hcontig_166_001   11564   12064         gene_model_hcontig_0166_01.4   \n",
       " 17173  hcontig_166_001   16530   17030         gene_model_hcontig_0166_01.5   \n",
       " 17174  hcontig_166_001   32465   32965         gene_model_hcontig_0166_01.6   \n",
       " 17175  hcontig_166_001   34223   34723         gene_model_hcontig_0166_01.7   \n",
       " 17176  hcontig_166_001   35538   36038         gene_model_hcontig_0166_01.8   \n",
       " 17177  hcontig_184_001   14584   15084   EVM prediction%2hcontig_0184_001.1   \n",
       " 17178  hcontig_184_001   17715   18215   EVM prediction%2hcontig_0184_001.2   \n",
       " 17179  hcontig_184_001   18634   19134         gene_model_hcontig_0184_01.3   \n",
       " 17180  hcontig_184_001   19820   20320   EVM prediction%2hcontig_0184_001.4   \n",
       " 17181  hcontig_184_001   20692   21192   EVM prediction%2hcontig_0184_001.5   \n",
       " 17182  hcontig_189_001      19     519         gene_model_hcontig_0189_01.1   \n",
       " 17183  hcontig_189_001    3135    3635         gene_model_hcontig_0189_01.2   \n",
       " 17184  hcontig_189_001    4758    5258         gene_model_hcontig_0189_01.3   \n",
       " 17185  hcontig_189_001    7474    7974         gene_model_hcontig_0189_01.4   \n",
       " 17186  hcontig_189_001    9153    9653         gene_model_hcontig_0189_01.5   \n",
       " 17187  hcontig_189_001   14376   14876         gene_model_hcontig_0189_01.6   \n",
       " 17188  hcontig_189_001   20435   20935         gene_model_hcontig_0189_01.7   \n",
       " 17189  hcontig_193_002   31821   32321         gene_model_hcontig_0193_02.1   \n",
       " 17190  hcontig_225_001    1064    1564         gene_model_hcontig_0225_01.1   \n",
       " 17191  hcontig_225_001    1956    2456         gene_model_hcontig_0225_01.2   \n",
       " 17192  hcontig_225_001    3913    4413         gene_model_hcontig_0225_01.3   \n",
       " 17193  hcontig_225_001    6485    6985         gene_model_hcontig_0225_01.4   \n",
       " 17194  hcontig_225_001   15959   16459         gene_model_hcontig_0225_01.5   \n",
       " 17195  hcontig_225_001   19857   20357         gene_model_hcontig_0225_01.6   \n",
       " 17196  hcontig_225_001   21253   21753         gene_model_hcontig_0225_01.7   \n",
       " 17197  hcontig_225_001   24890   25390         gene_model_hcontig_0225_01.8   \n",
       " 17198  hcontig_225_001   26312   26812         gene_model_hcontig_0225_01.9   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 20             0.040  \n",
       " 1                  0             0.000  \n",
       " 2                  2             0.004  \n",
       " 3                  1             0.002  \n",
       " 4                  1             0.002  \n",
       " 5                  1             0.002  \n",
       " 6                  0             0.000  \n",
       " 7                  2             0.004  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 2             0.004  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 17169              0             0.000  \n",
       " 17170             16             0.032  \n",
       " 17171             15             0.030  \n",
       " 17172              0             0.000  \n",
       " 17173              2             0.004  \n",
       " 17174              0             0.000  \n",
       " 17175             23             0.046  \n",
       " 17176             22             0.044  \n",
       " 17177             15             0.030  \n",
       " 17178             32             0.064  \n",
       " 17179             74             0.148  \n",
       " 17180             34             0.068  \n",
       " 17181             59             0.118  \n",
       " 17182             49             0.098  \n",
       " 17183              0             0.000  \n",
       " 17184              1             0.002  \n",
       " 17185              0             0.000  \n",
       " 17186              2             0.004  \n",
       " 17187              1             0.002  \n",
       " 17188             13             0.026  \n",
       " 17189             23             0.046  \n",
       " 17190              4             0.008  \n",
       " 17191              0             0.000  \n",
       " 17192              1             0.002  \n",
       " 17193              1             0.002  \n",
       " 17194              6             0.012  \n",
       " 17195              1             0.002  \n",
       " 17196              0             0.000  \n",
       " 17197             21             0.042  \n",
       " 17198             12             0.024  \n",
       " \n",
       " [17199 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               120             0.240  \n",
       " 1               124             0.248  \n",
       " 2               122             0.244  \n",
       " 3               134             0.268  \n",
       " 4               140             0.280  \n",
       " 5               132             0.264  \n",
       " 6               123             0.246  \n",
       " 7               104             0.208  \n",
       " 8                64             0.128  \n",
       " 9                86             0.172  \n",
       " 10              100             0.200  \n",
       " 11               98             0.196  \n",
       " 12              114             0.228  \n",
       " 13              115             0.230  \n",
       " 14              139             0.278  \n",
       " 15               97             0.194  \n",
       " 16              141             0.282  \n",
       " 17               82             0.164  \n",
       " 18              100             0.200  \n",
       " 19               87             0.174  \n",
       " 20              118             0.236  \n",
       " 21              116             0.232  \n",
       " 22              116             0.232  \n",
       " 23              176             0.352  \n",
       " 24               96             0.192  \n",
       " 25              107             0.214  \n",
       " 26               82             0.164  \n",
       " 27              173             0.346  \n",
       " 28               91             0.182  \n",
       " 29               93             0.186  \n",
       " ...             ...               ...  \n",
       " 3390            101             0.202  \n",
       " 3391            122             0.244  \n",
       " 3392            113             0.226  \n",
       " 3393            121             0.242  \n",
       " 3394            110             0.220  \n",
       " 3395            158             0.316  \n",
       " 3396              0             0.000  \n",
       " 3397            130             0.260  \n",
       " 3398            113             0.226  \n",
       " 3399            136             0.272  \n",
       " 3400            160             0.320  \n",
       " 3401            149             0.298  \n",
       " 3402            100             0.200  \n",
       " 3403            128             0.256  \n",
       " 3404            165             0.330  \n",
       " 3405            119             0.238  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410            156             0.312  \n",
       " 3411            142             0.284  \n",
       " 3412            142             0.284  \n",
       " 3413            110             0.220  \n",
       " 3414            154             0.308  \n",
       " 3415              6             0.012  \n",
       " 3416            129             0.258  \n",
       " 3417            138             0.276  \n",
       " 3418            157             0.314  \n",
       " 3419            137             0.274  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                60             0.120  \n",
       " 1                67             0.134  \n",
       " 2                58             0.116  \n",
       " 3                77             0.154  \n",
       " 4                83             0.166  \n",
       " 5                64             0.128  \n",
       " 6                95             0.190  \n",
       " 7                79             0.158  \n",
       " 8                 7             0.014  \n",
       " 9                12             0.024  \n",
       " 10               69             0.138  \n",
       " 11               53             0.106  \n",
       " 12               48             0.096  \n",
       " 13               61             0.122  \n",
       " 14              105             0.210  \n",
       " 15               61             0.122  \n",
       " 16               26             0.052  \n",
       " 17               30             0.060  \n",
       " 18               26             0.052  \n",
       " 19               25             0.050  \n",
       " 20               68             0.136  \n",
       " 21               62             0.124  \n",
       " 22               70             0.140  \n",
       " 23               54             0.108  \n",
       " 24               47             0.094  \n",
       " 25               41             0.082  \n",
       " 26               53             0.106  \n",
       " 27               95             0.190  \n",
       " 28               66             0.132  \n",
       " 29               67             0.134  \n",
       " ...             ...               ...  \n",
       " 3390             51             0.102  \n",
       " 3391             94             0.188  \n",
       " 3392             78             0.156  \n",
       " 3393             78             0.156  \n",
       " 3394             89             0.178  \n",
       " 3395            158             0.316  \n",
       " 3396              0             0.000  \n",
       " 3397             73             0.146  \n",
       " 3398             67             0.134  \n",
       " 3399             73             0.146  \n",
       " 3400             89             0.178  \n",
       " 3401             76             0.152  \n",
       " 3402             85             0.170  \n",
       " 3403            103             0.206  \n",
       " 3404            128             0.256  \n",
       " 3405            119             0.238  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410            122             0.244  \n",
       " 3411            117             0.234  \n",
       " 3412            120             0.240  \n",
       " 3413             92             0.184  \n",
       " 3414            121             0.242  \n",
       " 3415              6             0.012  \n",
       " 3416             69             0.138  \n",
       " 3417             58             0.116  \n",
       " 3418             71             0.142  \n",
       " 3419             69             0.138  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 2             0.004  \n",
       " 4                 5             0.010  \n",
       " 5                 0             0.000  \n",
       " 6                 8             0.016  \n",
       " 7                 3             0.006  \n",
       " 8                 0             0.000  \n",
       " 9                 3             0.006  \n",
       " 10                1             0.002  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                6             0.012  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                3             0.006  \n",
       " 18                0             0.000  \n",
       " 19                1             0.002  \n",
       " 20                1             0.002  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                1             0.002  \n",
       " 24                2             0.004  \n",
       " 25                1             0.002  \n",
       " 26                2             0.004  \n",
       " 27                2             0.004  \n",
       " 28                3             0.006  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 3390              1             0.002  \n",
       " 3391              8             0.016  \n",
       " 3392              9             0.018  \n",
       " 3393              0             0.000  \n",
       " 3394             12             0.024  \n",
       " 3395            116             0.232  \n",
       " 3396              0             0.000  \n",
       " 3397              0             0.000  \n",
       " 3398              0             0.000  \n",
       " 3399              0             0.000  \n",
       " 3400              0             0.000  \n",
       " 3401              1             0.002  \n",
       " 3402              0             0.000  \n",
       " 3403              7             0.014  \n",
       " 3404              1             0.002  \n",
       " 3405              9             0.018  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410             20             0.040  \n",
       " 3411              7             0.014  \n",
       " 3412             13             0.026  \n",
       " 3413              9             0.018  \n",
       " 3414              7             0.014  \n",
       " 3415              6             0.012  \n",
       " 3416              0             0.000  \n",
       " 3417              2             0.004  \n",
       " 3418              2             0.004  \n",
       " 3419              0             0.000  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 0             0.000  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 1             0.002  \n",
       " 5                 0             0.000  \n",
       " 6                 2             0.004  \n",
       " 7                 1             0.002  \n",
       " 8                 0             0.000  \n",
       " 9                 1             0.002  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                6             0.012  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                1             0.002  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                1             0.002  \n",
       " 24                2             0.004  \n",
       " 25                0             0.000  \n",
       " 26                1             0.002  \n",
       " 27                0             0.000  \n",
       " 28                0             0.000  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 3390              0             0.000  \n",
       " 3391              8             0.016  \n",
       " 3392              9             0.018  \n",
       " 3393              0             0.000  \n",
       " 3394             12             0.024  \n",
       " 3395            116             0.232  \n",
       " 3396              0             0.000  \n",
       " 3397              0             0.000  \n",
       " 3398              0             0.000  \n",
       " 3399              0             0.000  \n",
       " 3400              0             0.000  \n",
       " 3401              0             0.000  \n",
       " 3402              0             0.000  \n",
       " 3403              7             0.014  \n",
       " 3404              0             0.000  \n",
       " 3405              9             0.018  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410             20             0.040  \n",
       " 3411              0             0.000  \n",
       " 3412              1             0.002  \n",
       " 3413              2             0.004  \n",
       " 3414              2             0.004  \n",
       " 3415              6             0.012  \n",
       " 3416              0             0.000  \n",
       " 3417              0             0.000  \n",
       " 3418              0             0.000  \n",
       " 3419              0             0.000  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               157             0.314  \n",
       " 1               126             0.252  \n",
       " 2               127             0.254  \n",
       " 3               133             0.266  \n",
       " 4               130             0.260  \n",
       " 5               131             0.262  \n",
       " 6               169             0.338  \n",
       " 7               151             0.302  \n",
       " 8               197             0.394  \n",
       " 9               192             0.384  \n",
       " 10              204             0.408  \n",
       " 11              182             0.364  \n",
       " 12              131             0.262  \n",
       " 13              156             0.312  \n",
       " 14              127             0.254  \n",
       " 15              156             0.312  \n",
       " 16              138             0.276  \n",
       " 17              179             0.358  \n",
       " 18              155             0.310  \n",
       " 19              162             0.324  \n",
       " 20              169             0.338  \n",
       " 21              157             0.314  \n",
       " 22              149             0.298  \n",
       " 23              141             0.282  \n",
       " 24              155             0.310  \n",
       " 25              135             0.270  \n",
       " 26              148             0.296  \n",
       " 27              134             0.268  \n",
       " 28              161             0.322  \n",
       " 29              157             0.314  \n",
       " ...             ...               ...  \n",
       " 3390            144             0.288  \n",
       " 3391            109             0.218  \n",
       " 3392            115             0.230  \n",
       " 3393            127             0.254  \n",
       " 3394            156             0.312  \n",
       " 3395             69             0.138  \n",
       " 3396              0             0.000  \n",
       " 3397            133             0.266  \n",
       " 3398            161             0.322  \n",
       " 3399            143             0.286  \n",
       " 3400            116             0.232  \n",
       " 3401            110             0.220  \n",
       " 3402            187             0.374  \n",
       " 3403            111             0.222  \n",
       " 3404            118             0.236  \n",
       " 3405             98             0.196  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410             82             0.164  \n",
       " 3411            173             0.346  \n",
       " 3412            130             0.260  \n",
       " 3413            156             0.312  \n",
       " 3414            126             0.252  \n",
       " 3415              4             0.008  \n",
       " 3416            160             0.320  \n",
       " 3417            144             0.288  \n",
       " 3418            112             0.224  \n",
       " 3419            101             0.202  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                73             0.146  \n",
       " 1                61             0.122  \n",
       " 2                47             0.094  \n",
       " 3                62             0.124  \n",
       " 4                67             0.134  \n",
       " 5                43             0.086  \n",
       " 6                98             0.196  \n",
       " 7                71             0.142  \n",
       " 8                67             0.134  \n",
       " 9                66             0.132  \n",
       " 10               65             0.130  \n",
       " 11               42             0.084  \n",
       " 12               54             0.108  \n",
       " 13               67             0.134  \n",
       " 14               61             0.122  \n",
       " 15               56             0.112  \n",
       " 16               63             0.126  \n",
       " 17               72             0.144  \n",
       " 18               73             0.146  \n",
       " 19               76             0.152  \n",
       " 20               50             0.100  \n",
       " 21               39             0.078  \n",
       " 22               63             0.126  \n",
       " 23               31             0.062  \n",
       " 24               49             0.098  \n",
       " 25               34             0.068  \n",
       " 26               66             0.132  \n",
       " 27               55             0.110  \n",
       " 28               66             0.132  \n",
       " 29               75             0.150  \n",
       " ...             ...               ...  \n",
       " 3390             82             0.164  \n",
       " 3391             70             0.140  \n",
       " 3392             32             0.064  \n",
       " 3393             72             0.144  \n",
       " 3394             82             0.164  \n",
       " 3395             69             0.138  \n",
       " 3396              0             0.000  \n",
       " 3397             20             0.040  \n",
       " 3398             72             0.144  \n",
       " 3399             75             0.150  \n",
       " 3400             50             0.100  \n",
       " 3401             41             0.082  \n",
       " 3402             44             0.088  \n",
       " 3403             65             0.130  \n",
       " 3404             34             0.068  \n",
       " 3405             98             0.196  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410             40             0.080  \n",
       " 3411            117             0.234  \n",
       " 3412             75             0.150  \n",
       " 3413             90             0.180  \n",
       " 3414             57             0.114  \n",
       " 3415              4             0.008  \n",
       " 3416             98             0.196  \n",
       " 3417            104             0.208  \n",
       " 3418             79             0.158  \n",
       " 3419             35             0.070  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 1             0.002  \n",
       " 5                 0             0.000  \n",
       " 6                14             0.028  \n",
       " 7                 1             0.002  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                2             0.004  \n",
       " 15                3             0.006  \n",
       " 16                1             0.002  \n",
       " 17                1             0.002  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                4             0.008  \n",
       " 22                0             0.000  \n",
       " 23                1             0.002  \n",
       " 24                1             0.002  \n",
       " 25                2             0.004  \n",
       " 26                2             0.004  \n",
       " 27                0             0.000  \n",
       " 28                1             0.002  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 3390              6             0.012  \n",
       " 3391              1             0.002  \n",
       " 3392              2             0.004  \n",
       " 3393              3             0.006  \n",
       " 3394              5             0.010  \n",
       " 3395             38             0.076  \n",
       " 3396              0             0.000  \n",
       " 3397              0             0.000  \n",
       " 3398              0             0.000  \n",
       " 3399              0             0.000  \n",
       " 3400              0             0.000  \n",
       " 3401              0             0.000  \n",
       " 3402              2             0.004  \n",
       " 3403              7             0.014  \n",
       " 3404              0             0.000  \n",
       " 3405             33             0.066  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410              3             0.006  \n",
       " 3411              9             0.018  \n",
       " 3412              2             0.004  \n",
       " 3413              5             0.010  \n",
       " 3414              1             0.002  \n",
       " 3415              4             0.008  \n",
       " 3416              0             0.000  \n",
       " 3417              2             0.004  \n",
       " 3418              3             0.006  \n",
       " 3419              2             0.004  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':             chrom    start      end                                name  \\\n",
       " 0     pcontig_000   135539   136039     EVM prediction%2pcontig_0000.38   \n",
       " 1     pcontig_000   137364   137864     EVM prediction%2pcontig_0000.39   \n",
       " 2     pcontig_000   137910   138410     EVM prediction%2pcontig_0000.40   \n",
       " 3     pcontig_000   145098   145598     EVM prediction%2pcontig_0000.42   \n",
       " 4     pcontig_000   239086   239586     EVM prediction%2pcontig_0000.64   \n",
       " 5     pcontig_000   298326   298826     EVM prediction%2pcontig_0000.76   \n",
       " 6     pcontig_000   312567   313067     EVM prediction%2pcontig_0000.79   \n",
       " 7     pcontig_000   313921   314421     EVM prediction%2pcontig_0000.80   \n",
       " 8     pcontig_000   410603   411103    EVM prediction%2pcontig_0000.106   \n",
       " 9     pcontig_000   414047   414547    EVM prediction%2pcontig_0000.107   \n",
       " 10    pcontig_000   483629   484129    EVM prediction%2pcontig_0000.122   \n",
       " 11    pcontig_000   487349   487849    EVM prediction%2pcontig_0000.123   \n",
       " 12    pcontig_000   502340   502840    EVM prediction%2pcontig_0000.125   \n",
       " 13    pcontig_000   504596   505096    EVM prediction%2pcontig_0000.126   \n",
       " 14    pcontig_000   511150   511650    EVM prediction%2pcontig_0000.127   \n",
       " 15    pcontig_000   608891   609391    EVM prediction%2pcontig_0000.148   \n",
       " 16    pcontig_000   623314   623814    EVM prediction%2pcontig_0000.153   \n",
       " 17    pcontig_000   626035   626535    EVM prediction%2pcontig_0000.154   \n",
       " 18    pcontig_000   627313   627813    EVM prediction%2pcontig_0000.155   \n",
       " 19    pcontig_000   628219   628719    EVM prediction%2pcontig_0000.156   \n",
       " 20    pcontig_000   631333   631833    EVM prediction%2pcontig_0000.157   \n",
       " 21    pcontig_000   633436   633936    EVM prediction%2pcontig_0000.158   \n",
       " 22    pcontig_000   635447   635947    EVM prediction%2pcontig_0000.159   \n",
       " 23    pcontig_000   655602   656102    EVM prediction%2pcontig_0000.165   \n",
       " 24    pcontig_000   740056   740556    EVM prediction%2pcontig_0000.183   \n",
       " 25    pcontig_000   741412   741912    EVM prediction%2pcontig_0000.184   \n",
       " 26    pcontig_000  1000420  1000920    EVM prediction%2pcontig_0000.241   \n",
       " 27    pcontig_000  1002190  1002690    EVM prediction%2pcontig_0000.242   \n",
       " 28    pcontig_000  1005543  1006043    EVM prediction%2pcontig_0000.243   \n",
       " 29    pcontig_000  1006404  1006904    EVM prediction%2pcontig_0000.244   \n",
       " ...           ...      ...      ...                                 ...   \n",
       " 3390  pcontig_187    10770    11270      EVM prediction%2pcontig_0187.3   \n",
       " 3391  pcontig_187    21061    21561      EVM prediction%2pcontig_0187.6   \n",
       " 3392  pcontig_187    22527    23027      EVM prediction%2pcontig_0187.7   \n",
       " 3393  pcontig_187    23227    23727      EVM prediction%2pcontig_0187.8   \n",
       " 3394  pcontig_187    24397    24897      EVM prediction%2pcontig_0187.9   \n",
       " 3395  pcontig_187    37151    37651     EVM prediction%2pcontig_0187.12   \n",
       " 3396  pcontig_187    39461    39961     EVM prediction%2pcontig_0187.13   \n",
       " 3397  pcontig_193    44797    45297      EVM prediction%2pcontig_0193.1   \n",
       " 3398  pcontig_193    47929    48429      EVM prediction%2pcontig_0193.2   \n",
       " 3399  pcontig_193    49886    50386      EVM prediction%2pcontig_0193.3   \n",
       " 3400  pcontig_193    50908    51408      EVM prediction%2pcontig_0193.4   \n",
       " 3401  pcontig_193    53586    54086      EVM prediction%2pcontig_0193.5   \n",
       " 3402  pcontig_202     5884     6384      EVM prediction%2pcontig_0202.2   \n",
       " 3403  pcontig_204    15198    15698      EVM prediction%2pcontig_0204.4   \n",
       " 3404  pcontig_205    17162    17662      EVM prediction%2pcontig_0205.1   \n",
       " 3405  pcontig_207     2585     3085      EVM prediction%2pcontig_0207.2   \n",
       " 3406  pcontig_207    18938    19438      EVM prediction%2pcontig_0207.7   \n",
       " 3407  pcontig_207    19523    20023      EVM prediction%2pcontig_0207.8   \n",
       " 3408  pcontig_207    20776    21276      EVM prediction%2pcontig_0207.9   \n",
       " 3409  pcontig_207    21072    21572     EVM prediction%2pcontig_0207.10   \n",
       " 3410  pcontig_210     5456     5956      EVM prediction%2pcontig_0210.1   \n",
       " 3411  pcontig_214    10246    10746      EVM prediction%2pcontig_0214.2   \n",
       " 3412  pcontig_214    10761    11261      EVM prediction%2pcontig_0214.3   \n",
       " 3413  pcontig_218    13759    14259      EVM prediction%2pcontig_0218.4   \n",
       " 3414  pcontig_218    15627    16127      EVM prediction%2pcontig_0218.5   \n",
       " 3415  pcontig_230      236      736  EVM prediction%2hcontig_0230_001.1   \n",
       " 3416  pcontig_230    23876    24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 3417  pcontig_231    22945    23445      EVM prediction%2pcontig_0231.1   \n",
       " 3418  pcontig_231    24538    25038      EVM prediction%2pcontig_0231.2   \n",
       " 3419  pcontig_235    13217    13717      EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 0             0.000  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 4             0.008  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                2             0.004  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                1             0.002  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                0             0.000  \n",
       " 24                1             0.002  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                0             0.000  \n",
       " 28                0             0.000  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 3390              1             0.002  \n",
       " 3391              1             0.002  \n",
       " 3392              2             0.004  \n",
       " 3393              3             0.006  \n",
       " 3394              5             0.010  \n",
       " 3395             38             0.076  \n",
       " 3396              0             0.000  \n",
       " 3397              0             0.000  \n",
       " 3398              0             0.000  \n",
       " 3399              0             0.000  \n",
       " 3400              0             0.000  \n",
       " 3401              0             0.000  \n",
       " 3402              0             0.000  \n",
       " 3403              7             0.014  \n",
       " 3404              0             0.000  \n",
       " 3405             33             0.066  \n",
       " 3406              0             0.000  \n",
       " 3407              0             0.000  \n",
       " 3408              0             0.000  \n",
       " 3409              0             0.000  \n",
       " 3410              3             0.006  \n",
       " 3411              0             0.000  \n",
       " 3412              0             0.000  \n",
       " 3413              0             0.000  \n",
       " 3414              0             0.000  \n",
       " 3415              4             0.008  \n",
       " 3416              0             0.000  \n",
       " 3417              0             0.000  \n",
       " 3418              2             0.004  \n",
       " 3419              2             0.004  \n",
       " \n",
       " [3420 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                152             0.304  \n",
       " 1                136             0.272  \n",
       " 2                160             0.320  \n",
       " 3                179             0.358  \n",
       " 4                151             0.302  \n",
       " 5                112             0.224  \n",
       " 6                110             0.220  \n",
       " 7                115             0.230  \n",
       " 8                142             0.284  \n",
       " 9                156             0.312  \n",
       " 10               127             0.254  \n",
       " 11               150             0.300  \n",
       " 12               118             0.236  \n",
       " 13               114             0.228  \n",
       " 14               124             0.248  \n",
       " 15               122             0.244  \n",
       " 16                99             0.198  \n",
       " 17               138             0.276  \n",
       " 18               127             0.254  \n",
       " 19               168             0.336  \n",
       " 20               123             0.246  \n",
       " 21               107             0.214  \n",
       " 22               160             0.320  \n",
       " 23               126             0.252  \n",
       " 24               115             0.230  \n",
       " 25               130             0.260  \n",
       " 26               112             0.224  \n",
       " 27               123             0.246  \n",
       " 28                56             0.112  \n",
       " 29               116             0.232  \n",
       " ...              ...               ...  \n",
       " 19318            133             0.266  \n",
       " 19319            127             0.254  \n",
       " 19320            123             0.246  \n",
       " 19321            124             0.248  \n",
       " 19322            158             0.316  \n",
       " 19323            117             0.234  \n",
       " 19324              6             0.012  \n",
       " 19325            129             0.258  \n",
       " 19326            138             0.276  \n",
       " 19327            157             0.314  \n",
       " 19328            143             0.286  \n",
       " 19329            141             0.282  \n",
       " 19330            120             0.240  \n",
       " 19331            121             0.242  \n",
       " 19332            102             0.204  \n",
       " 19333            142             0.284  \n",
       " 19334             97             0.194  \n",
       " 19335            123             0.246  \n",
       " 19336            133             0.266  \n",
       " 19337            141             0.282  \n",
       " 19338            137             0.274  \n",
       " 19339            117             0.234  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342            100             0.200  \n",
       " 19343            136             0.272  \n",
       " 19344             69             0.138  \n",
       " 19345             72             0.144  \n",
       " 19346            132             0.264  \n",
       " 19347            134             0.268  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                141             0.282  \n",
       " 1                114             0.228  \n",
       " 2                105             0.210  \n",
       " 3                132             0.264  \n",
       " 4                113             0.226  \n",
       " 5                 53             0.106  \n",
       " 6                 43             0.086  \n",
       " 7                 47             0.094  \n",
       " 8                 57             0.114  \n",
       " 9                 77             0.154  \n",
       " 10                49             0.098  \n",
       " 11                97             0.194  \n",
       " 12                70             0.140  \n",
       " 13                59             0.118  \n",
       " 14                54             0.108  \n",
       " 15                70             0.140  \n",
       " 16                53             0.106  \n",
       " 17               102             0.204  \n",
       " 18                46             0.092  \n",
       " 19               120             0.240  \n",
       " 20                64             0.128  \n",
       " 21                69             0.138  \n",
       " 22               100             0.200  \n",
       " 23                78             0.156  \n",
       " 24                40             0.080  \n",
       " 25                27             0.054  \n",
       " 26                58             0.116  \n",
       " 27                51             0.102  \n",
       " 28                11             0.022  \n",
       " 29                67             0.134  \n",
       " ...              ...               ...  \n",
       " 19318            115             0.230  \n",
       " 19319             95             0.190  \n",
       " 19320             73             0.146  \n",
       " 19321             81             0.162  \n",
       " 19322             76             0.152  \n",
       " 19323             59             0.118  \n",
       " 19324              6             0.012  \n",
       " 19325             69             0.138  \n",
       " 19326             58             0.116  \n",
       " 19327             71             0.142  \n",
       " 19328            106             0.212  \n",
       " 19329             66             0.132  \n",
       " 19330             73             0.146  \n",
       " 19331             87             0.174  \n",
       " 19332             55             0.110  \n",
       " 19333             29             0.058  \n",
       " 19334             97             0.194  \n",
       " 19335             91             0.182  \n",
       " 19336             93             0.186  \n",
       " 19337             92             0.184  \n",
       " 19338             69             0.138  \n",
       " 19339             88             0.176  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342             30             0.060  \n",
       " 19343            126             0.252  \n",
       " 19344             59             0.118  \n",
       " 19345             61             0.122  \n",
       " 19346            117             0.234  \n",
       " 19347            131             0.262  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 76             0.152  \n",
       " 1                  8             0.016  \n",
       " 2                  6             0.012  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 1             0.002  \n",
       " 14                 0             0.000  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 2             0.004  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 19318              4             0.008  \n",
       " 19319              0             0.000  \n",
       " 19320              2             0.004  \n",
       " 19321              0             0.000  \n",
       " 19322              0             0.000  \n",
       " 19323              0             0.000  \n",
       " 19324              6             0.012  \n",
       " 19325              0             0.000  \n",
       " 19326              2             0.004  \n",
       " 19327              2             0.004  \n",
       " 19328             10             0.020  \n",
       " 19329              0             0.000  \n",
       " 19330              0             0.000  \n",
       " 19331              0             0.000  \n",
       " 19332              0             0.000  \n",
       " 19333              0             0.000  \n",
       " 19334              4             0.008  \n",
       " 19335              4             0.008  \n",
       " 19336              5             0.010  \n",
       " 19337              1             0.002  \n",
       " 19338              0             0.000  \n",
       " 19339              1             0.002  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342             10             0.020  \n",
       " 19343             16             0.032  \n",
       " 19344             22             0.044  \n",
       " 19345             35             0.070  \n",
       " 19346             69             0.138  \n",
       " 19347             92             0.184  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 36             0.072  \n",
       " 1                  8             0.016  \n",
       " 2                  1             0.002  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 19318              1             0.002  \n",
       " 19319              0             0.000  \n",
       " 19320              0             0.000  \n",
       " 19321              0             0.000  \n",
       " 19322              0             0.000  \n",
       " 19323              0             0.000  \n",
       " 19324              6             0.012  \n",
       " 19325              0             0.000  \n",
       " 19326              0             0.000  \n",
       " 19327              0             0.000  \n",
       " 19328             10             0.020  \n",
       " 19329              0             0.000  \n",
       " 19330              0             0.000  \n",
       " 19331              0             0.000  \n",
       " 19332              0             0.000  \n",
       " 19333              0             0.000  \n",
       " 19334              4             0.008  \n",
       " 19335              4             0.008  \n",
       " 19336              5             0.010  \n",
       " 19337              0             0.000  \n",
       " 19338              0             0.000  \n",
       " 19339              1             0.002  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342             10             0.020  \n",
       " 19343             16             0.032  \n",
       " 19344             22             0.044  \n",
       " 19345             35             0.070  \n",
       " 19346             69             0.138  \n",
       " 19347             92             0.184  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                135             0.270  \n",
       " 1                121             0.242  \n",
       " 2                108             0.216  \n",
       " 3                144             0.288  \n",
       " 4                137             0.274  \n",
       " 5                130             0.260  \n",
       " 6                140             0.280  \n",
       " 7                175             0.350  \n",
       " 8                115             0.230  \n",
       " 9                111             0.222  \n",
       " 10               132             0.264  \n",
       " 11               125             0.250  \n",
       " 12               140             0.280  \n",
       " 13               135             0.270  \n",
       " 14               146             0.292  \n",
       " 15               146             0.292  \n",
       " 16               157             0.314  \n",
       " 17               122             0.244  \n",
       " 18               160             0.320  \n",
       " 19               111             0.222  \n",
       " 20               122             0.244  \n",
       " 21               139             0.278  \n",
       " 22               121             0.242  \n",
       " 23               143             0.286  \n",
       " 24               122             0.244  \n",
       " 25               150             0.300  \n",
       " 26               147             0.294  \n",
       " 27               165             0.330  \n",
       " 28               214             0.428  \n",
       " 29               143             0.286  \n",
       " ...              ...               ...  \n",
       " 19318            152             0.304  \n",
       " 19319            139             0.278  \n",
       " 19320            163             0.326  \n",
       " 19321            128             0.256  \n",
       " 19322            115             0.230  \n",
       " 19323            136             0.272  \n",
       " 19324              4             0.008  \n",
       " 19325            160             0.320  \n",
       " 19326            144             0.288  \n",
       " 19327            112             0.224  \n",
       " 19328            132             0.264  \n",
       " 19329            137             0.274  \n",
       " 19330            144             0.288  \n",
       " 19331            160             0.320  \n",
       " 19332            161             0.322  \n",
       " 19333            111             0.222  \n",
       " 19334            117             0.234  \n",
       " 19335            113             0.226  \n",
       " 19336            147             0.294  \n",
       " 19337            131             0.262  \n",
       " 19338            101             0.202  \n",
       " 19339            154             0.308  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342             93             0.186  \n",
       " 19343             95             0.190  \n",
       " 19344            111             0.222  \n",
       " 19345            115             0.230  \n",
       " 19346             88             0.176  \n",
       " 19347             56             0.112  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 83             0.166  \n",
       " 1                 64             0.128  \n",
       " 2                 71             0.142  \n",
       " 3                 41             0.082  \n",
       " 4                 36             0.072  \n",
       " 5                 69             0.138  \n",
       " 6                 73             0.146  \n",
       " 7                 81             0.162  \n",
       " 8                 19             0.038  \n",
       " 9                 30             0.060  \n",
       " 10                73             0.146  \n",
       " 11                46             0.092  \n",
       " 12                62             0.124  \n",
       " 13                65             0.130  \n",
       " 14                71             0.142  \n",
       " 15                40             0.080  \n",
       " 16                43             0.086  \n",
       " 17                50             0.100  \n",
       " 18                68             0.136  \n",
       " 19                43             0.086  \n",
       " 20                43             0.086  \n",
       " 21                61             0.122  \n",
       " 22                33             0.066  \n",
       " 23                60             0.120  \n",
       " 24                57             0.114  \n",
       " 25                57             0.114  \n",
       " 26                47             0.094  \n",
       " 27                67             0.134  \n",
       " 28               106             0.212  \n",
       " 29                53             0.106  \n",
       " ...              ...               ...  \n",
       " 19318             63             0.126  \n",
       " 19319             52             0.104  \n",
       " 19320             77             0.154  \n",
       " 19321             43             0.086  \n",
       " 19322             36             0.072  \n",
       " 19323             40             0.080  \n",
       " 19324              4             0.008  \n",
       " 19325             98             0.196  \n",
       " 19326            104             0.208  \n",
       " 19327             79             0.158  \n",
       " 19328             59             0.118  \n",
       " 19329             57             0.114  \n",
       " 19330             81             0.162  \n",
       " 19331             50             0.100  \n",
       " 19332             64             0.128  \n",
       " 19333             33             0.066  \n",
       " 19334            117             0.234  \n",
       " 19335             55             0.110  \n",
       " 19336             57             0.114  \n",
       " 19337             44             0.088  \n",
       " 19338             35             0.070  \n",
       " 19339             84             0.168  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342             20             0.040  \n",
       " 19343             51             0.102  \n",
       " 19344             69             0.138  \n",
       " 19345             66             0.132  \n",
       " 19346             51             0.102  \n",
       " 19347             46             0.092  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 14             0.028  \n",
       " 1                  0             0.000  \n",
       " 2                  3             0.006  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  2             0.004  \n",
       " 7                  1             0.002  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 2             0.004  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 1             0.002  \n",
       " 25                 1             0.002  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 1             0.002  \n",
       " ...              ...               ...  \n",
       " 19318              1             0.002  \n",
       " 19319              1             0.002  \n",
       " 19320              2             0.004  \n",
       " 19321              0             0.000  \n",
       " 19322              0             0.000  \n",
       " 19323              0             0.000  \n",
       " 19324              4             0.008  \n",
       " 19325              0             0.000  \n",
       " 19326              2             0.004  \n",
       " 19327              3             0.006  \n",
       " 19328              2             0.004  \n",
       " 19329              0             0.000  \n",
       " 19330              0             0.000  \n",
       " 19331              1             0.002  \n",
       " 19332              0             0.000  \n",
       " 19333              2             0.004  \n",
       " 19334             29             0.058  \n",
       " 19335              4             0.008  \n",
       " 19336              1             0.002  \n",
       " 19337              1             0.002  \n",
       " 19338              2             0.004  \n",
       " 19339              1             0.002  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342              3             0.006  \n",
       " 19343              2             0.004  \n",
       " 19344             23             0.046  \n",
       " 19345             29             0.058  \n",
       " 19346             18             0.036  \n",
       " 19347             14             0.028  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':              chrom   start     end                                name  \\\n",
       " 0      pcontig_000     766    1266            gene_model_pcontig_000.1   \n",
       " 1      pcontig_000    5108    5608            gene_model_pcontig_000.2   \n",
       " 2      pcontig_000    8489    8989            gene_model_pcontig_000.3   \n",
       " 3      pcontig_000   12148   12648            gene_model_pcontig_000.4   \n",
       " 4      pcontig_000   22053   22553            gene_model_pcontig_000.5   \n",
       " 5      pcontig_000   28351   28851            gene_model_pcontig_000.6   \n",
       " 6      pcontig_000   31591   32091            gene_model_pcontig_000.7   \n",
       " 7      pcontig_000   31989   32489            gene_model_pcontig_000.8   \n",
       " 8      pcontig_000   34648   35148            gene_model_pcontig_000.9   \n",
       " 9      pcontig_000   38446   38946           gene_model_pcontig_000.10   \n",
       " 10     pcontig_000   40765   41265           gene_model_pcontig_000.11   \n",
       " 11     pcontig_000   43851   44351           gene_model_pcontig_000.12   \n",
       " 12     pcontig_000   46945   47445           gene_model_pcontig_000.13   \n",
       " 13     pcontig_000   49547   50047           gene_model_pcontig_000.14   \n",
       " 14     pcontig_000   52604   53104           gene_model_pcontig_000.15   \n",
       " 15     pcontig_000   54854   55354           gene_model_pcontig_000.16   \n",
       " 16     pcontig_000   57424   57924           gene_model_pcontig_000.17   \n",
       " 17     pcontig_000   60472   60972           gene_model_pcontig_000.18   \n",
       " 18     pcontig_000   66836   67336           gene_model_pcontig_000.19   \n",
       " 19     pcontig_000   71673   72173           gene_model_pcontig_000.20   \n",
       " 20     pcontig_000   76029   76529           gene_model_pcontig_000.21   \n",
       " 21     pcontig_000   81209   81709           gene_model_pcontig_000.22   \n",
       " 22     pcontig_000   82757   83257           gene_model_pcontig_000.23   \n",
       " 23     pcontig_000   84284   84784           gene_model_pcontig_000.24   \n",
       " 24     pcontig_000   84827   85327           gene_model_pcontig_000.25   \n",
       " 25     pcontig_000   88628   89128           gene_model_pcontig_000.26   \n",
       " 26     pcontig_000   91662   92162           gene_model_pcontig_000.27   \n",
       " 27     pcontig_000   97055   97555           gene_model_pcontig_000.28   \n",
       " 28     pcontig_000  108065  108565           gene_model_pcontig_000.29   \n",
       " 29     pcontig_000  114353  114853           gene_model_pcontig_000.30   \n",
       " ...            ...     ...     ...                                 ...   \n",
       " 19318  pcontig_225   13374   13874            gene_model_pcontig_225.4   \n",
       " 19319  pcontig_225   17271   17771            gene_model_pcontig_225.5   \n",
       " 19320  pcontig_225   18671   19171            gene_model_pcontig_225.6   \n",
       " 19321  pcontig_225   22307   22807            gene_model_pcontig_225.7   \n",
       " 19322  pcontig_225   23726   24226            gene_model_pcontig_225.8   \n",
       " 19323  pcontig_225   24708   25208            gene_model_pcontig_225.9   \n",
       " 19324  pcontig_230     236     736  EVM prediction%2hcontig_0230_001.1   \n",
       " 19325  pcontig_230   23876   24376  EVM prediction%2hcontig_0230_001.2   \n",
       " 19326  pcontig_231   22945   23445      EVM prediction%2pcontig_0231.1   \n",
       " 19327  pcontig_231   24538   25038      EVM prediction%2pcontig_0231.2   \n",
       " 19328  pcontig_233    2633    3133            gene_model_pcontig_233.1   \n",
       " 19329  pcontig_233    9598   10098            gene_model_pcontig_233.2   \n",
       " 19330  pcontig_233   11955   12455            gene_model_pcontig_233.3   \n",
       " 19331  pcontig_233   15137   15637            gene_model_pcontig_233.4   \n",
       " 19332  pcontig_233   19085   19585            gene_model_pcontig_233.5   \n",
       " 19333  pcontig_233   24903   25403            gene_model_pcontig_233.6   \n",
       " 19334  pcontig_235    1209    1709            gene_model_pcontig_235.1   \n",
       " 19335  pcontig_235    3517    4017            gene_model_pcontig_235.2   \n",
       " 19336  pcontig_235    9863   10363            gene_model_pcontig_235.3   \n",
       " 19337  pcontig_235   12336   12836            gene_model_pcontig_235.4   \n",
       " 19338  pcontig_235   13217   13717      EVM prediction%2pcontig_0235.5   \n",
       " 19339  pcontig_235   20192   20692            gene_model_pcontig_235.6   \n",
       " 19340  pcontig_235   24768   25268            gene_model_pcontig_235.7   \n",
       " 19341  pcontig_248    1242    1742            gene_model_pcontig_248.1   \n",
       " 19342  pcontig_248    3541    4041            gene_model_pcontig_248.2   \n",
       " 19343  pcontig_248    8183    8683            gene_model_pcontig_248.3   \n",
       " 19344  pcontig_248    9497    9997            gene_model_pcontig_248.4   \n",
       " 19345  pcontig_248   11216   11716            gene_model_pcontig_248.5   \n",
       " 19346  pcontig_248   13257   13757            gene_model_pcontig_248.6   \n",
       " 19347  pcontig_248   17155   17655            gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  5             0.010  \n",
       " 1                  0             0.000  \n",
       " 2                  0             0.000  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 1             0.002  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 19318              0             0.000  \n",
       " 19319              0             0.000  \n",
       " 19320              0             0.000  \n",
       " 19321              0             0.000  \n",
       " 19322              0             0.000  \n",
       " 19323              0             0.000  \n",
       " 19324              4             0.008  \n",
       " 19325              0             0.000  \n",
       " 19326              0             0.000  \n",
       " 19327              2             0.004  \n",
       " 19328              2             0.004  \n",
       " 19329              0             0.000  \n",
       " 19330              0             0.000  \n",
       " 19331              0             0.000  \n",
       " 19332              0             0.000  \n",
       " 19333              2             0.004  \n",
       " 19334             29             0.058  \n",
       " 19335              4             0.008  \n",
       " 19336              1             0.002  \n",
       " 19337              1             0.002  \n",
       " 19338              2             0.004  \n",
       " 19339              1             0.002  \n",
       " 19340              0             0.000  \n",
       " 19341              0             0.000  \n",
       " 19342              3             0.006  \n",
       " 19343              2             0.004  \n",
       " 19344             23             0.046  \n",
       " 19345             29             0.058  \n",
       " 19346             18             0.036  \n",
       " 19347             14             0.028  \n",
       " \n",
       " [19348 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                93             0.186  \n",
       " 1               112             0.224  \n",
       " 2               123             0.246  \n",
       " 3                98             0.196  \n",
       " 4               155             0.310  \n",
       " 5               119             0.238  \n",
       " 6               127             0.254  \n",
       " 7               121             0.242  \n",
       " 8               134             0.268  \n",
       " 9                73             0.146  \n",
       " 10              130             0.260  \n",
       " 11              126             0.252  \n",
       " 12              103             0.206  \n",
       " 13              111             0.222  \n",
       " 14               88             0.176  \n",
       " 15              163             0.326  \n",
       " 16               65             0.130  \n",
       " 17               86             0.172  \n",
       " 18              103             0.206  \n",
       " 19               86             0.172  \n",
       " 20              110             0.220  \n",
       " 21               74             0.148  \n",
       " 22              133             0.266  \n",
       " 23              127             0.254  \n",
       " 24              111             0.222  \n",
       " 25              121             0.242  \n",
       " 26              117             0.234  \n",
       " 27               95             0.190  \n",
       " 28               82             0.164  \n",
       " 29               88             0.176  \n",
       " ...             ...               ...  \n",
       " 6268            101             0.202  \n",
       " 6269            122             0.244  \n",
       " 6270            113             0.226  \n",
       " 6271            121             0.242  \n",
       " 6272            110             0.220  \n",
       " 6273            158             0.316  \n",
       " 6274              0             0.000  \n",
       " 6275            130             0.260  \n",
       " 6276            113             0.226  \n",
       " 6277            136             0.272  \n",
       " 6278            160             0.320  \n",
       " 6279            149             0.298  \n",
       " 6280            100             0.200  \n",
       " 6281            128             0.256  \n",
       " 6282            165             0.330  \n",
       " 6283            119             0.238  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288            156             0.312  \n",
       " 6289            142             0.284  \n",
       " 6290            142             0.284  \n",
       " 6291            110             0.220  \n",
       " 6292            154             0.308  \n",
       " 6293              6             0.012  \n",
       " 6294            129             0.258  \n",
       " 6295            138             0.276  \n",
       " 6296            157             0.314  \n",
       " 6297            137             0.274  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                66             0.132  \n",
       " 1                48             0.096  \n",
       " 2                82             0.164  \n",
       " 3                68             0.136  \n",
       " 4               121             0.242  \n",
       " 5                61             0.122  \n",
       " 6                88             0.176  \n",
       " 7                75             0.150  \n",
       " 8               100             0.200  \n",
       " 9                25             0.050  \n",
       " 10               67             0.134  \n",
       " 11               74             0.148  \n",
       " 12               60             0.120  \n",
       " 13               54             0.108  \n",
       " 14               63             0.126  \n",
       " 15               81             0.162  \n",
       " 16               35             0.070  \n",
       " 17               46             0.092  \n",
       " 18               56             0.112  \n",
       " 19               41             0.082  \n",
       " 20               47             0.094  \n",
       " 21               29             0.058  \n",
       " 22               47             0.094  \n",
       " 23               68             0.136  \n",
       " 24               61             0.122  \n",
       " 25               65             0.130  \n",
       " 26               76             0.152  \n",
       " 27               33             0.066  \n",
       " 28               15             0.030  \n",
       " 29               15             0.030  \n",
       " ...             ...               ...  \n",
       " 6268             51             0.102  \n",
       " 6269             94             0.188  \n",
       " 6270             78             0.156  \n",
       " 6271             78             0.156  \n",
       " 6272             89             0.178  \n",
       " 6273            158             0.316  \n",
       " 6274              0             0.000  \n",
       " 6275             73             0.146  \n",
       " 6276             67             0.134  \n",
       " 6277             73             0.146  \n",
       " 6278             89             0.178  \n",
       " 6279             76             0.152  \n",
       " 6280             85             0.170  \n",
       " 6281            103             0.206  \n",
       " 6282            128             0.256  \n",
       " 6283            119             0.238  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288            122             0.244  \n",
       " 6289            117             0.234  \n",
       " 6290            120             0.240  \n",
       " 6291             92             0.184  \n",
       " 6292            121             0.242  \n",
       " 6293              6             0.012  \n",
       " 6294             69             0.138  \n",
       " 6295             58             0.116  \n",
       " 6296             71             0.142  \n",
       " 6297             69             0.138  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 2             0.004  \n",
       " 1                 2             0.004  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 3             0.006  \n",
       " 5                 3             0.006  \n",
       " 6                 8             0.016  \n",
       " 7                 7             0.014  \n",
       " 8                 4             0.008  \n",
       " 9                 1             0.002  \n",
       " 10                2             0.004  \n",
       " 11                1             0.002  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                1             0.002  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                4             0.008  \n",
       " 18                0             0.000  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                1             0.002  \n",
       " 23                3             0.006  \n",
       " 24                1             0.002  \n",
       " 25                0             0.000  \n",
       " 26                1             0.002  \n",
       " 27                0             0.000  \n",
       " 28                4             0.008  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 6268              1             0.002  \n",
       " 6269              8             0.016  \n",
       " 6270              9             0.018  \n",
       " 6271              0             0.000  \n",
       " 6272             12             0.024  \n",
       " 6273            116             0.232  \n",
       " 6274              0             0.000  \n",
       " 6275              0             0.000  \n",
       " 6276              0             0.000  \n",
       " 6277              0             0.000  \n",
       " 6278              0             0.000  \n",
       " 6279              1             0.002  \n",
       " 6280              0             0.000  \n",
       " 6281              7             0.014  \n",
       " 6282              1             0.002  \n",
       " 6283              9             0.018  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288             20             0.040  \n",
       " 6289              7             0.014  \n",
       " 6290             13             0.026  \n",
       " 6291              9             0.018  \n",
       " 6292              7             0.014  \n",
       " 6293              6             0.012  \n",
       " 6294              0             0.000  \n",
       " 6295              2             0.004  \n",
       " 6296              2             0.004  \n",
       " 6297              0             0.000  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 0             0.000  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 1             0.002  \n",
       " 6                 2             0.004  \n",
       " 7                 2             0.004  \n",
       " 8                 1             0.002  \n",
       " 9                 1             0.002  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                1             0.002  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                2             0.004  \n",
       " 24                1             0.002  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                0             0.000  \n",
       " 28                1             0.002  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 6268              0             0.000  \n",
       " 6269              8             0.016  \n",
       " 6270              9             0.018  \n",
       " 6271              0             0.000  \n",
       " 6272             12             0.024  \n",
       " 6273            116             0.232  \n",
       " 6274              0             0.000  \n",
       " 6275              0             0.000  \n",
       " 6276              0             0.000  \n",
       " 6277              0             0.000  \n",
       " 6278              0             0.000  \n",
       " 6279              0             0.000  \n",
       " 6280              0             0.000  \n",
       " 6281              7             0.014  \n",
       " 6282              0             0.000  \n",
       " 6283              9             0.018  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288             20             0.040  \n",
       " 6289              0             0.000  \n",
       " 6290              1             0.002  \n",
       " 6291              2             0.004  \n",
       " 6292              2             0.004  \n",
       " 6293              6             0.012  \n",
       " 6294              0             0.000  \n",
       " 6295              0             0.000  \n",
       " 6296              0             0.000  \n",
       " 6297              0             0.000  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               187             0.374  \n",
       " 1               174             0.348  \n",
       " 2               150             0.300  \n",
       " 3               163             0.326  \n",
       " 4               142             0.284  \n",
       " 5               155             0.310  \n",
       " 6               125             0.250  \n",
       " 7               127             0.254  \n",
       " 8               131             0.262  \n",
       " 9               165             0.330  \n",
       " 10              134             0.268  \n",
       " 11              173             0.346  \n",
       " 12              148             0.296  \n",
       " 13              108             0.216  \n",
       " 14              195             0.390  \n",
       " 15              117             0.234  \n",
       " 16              202             0.404  \n",
       " 17              191             0.382  \n",
       " 18              171             0.342  \n",
       " 19              164             0.328  \n",
       " 20              166             0.332  \n",
       " 21              181             0.362  \n",
       " 22              171             0.342  \n",
       " 23              151             0.302  \n",
       " 24              134             0.268  \n",
       " 25              144             0.288  \n",
       " 26              159             0.318  \n",
       " 27              153             0.306  \n",
       " 28              179             0.358  \n",
       " 29              159             0.318  \n",
       " ...             ...               ...  \n",
       " 6268            144             0.288  \n",
       " 6269            109             0.218  \n",
       " 6270            115             0.230  \n",
       " 6271            127             0.254  \n",
       " 6272            156             0.312  \n",
       " 6273             69             0.138  \n",
       " 6274              0             0.000  \n",
       " 6275            133             0.266  \n",
       " 6276            161             0.322  \n",
       " 6277            143             0.286  \n",
       " 6278            116             0.232  \n",
       " 6279            110             0.220  \n",
       " 6280            187             0.374  \n",
       " 6281            111             0.222  \n",
       " 6282            118             0.236  \n",
       " 6283             98             0.196  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288             82             0.164  \n",
       " 6289            173             0.346  \n",
       " 6290            130             0.260  \n",
       " 6291            156             0.312  \n",
       " 6292            126             0.252  \n",
       " 6293              4             0.008  \n",
       " 6294            160             0.320  \n",
       " 6295            144             0.288  \n",
       " 6296            112             0.224  \n",
       " 6297            101             0.202  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               125             0.250  \n",
       " 1               104             0.208  \n",
       " 2                55             0.110  \n",
       " 3                47             0.094  \n",
       " 4                31             0.062  \n",
       " 5                79             0.158  \n",
       " 6                61             0.122  \n",
       " 7                54             0.108  \n",
       " 8                64             0.128  \n",
       " 9                60             0.120  \n",
       " 10               50             0.100  \n",
       " 11               77             0.154  \n",
       " 12               49             0.098  \n",
       " 13               30             0.060  \n",
       " 14               68             0.136  \n",
       " 15               31             0.062  \n",
       " 16              112             0.224  \n",
       " 17              119             0.238  \n",
       " 18              112             0.224  \n",
       " 19               97             0.194  \n",
       " 20              108             0.216  \n",
       " 21              106             0.212  \n",
       " 22               56             0.112  \n",
       " 23               57             0.114  \n",
       " 24               28             0.056  \n",
       " 25               45             0.090  \n",
       " 26               90             0.180  \n",
       " 27               36             0.072  \n",
       " 28               30             0.060  \n",
       " 29               37             0.074  \n",
       " ...             ...               ...  \n",
       " 6268             82             0.164  \n",
       " 6269             70             0.140  \n",
       " 6270             32             0.064  \n",
       " 6271             72             0.144  \n",
       " 6272             82             0.164  \n",
       " 6273             69             0.138  \n",
       " 6274              0             0.000  \n",
       " 6275             20             0.040  \n",
       " 6276             72             0.144  \n",
       " 6277             75             0.150  \n",
       " 6278             50             0.100  \n",
       " 6279             41             0.082  \n",
       " 6280             44             0.088  \n",
       " 6281             65             0.130  \n",
       " 6282             34             0.068  \n",
       " 6283             98             0.196  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288             40             0.080  \n",
       " 6289            117             0.234  \n",
       " 6290             75             0.150  \n",
       " 6291             90             0.180  \n",
       " 6292             57             0.114  \n",
       " 6293              4             0.008  \n",
       " 6294             98             0.196  \n",
       " 6295            104             0.208  \n",
       " 6296             79             0.158  \n",
       " 6297             35             0.070  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 7             0.014  \n",
       " 1                 4             0.008  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 4             0.008  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 2             0.004  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                4             0.008  \n",
       " 18                0             0.000  \n",
       " 19                8             0.016  \n",
       " 20                2             0.004  \n",
       " 21                1             0.002  \n",
       " 22                1             0.002  \n",
       " 23                4             0.008  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                1             0.002  \n",
       " 27                1             0.002  \n",
       " 28                2             0.004  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 6268              6             0.012  \n",
       " 6269              1             0.002  \n",
       " 6270              2             0.004  \n",
       " 6271              3             0.006  \n",
       " 6272              5             0.010  \n",
       " 6273             38             0.076  \n",
       " 6274              0             0.000  \n",
       " 6275              0             0.000  \n",
       " 6276              0             0.000  \n",
       " 6277              0             0.000  \n",
       " 6278              0             0.000  \n",
       " 6279              0             0.000  \n",
       " 6280              2             0.004  \n",
       " 6281              7             0.014  \n",
       " 6282              0             0.000  \n",
       " 6283             33             0.066  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288              3             0.006  \n",
       " 6289              9             0.018  \n",
       " 6290              2             0.004  \n",
       " 6291              5             0.010  \n",
       " 6292              1             0.002  \n",
       " 6293              4             0.008  \n",
       " 6294              0             0.000  \n",
       " 6295              2             0.004  \n",
       " 6296              3             0.006  \n",
       " 6297              2             0.004  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom   start     end                                  name  \\\n",
       " 0     hcontig_000_003   14562   15062    EVM prediction%2hcontig_0000_003.5   \n",
       " 1     hcontig_000_003   17795   18295    EVM prediction%2hcontig_0000_003.6   \n",
       " 2     hcontig_000_003   80525   81025   EVM prediction%2hcontig_0000_003.21   \n",
       " 3     hcontig_000_003   81736   82236   EVM prediction%2hcontig_0000_003.22   \n",
       " 4     hcontig_000_003  108981  109481   EVM prediction%2hcontig_0000_003.25   \n",
       " 5     hcontig_000_003  180425  180925   EVM prediction%2hcontig_0000_003.45   \n",
       " 6     hcontig_000_003  182250  182750   EVM prediction%2hcontig_0000_003.46   \n",
       " 7     hcontig_000_003  182796  183296   EVM prediction%2hcontig_0000_003.47   \n",
       " 8     hcontig_000_003  189995  190495   EVM prediction%2hcontig_0000_003.48   \n",
       " 9     hcontig_000_003  269647  270147   EVM prediction%2hcontig_0000_003.69   \n",
       " 10    hcontig_000_003  329808  330308   EVM prediction%2hcontig_0000_003.81   \n",
       " 11    hcontig_000_003  344036  344536   EVM prediction%2hcontig_0000_003.84   \n",
       " 12    hcontig_000_003  345369  345869   EVM prediction%2hcontig_0000_003.85   \n",
       " 13    hcontig_000_003  402060  402560  EVM prediction%2hcontig_0000_003.100   \n",
       " 14    hcontig_000_003  406700  407200  EVM prediction%2hcontig_0000_003.102   \n",
       " 15    hcontig_000_003  452052  452552  EVM prediction%2hcontig_0000_003.113   \n",
       " 16    hcontig_000_003  461215  461715  EVM prediction%2hcontig_0000_003.114   \n",
       " 17    hcontig_000_003  464659  465159  EVM prediction%2hcontig_0000_003.115   \n",
       " 18    hcontig_000_003  524253  524753  EVM prediction%2hcontig_0000_003.128   \n",
       " 19    hcontig_000_003  524807  525307  EVM prediction%2hcontig_0000_003.129   \n",
       " 20    hcontig_000_003  525753  526253  EVM prediction%2hcontig_0000_003.130   \n",
       " 21    hcontig_000_003  530399  530899  EVM prediction%2hcontig_0000_003.132   \n",
       " 22    hcontig_000_003  538838  539338  EVM prediction%2hcontig_0000_003.134   \n",
       " 23    hcontig_000_003  547041  547541  EVM prediction%2hcontig_0000_003.136   \n",
       " 24    hcontig_000_003  570436  570936  EVM prediction%2hcontig_0000_003.140   \n",
       " 25    hcontig_000_003  571657  572157  EVM prediction%2hcontig_0000_003.141   \n",
       " 26    hcontig_000_003  575825  576325  EVM prediction%2hcontig_0000_003.143   \n",
       " 27    hcontig_000_003  670734  671234  EVM prediction%2hcontig_0000_003.161   \n",
       " 28    hcontig_000_003  687456  687956  EVM prediction%2hcontig_0000_003.166   \n",
       " 29    hcontig_000_003  689640  690140  EVM prediction%2hcontig_0000_003.167   \n",
       " ...               ...     ...     ...                                   ...   \n",
       " 6268      pcontig_187   10770   11270        EVM prediction%2pcontig_0187.3   \n",
       " 6269      pcontig_187   21061   21561        EVM prediction%2pcontig_0187.6   \n",
       " 6270      pcontig_187   22527   23027        EVM prediction%2pcontig_0187.7   \n",
       " 6271      pcontig_187   23227   23727        EVM prediction%2pcontig_0187.8   \n",
       " 6272      pcontig_187   24397   24897        EVM prediction%2pcontig_0187.9   \n",
       " 6273      pcontig_187   37151   37651       EVM prediction%2pcontig_0187.12   \n",
       " 6274      pcontig_187   39461   39961       EVM prediction%2pcontig_0187.13   \n",
       " 6275      pcontig_193   44797   45297        EVM prediction%2pcontig_0193.1   \n",
       " 6276      pcontig_193   47929   48429        EVM prediction%2pcontig_0193.2   \n",
       " 6277      pcontig_193   49886   50386        EVM prediction%2pcontig_0193.3   \n",
       " 6278      pcontig_193   50908   51408        EVM prediction%2pcontig_0193.4   \n",
       " 6279      pcontig_193   53586   54086        EVM prediction%2pcontig_0193.5   \n",
       " 6280      pcontig_202    5884    6384        EVM prediction%2pcontig_0202.2   \n",
       " 6281      pcontig_204   15198   15698        EVM prediction%2pcontig_0204.4   \n",
       " 6282      pcontig_205   17162   17662        EVM prediction%2pcontig_0205.1   \n",
       " 6283      pcontig_207    2585    3085        EVM prediction%2pcontig_0207.2   \n",
       " 6284      pcontig_207   18938   19438        EVM prediction%2pcontig_0207.7   \n",
       " 6285      pcontig_207   19523   20023        EVM prediction%2pcontig_0207.8   \n",
       " 6286      pcontig_207   20776   21276        EVM prediction%2pcontig_0207.9   \n",
       " 6287      pcontig_207   21072   21572       EVM prediction%2pcontig_0207.10   \n",
       " 6288      pcontig_210    5456    5956        EVM prediction%2pcontig_0210.1   \n",
       " 6289      pcontig_214   10246   10746        EVM prediction%2pcontig_0214.2   \n",
       " 6290      pcontig_214   10761   11261        EVM prediction%2pcontig_0214.3   \n",
       " 6291      pcontig_218   13759   14259        EVM prediction%2pcontig_0218.4   \n",
       " 6292      pcontig_218   15627   16127        EVM prediction%2pcontig_0218.5   \n",
       " 6293      pcontig_230     236     736    EVM prediction%2hcontig_0230_001.1   \n",
       " 6294      pcontig_230   23876   24376    EVM prediction%2hcontig_0230_001.2   \n",
       " 6295      pcontig_231   22945   23445        EVM prediction%2pcontig_0231.1   \n",
       " 6296      pcontig_231   24538   25038        EVM prediction%2pcontig_0231.2   \n",
       " 6297      pcontig_235   13217   13717        EVM prediction%2pcontig_0235.5   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                 1             0.002  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                2             0.004  \n",
       " 18                0             0.000  \n",
       " 19                1             0.002  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                2             0.004  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                1             0.002  \n",
       " 28                0             0.000  \n",
       " 29                1             0.002  \n",
       " ...             ...               ...  \n",
       " 6268              1             0.002  \n",
       " 6269              1             0.002  \n",
       " 6270              2             0.004  \n",
       " 6271              3             0.006  \n",
       " 6272              5             0.010  \n",
       " 6273             38             0.076  \n",
       " 6274              0             0.000  \n",
       " 6275              0             0.000  \n",
       " 6276              0             0.000  \n",
       " 6277              0             0.000  \n",
       " 6278              0             0.000  \n",
       " 6279              0             0.000  \n",
       " 6280              0             0.000  \n",
       " 6281              7             0.014  \n",
       " 6282              0             0.000  \n",
       " 6283             33             0.066  \n",
       " 6284              0             0.000  \n",
       " 6285              0             0.000  \n",
       " 6286              0             0.000  \n",
       " 6287              0             0.000  \n",
       " 6288              3             0.006  \n",
       " 6289              0             0.000  \n",
       " 6290              0             0.000  \n",
       " 6291              0             0.000  \n",
       " 6292              0             0.000  \n",
       " 6293              4             0.008  \n",
       " 6294              0             0.000  \n",
       " 6295              0             0.000  \n",
       " 6296              2             0.004  \n",
       " 6297              2             0.004  \n",
       " \n",
       " [6298 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               131             0.262  \n",
       " 1               143             0.286  \n",
       " 2               114             0.228  \n",
       " 3               119             0.238  \n",
       " 4               161             0.322  \n",
       " 5               112             0.224  \n",
       " 6               120             0.240  \n",
       " 7               127             0.254  \n",
       " 8               119             0.238  \n",
       " 9               132             0.264  \n",
       " 10              130             0.260  \n",
       " 11              127             0.254  \n",
       " 12              126             0.252  \n",
       " 13              151             0.302  \n",
       " 14              109             0.218  \n",
       " 15              131             0.262  \n",
       " 16              143             0.286  \n",
       " 17              135             0.270  \n",
       " 18              122             0.244  \n",
       " 19              131             0.262  \n",
       " 20               99             0.198  \n",
       " 21              131             0.262  \n",
       " 22              159             0.318  \n",
       " 23              157             0.314  \n",
       " 24              117             0.234  \n",
       " 25              113             0.226  \n",
       " 26              146             0.292  \n",
       " 27              134             0.268  \n",
       " 28              147             0.294  \n",
       " 29              125             0.250  \n",
       " ...             ...               ...  \n",
       " 2008            126             0.252  \n",
       " 2009             85             0.170  \n",
       " 2010              0             0.000  \n",
       " 2011            120             0.240  \n",
       " 2012            151             0.302  \n",
       " 2013            131             0.262  \n",
       " 2014             91             0.182  \n",
       " 2015            122             0.244  \n",
       " 2016             96             0.192  \n",
       " 2017             96             0.192  \n",
       " 2018             93             0.186  \n",
       " 2019             93             0.186  \n",
       " 2020            111             0.222  \n",
       " 2021            134             0.268  \n",
       " 2022            118             0.236  \n",
       " 2023            120             0.240  \n",
       " 2024             98             0.196  \n",
       " 2025              0             0.000  \n",
       " 2026            124             0.248  \n",
       " 2027            147             0.294  \n",
       " 2028            118             0.236  \n",
       " 2029            121             0.242  \n",
       " 2030            109             0.218  \n",
       " 2031            112             0.224  \n",
       " 2032            141             0.282  \n",
       " 2033            109             0.218  \n",
       " 2034            104             0.208  \n",
       " 2035            104             0.208  \n",
       " 2036            142             0.284  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               119             0.238  \n",
       " 1                86             0.172  \n",
       " 2                37             0.074  \n",
       " 3                18             0.036  \n",
       " 4               102             0.204  \n",
       " 5                45             0.090  \n",
       " 6                76             0.152  \n",
       " 7                70             0.140  \n",
       " 8                67             0.134  \n",
       " 9                26             0.052  \n",
       " 10               71             0.142  \n",
       " 11               46             0.092  \n",
       " 12               43             0.086  \n",
       " 13               42             0.084  \n",
       " 14               63             0.126  \n",
       " 15               14             0.028  \n",
       " 16               16             0.032  \n",
       " 17              114             0.228  \n",
       " 18               46             0.092  \n",
       " 19               39             0.078  \n",
       " 20               32             0.064  \n",
       " 21               29             0.058  \n",
       " 22               83             0.166  \n",
       " 23               89             0.178  \n",
       " 24               71             0.142  \n",
       " 25               42             0.084  \n",
       " 26              100             0.200  \n",
       " 27               41             0.082  \n",
       " 28               90             0.180  \n",
       " 29               70             0.140  \n",
       " ...             ...               ...  \n",
       " 2008             69             0.138  \n",
       " 2009             15             0.030  \n",
       " 2010              0             0.000  \n",
       " 2011             48             0.096  \n",
       " 2012             62             0.124  \n",
       " 2013             83             0.166  \n",
       " 2014             49             0.098  \n",
       " 2015             58             0.116  \n",
       " 2016             60             0.120  \n",
       " 2017             39             0.078  \n",
       " 2018             33             0.066  \n",
       " 2019             34             0.068  \n",
       " 2020             51             0.102  \n",
       " 2021             44             0.088  \n",
       " 2022             96             0.192  \n",
       " 2023            101             0.202  \n",
       " 2024             32             0.064  \n",
       " 2025              0             0.000  \n",
       " 2026            107             0.214  \n",
       " 2027            125             0.250  \n",
       " 2028            105             0.210  \n",
       " 2029            108             0.216  \n",
       " 2030             39             0.078  \n",
       " 2031             23             0.046  \n",
       " 2032             87             0.174  \n",
       " 2033             53             0.106  \n",
       " 2034             54             0.108  \n",
       " 2035             17             0.034  \n",
       " 2036             29             0.058  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                53             0.106  \n",
       " 1                 1             0.002  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 1             0.002  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                1             0.002  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17               21             0.042  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                2             0.004  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                2             0.004  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                5             0.010  \n",
       " 27                0             0.000  \n",
       " 28                0             0.000  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2008              0             0.000  \n",
       " 2009              0             0.000  \n",
       " 2010              0             0.000  \n",
       " 2011              0             0.000  \n",
       " 2012              0             0.000  \n",
       " 2013              0             0.000  \n",
       " 2014              0             0.000  \n",
       " 2015              0             0.000  \n",
       " 2016              0             0.000  \n",
       " 2017              0             0.000  \n",
       " 2018              0             0.000  \n",
       " 2019              0             0.000  \n",
       " 2020              0             0.000  \n",
       " 2021              0             0.000  \n",
       " 2022              0             0.000  \n",
       " 2023              3             0.006  \n",
       " 2024              7             0.014  \n",
       " 2025              0             0.000  \n",
       " 2026              2             0.004  \n",
       " 2027             26             0.052  \n",
       " 2028             33             0.066  \n",
       " 2029             21             0.042  \n",
       " 2030              1             0.002  \n",
       " 2031              0             0.000  \n",
       " 2032              0             0.000  \n",
       " 2033              0             0.000  \n",
       " 2034              0             0.000  \n",
       " 2035              0             0.000  \n",
       " 2036              0             0.000  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                53             0.106  \n",
       " 1                 0             0.000  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17               21             0.042  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                0             0.000  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                0             0.000  \n",
       " 28                0             0.000  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2008              0             0.000  \n",
       " 2009              0             0.000  \n",
       " 2010              0             0.000  \n",
       " 2011              0             0.000  \n",
       " 2012              0             0.000  \n",
       " 2013              0             0.000  \n",
       " 2014              0             0.000  \n",
       " 2015              0             0.000  \n",
       " 2016              0             0.000  \n",
       " 2017              0             0.000  \n",
       " 2018              0             0.000  \n",
       " 2019              0             0.000  \n",
       " 2020              0             0.000  \n",
       " 2021              0             0.000  \n",
       " 2022              0             0.000  \n",
       " 2023              0             0.000  \n",
       " 2024              7             0.014  \n",
       " 2025              0             0.000  \n",
       " 2026              2             0.004  \n",
       " 2027             26             0.052  \n",
       " 2028             33             0.066  \n",
       " 2029             21             0.042  \n",
       " 2030              0             0.000  \n",
       " 2031              0             0.000  \n",
       " 2032              0             0.000  \n",
       " 2033              0             0.000  \n",
       " 2034              0             0.000  \n",
       " 2035              0             0.000  \n",
       " 2036              0             0.000  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0               109             0.218  \n",
       " 1               116             0.232  \n",
       " 2               187             0.374  \n",
       " 3               112             0.224  \n",
       " 4               104             0.208  \n",
       " 5               143             0.286  \n",
       " 6               127             0.254  \n",
       " 7               127             0.254  \n",
       " 8               123             0.246  \n",
       " 9               112             0.224  \n",
       " 10              121             0.242  \n",
       " 11              141             0.282  \n",
       " 12              127             0.254  \n",
       " 13              127             0.254  \n",
       " 14              153             0.306  \n",
       " 15              122             0.244  \n",
       " 16              120             0.240  \n",
       " 17              100             0.200  \n",
       " 18              114             0.228  \n",
       " 19              118             0.236  \n",
       " 20              148             0.296  \n",
       " 21              128             0.256  \n",
       " 22              111             0.222  \n",
       " 23               84             0.168  \n",
       " 24              145             0.290  \n",
       " 25              149             0.298  \n",
       " 26              120             0.240  \n",
       " 27              147             0.294  \n",
       " 28              105             0.210  \n",
       " 29              135             0.270  \n",
       " ...             ...               ...  \n",
       " 2008            104             0.208  \n",
       " 2009            129             0.258  \n",
       " 2010              0             0.000  \n",
       " 2011            135             0.270  \n",
       " 2012            130             0.260  \n",
       " 2013             81             0.162  \n",
       " 2014            148             0.296  \n",
       " 2015            146             0.292  \n",
       " 2016            155             0.310  \n",
       " 2017            151             0.302  \n",
       " 2018            164             0.328  \n",
       " 2019            163             0.326  \n",
       " 2020            138             0.276  \n",
       " 2021            135             0.270  \n",
       " 2022            129             0.258  \n",
       " 2023            135             0.270  \n",
       " 2024            112             0.224  \n",
       " 2025              0             0.000  \n",
       " 2026            145             0.290  \n",
       " 2027             94             0.188  \n",
       " 2028            107             0.214  \n",
       " 2029            140             0.280  \n",
       " 2030            122             0.244  \n",
       " 2031            126             0.252  \n",
       " 2032            134             0.268  \n",
       " 2033            153             0.306  \n",
       " 2034            134             0.268  \n",
       " 2035            130             0.260  \n",
       " 2036            111             0.222  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                74             0.148  \n",
       " 1                30             0.060  \n",
       " 2                59             0.118  \n",
       " 3                31             0.062  \n",
       " 4                46             0.092  \n",
       " 5                61             0.122  \n",
       " 6                29             0.058  \n",
       " 7                56             0.112  \n",
       " 8                54             0.108  \n",
       " 9                22             0.044  \n",
       " 10               36             0.072  \n",
       " 11               75             0.150  \n",
       " 12               66             0.132  \n",
       " 13               34             0.068  \n",
       " 14               56             0.112  \n",
       " 15               38             0.076  \n",
       " 16               25             0.050  \n",
       " 17               56             0.112  \n",
       " 18               19             0.038  \n",
       " 19               16             0.032  \n",
       " 20               61             0.122  \n",
       " 21               34             0.068  \n",
       " 22               58             0.116  \n",
       " 23               41             0.082  \n",
       " 24               39             0.078  \n",
       " 25               34             0.068  \n",
       " 26               68             0.136  \n",
       " 27               37             0.074  \n",
       " 28               38             0.076  \n",
       " 29               39             0.078  \n",
       " ...             ...               ...  \n",
       " 2008             46             0.092  \n",
       " 2009             17             0.034  \n",
       " 2010              0             0.000  \n",
       " 2011             40             0.080  \n",
       " 2012             35             0.070  \n",
       " 2013             16             0.032  \n",
       " 2014             36             0.072  \n",
       " 2015             30             0.060  \n",
       " 2016             61             0.122  \n",
       " 2017             38             0.076  \n",
       " 2018             24             0.048  \n",
       " 2019             27             0.054  \n",
       " 2020             52             0.104  \n",
       " 2021             80             0.160  \n",
       " 2022             51             0.102  \n",
       " 2023             65             0.130  \n",
       " 2024             34             0.068  \n",
       " 2025              0             0.000  \n",
       " 2026             78             0.156  \n",
       " 2027             47             0.094  \n",
       " 2028             83             0.166  \n",
       " 2029            109             0.218  \n",
       " 2030             57             0.114  \n",
       " 2031             49             0.098  \n",
       " 2032             75             0.150  \n",
       " 2033             55             0.110  \n",
       " 2034             65             0.130  \n",
       " 2035             43             0.086  \n",
       " 2036             33             0.066  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                20             0.040  \n",
       " 1                 0             0.000  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 0             0.000  \n",
       " 7                 1             0.002  \n",
       " 8                 1             0.002  \n",
       " 9                 0             0.000  \n",
       " 10                1             0.002  \n",
       " 11                1             0.002  \n",
       " 12                1             0.002  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                1             0.002  \n",
       " 16                0             0.000  \n",
       " 17                6             0.012  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                3             0.006  \n",
       " 21                0             0.000  \n",
       " 22                5             0.010  \n",
       " 23                0             0.000  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                4             0.008  \n",
       " 27                0             0.000  \n",
       " 28                0             0.000  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2008              0             0.000  \n",
       " 2009              0             0.000  \n",
       " 2010              0             0.000  \n",
       " 2011              0             0.000  \n",
       " 2012              1             0.002  \n",
       " 2013              0             0.000  \n",
       " 2014              0             0.000  \n",
       " 2015              0             0.000  \n",
       " 2016              0             0.000  \n",
       " 2017              0             0.000  \n",
       " 2018              0             0.000  \n",
       " 2019              0             0.000  \n",
       " 2020              0             0.000  \n",
       " 2021              3             0.006  \n",
       " 2022              0             0.000  \n",
       " 2023              1             0.002  \n",
       " 2024             11             0.022  \n",
       " 2025              0             0.000  \n",
       " 2026              2             0.004  \n",
       " 2027              9             0.018  \n",
       " 2028              8             0.016  \n",
       " 2029             11             0.022  \n",
       " 2030              1             0.002  \n",
       " 2031              1             0.002  \n",
       " 2032              4             0.008  \n",
       " 2033              1             0.002  \n",
       " 2034              0             0.000  \n",
       " 2035              1             0.002  \n",
       " 2036              2             0.004  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                 chrom    start      end                            name  \\\n",
       " 0     hcontig_000_003     1022     1522    gene_model_hcontig_0000_03.1   \n",
       " 1     hcontig_000_003    39749    40249   gene_model_hcontig_0000_03.10   \n",
       " 2     hcontig_000_003   488402   488902  gene_model_hcontig_0000_03.120   \n",
       " 3     hcontig_000_003   655466   655966  gene_model_hcontig_0000_03.158   \n",
       " 4     hcontig_000_003     4849     5349    gene_model_hcontig_0000_03.2   \n",
       " 5     hcontig_000_003    73712    74212   gene_model_hcontig_0000_03.20   \n",
       " 6     hcontig_000_003   118472   118972   gene_model_hcontig_0000_03.26   \n",
       " 7     hcontig_000_003  1312418  1312918  gene_model_hcontig_0000_03.314   \n",
       " 8     hcontig_000_003  1424627  1425127  gene_model_hcontig_0000_03.340   \n",
       " 9     hcontig_000_003  1699313  1699813  gene_model_hcontig_0000_03.380   \n",
       " 10    hcontig_000_003  1792279  1792779  gene_model_hcontig_0000_03.402   \n",
       " 11    hcontig_000_003  1881259  1881759  gene_model_hcontig_0000_03.419   \n",
       " 12    hcontig_000_003  1897873  1898373  gene_model_hcontig_0000_03.421   \n",
       " 13    hcontig_000_003  1904502  1905002  gene_model_hcontig_0000_03.423   \n",
       " 14    hcontig_000_003  2002834  2003334  gene_model_hcontig_0000_03.444   \n",
       " 15    hcontig_000_003  2017347  2017847  gene_model_hcontig_0000_03.450   \n",
       " 16    hcontig_000_003   369677   370177   gene_model_hcontig_0000_03.90   \n",
       " 17    hcontig_000_031    13319    13819    gene_model_hcontig_0000_31.4   \n",
       " 18    hcontig_000_050   477118   477618  gene_model_hcontig_0000_50.114   \n",
       " 19    hcontig_000_050   519662   520162  gene_model_hcontig_0000_50.122   \n",
       " 20    hcontig_000_050   611917   612417  gene_model_hcontig_0000_50.141   \n",
       " 21    hcontig_000_050   637205   637705  gene_model_hcontig_0000_50.144   \n",
       " 22    hcontig_000_050   659227   659727  gene_model_hcontig_0000_50.149   \n",
       " 23    hcontig_000_050   147057   147557   gene_model_hcontig_0000_50.34   \n",
       " 24    hcontig_000_050   353176   353676   gene_model_hcontig_0000_50.85   \n",
       " 25    hcontig_000_050   366925   367425   gene_model_hcontig_0000_50.87   \n",
       " 26    hcontig_000_050    47495    47995    gene_model_hcontig_0000_50.9   \n",
       " 27    hcontig_000_050   396033   396533   gene_model_hcontig_0000_50.93   \n",
       " 28    hcontig_000_054    55841    56341   gene_model_hcontig_0000_54.13   \n",
       " 29    hcontig_000_054   186077   186577   gene_model_hcontig_0000_54.43   \n",
       " ...               ...      ...      ...                             ...   \n",
       " 2008      pcontig_112     1431     1931        gene_model_pcontig_112.1   \n",
       " 2009      pcontig_115   109549   110049       gene_model_pcontig_115.25   \n",
       " 2010      pcontig_120    89229    89729       gene_model_pcontig_120.23   \n",
       " 2011      pcontig_130    66050    66550       gene_model_pcontig_130.20   \n",
       " 2012      pcontig_130    22481    22981        gene_model_pcontig_130.9   \n",
       " 2013      pcontig_147    43710    44210        gene_model_pcontig_147.9   \n",
       " 2014      pcontig_148    30628    31128       gene_model_pcontig_148.11   \n",
       " 2015      pcontig_148    24170    24670        gene_model_pcontig_148.9   \n",
       " 2016      pcontig_149    36248    36748       gene_model_pcontig_149.10   \n",
       " 2017      pcontig_157     9038     9538        gene_model_pcontig_157.3   \n",
       " 2018      pcontig_157    11811    12311        gene_model_pcontig_157.4   \n",
       " 2019      pcontig_157    25381    25881        gene_model_pcontig_157.8   \n",
       " 2020      pcontig_165    16197    16697        gene_model_pcontig_165.8   \n",
       " 2021      pcontig_170     4093     4593        gene_model_pcontig_170.3   \n",
       " 2022      pcontig_173     7365     7865        gene_model_pcontig_173.4   \n",
       " 2023      pcontig_173    11061    11561        gene_model_pcontig_173.5   \n",
       " 2024      pcontig_174     1124     1624        gene_model_pcontig_174.1   \n",
       " 2025      pcontig_181    37581    38081        gene_model_pcontig_181.9   \n",
       " 2026      pcontig_186    12234    12734        gene_model_pcontig_186.2   \n",
       " 2027      pcontig_186    22885    23385        gene_model_pcontig_186.6   \n",
       " 2028      pcontig_186    25642    26142        gene_model_pcontig_186.7   \n",
       " 2029      pcontig_186    30938    31438        gene_model_pcontig_186.9   \n",
       " 2030      pcontig_188     9426     9926        gene_model_pcontig_188.5   \n",
       " 2031      pcontig_188    13701    14201        gene_model_pcontig_188.6   \n",
       " 2032      pcontig_189    26708    27208        gene_model_pcontig_189.7   \n",
       " 2033      pcontig_203    18228    18728        gene_model_pcontig_203.5   \n",
       " 2034      pcontig_203    19977    20477        gene_model_pcontig_203.6   \n",
       " 2035      pcontig_207     1294     1794        gene_model_pcontig_207.1   \n",
       " 2036      pcontig_233    24903    25403        gene_model_pcontig_233.6   \n",
       " 2037      pcontig_235    24768    25268        gene_model_pcontig_235.7   \n",
       " \n",
       "       overlap_count  overlap_fraction  \n",
       " 0                20             0.040  \n",
       " 1                 0             0.000  \n",
       " 2                 0             0.000  \n",
       " 3                 0             0.000  \n",
       " 4                 0             0.000  \n",
       " 5                 0             0.000  \n",
       " 6                 0             0.000  \n",
       " 7                 0             0.000  \n",
       " 8                 0             0.000  \n",
       " 9                 0             0.000  \n",
       " 10                0             0.000  \n",
       " 11                0             0.000  \n",
       " 12                0             0.000  \n",
       " 13                0             0.000  \n",
       " 14                0             0.000  \n",
       " 15                0             0.000  \n",
       " 16                0             0.000  \n",
       " 17                6             0.012  \n",
       " 18                0             0.000  \n",
       " 19                0             0.000  \n",
       " 20                0             0.000  \n",
       " 21                0             0.000  \n",
       " 22                0             0.000  \n",
       " 23                0             0.000  \n",
       " 24                0             0.000  \n",
       " 25                0             0.000  \n",
       " 26                0             0.000  \n",
       " 27                0             0.000  \n",
       " 28                0             0.000  \n",
       " 29                0             0.000  \n",
       " ...             ...               ...  \n",
       " 2008              0             0.000  \n",
       " 2009              0             0.000  \n",
       " 2010              0             0.000  \n",
       " 2011              0             0.000  \n",
       " 2012              0             0.000  \n",
       " 2013              0             0.000  \n",
       " 2014              0             0.000  \n",
       " 2015              0             0.000  \n",
       " 2016              0             0.000  \n",
       " 2017              0             0.000  \n",
       " 2018              0             0.000  \n",
       " 2019              0             0.000  \n",
       " 2020              0             0.000  \n",
       " 2021              0             0.000  \n",
       " 2022              0             0.000  \n",
       " 2023              0             0.000  \n",
       " 2024             11             0.022  \n",
       " 2025              0             0.000  \n",
       " 2026              2             0.004  \n",
       " 2027              9             0.018  \n",
       " 2028              8             0.016  \n",
       " 2029             11             0.022  \n",
       " 2030              0             0.000  \n",
       " 2031              0             0.000  \n",
       " 2032              0             0.000  \n",
       " 2033              0             0.000  \n",
       " 2034              0             0.000  \n",
       " 2035              1             0.002  \n",
       " 2036              2             0.004  \n",
       " 2037              0             0.000  \n",
       " \n",
       " [2038 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                177             0.354  \n",
       " 1                142             0.284  \n",
       " 2                151             0.302  \n",
       " 3                113             0.226  \n",
       " 4                110             0.220  \n",
       " 5                133             0.266  \n",
       " 6                127             0.254  \n",
       " 7                152             0.304  \n",
       " 8                116             0.232  \n",
       " 9                116             0.232  \n",
       " 10               116             0.232  \n",
       " 11               119             0.238  \n",
       " 12               101             0.202  \n",
       " 13               136             0.272  \n",
       " 14               129             0.258  \n",
       " 15               117             0.234  \n",
       " 16               176             0.352  \n",
       " 17               123             0.246  \n",
       " 18               104             0.208  \n",
       " 19               169             0.338  \n",
       " 20               114             0.228  \n",
       " 21               152             0.304  \n",
       " 22               130             0.260  \n",
       " 23               113             0.226  \n",
       " 24               121             0.242  \n",
       " 25               105             0.210  \n",
       " 26                47             0.094  \n",
       " 27               115             0.230  \n",
       " 28               123             0.246  \n",
       " 29               123             0.246  \n",
       " ...              ...               ...  \n",
       " 28181             96             0.192  \n",
       " 28182            100             0.200  \n",
       " 28183            124             0.248  \n",
       " 28184             91             0.182  \n",
       " 28185             56             0.112  \n",
       " 28186             75             0.150  \n",
       " 28187             93             0.186  \n",
       " 28188            133             0.266  \n",
       " 28189            127             0.254  \n",
       " 28190            123             0.246  \n",
       " 28191            124             0.248  \n",
       " 28192            158             0.316  \n",
       " 28193            117             0.234  \n",
       " 28194            143             0.286  \n",
       " 28195            141             0.282  \n",
       " 28196            120             0.240  \n",
       " 28197            121             0.242  \n",
       " 28198            102             0.204  \n",
       " 28199             97             0.194  \n",
       " 28200            123             0.246  \n",
       " 28201            133             0.266  \n",
       " 28202            141             0.282  \n",
       " 28203            117             0.234  \n",
       " 28204              0             0.000  \n",
       " 28205            100             0.200  \n",
       " 28206            136             0.272  \n",
       " 28207             69             0.138  \n",
       " 28208             72             0.144  \n",
       " 28209            132             0.264  \n",
       " 28210            134             0.268  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 84             0.168  \n",
       " 1                112             0.224  \n",
       " 2                121             0.242  \n",
       " 3                 57             0.114  \n",
       " 4                 50             0.100  \n",
       " 5                 71             0.142  \n",
       " 6                 52             0.104  \n",
       " 7                118             0.236  \n",
       " 8                 21             0.042  \n",
       " 9                 23             0.046  \n",
       " 10                25             0.050  \n",
       " 11                25             0.050  \n",
       " 12                13             0.026  \n",
       " 13                95             0.190  \n",
       " 14                42             0.084  \n",
       " 15                55             0.110  \n",
       " 16               125             0.250  \n",
       " 17                82             0.164  \n",
       " 18                72             0.144  \n",
       " 19                 7             0.014  \n",
       " 20                63             0.126  \n",
       " 21                17             0.034  \n",
       " 22                75             0.150  \n",
       " 23                27             0.054  \n",
       " 24                24             0.048  \n",
       " 25                64             0.128  \n",
       " 26                23             0.046  \n",
       " 27                51             0.102  \n",
       " 28                41             0.082  \n",
       " 29                25             0.050  \n",
       " ...              ...               ...  \n",
       " 28181             84             0.168  \n",
       " 28182             73             0.146  \n",
       " 28183             87             0.174  \n",
       " 28184             70             0.140  \n",
       " 28185             56             0.112  \n",
       " 28186             75             0.150  \n",
       " 28187             62             0.124  \n",
       " 28188            115             0.230  \n",
       " 28189             95             0.190  \n",
       " 28190             73             0.146  \n",
       " 28191             81             0.162  \n",
       " 28192             76             0.152  \n",
       " 28193             59             0.118  \n",
       " 28194            106             0.212  \n",
       " 28195             66             0.132  \n",
       " 28196             73             0.146  \n",
       " 28197             87             0.174  \n",
       " 28198             55             0.110  \n",
       " 28199             97             0.194  \n",
       " 28200             91             0.182  \n",
       " 28201             93             0.186  \n",
       " 28202             92             0.184  \n",
       " 28203             88             0.176  \n",
       " 28204              0             0.000  \n",
       " 28205             30             0.060  \n",
       " 28206            126             0.252  \n",
       " 28207             59             0.118  \n",
       " 28208             61             0.122  \n",
       " 28209            117             0.234  \n",
       " 28210            131             0.262  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  0             0.000  \n",
       " 1                  5             0.010  \n",
       " 2                  0             0.000  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 1             0.002  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 1             0.002  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 2             0.004  \n",
       " 25                 0             0.000  \n",
       " 26                 1             0.002  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 28181              4             0.008  \n",
       " 28182              2             0.004  \n",
       " 28183              7             0.014  \n",
       " 28184              2             0.004  \n",
       " 28185             56             0.112  \n",
       " 28186             49             0.098  \n",
       " 28187              2             0.004  \n",
       " 28188              4             0.008  \n",
       " 28189              0             0.000  \n",
       " 28190              2             0.004  \n",
       " 28191              0             0.000  \n",
       " 28192              0             0.000  \n",
       " 28193              0             0.000  \n",
       " 28194             10             0.020  \n",
       " 28195              0             0.000  \n",
       " 28196              0             0.000  \n",
       " 28197              0             0.000  \n",
       " 28198              0             0.000  \n",
       " 28199              4             0.008  \n",
       " 28200              4             0.008  \n",
       " 28201              5             0.010  \n",
       " 28202              1             0.002  \n",
       " 28203              1             0.002  \n",
       " 28204              0             0.000  \n",
       " 28205             10             0.020  \n",
       " 28206             16             0.032  \n",
       " 28207             22             0.044  \n",
       " 28208             35             0.070  \n",
       " 28209             69             0.138  \n",
       " 28210             92             0.184  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  0             0.000  \n",
       " 1                  1             0.002  \n",
       " 2                  0             0.000  \n",
       " 3                  0             0.000  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 1             0.002  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 1             0.002  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 28181              0             0.000  \n",
       " 28182              0             0.000  \n",
       " 28183              0             0.000  \n",
       " 28184              2             0.004  \n",
       " 28185             56             0.112  \n",
       " 28186             49             0.098  \n",
       " 28187              2             0.004  \n",
       " 28188              1             0.002  \n",
       " 28189              0             0.000  \n",
       " 28190              0             0.000  \n",
       " 28191              0             0.000  \n",
       " 28192              0             0.000  \n",
       " 28193              0             0.000  \n",
       " 28194             10             0.020  \n",
       " 28195              0             0.000  \n",
       " 28196              0             0.000  \n",
       " 28197              0             0.000  \n",
       " 28198              0             0.000  \n",
       " 28199              4             0.008  \n",
       " 28200              4             0.008  \n",
       " 28201              5             0.010  \n",
       " 28202              0             0.000  \n",
       " 28203              1             0.002  \n",
       " 28204              0             0.000  \n",
       " 28205             10             0.020  \n",
       " 28206             16             0.032  \n",
       " 28207             22             0.044  \n",
       " 28208             35             0.070  \n",
       " 28209             69             0.138  \n",
       " 28210             92             0.184  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                139             0.278  \n",
       " 1                134             0.268  \n",
       " 2                134             0.268  \n",
       " 3                129             0.258  \n",
       " 4                142             0.284  \n",
       " 5                144             0.288  \n",
       " 6                132             0.264  \n",
       " 7                120             0.240  \n",
       " 8                140             0.280  \n",
       " 9                138             0.276  \n",
       " 10               142             0.284  \n",
       " 11               142             0.284  \n",
       " 12               157             0.314  \n",
       " 13               118             0.236  \n",
       " 14               156             0.312  \n",
       " 15               154             0.308  \n",
       " 16               107             0.214  \n",
       " 17               122             0.244  \n",
       " 18               139             0.278  \n",
       " 19               122             0.244  \n",
       " 20               133             0.266  \n",
       " 21               113             0.226  \n",
       " 22               149             0.298  \n",
       " 23               146             0.292  \n",
       " 24               166             0.332  \n",
       " 25               167             0.334  \n",
       " 26               221             0.442  \n",
       " 27               135             0.270  \n",
       " 28               142             0.284  \n",
       " 29               141             0.282  \n",
       " ...              ...               ...  \n",
       " 28181            157             0.314  \n",
       " 28182            151             0.302  \n",
       " 28183            146             0.292  \n",
       " 28184            141             0.282  \n",
       " 28185             74             0.148  \n",
       " 28186             56             0.112  \n",
       " 28187            127             0.254  \n",
       " 28188            152             0.304  \n",
       " 28189            139             0.278  \n",
       " 28190            163             0.326  \n",
       " 28191            128             0.256  \n",
       " 28192            115             0.230  \n",
       " 28193            136             0.272  \n",
       " 28194            132             0.264  \n",
       " 28195            137             0.274  \n",
       " 28196            144             0.288  \n",
       " 28197            160             0.320  \n",
       " 28198            161             0.322  \n",
       " 28199            117             0.234  \n",
       " 28200            113             0.226  \n",
       " 28201            147             0.294  \n",
       " 28202            131             0.262  \n",
       " 28203            154             0.308  \n",
       " 28204              0             0.000  \n",
       " 28205             93             0.186  \n",
       " 28206             95             0.190  \n",
       " 28207            111             0.222  \n",
       " 28208            115             0.230  \n",
       " 28209             88             0.176  \n",
       " 28210             56             0.112  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 45             0.090  \n",
       " 1                 66             0.132  \n",
       " 2                 54             0.108  \n",
       " 3                 80             0.160  \n",
       " 4                 86             0.172  \n",
       " 5                 44             0.088  \n",
       " 6                 83             0.166  \n",
       " 7                 36             0.072  \n",
       " 8                 67             0.134  \n",
       " 9                 53             0.106  \n",
       " 10                66             0.132  \n",
       " 11                38             0.076  \n",
       " 12                34             0.068  \n",
       " 13                51             0.102  \n",
       " 14                25             0.050  \n",
       " 15               107             0.214  \n",
       " 16                34             0.068  \n",
       " 17                48             0.096  \n",
       " 18                82             0.164  \n",
       " 19                13             0.026  \n",
       " 20                45             0.090  \n",
       " 21                17             0.034  \n",
       " 22                36             0.072  \n",
       " 23                38             0.076  \n",
       " 24                36             0.072  \n",
       " 25                30             0.060  \n",
       " 26               118             0.236  \n",
       " 27                59             0.118  \n",
       " 28                55             0.110  \n",
       " 29                28             0.056  \n",
       " ...              ...               ...  \n",
       " 28181             66             0.132  \n",
       " 28182             57             0.114  \n",
       " 28183             46             0.092  \n",
       " 28184            103             0.206  \n",
       " 28185             74             0.148  \n",
       " 28186             56             0.112  \n",
       " 28187             84             0.168  \n",
       " 28188             63             0.126  \n",
       " 28189             52             0.104  \n",
       " 28190             77             0.154  \n",
       " 28191             43             0.086  \n",
       " 28192             36             0.072  \n",
       " 28193             40             0.080  \n",
       " 28194             59             0.118  \n",
       " 28195             57             0.114  \n",
       " 28196             81             0.162  \n",
       " 28197             50             0.100  \n",
       " 28198             64             0.128  \n",
       " 28199            117             0.234  \n",
       " 28200             55             0.110  \n",
       " 28201             57             0.114  \n",
       " 28202             44             0.088  \n",
       " 28203             84             0.168  \n",
       " 28204              0             0.000  \n",
       " 28205             20             0.040  \n",
       " 28206             51             0.102  \n",
       " 28207             69             0.138  \n",
       " 28208             66             0.132  \n",
       " 28209             51             0.102  \n",
       " 28210             46             0.092  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  2             0.004  \n",
       " 1                  5             0.010  \n",
       " 2                  0             0.000  \n",
       " 3                  2             0.004  \n",
       " 4                  3             0.006  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  2             0.004  \n",
       " 10                 2             0.004  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 1             0.002  \n",
       " 25                 0             0.000  \n",
       " 26                 9             0.018  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 28181              0             0.000  \n",
       " 28182              0             0.000  \n",
       " 28183              2             0.004  \n",
       " 28184             12             0.024  \n",
       " 28185             74             0.148  \n",
       " 28186             38             0.076  \n",
       " 28187             13             0.026  \n",
       " 28188              1             0.002  \n",
       " 28189              1             0.002  \n",
       " 28190              2             0.004  \n",
       " 28191              0             0.000  \n",
       " 28192              0             0.000  \n",
       " 28193              0             0.000  \n",
       " 28194              2             0.004  \n",
       " 28195              0             0.000  \n",
       " 28196              0             0.000  \n",
       " 28197              1             0.002  \n",
       " 28198              0             0.000  \n",
       " 28199             29             0.058  \n",
       " 28200              4             0.008  \n",
       " 28201              1             0.002  \n",
       " 28202              1             0.002  \n",
       " 28203              1             0.002  \n",
       " 28204              0             0.000  \n",
       " 28205              3             0.006  \n",
       " 28206              2             0.004  \n",
       " 28207             23             0.046  \n",
       " 28208             29             0.058  \n",
       " 28209             18             0.036  \n",
       " 28210             14             0.028  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                           name  \\\n",
       " 0      hcontig_000_003    7889    8389   gene_model_hcontig_0000_03.3   \n",
       " 1      hcontig_000_003   11922   12422   gene_model_hcontig_0000_03.4   \n",
       " 2      hcontig_000_003   27259   27759   gene_model_hcontig_0000_03.7   \n",
       " 3      hcontig_000_003   33454   33954   gene_model_hcontig_0000_03.8   \n",
       " 4      hcontig_000_003   36694   37194   gene_model_hcontig_0000_03.9   \n",
       " 5      hcontig_000_003   43215   43715  gene_model_hcontig_0000_03.11   \n",
       " 6      hcontig_000_003   45861   46361  gene_model_hcontig_0000_03.12   \n",
       " 7      hcontig_000_003   48953   49453  gene_model_hcontig_0000_03.13   \n",
       " 8      hcontig_000_003   52047   52547  gene_model_hcontig_0000_03.14   \n",
       " 9      hcontig_000_003   54649   55149  gene_model_hcontig_0000_03.15   \n",
       " 10     hcontig_000_003   57706   58206  gene_model_hcontig_0000_03.16   \n",
       " 11     hcontig_000_003   59932   60432  gene_model_hcontig_0000_03.17   \n",
       " 12     hcontig_000_003   62498   62998  gene_model_hcontig_0000_03.18   \n",
       " 13     hcontig_000_003   65506   66006  gene_model_hcontig_0000_03.19   \n",
       " 14     hcontig_000_003   97449   97949  gene_model_hcontig_0000_03.23   \n",
       " 15     hcontig_000_003  102059  102559  gene_model_hcontig_0000_03.24   \n",
       " 16     hcontig_000_003  121489  121989  gene_model_hcontig_0000_03.27   \n",
       " 17     hcontig_000_003  125841  126341  gene_model_hcontig_0000_03.28   \n",
       " 18     hcontig_000_003  131016  131516  gene_model_hcontig_0000_03.29   \n",
       " 19     hcontig_000_003  132650  133150  gene_model_hcontig_0000_03.30   \n",
       " 20     hcontig_000_003  134537  135037  gene_model_hcontig_0000_03.31   \n",
       " 21     hcontig_000_003  136805  137305  gene_model_hcontig_0000_03.32   \n",
       " 22     hcontig_000_003  138521  139021  gene_model_hcontig_0000_03.33   \n",
       " 23     hcontig_000_003  141557  142057  gene_model_hcontig_0000_03.34   \n",
       " 24     hcontig_000_003  146950  147450  gene_model_hcontig_0000_03.35   \n",
       " 25     hcontig_000_003  154162  154662  gene_model_hcontig_0000_03.36   \n",
       " 26     hcontig_000_003  158136  158636  gene_model_hcontig_0000_03.37   \n",
       " 27     hcontig_000_003  164450  164950  gene_model_hcontig_0000_03.38   \n",
       " 28     hcontig_000_003  166538  167038  gene_model_hcontig_0000_03.39   \n",
       " 29     hcontig_000_003  169767  170267  gene_model_hcontig_0000_03.40   \n",
       " ...                ...     ...     ...                            ...   \n",
       " 28181      pcontig_223   15475   15975       gene_model_pcontig_223.7   \n",
       " 28182      pcontig_223   16555   17055       gene_model_pcontig_223.8   \n",
       " 28183      pcontig_223   21224   21724       gene_model_pcontig_223.9   \n",
       " 28184      pcontig_223   25049   25549      gene_model_pcontig_223.10   \n",
       " 28185      pcontig_225    2066    2566       gene_model_pcontig_225.1   \n",
       " 28186      pcontig_225    2657    3157       gene_model_pcontig_225.2   \n",
       " 28187      pcontig_225    3998    4498       gene_model_pcontig_225.3   \n",
       " 28188      pcontig_225   13374   13874       gene_model_pcontig_225.4   \n",
       " 28189      pcontig_225   17271   17771       gene_model_pcontig_225.5   \n",
       " 28190      pcontig_225   18671   19171       gene_model_pcontig_225.6   \n",
       " 28191      pcontig_225   22307   22807       gene_model_pcontig_225.7   \n",
       " 28192      pcontig_225   23726   24226       gene_model_pcontig_225.8   \n",
       " 28193      pcontig_225   24708   25208       gene_model_pcontig_225.9   \n",
       " 28194      pcontig_233    2633    3133       gene_model_pcontig_233.1   \n",
       " 28195      pcontig_233    9598   10098       gene_model_pcontig_233.2   \n",
       " 28196      pcontig_233   11955   12455       gene_model_pcontig_233.3   \n",
       " 28197      pcontig_233   15137   15637       gene_model_pcontig_233.4   \n",
       " 28198      pcontig_233   19085   19585       gene_model_pcontig_233.5   \n",
       " 28199      pcontig_235    1209    1709       gene_model_pcontig_235.1   \n",
       " 28200      pcontig_235    3517    4017       gene_model_pcontig_235.2   \n",
       " 28201      pcontig_235    9863   10363       gene_model_pcontig_235.3   \n",
       " 28202      pcontig_235   12336   12836       gene_model_pcontig_235.4   \n",
       " 28203      pcontig_235   20192   20692       gene_model_pcontig_235.6   \n",
       " 28204      pcontig_248    1242    1742       gene_model_pcontig_248.1   \n",
       " 28205      pcontig_248    3541    4041       gene_model_pcontig_248.2   \n",
       " 28206      pcontig_248    8183    8683       gene_model_pcontig_248.3   \n",
       " 28207      pcontig_248    9497    9997       gene_model_pcontig_248.4   \n",
       " 28208      pcontig_248   11216   11716       gene_model_pcontig_248.5   \n",
       " 28209      pcontig_248   13257   13757       gene_model_pcontig_248.6   \n",
       " 28210      pcontig_248   17155   17655       gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                  2             0.004  \n",
       " 1                  1             0.002  \n",
       " 2                  0             0.000  \n",
       " 3                  2             0.004  \n",
       " 4                  0             0.000  \n",
       " 5                  0             0.000  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 2             0.004  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 0             0.000  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 2             0.004  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 28181              0             0.000  \n",
       " 28182              0             0.000  \n",
       " 28183              0             0.000  \n",
       " 28184             12             0.024  \n",
       " 28185             74             0.148  \n",
       " 28186             38             0.076  \n",
       " 28187             13             0.026  \n",
       " 28188              0             0.000  \n",
       " 28189              0             0.000  \n",
       " 28190              0             0.000  \n",
       " 28191              0             0.000  \n",
       " 28192              0             0.000  \n",
       " 28193              0             0.000  \n",
       " 28194              2             0.004  \n",
       " 28195              0             0.000  \n",
       " 28196              0             0.000  \n",
       " 28197              0             0.000  \n",
       " 28198              0             0.000  \n",
       " 28199             29             0.058  \n",
       " 28200              4             0.008  \n",
       " 28201              1             0.002  \n",
       " 28202              1             0.002  \n",
       " 28203              1             0.002  \n",
       " 28204              0             0.000  \n",
       " 28205              3             0.006  \n",
       " 28206              2             0.004  \n",
       " 28207             23             0.046  \n",
       " 28208             29             0.058  \n",
       " 28209             18             0.036  \n",
       " 28210             14             0.028  \n",
       " \n",
       " [28211 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                131             0.262  \n",
       " 1                161             0.322  \n",
       " 2                177             0.354  \n",
       " 3                142             0.284  \n",
       " 4                 93             0.186  \n",
       " 5                112             0.224  \n",
       " 6                151             0.302  \n",
       " 7                113             0.226  \n",
       " 8                110             0.220  \n",
       " 9                143             0.286  \n",
       " 10               133             0.266  \n",
       " 11               127             0.254  \n",
       " 12               152             0.304  \n",
       " 13               116             0.232  \n",
       " 14               116             0.232  \n",
       " 15               116             0.232  \n",
       " 16               119             0.238  \n",
       " 17               101             0.202  \n",
       " 18               136             0.272  \n",
       " 19               112             0.224  \n",
       " 20               123             0.246  \n",
       " 21                98             0.196  \n",
       " 22               129             0.258  \n",
       " 23               117             0.234  \n",
       " 24               155             0.310  \n",
       " 25               120             0.240  \n",
       " 26               176             0.352  \n",
       " 27               123             0.246  \n",
       " 28               104             0.208  \n",
       " 29               169             0.338  \n",
       " ...              ...               ...  \n",
       " 36517            133             0.266  \n",
       " 36518            127             0.254  \n",
       " 36519            123             0.246  \n",
       " 36520            124             0.248  \n",
       " 36521            158             0.316  \n",
       " 36522            117             0.234  \n",
       " 36523              6             0.012  \n",
       " 36524            129             0.258  \n",
       " 36525            138             0.276  \n",
       " 36526            157             0.314  \n",
       " 36527            143             0.286  \n",
       " 36528            141             0.282  \n",
       " 36529            120             0.240  \n",
       " 36530            121             0.242  \n",
       " 36531            102             0.204  \n",
       " 36532            142             0.284  \n",
       " 36533             97             0.194  \n",
       " 36534            123             0.246  \n",
       " 36535            133             0.266  \n",
       " 36536            141             0.282  \n",
       " 36537            137             0.274  \n",
       " 36538            117             0.234  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541            100             0.200  \n",
       " 36542            136             0.272  \n",
       " 36543             69             0.138  \n",
       " 36544             72             0.144  \n",
       " 36545            132             0.264  \n",
       " 36546            134             0.268  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                119             0.238  \n",
       " 1                102             0.204  \n",
       " 2                 84             0.168  \n",
       " 3                112             0.224  \n",
       " 4                 66             0.132  \n",
       " 5                 48             0.096  \n",
       " 6                121             0.242  \n",
       " 7                 57             0.114  \n",
       " 8                 50             0.100  \n",
       " 9                 86             0.172  \n",
       " 10                71             0.142  \n",
       " 11                52             0.104  \n",
       " 12               118             0.236  \n",
       " 13                21             0.042  \n",
       " 14                23             0.046  \n",
       " 15                25             0.050  \n",
       " 16                25             0.050  \n",
       " 17                13             0.026  \n",
       " 18                95             0.190  \n",
       " 19                45             0.090  \n",
       " 20                82             0.164  \n",
       " 21                68             0.136  \n",
       " 22                42             0.084  \n",
       " 23                55             0.110  \n",
       " 24               121             0.242  \n",
       " 25                76             0.152  \n",
       " 26               125             0.250  \n",
       " 27                82             0.164  \n",
       " 28                72             0.144  \n",
       " 29                 7             0.014  \n",
       " ...              ...               ...  \n",
       " 36517            115             0.230  \n",
       " 36518             95             0.190  \n",
       " 36519             73             0.146  \n",
       " 36520             81             0.162  \n",
       " 36521             76             0.152  \n",
       " 36522             59             0.118  \n",
       " 36523              6             0.012  \n",
       " 36524             69             0.138  \n",
       " 36525             58             0.116  \n",
       " 36526             71             0.142  \n",
       " 36527            106             0.212  \n",
       " 36528             66             0.132  \n",
       " 36529             73             0.146  \n",
       " 36530             87             0.174  \n",
       " 36531             55             0.110  \n",
       " 36532             29             0.058  \n",
       " 36533             97             0.194  \n",
       " 36534             91             0.182  \n",
       " 36535             93             0.186  \n",
       " 36536             92             0.184  \n",
       " 36537             69             0.138  \n",
       " 36538             88             0.176  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541             30             0.060  \n",
       " 36542            126             0.252  \n",
       " 36543             59             0.118  \n",
       " 36544             61             0.122  \n",
       " 36545            117             0.234  \n",
       " 36546            131             0.262  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 53             0.106  \n",
       " 1                  0             0.000  \n",
       " 2                  0             0.000  \n",
       " 3                  5             0.010  \n",
       " 4                  2             0.004  \n",
       " 5                  2             0.004  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  1             0.002  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 1             0.002  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 3             0.006  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 36517              4             0.008  \n",
       " 36518              0             0.000  \n",
       " 36519              2             0.004  \n",
       " 36520              0             0.000  \n",
       " 36521              0             0.000  \n",
       " 36522              0             0.000  \n",
       " 36523              6             0.012  \n",
       " 36524              0             0.000  \n",
       " 36525              2             0.004  \n",
       " 36526              2             0.004  \n",
       " 36527             10             0.020  \n",
       " 36528              0             0.000  \n",
       " 36529              0             0.000  \n",
       " 36530              0             0.000  \n",
       " 36531              0             0.000  \n",
       " 36532              0             0.000  \n",
       " 36533              4             0.008  \n",
       " 36534              4             0.008  \n",
       " 36535              5             0.010  \n",
       " 36536              1             0.002  \n",
       " 36537              0             0.000  \n",
       " 36538              1             0.002  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541             10             0.020  \n",
       " 36542             16             0.032  \n",
       " 36543             22             0.044  \n",
       " 36544             35             0.070  \n",
       " 36545             69             0.138  \n",
       " 36546             92             0.184  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 53             0.106  \n",
       " 1                  0             0.000  \n",
       " 2                  0             0.000  \n",
       " 3                  1             0.002  \n",
       " 4                  0             0.000  \n",
       " 5                  1             0.002  \n",
       " 6                  0             0.000  \n",
       " 7                  0             0.000  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 1             0.002  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 36517              1             0.002  \n",
       " 36518              0             0.000  \n",
       " 36519              0             0.000  \n",
       " 36520              0             0.000  \n",
       " 36521              0             0.000  \n",
       " 36522              0             0.000  \n",
       " 36523              6             0.012  \n",
       " 36524              0             0.000  \n",
       " 36525              0             0.000  \n",
       " 36526              0             0.000  \n",
       " 36527             10             0.020  \n",
       " 36528              0             0.000  \n",
       " 36529              0             0.000  \n",
       " 36530              0             0.000  \n",
       " 36531              0             0.000  \n",
       " 36532              0             0.000  \n",
       " 36533              4             0.008  \n",
       " 36534              4             0.008  \n",
       " 36535              5             0.010  \n",
       " 36536              0             0.000  \n",
       " 36537              0             0.000  \n",
       " 36538              1             0.002  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541             10             0.020  \n",
       " 36542             16             0.032  \n",
       " 36543             22             0.044  \n",
       " 36544             35             0.070  \n",
       " 36545             69             0.138  \n",
       " 36546             92             0.184  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                109             0.218  \n",
       " 1                104             0.208  \n",
       " 2                139             0.278  \n",
       " 3                134             0.268  \n",
       " 4                187             0.374  \n",
       " 5                174             0.348  \n",
       " 6                134             0.268  \n",
       " 7                129             0.258  \n",
       " 8                142             0.284  \n",
       " 9                116             0.232  \n",
       " 10               144             0.288  \n",
       " 11               132             0.264  \n",
       " 12               120             0.240  \n",
       " 13               140             0.280  \n",
       " 14               138             0.276  \n",
       " 15               142             0.284  \n",
       " 16               142             0.284  \n",
       " 17               157             0.314  \n",
       " 18               118             0.236  \n",
       " 19               143             0.286  \n",
       " 20               150             0.300  \n",
       " 21               163             0.326  \n",
       " 22               156             0.312  \n",
       " 23               154             0.308  \n",
       " 24               142             0.284  \n",
       " 25               127             0.254  \n",
       " 26               107             0.214  \n",
       " 27               122             0.244  \n",
       " 28               139             0.278  \n",
       " 29               122             0.244  \n",
       " ...              ...               ...  \n",
       " 36517            152             0.304  \n",
       " 36518            139             0.278  \n",
       " 36519            163             0.326  \n",
       " 36520            128             0.256  \n",
       " 36521            115             0.230  \n",
       " 36522            136             0.272  \n",
       " 36523              4             0.008  \n",
       " 36524            160             0.320  \n",
       " 36525            144             0.288  \n",
       " 36526            112             0.224  \n",
       " 36527            132             0.264  \n",
       " 36528            137             0.274  \n",
       " 36529            144             0.288  \n",
       " 36530            160             0.320  \n",
       " 36531            161             0.322  \n",
       " 36532            111             0.222  \n",
       " 36533            117             0.234  \n",
       " 36534            113             0.226  \n",
       " 36535            147             0.294  \n",
       " 36536            131             0.262  \n",
       " 36537            101             0.202  \n",
       " 36538            154             0.308  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541             93             0.186  \n",
       " 36542             95             0.190  \n",
       " 36543            111             0.222  \n",
       " 36544            115             0.230  \n",
       " 36545             88             0.176  \n",
       " 36546             56             0.112  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 74             0.148  \n",
       " 1                 46             0.092  \n",
       " 2                 45             0.090  \n",
       " 3                 66             0.132  \n",
       " 4                125             0.250  \n",
       " 5                104             0.208  \n",
       " 6                 54             0.108  \n",
       " 7                 80             0.160  \n",
       " 8                 86             0.172  \n",
       " 9                 30             0.060  \n",
       " 10                44             0.088  \n",
       " 11                83             0.166  \n",
       " 12                36             0.072  \n",
       " 13                67             0.134  \n",
       " 14                53             0.106  \n",
       " 15                66             0.132  \n",
       " 16                38             0.076  \n",
       " 17                34             0.068  \n",
       " 18                51             0.102  \n",
       " 19                61             0.122  \n",
       " 20                55             0.110  \n",
       " 21                47             0.094  \n",
       " 22                25             0.050  \n",
       " 23               107             0.214  \n",
       " 24                31             0.062  \n",
       " 25                29             0.058  \n",
       " 26                34             0.068  \n",
       " 27                48             0.096  \n",
       " 28                82             0.164  \n",
       " 29                13             0.026  \n",
       " ...              ...               ...  \n",
       " 36517             63             0.126  \n",
       " 36518             52             0.104  \n",
       " 36519             77             0.154  \n",
       " 36520             43             0.086  \n",
       " 36521             36             0.072  \n",
       " 36522             40             0.080  \n",
       " 36523              4             0.008  \n",
       " 36524             98             0.196  \n",
       " 36525            104             0.208  \n",
       " 36526             79             0.158  \n",
       " 36527             59             0.118  \n",
       " 36528             57             0.114  \n",
       " 36529             81             0.162  \n",
       " 36530             50             0.100  \n",
       " 36531             64             0.128  \n",
       " 36532             33             0.066  \n",
       " 36533            117             0.234  \n",
       " 36534             55             0.110  \n",
       " 36535             57             0.114  \n",
       " 36536             44             0.088  \n",
       " 36537             35             0.070  \n",
       " 36538             84             0.168  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541             20             0.040  \n",
       " 36542             51             0.102  \n",
       " 36543             69             0.138  \n",
       " 36544             66             0.132  \n",
       " 36545             51             0.102  \n",
       " 36546             46             0.092  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 20             0.040  \n",
       " 1                  0             0.000  \n",
       " 2                  2             0.004  \n",
       " 3                  5             0.010  \n",
       " 4                  7             0.014  \n",
       " 5                  4             0.008  \n",
       " 6                  0             0.000  \n",
       " 7                  2             0.004  \n",
       " 8                  3             0.006  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 2             0.004  \n",
       " 15                 2             0.004  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 36517              1             0.002  \n",
       " 36518              1             0.002  \n",
       " 36519              2             0.004  \n",
       " 36520              0             0.000  \n",
       " 36521              0             0.000  \n",
       " 36522              0             0.000  \n",
       " 36523              4             0.008  \n",
       " 36524              0             0.000  \n",
       " 36525              2             0.004  \n",
       " 36526              3             0.006  \n",
       " 36527              2             0.004  \n",
       " 36528              0             0.000  \n",
       " 36529              0             0.000  \n",
       " 36530              1             0.002  \n",
       " 36531              0             0.000  \n",
       " 36532              2             0.004  \n",
       " 36533             29             0.058  \n",
       " 36534              4             0.008  \n",
       " 36535              1             0.002  \n",
       " 36536              1             0.002  \n",
       " 36537              2             0.004  \n",
       " 36538              1             0.002  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541              3             0.006  \n",
       " 36542              2             0.004  \n",
       " 36543             23             0.046  \n",
       " 36544             29             0.058  \n",
       " 36545             18             0.036  \n",
       " 36546             14             0.028  \n",
       " \n",
       " [36547 rows x 6 columns],\n",
       " 'Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed':                  chrom   start     end                                 name  \\\n",
       " 0      hcontig_000_003    1022    1522         gene_model_hcontig_0000_03.1   \n",
       " 1      hcontig_000_003    4849    5349         gene_model_hcontig_0000_03.2   \n",
       " 2      hcontig_000_003    7889    8389         gene_model_hcontig_0000_03.3   \n",
       " 3      hcontig_000_003   11922   12422         gene_model_hcontig_0000_03.4   \n",
       " 4      hcontig_000_003   14562   15062   EVM prediction%2hcontig_0000_003.5   \n",
       " 5      hcontig_000_003   17795   18295   EVM prediction%2hcontig_0000_003.6   \n",
       " 6      hcontig_000_003   27259   27759         gene_model_hcontig_0000_03.7   \n",
       " 7      hcontig_000_003   33454   33954         gene_model_hcontig_0000_03.8   \n",
       " 8      hcontig_000_003   36694   37194         gene_model_hcontig_0000_03.9   \n",
       " 9      hcontig_000_003   39749   40249        gene_model_hcontig_0000_03.10   \n",
       " 10     hcontig_000_003   43215   43715        gene_model_hcontig_0000_03.11   \n",
       " 11     hcontig_000_003   45861   46361        gene_model_hcontig_0000_03.12   \n",
       " 12     hcontig_000_003   48953   49453        gene_model_hcontig_0000_03.13   \n",
       " 13     hcontig_000_003   52047   52547        gene_model_hcontig_0000_03.14   \n",
       " 14     hcontig_000_003   54649   55149        gene_model_hcontig_0000_03.15   \n",
       " 15     hcontig_000_003   57706   58206        gene_model_hcontig_0000_03.16   \n",
       " 16     hcontig_000_003   59932   60432        gene_model_hcontig_0000_03.17   \n",
       " 17     hcontig_000_003   62498   62998        gene_model_hcontig_0000_03.18   \n",
       " 18     hcontig_000_003   65506   66006        gene_model_hcontig_0000_03.19   \n",
       " 19     hcontig_000_003   73712   74212        gene_model_hcontig_0000_03.20   \n",
       " 20     hcontig_000_003   80525   81025  EVM prediction%2hcontig_0000_003.21   \n",
       " 21     hcontig_000_003   81736   82236  EVM prediction%2hcontig_0000_003.22   \n",
       " 22     hcontig_000_003   97449   97949        gene_model_hcontig_0000_03.23   \n",
       " 23     hcontig_000_003  102059  102559        gene_model_hcontig_0000_03.24   \n",
       " 24     hcontig_000_003  108981  109481  EVM prediction%2hcontig_0000_003.25   \n",
       " 25     hcontig_000_003  118472  118972        gene_model_hcontig_0000_03.26   \n",
       " 26     hcontig_000_003  121489  121989        gene_model_hcontig_0000_03.27   \n",
       " 27     hcontig_000_003  125841  126341        gene_model_hcontig_0000_03.28   \n",
       " 28     hcontig_000_003  131016  131516        gene_model_hcontig_0000_03.29   \n",
       " 29     hcontig_000_003  132650  133150        gene_model_hcontig_0000_03.30   \n",
       " ...                ...     ...     ...                                  ...   \n",
       " 36517      pcontig_225   13374   13874             gene_model_pcontig_225.4   \n",
       " 36518      pcontig_225   17271   17771             gene_model_pcontig_225.5   \n",
       " 36519      pcontig_225   18671   19171             gene_model_pcontig_225.6   \n",
       " 36520      pcontig_225   22307   22807             gene_model_pcontig_225.7   \n",
       " 36521      pcontig_225   23726   24226             gene_model_pcontig_225.8   \n",
       " 36522      pcontig_225   24708   25208             gene_model_pcontig_225.9   \n",
       " 36523      pcontig_230     236     736   EVM prediction%2hcontig_0230_001.1   \n",
       " 36524      pcontig_230   23876   24376   EVM prediction%2hcontig_0230_001.2   \n",
       " 36525      pcontig_231   22945   23445       EVM prediction%2pcontig_0231.1   \n",
       " 36526      pcontig_231   24538   25038       EVM prediction%2pcontig_0231.2   \n",
       " 36527      pcontig_233    2633    3133             gene_model_pcontig_233.1   \n",
       " 36528      pcontig_233    9598   10098             gene_model_pcontig_233.2   \n",
       " 36529      pcontig_233   11955   12455             gene_model_pcontig_233.3   \n",
       " 36530      pcontig_233   15137   15637             gene_model_pcontig_233.4   \n",
       " 36531      pcontig_233   19085   19585             gene_model_pcontig_233.5   \n",
       " 36532      pcontig_233   24903   25403             gene_model_pcontig_233.6   \n",
       " 36533      pcontig_235    1209    1709             gene_model_pcontig_235.1   \n",
       " 36534      pcontig_235    3517    4017             gene_model_pcontig_235.2   \n",
       " 36535      pcontig_235    9863   10363             gene_model_pcontig_235.3   \n",
       " 36536      pcontig_235   12336   12836             gene_model_pcontig_235.4   \n",
       " 36537      pcontig_235   13217   13717       EVM prediction%2pcontig_0235.5   \n",
       " 36538      pcontig_235   20192   20692             gene_model_pcontig_235.6   \n",
       " 36539      pcontig_235   24768   25268             gene_model_pcontig_235.7   \n",
       " 36540      pcontig_248    1242    1742             gene_model_pcontig_248.1   \n",
       " 36541      pcontig_248    3541    4041             gene_model_pcontig_248.2   \n",
       " 36542      pcontig_248    8183    8683             gene_model_pcontig_248.3   \n",
       " 36543      pcontig_248    9497    9997             gene_model_pcontig_248.4   \n",
       " 36544      pcontig_248   11216   11716             gene_model_pcontig_248.5   \n",
       " 36545      pcontig_248   13257   13757             gene_model_pcontig_248.6   \n",
       " 36546      pcontig_248   17155   17655             gene_model_pcontig_248.7   \n",
       " \n",
       "        overlap_count  overlap_fraction  \n",
       " 0                 20             0.040  \n",
       " 1                  0             0.000  \n",
       " 2                  2             0.004  \n",
       " 3                  1             0.002  \n",
       " 4                  1             0.002  \n",
       " 5                  1             0.002  \n",
       " 6                  0             0.000  \n",
       " 7                  2             0.004  \n",
       " 8                  0             0.000  \n",
       " 9                  0             0.000  \n",
       " 10                 0             0.000  \n",
       " 11                 0             0.000  \n",
       " 12                 0             0.000  \n",
       " 13                 0             0.000  \n",
       " 14                 0             0.000  \n",
       " 15                 2             0.004  \n",
       " 16                 0             0.000  \n",
       " 17                 0             0.000  \n",
       " 18                 0             0.000  \n",
       " 19                 0             0.000  \n",
       " 20                 0             0.000  \n",
       " 21                 0             0.000  \n",
       " 22                 0             0.000  \n",
       " 23                 0             0.000  \n",
       " 24                 0             0.000  \n",
       " 25                 0             0.000  \n",
       " 26                 0             0.000  \n",
       " 27                 0             0.000  \n",
       " 28                 0             0.000  \n",
       " 29                 0             0.000  \n",
       " ...              ...               ...  \n",
       " 36517              0             0.000  \n",
       " 36518              0             0.000  \n",
       " 36519              0             0.000  \n",
       " 36520              0             0.000  \n",
       " 36521              0             0.000  \n",
       " 36522              0             0.000  \n",
       " 36523              4             0.008  \n",
       " 36524              0             0.000  \n",
       " 36525              0             0.000  \n",
       " 36526              2             0.004  \n",
       " 36527              2             0.004  \n",
       " 36528              0             0.000  \n",
       " 36529              0             0.000  \n",
       " 36530              0             0.000  \n",
       " 36531              0             0.000  \n",
       " 36532              2             0.004  \n",
       " 36533             29             0.058  \n",
       " 36534              4             0.008  \n",
       " 36535              1             0.002  \n",
       " 36536              1             0.002  \n",
       " 36537              2             0.004  \n",
       " 36538              1             0.002  \n",
       " 36539              0             0.000  \n",
       " 36540              0             0.000  \n",
       " 36541              3             0.006  \n",
       " 36542              2             0.004  \n",
       " 36543             23             0.046  \n",
       " 36544             29             0.058  \n",
       " 36545             18             0.036  \n",
       " 36546             14             0.028  \n",
       " \n",
       " [36547 rows x 6 columns]}"
      ]
     },
     "execution_count": 2022,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tss_overlap_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#14c4ff'> 6.D Improving biological significance of gene coverage files. <span/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <span style='color:#14c4ff'> 6.E Testing statistical significance of gene coverage files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 6mA: Wilcoxon for each cutoff, see if there's a difference"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "wilcoxon_coverage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2024,
   "metadata": {},
   "outputs": [],
   "source": [
    "def coverage_wilcoxon_same_key(obs_df_dict, exp_df_dict):\n",
    "    \"\"\"This function returns a dictionary of Wilcoxon statistic and p-value for a test of observed and randomised sites.\"\"\"\n",
    "    wilcoxon_dict = {}\n",
    "    for okey, ovalue in obs_df_dict.items():\n",
    "        for ekey, evalue in exp_df_dict.items():\n",
    "            if okey == ekey:\n",
    "                o_df = ovalue\n",
    "                e_df = evalue\n",
    "                obs = o_df['overlap_fraction']\n",
    "                exp = e_df['overlap_fraction']\n",
    "                stat, p = wilcoxon(obs, exp)\n",
    "                wilcoxon_dict[okey] = stat, p\n",
    "    return wilcoxon_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2039,
   "metadata": {},
   "outputs": [],
   "source": [
    "tss_wilcoxon = coverage_wilcoxon_same_key(tss_overlap_dict, tss_rand_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2040,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_h_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_p_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_p_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.non_effectors.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_h_ctg_combined_sorted_anno.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.TE.sorted.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.00.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.90.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.99.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.5mC_hc_tombo_sorted.cutoff.0.50.overlap.bed\n",
      "Pst_104E_v13_ph_ctg.effectors.tss.6mA_hc_tombo_sorted.cutoff.0.50.overlap.bed\n"
     ]
    }
   ],
   "source": [
    "print(*tss_rand_dict, sep = '\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#15c66e'> 7. Analysing gene expression files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1534,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare inputs\n",
    "\n",
    "rnaseq_list = sorted([fn for fn in glob.iglob('%s/*_average.UG.sorted.tsv' % DIRS['MEAN_STD'], recursive=True)])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 6mA: Spearman's rho between ranked methylated genes in df and ranked expressed genes in df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# write a function combining these steps:\n",
    "# Make a dataframe of the gene methylation fractions\n",
    "# Slice it, so gene name is column 0 and fraction is column 1, save as a new df\n",
    "# Sort this df by gene name alphabetically\n",
    "\n",
    "# Make a dataframe of RNA expression\n",
    "# Sort this by gene name alphabetically\n",
    "\n",
    "# Run spearman's correlation between these 2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Ignore both_u_d files for now!\n",
    "# For the both_u_d files:\n",
    "# iterrows through df and if name == name:\n",
    "# add the frac1 + frac2 and divide by 2\n",
    "# append the average to a list\n",
    "# make a new df of ... ok this is too hard I'll ask Ben on Monday"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1535,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['/home/anjuni/analysis/rna_counts/average_and_stdev/Pst_104E_v13_h_exon_rpkm_average.UG.sorted.tsv',\n",
       " '/home/anjuni/analysis/rna_counts/average_and_stdev/Pst_104E_v13_h_gene_rpkm_average.UG.sorted.tsv',\n",
       " '/home/anjuni/analysis/rna_counts/average_and_stdev/Pst_104E_v13_p_exon_rpkm_average.UG.sorted.tsv',\n",
       " '/home/anjuni/analysis/rna_counts/average_and_stdev/Pst_104E_v13_p_gene_rpkm_average.UG.sorted.tsv',\n",
       " '/home/anjuni/analysis/rna_counts/average_and_stdev/Pst_104E_v13_ph_exon_rpkm_average.UG.sorted.tsv',\n",
       " '/home/anjuni/analysis/rna_counts/average_and_stdev/Pst_104E_v13_ph_gene_rpkm_average.UG.sorted.tsv']"
      ]
     },
     "execution_count": 1535,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rnaseq_list"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make files ranking genes expressed in each life cycle stage from high to low expression\n",
    "\n",
    "# Make a new file of the highly expressed genes for each life cycle stage\n",
    "# Make a new file of the lowly expressed genes for each life cycle stage\n",
    "\n",
    "# Make a file of genes that are highly expressed in all stages -> for loop that checks whether gene is in all \"highly expressed\" files\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Figure out how to identify genes that are differentially expressed in each life cycle stage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Are RNA mapping files for all genes, or just expressed genes?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#9ac615'> 8. Intersecting methylation with transposons expression files. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#ffa347'> 10. Comparing methylated transposons and genes. <span/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#ff4f14'> 11. Expression of methylation machinery throughout Pst life cycle. <span/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## <span style='color:#ff4f14'>  12. Data processing for tables and graphs. <span/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make cutoff files for Nanopolish and SMRTLink, for the table\n",
    "\n",
    "# Saving out a csv file of cutoff vs. num_sites_file1, num_sites_file2, num_overlap_sites, %_overlap for each initial file\n",
    "# Already done for tombo 6mA and all overlaps, need to do for NP, Tombo CpG and PB cutoff files\n",
    "# The csv will be formatted in excel :)\n",
    "\n",
    "# Ask Ben how to format headings properly\n",
    "\n",
    "header_6mA = ['Cutoff', 'Nanopore', 'PacBio', 'Overlapping sites', 'Percentage Overlap']\n",
    "\n",
    "header_5mC = ['Cutoff', 'Tombo', 'Nanopolish', 'Overlapping sites', 'Percentage Overlap']\n",
    "\n",
    "for akey, avalue in dict_a.items():\n",
    "    for bkey, bvalue in dict_b.items():\n",
    "        linakey= []\n",
    "        if akey == bkey:\n",
    "            cutoff = akey[x:x]\n",
    "            line.append(cutoff, avalue, bvalue)\n",
    "            "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
